\chapter{Network graph and GWAS samples}




The previous chapter described how an analysis of the post synaptic proteome network might be used to augment the analysis of GWA studies of traits or diseases that involve the synapse. This in turn could show if network methods are useful in this context.  Important nodes are tested for genetic variants associated with the phenotype (differences in intelligence or educational attainment) and GSA tests the association of  groups of genes encoding proteins forming meso-scale network structures. 

This chapter describes the generation of the post synaptic proteome network and how the discovery and replication GWAS samples were constructed  for intelligence and educational attainment.
% This thesis will address how network studies inform two of the approaches to the understanding of differences in intelligence and their association with models of the synapse.
% \todo{perhaps change this to cohort generation and network graph and add the network level statistics from the next chapter to this or split into two chapters the network grapha nd network level statistics and cohort generation and gene level results}
% \todo{you could also add the Hill revisited GSA and the gsa from the MAGMA site gene set synaptic into this section}


% ``Pathway" based approaches study for genetic variants the genes that are associated with cognition in model organisms \cite{deary2009genetic} (the ``genes to cognition" project being an example). These have therefore primarily addressed the consequence of molecular perturbation primarily of synaptic proteins. 

% The other, population genomic approach examines large numbers of individuals in Genome Wide Association Studies and determines how the specific genetic differences they have influence differences in educational attainment and intelligence. These require large studies to develop sufficient power as the individual effect of these genetic differences is small \cite{plomin2015genetics}. In the course of completing this thesis the number of individuals included in the samples  available to me has increased dramatically (see fig) \todo{add figure}. Before seeing how these studies are correlated with the structure of the synaptic network we must first generate the network. 


\section{Network Graph of the Post Synaptic Proteome}
\label{sec:network graph generation}
The post-synaptic proteome (PSP) network was developed as part of an ongoing project to provide comprehensive maps of the synaptic proteome, synaptosome and presynaptic proteome. Generation of the network required the integration of the synaptic proteomic literature and protein-protein interaction databases and was the work over several years of members of the Armstrong groups and Simpson groups at the University of Edinburgh(see acknowledgements~\ref{sec:ack}). The network presented here represents a ``freeze'' of the evolving network data, albeit at a point where the rate of discovery of new network elements had significantly slowed \cite{heil2018systems}.

An undirected, unweighted interaction network graph was used to represent the interactions of the post synaptic density and closely associated proteins (post synaptic proteome PSP). The nodes (vertices) in the network represent genes encoding proteins and were derived from curation of recent proteomic studies of the PSP. An edge occurs between genes if there is direct experimental evidence, recorded in protein-protein interaction databases, that their protein products interact.  Figure~\ref{fig:nrcmasc} shows the network of the NRC-MASC complex from an earlier paper\cite{pocklington2006organization},\cite{mclean2016improved}, 

\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/nrc_arrow.png}
        \caption[Network graph of the NRC MASC complex.]{Network graph of the NRC MASC complex. Arrows show a node or vertex which correspond to a gene encoding NRC MASC protein and an edge or link between two nodes which shows that the proteins interact. The network is that described by Pocklington et al. \cite{pocklington2006proteomes} used in the paper by McLean\cite{mclean2016improved} and the gml file is provided by the first author. Graph visualisation in Gephi\cite{bastian2009gephi} with colours corresponding to community membership. Node size is proportional to degree (the number of edges connected to a node)}
    \label{fig:nrcmasc}
\end{figure}


Protein-protein interaction networks are built using evidence of molecular interaction curated in protein-protein interaction databases. Human protein-protein interaction databases include those that map to primary proteomic experiments:   the Dictionary of Interacting Proteins (DIP) \cite{salwinski2004database}, MIntAct\cite{orchard2014mintact} and BioGRID \textit{(Bio}logical\textit{G}eneral \textit{R}epository for \textit{I}nteraction \textit{D}atasets)\cite{chatr2017biogrid}; those that contain computational predictions of interactions; and those that contain both, such as STRING\cite{szklarczyk2019string}.

 Alternatively one can distinguish between primary databases reporting the results of proteomic experiments and  meta databases that collate primary databases and may also include computational predictions\cite{dong2018analyses}. Experimental evidence of protein-protein interaction comes from high throughput or low throughput studies. High throughput methods include affinity purification mass spectrometry and two-hybrid screening\cite{dong2018analyses}. High throughput studies constitute the minority of publications curated in databases but account for most of the interactions. For example, in the IntAct database\cite{hermjakob2004intact}, 78\% of publications report ten or fewer interactions, but these constitute only 4\% of the interactions in the database \cite{dong2018analyses}. Conversely, only 2\% of studies report over 100 interactions, but these supply 77\% of the interactions in the database \cite{dong2018analyses}. 
 
 To permit efficient data sharing a common database schema was developed by the Molecular Interactions workgroup (MI) of the Human Proteome Organisation (HUPO) Proteomics Standard Initiative (HUPO-PSI-MI) in 2002. Protein-protein interactions are documented using a controlled vocabulary and indexed by accession numbers\cite{orchard2012molecular}. Both a tab-delimited text format (MITAB) and stable XML format (PSI-MI XML) are available allowing a variety of tools to parse information from these databases \cite{orchard2012molecular}.  Protein-protein interaction databases differ due to both differences in detection technology and in how experimental data is recorded (curation). The International Molecular Exchange (IMEx) consortium seeks to establish common curation rules and avoid redundant curation of interactions by coordination across members \cite{orchard2012molecular}, \cite{orchard2012protein}.
 
 \begin{figure}
     \centering
     \includegraphics[width=\textwidth]{images/chapter_community_detection/mac/getting_data.png}
     \caption{Sources of data for PPI network.Image credit creative commons \url{https://www.ebi.ac.uk/training/online/courses/network-analysis-of-protein-interaction-data-an-introduction/building-and-analysing-ppins/sources-of-ppi-data/}}
     
     \label{fig:sources of data for PPI network}
 \end{figure}

Despite the use of a standard, machine-readable vocabulary, there is only moderate overlap between protein-protein interaction databases\cite{dong2018analyses}\footnote{although some of this is due to non-redundant curation}(see table~\ref{tab:overlap between protein interaction databases}). Generating  a comprehensive network model of a particular proteome therefore requires harmonising data from multiple databases\cite{alonso2019apid}.  % \textcolor{red}{New} 
Although meta databases combine information from primary databases, they may also include data from computational predictions and other less reliable data sources. Further data curation is necessary to construct a network with reliable, consistent interactions.

This is a significant task due to well-documented problems including the use of multiple identifiers, incomplete overlap in experiments, differences between matrix and spokes models and different confidence thresholds \cite{lehne2009protein}. An overview of network curation is shown in figure~\ref{fig:sources of data for PPI } and is reviewed in depth by Dong and Provart \cite{dong2018analyses}.

  \begin{table}[]
       \centering
       \begin{tabular}{lllll}
       \toprule
    &    BioGRID    & DIP & IntAct & MINT   \\
    \midrule
    \\
    Interactions & 506,234 & 81,731 & 659,284 & 125,464\\
    \midrule
    \\
    Species & 62 & 834 & 980 & 611 \\
    \midrule
    BioGRID &  & 13\% & 41\%& 18\%      \\
    DIP & 19\% & & 32\% &20\% \\
    IntAct& 32\% & 10\% & &23\% \\
    MINT& 48\% & 26\% & 100 \% & \\
    \bottomrule
       \end{tabular}
       \caption[Overlap between protein-protein interaction databases]{Overlap in records between protein protein interaction databases. The table includes all databases used in the final PSP network with the addition of MINT for purposes of illustration. The table is asymmetric and should be read down the columns as the amount of the \textit{row name} (for example \textit{MINT}) contained in the \textit{column name} (for this example \textit{IntAct}; 100\%). MINT has been included as an example and is a subset of IntACT. All of MINTs interactions are in IntAct so the row MINT, column IntAct; is 100\% but the row IntAct, column MINT is 23\% as the contents of MINT account for this amount of IntAct. Modified from figure 3 in Dong and Provart (2018) \cite{dong2018analyses}. The date of collection of this data (July 2017) is within 6 months of the generation of the PSP graph.}
       \label{tab:overlap between protein interaction databases}
   \end{table}
 \subsection{Documenting interactions}
 Protein-protein interactions for the PSP network were obtained from the mining and harmonisation of data from the Database of Interacting Proteins (DIP) \cite{xenarios2002dip},  IntAct database \cite{orchard2014mintact},  BioGRID \cite{chatr2017biogrid}, and HIPPIE\cite{schaefer2012hippie}. HPRD\cite{keshava2009human} was excluded as it did not comply to PSI-MI TAB  standards at the time of compiling the network. HIPPIE was ultimately also excluded from further analysis due to a lack of specific interaction information (see Heil(2018)\cite{heil2018systems} chapter 4 for full details). 
 
 The mapping of identifiers and accession numbers across different databases is a key bioinformatics task\cite{huang2011comprehensive}. These can be different identifiers for a particular object (such as HGNC Gene Symbols or Entrez Gene ID) or mapping identifiers between related objects (e.g. between a protein accession identifier and that of the encoding gene). The mapping of identifiers from UniprotKB to Entrez Gene ID was the most commonly used task in a protein identifier mapping service\cite{huang2011comprehensive}.
 
 A bidirectional map was created between protein identifications (Uniprot accession numbers\cite{uniprot2017uniprot}), and gene identifiers (the GeneID provided by the Entrez Gene portal at the National Centre for Biotechnology Information - NCBI). NCBI Entrez Gene GeneIDs are unique integer identifiers and were used as the main identifier for all subsequent data tables and analyses \cite{maglott2005entrez}. For brevity I use Entrez GeneID synonymously with NCBI Entrez Gene GeneID. The widespread use of Entrez Gene identifiers in genomic studies and genomics software \cite{huang2011comprehensive} makes them a suitable primary identifier for network nodes.

Protein-protein interactions were included if there was high quality evidence for direct physical interaction in humans. The combined interaction list was filtered to ensure all proteins had a human taxonomy ID of 9606. The details of interactions were recorded in PSI-MI TAB 2.5 format (MITAB25)\footnote{MITAB25 is part of  PSI-MI 2.5 \url{https://github.com/HUPO-PSI/miTab/blob/master/PSI-MITAB25Format.md}}\cite{isserlin2011biomolecular},\cite{heil2018systems}. Where mirrored duplicates occurred (e.g. A-B and B-A) only one was retained. The  resulting data contained 396,837 rows and 237,789 unique PPI.  The interaction list was further filtered to generate the protein interactions represented by edges in the network graph. Protein-protein interactions were excluded if there was no direct experimental evidence of interaction documented by Proteomics Standard Initiative - Molecular Interaction ontology terms in the databases. PSI-MI is a structured ontology used to describe the evidence for protein-protein interactions.\cite{isserlin2011biomolecular} PPIs based on ``genetic interactions'' (MI:0208) , ``genetic interference'' (MI:0254), ``genetic interaction'' (MI:0208)\footnote{which includes suppression (MI:0796 and synthetic MI:0794)} and ``co-localisation'' were excluded (MI:0403). Sixty six interaction MI accession codes were identified representing evidence of direct interaction including one obsolete term, `physical interaction' (MI:0218). The final filtered list contained 352,325 unique rows representing 205,461 binary interactions.




\subsubsection{Proteomics data}
\label{sec:proteomics data}
%\todo{there is a repetition somewhere below of the number of studies}
Proteomic studies conducted between 2000-2017 of the human, mouse and rat synapse ($n=39$) were curated into a database of 6,500 potential synaptic genes  mapped to human Entrez Gene GeneIDs\cite{heil2018systems},\cite{maglott2005entrez}.  Of these studies, seventeen examined the post synaptic density (PSD) and are shown in table~\ref{tab:Katharina_phd_studies_oksana_studies}.
%an electron dense structure beneath the post synaptic membrane \cite{PALAY1956}  consisting of complexes of neurotransmitter receptors and their supporting proteins. \cite{kennedy2000signal}  Affinity purification against common PSD proteins improves their identification. \cite{ewing2007large}  
The human proteins and human orthologues of murine proteins found in these studies were used as a ``parts list'' for the PSD and associated interacting proteins expressed in the post synaptic area (PSP). 

 The nodes in the network used in this thesis are genes encoding proteins discovered in the proteomic studies in table~\ref{tab:Katharina_phd_studies_oksana_studies} which have high quality experimental direct interaction data in humans. 
 
% [7] "PENG  2002" \cite{peng2004semiquantitative}  is 2004
% "SATON  2002"   is SATOH 2002 \cite{satoh2002identification}

% "SCHWENK  2013"  \cite{schwenk2012high} 2012
% "SELIMI   2009" \cite{selimi2009proteomic} 
% "TRINIDAD  2005 \cite{trinidad2005phosphorylation}"  
% "TRINIDAD  2008" \cite{trinidad2008quantitative} 
% [13] "WALIKONIS  2000" \cite{walikonis2000identification} 
% "YOSHIMURA" 2004 \cite{yoshimura2004molecular}   
% "FROMER 2014" Not in Heil I only find fromer in an oksana paper on PSD and it is on scz I assume this is Focking and the one cited in Heil is bipolar
% and they have 2033 PSD proteins Focking \cite{focking2016proteomic}
% "FARR.2004" \cite{farr2004proteomic}
% "Distler.PSDII" \cite{distler2014depth}
% "BAYES 2011"   \cite{bayes2011characterization}  
% [19] "BAYES 2012" \cite{bayes2012comparative}    
% "COLLINS 2006" \cite{collins2006molecular}
% "DOSEMECI  2007" \cite{dosemeci2007composition}
% "FERNANDES  2009" \cite{fernandez2009targeted}
% "BAYES 2014"  \cite{bayes2014human}




%  "PENG  2002" \cite{peng2004semiquantitative}  is 2004
% "SATON  2002"   is SATOH 2002 \cite{satoh2002identification}

% "SCHWENK  2013"  \cite{schwenk2012high} 2012
% "SELIMI   2009" \cite{selimi2009proteomic} 
% "TRINIDAD  2005 \cite{trinidad2005phosphorylation}"  
% "TRINIDAD  2008" \cite{trinidad2008quantitative} 
% [13] "WALIKONIS  2000" \cite{walikonis2000identification} 
% "YOSHIMURA" 2004 \cite{yoshimura2004molecular}   
% "FROMER 2014" Not in Heil I only find fromer in an oksana paper on PSD and it is on scz I assume this is Focking and the one cited in Heil is bipolar
% and they have 2033 PSD proteins Focking \cite{focking2016proteomic}
% "FARR.2004" \cite{farr2004proteomic}
% "Distler.PSDII" \cite{distler2014depth}
% "BAYES 2011"   \cite{bayes2011characterization}  
% [19] "BAYES 2012" \cite{bayes2012comparative}    
% "COLLINS 2006" \cite{collins2006molecular}
% "DOSEMECI  2007" \cite{dosemeci2007composition}
% "FERNANDES  2009" \cite{fernandez2009targeted}
% "BAYES 2014"  \cite{bayes2014human}
% in Katharina 
% Doscemi 2006




% \begin{table}[]
%     \centering
%     \begin{tabular}{lllllll}
%     \toprule
%      & First author & Year & Reference & Region & Species & $n$ \\
%     \midrule
% 1 &    WALIKONIS &2000 &Walikonis et al. (2000)\cite{walikonis2000identification} &postsynapse & rat & 29 \\
% 2 &PENG&2004&Peng et al. (2004) \cite{peng2004semiquantitative}&postsynapse& rat& 325\\
% 3 &SATOH&2002&Satoh et al. (2002)\cite{satoh2002identification}&postsynapse &mouse &45\\
% 4 &YOSHIMURA&2004&Yoshimura et al. (2004)\cite{yoshimura2004molecular} &postsynapse& rat &435\\
% 5 &FARR&2004&Farr et al. (2004)\cite{farr2004proteomic}&postsynapse &rat &71 \\
% % 6 &JORDAN&2004&Jordan et al. (2004)&postsynapse &mouse and rat &390& NOT\\
% % 7&LI&2004 &wan Li et al. (2003)&postsynapse &rat& 137& NOT\\
% 6&TRINIDAD&2005 &Trinidad et al. (2005) \cite{trinidad2005phosphorylation}&postsynapse&mouse& 234\\
% % 9&CHENG&2006&Cheng et al. (2006)&postsynapse& rat& 288& NO\\
% 7&COLLINS&2006&Collins et al. (2006)\cite{collins2006molecular}&postsynapse &mouse& 717\\
% % 11&DOSEMESI&2006&Dosemeci et al. (2006)&postsynapse& rat& 113& NO\\
% 8&DOSEMESI&2007&Dosemeci et al. (2007)\cite{dosemeci2007composition}&postsynapse& rat& 548\\
% 9&TRINIDAD&2008&Trinidad et al. (2008) \cite{trinidad2008quantitative}&postsynapse& mouse& 2150\\
% 10&SELIMI&2009&Selimi et al. (2009)\cite{selimi2009proteomic} &postsynapse &mouse &61\\
% 11&FERNANDEZ&2009&Fernández et al. (2009)&postsynapse& mouse &292\\
% 12&BAYES&2011&Bayés et al. (2011)\cite{bayes2011characterization}&postsynapse &human &1441\\
% 13&BAYES&2012&Bayés et al. (2012)\cite{bayes2012comparative}  &postsynapse &mouse &1545\\
% 14&SCHWENK&2012&Schwenk et al. (2012)\cite{schwenk2012high} &postsynapse &unknown &34\\
% % 19&DISTLER PSD1*&2014&Distler et al. (2014)&postsynapse& mouse& 3545& NO \footnote{intended per email}\\
% 15&DISTLER PSD2*&2014&Distler et al. (2014)\cite{distler2014depth}&postsynapse& mouse& 2092*\\
% 16&BAYES&2014&Bayés et al. (2014) \cite{bayes2014human}&postsynapse& human &1141\\
% % 22&UEZU&2016&Uezu et al. (2016)&postsynapse &mouse &1111 &NO\\
%  17&FOCKING&2016&Föcking et al. (2016)\cite{focking2016proteomic}&postsynapse& human &2026\\

% \bottomrule
%     \end{tabular}
%     \caption[Primary Proteomic studies contributing to the PSP - from Heil (2018)]{Primary proteomic studies contributing to the PSP Clean Published network adapted from Heil 2018\cite{heil2018systems} PhD p84 table 5.1. }
%     \label{tab:Katharina_phd_studies_oksana_studies}
% \end{table}


\begin{table}[]
    \centering
     \setlength{\extrarowheight}{2pt}
    \begin{tabular}{lllllll}
    \toprule
     & First author & Year & Reference & Species & $n$ \\
    \midrule
1 &    WALIKONIS &2000 &Walikonis et al. (2000)\cite{walikonis2000identification}   & rat & 29 \\
2 &PENG&2004&Peng et al. (2004) \cite{peng2004semiquantitative} & rat& 325\\
3 &SATOH&2002&Satoh et al. (2002)\cite{satoh2002identification}  &mouse &45\\
4 &YOSHIMURA&2004&Yoshimura et al. (2004)\cite{yoshimura2004molecular}  & rat &435\\
5 &FARR&2004&Farr et al. (2004)\cite{farr2004proteomic}  &rat &71 \\
% 6 &JORDAN&2004&Jordan et al. (2004)  &mouse and rat &390& NOT\\
% 7&LI&2004 &wan Li et al. (2003)  &rat& 137& NOT\\
6&TRINIDAD&2005 &Trinidad et al. (2005) \cite{trinidad2005phosphorylation} &mouse& 234\\
% 9&CHENG&2006&Cheng et al. (2006) & rat& 288& NO\\
7&COLLINS&2006&Collins et al. (2006)\cite{collins2006molecular}  &mouse& 717\\
% 11&DOSEMESI&2006&Dosemeci et al. (2006) & rat& 113& NO\\
8&DOSEMESI&2007&Dosemeci et al. (2007)\cite{dosemeci2007composition} & rat& 548\\
9&TRINIDAD&2008&Trinidad et al. (2008) \cite{trinidad2008quantitative} & mouse& 2150\\
10&SELIMI&2009&Selimi et al. (2009)\cite{selimi2009proteomic}   &mouse &61\\
11&FERNANDEZ&2009&Fernández et al. (2009) & mouse &292\\
12&BAYES&2011&Bayés et al. (2011)\cite{bayes2011characterization}  &human &1441\\
13&BAYES&2012&Bayés et al. (2012)\cite{bayes2012comparative}    &mouse &1545\\
14&SCHWENK&2012&Schwenk et al. (2012)\cite{schwenk2012high}   &unknown &34\\
% 19&DISTLER PSD1*&2014&Distler et al. (2014) & mouse& 3545& NO \footnote{intended per email}\\
15&DISTLER PSD2*&2014&Distler et al. (2014)\cite{distler2014depth} & mouse& 2092*\\
16&BAYES&2014&Bayés et al. (2014) \cite{bayes2014human} & human &1141\\
% 22&UEZU&2016&Uezu et al. (2016)&postsynapse &mouse &1111 &NO\\
 17&FOCKING&2016&Föcking et al. (2016)\cite{focking2016proteomic}& human &2026\\

\bottomrule
    \end{tabular}
    \caption[Primary Proteomic studies contributing to the PSP - from Heil (2018)]{Primary proteomic studies contributing to the PSP Clean Published network adapted from Heil 2018\cite{heil2018systems} PhD p84 table 5.1. }
    \label{tab:Katharina_phd_studies_oksana_studies}
\end{table}




 


 
  




 
 
The largest connected component of the resulting graph was used for community detection and in calculating network statistics (3,457 nodes, 30,498 edges). The details of the small number of excluded vertices not part of the largest connected component ($n=16$) are in section~\ref{sec: PSP graph connected component and missing} below.

The network was stored as a named edge list consisting of two columns. Each row represents an interacting protein pair and each protein is identified using the Entrez Gene Id of its encoding gene. Network metadata and interactions were stored as a graph modelling language (GML) file generated from this edge list\cite{himsolt1997gml}. 
GML is an ASCII based, hierarchical, lightweight format supported by igraph \cite{csardi2006igraph}, Gephi \cite{bastian2009gephi} , NetworkX \cite{hagberg2008exploring} and graph-tool \cite{peixoto_graph-tool_2014}.  A master copy of the original file supplied to me was stored on the University of Edinburgh informatics server. The original \texttt{gml} file was generated by Dr Colin McLean and also contains community detection data and is included in the supplementary material.

%\textcolor{red}{commented out}
% For certain packages the edge list was used or the edge list transformed to an adjacency matrix or other graph object (eg SNAP format)

% Previous note commented out below (list from colin with the 27 had 5101 genes)\footnote{
% for combined list of genes (5101) including both Distler, we have got the PPI information for 3457 genes, which resulted in the graph of 3457 nodes and 30499 edges. Of this - 3457 nodes and 30498 edges contributed to Largest connected component (I guess the same size as initial graph). The majority of genes that survive quality control are expressed in more than one study) Of those in only one study 489/3457 = 14.1\%. 352 appear in the 2014 paper by Distler et al. You have a total of 5101 and of these 1640 appear only once of the 1643 genes that did not makes the final graph 1151 (70\%) were expressed only once 286 twice and 76 three times "

% Those that appeared in the overall graph but not in the LCC appeared on one occasion 2, 2 occasion 2, 3 occasion 6, 7 occasion 1, 8 occasion 2, 9 occasion 3) so there were nodes that we had PPI information for that still did not make it into the LCC so a lot of the ones that did not make it is because we did not have PPI information that met PPI or the genes missed other qc. 

% Yeah cos the ones that appear once we have information on we might even have their degree they are not all going to have degree one so its a subgroup of the proteomics where we have valid PPI - got it. Comparisons between the number of representations in the literature and gene features are described in depth in Heil} \cite{heil2018systems}.\textcolor{red}{endremove}
% \todo{could do correlation in number of representations and pLI and GTex}

\subsubsection{Value of tissue specific networks}
\label{sec:value_of_tissue_specific_networks}

The post synaptic proteome network is tissue specific; it is the network of interacting proteins found in the post synaptic area. I will, as part of overall quality control confirm in section~\ref{sec:gtex profile of PSP genes} that these genes are differentially expressed in the CNS in humans given the expansion in the number of genes in the PSP network compared to previous models\cite{mclean2016improved}.

A tissue specific network is desirable as different tissues contain different proteins and therefore different nodes and edges in their proteome network.  Some tissues are known to have very distinct gene expression patterns (e.g solid tissues and blood, CNS \cite{mele2015human}) and their proteome network will also differ. In addition, it has been reported that disease genes tend to be expressed in specific tissues (and to be on the periphery of cellular networks)\cite{barabasi2011network},\cite{kitsak2016tissue}. Finally, the synaptic proteome \textit{in particular} has a modular structure arising from the interaction of its protein components and giving rise to higher order structures and functions \cite{pocklington2006organization},\cite{grant2012synaptopathies} that are particular to the synapse. 

Two examples help illustrate the issues. Ghiassian et al. (2015)\cite{ghiassian2015disease} performed an analysis similar to the one presented here (see section~\ref{sec:intro other approaches}) before going on to describe a method for identifying disease modules. They concluded that topological units were not associated with disease, but that disease results from loss of function of an area of a \textit{global} interactome where a group of disease associated genes reside. The network used was general and composed of heterogenous elements in addition to protein-protein interactions. 

%A co-author of one of the papers discussing tissue specificity of disease genes\cite{kitsak2016tissue}) 
By contrast a review of systems biology \cite{parikshak2015systems} gives the use of a tissue specific cardiac PPI to investigate long QT syndrome (LQTS) by Lundby et al.\cite{lundby2014annotation} as an example of the value of tissue specificity. The authors used a mouse cardiac protein-protein interaction network derived from High Performance Liquid Chromatography Tandem Mass Spectrometry (HPLC-MS/MS) of five seed proteins encoded by genes identified in a GWAS of LQTS to generate candidate genes later confirmed in zebra fish. These genes could not be identified using more general networks and significantly the affected genes were those in the \textit{neighbourhood} of receptors and ion channels.


\subsection{Connected component}
\label{sec: PSP graph connected component and missing}

All of the nodes in the parts list other than sixteen are connected. In the simplest random generative models of networks (Erd{\H{o}}s - R{\'e}nyi\cite{erdHos1960evolution}) where nodes are connected with a uniform probability $p$, isolated nodes spontaneously aggregate to form a large connected component (connected in that all of its nodes can be reached via `hops' along its edges) containing the majority of nodes above a critical value of $p$\cite{newman2018networks}. In real world networks, the proportion ($S$) of nodes in the largest component varies but one component always dominates the others\cite{newman2018networks}.


\paragraph{Methods}
The network containing all nodes, including those not in the largest connected component, was generated by Dr Colin McLean (section~\ref{sec:ack}) and stored as a gml file. Components were identified using the \texttt{components} command for igraph version 1.2.5\cite{csardi2006igraph} for R and the components not part of the largest component recorded\footnote{\url{source('~/RProjects/network_graph/R/notLCC_gmt/genes_not_inLCC.R')}}. 

 igraph for R version 1.2.5 was used for the majority of graph processing tasks in this thesis\cite{csardi2006igraph}. R version 3.6.3\cite{ihaka1996r} is used throughout and  all statistical analyses are carried out in R unless otherwise stated. On occasion igraph for Python (v0.8; Python 3.6) was used, when methods from other Python packages were required. This is noted in the corresponding methods sections and associated scripts.

Nodes not in the connected component are excluded from further analysis, as community detection algorithms and most vertex statistics are not valid when there are unconnected components.

The composition of the excluded components was analysed to assess whether they contained a plurality of elements that might bias the subsequent analyses (see section~\ref{sec:analysis of genes not in largest component}). 

\paragraph{Results}
In addition to the large connected component ($n$=3,457 nodes), 15 smaller components were found within the PSP graph; these were all single isolated components other than one component of size two.  The  proteins encoded by these genes appear in the ``parts list" derived from proteomic studies but lack experimental evidence of direct interaction with the protein products of the genes in the large component of the PSP.


The sixteen genes not in the giant connected component are shown in table~\ref{table:notinLCC}. The only component with two genes contained Oligodendrocyte Myelin Glycoprotein (OMG) and Reticulon 4 Receptor (RTN4R)\footnote{member of growth cone pathway and implicated in schizophrenia}. 

 

These constitute 0.46\% ($S$= 0.995; see\cite{newman2018networks} p305 for a comparison with other networks) of the genes identified in proteomic experiments (section~\ref{sec:proteomics data}) with high quality experimental interaction data. This is comparable to that reported for undirected metabolic network ($S=0.996$) and high compared to earlier protein-protein interaction networks (n=2115,$S$=0.689)\cite{newman2018networks}\footnote{p 305}.

\subsubsection{Analysis of genes not in largest connected component}
\label{sec:analysis of genes not in largest component}
%\todo[inline]{what is a component}

Gene ontology enrichment analysis of the genes ($n=16$) not in the largest connected component was carried out to ensure they did not belong to a common system or component that would bias the analysis. Gene ontology analysis with ToppGene did not show any significant enrichment for terms compared to the rest of the PSP. Gene ontology enrichment analysis is widely used in the rest of this thesis and the methods introduced below.

\begin{table}[ht]
 \setlength{\extrarowheight}{2pt}
\centering
\begin{tabular}{llll}
\toprule
 
GeneID & Gene Symbol & Gene Name & $n$ \\ 
\midrule
 4974 & OMG & oligodendrocyte myelin glycoprotein & 2 \\ 
  65078 & RTN4R & reticulon 4 receptor & 2 \\ 
 98 & ACYP2 & acylphosphatase 2 & 1 \\ 
  166785 & MMAA & metabolism of cobalamin associated A & 1 \\ 
  549 & AUH & AU RNA binding methylglutaconyl-CoA hydratase & 1 \\ 
  55847 & CISD1 & CDGSH iron sulfur domain 1 & 1 \\ 
  23305 & ACSL6 & acyl-CoA synthetase long chain family member 6 & 1 \\ 
  26027 & ACOT11 & acyl-CoA thioesterase 11 & 1 \\ 
  7915 & ALDH5A1 & aldehyde dehydrogenase 5 family member A1 & 1 \\ 
  4685 & NCAM2 & neural cell adhesion molecule 2 & 1 \\ 
 
  5649 & RELN & reelin & 1 \\ 
  2741 & GLRA1 & glycine receptor alpha 1 & 1 \\ 
  152404 & IGSF11 & immunoglobulin superfamily member 11 & 1 \\ 
 
  27000 & DNAJC2 & DnaJ heat shock protein family (Hsp40) member C2 & 1 \\ 
  2686 & GGT7 & gamma-glutamyltransferase 7 & 1 \\ 
  55262 & MAP11 & microtubule associated protein 11 & 1 \\ 
   \bottomrule
\end{tabular}
\caption[Genes not in largest connected component]{Genes with cognate proteins in the PSP as defined by their presence in proteomic studies but that are not part of the giant connected component. These encode proteins that lack direct experimental evidence of interaction with the giant connected component. GeneID is Entrez Gene ID. $n$ is the number of genes in the component.} 
\label{table:notinLCC}
\end{table}

\subsection{Gene ontology analysis}
\label{sec: gene ontology analysis}
A gene ontology provides a controlled, maintained vocabulary to describe sets of genes according to their Biological Function, Cellular Component and Molecular Function. It is widely used in high throughput genetics to describe the function of a set of genes\cite{ashburner2000gene}. Gene ontology terms are structured forming a directed acyclic graph that shows a hierarchical relationship with more specific child terms arising from parent terms (see figure~\ref{fig:the DAG of Collateral sprouting}).

Gene ontology over-representation analysis tests whether a set of genes are more commonly annotated to a specific term  than would be expected by chance. Some difference in results is reported across different platforms and therefore more than one method is reported\cite{rhee2008use},\cite{khatri2005ontological}. 


\paragraph{Multiple testing}
Gene ontology terms overlap, particularly at different levels in the hierarchy, invalidating the assumption of independence made by multiple testing correction methods. The issue of multiple testing in Gene Ontology enrichment analysis is complex and beyond the scope of this thesis. Rhee et al. (2008) \cite{rhee2008use} provide a comprehensive summary and a more recent, technical analysis is found in Meijer et al (2016)\cite{meijer2016multiple}. A brief description will introduce the methods in this chapter and highlight potential issues common to pathway enrichment methods that use GO terms.

Rhee et al. (2008) \cite{rhee2008use} recommend the following approach to multiple testing:

`An unbiased search for significant GO associations can be done with a bottom up approach as follows: for every leaf term, calculate p-values with the genes directly associated to it. If any term is significant do not propagate its genes above. This would provide the most specific node that is significant in that particular branch. If a term is not significant, propagate the annotations to its parent and recalculate with the parent term. The genes will propagate upwards until a significant node is found or until the root is reached. A careful analysis is still necessary to properly correct for multiple comparisons.\cite{rhee2008use}'

 To my knowledge there is no widely available implementation of this method designed for genomic analysis. The closest approximation being the methods described by Alexa et al.\cite{alexa2006improved} and implemented in topGO\cite{alexa2009gene}.  Rhee et al.\cite{rhee2008use} argue that the Bonferroni correction is too conservative and that the Holm and FDR are better alternatives. A different method for correction is implemented in g:Profiler\cite{raudvere2019g} which uses the sgs method to increase power using the interdependence of GO terms. 
 
 Rhee et al. recommend minimising the number of tests wherever possible\cite{rhee2008use}. One way to reduce the number of tests is to use a restricted or clipped Ontology. This can be challenging to implement and although a neural-immunology clipped ontology is available\cite{geifman2010neural}, I am not aware of any current application using it for gene ontology enrichment. The SynGO project\cite{koopmans2019syngo} uses an ontology restricted to the synapse, with curated gene-term relationships supported by high quality experimental evidence. 

% In practical terms one should be aware that multiple testing using Bonferroni correction (as implemented in MAGMA) is too strict, second that the results of enrichment in hypothesis generating applications do not truly represent an absolute value of the likelihood of a test under repetition but a measure of confidence in specific findings. I will return to this in the discussion.


\subparagraph{elim method}
The \textit{elim} and \textit{weight} methods described by Alexa et al. (2006)   \cite{alexa2006improved} and implemented in the popular topGO R  package\cite{alexa2009gene} have the effect of decreasing the significance of very general GO terms (increase $p$) in comparison to more specific child terms. These methods start testing at child nodes and only move genes for testing to higher levels if they have not been scored. 

This decreases the significance of very general terms and makes specific terms more significant. The overall effect is to increase raw p values but to do this most for general terms. It does not correspond exactly to multiple testing correction but does prioritise general terms and reduce the significance of general ones. The authors state that most multiple comparison methods are too strict given the connected structure of GO and they prefer to regard the \textit{elim} results as already corrected \cite{alexa2020gene}.

\subparagraph{Background set}
The number of terms that are expected to be found in a group of genes depends on the background set and this plays an essential role in calculating over representation. Some packages allow the use of custom background sets (ToppGene has very recently added this feature  - section~\ref{sec:ToppGene GO enrichment}). If not indicated the background set is the default for that enrichment package usually the protein encoding genes in the genome.  

\subsection{Gene ontology enrichment methods}
\label{sec:GO enrichment methods}
Some methods use direct data entry to a web interface, others an API, and some methods can be locally installed. The main advantage of web based ORA is regular updating of the ontologies and a common computational platform. To reduce transcription errors, whenever text entry of a list of genes is used on a web interface a script is used to write the list of genes directly to the clipboard.

\subsubsection{PANTHER methods}
\label{sec:panther methods1}
\label{sec:methods GO Pather}
Gene ontology ORA was carried out using the PANTHER classification system (protein annotation through evolutionary relationship) version 15.0\cite{mi2019protocol} via the web-based user interface hosted at the Gene Ontology Consortium (GOC) website\footnote{\url{http://geneontology.org/}}.

%\todo{compress this and put into methods and results}\textcolor{red}{start compress}
PANTHER provides information on gene function at a genomic scale, and as a member of the Gene Ontology Consortium, provides accurate annotation with regular updates. Results can be displayed showing their relation in the gene ontology hierarchy\cite{mi2019protocol}.  The authors recommend the use of the False Discovery Rate to correct for multiple comparisons because of the correlation between tests involving related annotations \cite{mi2019protocol}. It can be accessed using an API or web application. It was initially used for tests where authoritative, accurate testing using an up to date ontology was required and where custom background sets for enrichment were required. The hierarchy view was also informative, however problems with the interface and gene registration caused me to switch to ToppGene once it became possible to use background sets in this application.

PANTHER 15.0 (released 2020/02/14)\cite{mi2013large} was used to test sets of genes that were significant at MAGMA GWGAS for over representation of terms from each of the three Gene Ontology clades (Cellular Component, Biological Process and Molecular Function). The PANTHER reporting details are as follows: The PANTHER Over-representation Test used implementing Fisher's Exact test (released 2020/07/28).  The Gene ontology database was DOI: 10.5281/zenodo.3873405 Released 2020-06-01.  The Benjamini-Hochberg procedure \cite{benjamini1995controlling} was used to calculate the False Discovery Rate (FDR) to account for multiple comparisons, and corrections  were applied for each clade.

\paragraph{Annotations}

Despite using Entrez Gene ID integer values with the recommended string prefix "GeneID:" (PANTHER interprets integers as HGNC identifiers by default), several significant genes from the population samples were not identified using Ensembl and Entrez Gene ID. 

% PANTHER introduced in section~\ref{sec:panther methods1} is a member of the gene ontology consortium and frequently updated. The results obtained contain links to easily view the GO tree and also present results in terms of the hierarchy. Mi et al\cite{mi2019protocol} in the most recent update acknowledge that given the relatedness of GO terms tests are non independent and therefore the Bonferroni correction is too conservative. As an alternative they recommend using the FDR method.





Using PANTHER, there were identifiers that were not recognised using Ensembl ID, Entrez ID and GeneID. The most robust appeared to be gene symbol. I used translation from Entrez ID to Gene Symbol including synonyms until all genes were identified. In addition to the code documenting the data transformation \footnote{code \url{ source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/EA2_FUMA2clipboard.R')}}, the gene mapping compatible with PANTHER is included in the supplementary materials.

PANTHER displays the hierarchy of the ontology terms that can be useful. Although the FDR values are only nominally significant the very specific term related to neurogenesis: positive regulation of collateral sprouting (FDR p 0.041) is prominent in comparison to the same data ordered by FDR shown in table~\ref{tab:GO biological process complete Education Replication FDR}.

\subsubsection{ToppGene}
\label{sec:ToppGene GO enrichment}
 Toppgene is a web based gene ontology enrichment service that also supports murine and human phenotypes and disease enrichment\cite{chen2009toppgene}. It has a regularly updated database ( see \cite{tomczak2018interpretation} for importance of regular updates). ToppGene uses Entrez ID as its default identifier which made it well suited for analysis of the PSP network.
 
 The ToppGene API was used for some translation of identifiers and where this has been done it is noted in the manuscript. Custom background gene sets can only be used with the web interface so this was used when required in chapters 3 and 4.The output tables for significant ToppGene analyses including terms searched are included as tab separated text files  in the supplementary materials.Database update 28-07-20. Gene Ontology Biological Process 16,465 annotations 20,148 genes. Cellular component  2,043 annotations, 20,584 genes, molecular function 4,745 Genes 20,584 with GO ontology from PANTHER DB.  
 
 


\subsubsection{FUMA Gene2Func package} 
\label{sec:FUMA go enrichment}

FUMA (Functional Mapping and Annotation of Genome-Wide Association Studies)\footnote{\url{fuma.ctglab.nl}} is a comprehensive web based annotation and analysis package\cite{watanabe2017functional} that has been widely used in the secondary analysis of GWA studies of intelligence\cite{hill2019combined},\cite{savage2018genome}. It comprises two modules: SNP2Gene takes summary SNP data as an input and implements gene level GWGAS using MAGMA, and annotates significant genes, and the Gene2Func module which takes a list of significant genes and provides over representation analysis.

I have used the GTEx expression enrichment analysis from the Gene2Func module to check that the PSP genes are expressed in the CNS (FUMA v1.3.6 was used throughout). This differs from the results and methods presented in section~\ref{sec:GTEx methods} in using a predetermined set of differentially expressed genes and testing if a set of genes are over-represented in these using the hypergeometric test and Bonferroni correction for multiple comparison.

This provides a quick, graphical representation of the differences in tissue expression of genes in the PSP and genome wide significant genes at GWGAS in the discovery and replication samples.

I have also used the Gene Set Analysis in the Gene2Func module which uses hypergeometric tests to assess over representation of terms. Data sets are provided, including gene ontology terms, Wiki Pathways and GWAS catalogue with Bonferroni correction per category. I have used the GWAS catalogue set enrichment to ensure that genes excluded from analysis are not potential sources of bias (ie reported in intelligence GWA or other neuro-cognitive phenotypes).

\subsubsection{g:Profiler} 
\label{sec:gProfiler GO enrichment}
g:Profiler, is a  Gene Ontology Consortium (GOC) backed Gene Ontology functional enrichment package. It has a recently updated web interface and an R API. Over representation of ontology terms is tested using the cumulative hypergeometric test\cite{raudvere2019g}. The g:SCS (set counts and sizes) method, which takes into account the overlapping between gene sets based on GO terms\cite{reimand2007g}, is used to correct for multiple comparisons.  A Manhattan like plot of over representation is provided (figure~\ref{fig:gProfiler 3 samples}),  and a comprehensive translation service that can be accessed from the R API. Although it includes other data from KEGG and wiki pathways, I have limited the analysis to GO enrichment terms (these are tested using ToppGene). 

Functional enrichment was carried out using g:Profiler version e100\_eg47\_p14\_7733820 (database updated on 07/07/2020). Multiple testing was controlled using g:SCS method applying a significance threshold of 0.05\footnote{recommended citation format from website} and not displaying  under-representation.



\paragraph{Results ORA of genes not in connected component}

There is no significant enrichment of pathways or ontology terms for the genes that are not in the connected component using ToppGene for ORA analysis and the PSP genes as background.

%\footnote{two genes are in dystonia, seven are mitochondrial}


%\todo[inline]{this must be in strontium}
%%% GENES EXCLUDED AS NOT IN LCC %%%

% latex table generated in R 3.6.2 by xtable 1.8-4 package
% Tue Dec 31 09:25:51 2019/home/grant/RProjects/network_graph

There is no evidence of any enrichment of an association between genetic variants in the set of 16 genes not in the largest connected component, and differences in intelligence or educational attainment in any of the discovery or replication samples (see section~\ref{sec:samples from paper section}) using  MAGMA Gene Set Analysis (see table~\ref{tab:notLCC}; details on methods for MAGMA GSA in section~\ref{sec:Synaptic component enrichment MAGMA}).

\begin{table}[]
    \centering
    \begin{tabular}{lll}
    \toprule
     Sample   & $\beta$ & $p$  \\
     \midrule
     Intelligence\textsubscript{Replication} & 0.30 & 0.10\\
     Education\textsubscript{Replication} & 0.11 & 0.32\\
    % ea3     & 0.307 & 0.19\\
    Intelligence\textsubscript{Discovery} & -0.09 & 0.64\\
    Education\textsubscript{Discovery} & 0.21 & 0.21 \\
    \bottomrule
    \end{tabular}
    \caption[MAGMA GSA for network vertices not in largest connected component]{MAGMA GSA enrichment of the set of genes not in LCC $n$ = 16. Testing the competitive hypothesis that genetic variants mapped to these genes are more associated the Intelligence or Educational Attainment phenotypes than the rest of the genome} 
    \tiny\url{source('~/RProjects/chapter2/R/LCC/write_not_lcc_gmt_out.R')}
    \label{tab:notLCC}
\end{table}
%%%
\clearpage

\subsection{GTEx profile of PSP genes}
\label{sec:gtex profile of PSP genes}
\subsubsection{Introduction to GTEx}
The gene tissue expression project (GTEx) is a repository of gene expression derived from RNA seq in over 1,000 individuals from 54 tissue areas\cite{lek2016analysis}. I have used this to confirm that the PSP genes are highly expressed in the brain, if there are significant regional differences in expression that might affect the analysis, and that the expression of PSP genes is as expected significantly greater than of those not in the PSP.

\subsubsection{GTEx Methods}
\label{sec:GTEx methods}
 Gene expression data (GTEx version 8 - dbGaP Accession phs000424.v8.p2) were obtained from the GTEx Portal on 07/03/2020. Median gene expression counts normalised by Transcripts Per Million (TPM) were downloaded from the GTEx Portal website\footnote{\url{https://storage.googleapis.com/gtex_analysis_v8/rna_seq_data/GTEx_Analysis_2017-06-05_v8_RNASeQCv1.1.9_gene_median_tpm.gct.gz}},\cite{gtex2015genotype},\cite{conesa2016survey}. The median gene expression values were calculated by the GTEx authors from the file \url{GTEx_Analysis_2017-06-05_v8_RNASeQCv1.1.9_gene_tpm.gct.gz}. 
 
 Two identifiers are provided for each gene, Ensembl Gene ID (ENSG) and HUGO gene nomenclature committee (HGNC) Gene Symbol\cite{gray2012genenames}. The ToppGene annotation service \cite{chen2009toppgene} was used to provide Entrez gene GeneIDs from Gene Symbol to allow the results to be combined with the synaptic network and cohort study results\footnote{\url{https://toppgene.cchmc.org/API/lookup}},\footnote{\url{source('~/RProjects/gtex/R/exploratory_gtex/load_toppgene_annotations.R')}}.


Fifty-six thousand two hundred rows were downloaded from the GTEx Portal indexed by Ensembl id, which included a version suffix. Each Ensembl ID, including version number was unique; 56,156 unique ID had only one Ensembl version.

One hundred ninety-nine duplicated gene names were identified in 1,608 duplicated GTEx entries. One hundred fourteen had a single duplicate (a total of Two Ensembl Id to one Gene Name). Seven hundred thirty-seven duplicates were identified as Y RNA (45.8\%); small non-coding RNA \cite{perreault2007ro}. 166 were metazoa signal recognition particles (SRP)\cite{keenan2001signal} (10.3\%). 



54,592 unique Gene Symbols could be associated with unique Ensembl id (97.1\%)\footnote{\url{source('~/RProjects/gtex/R/exploratory_gtex/first_load_gtex.R')}}. This is a lot more than the number of protein-encoding genes, and to exclude non-coding elements and reduce the data size and I excluded those not amongst the 18,390 genes that have gene-level results in at least one of four population samples calculated in MAGMA. 17,584 unique Entrez ID were identified from the remaining elements and used for subsequent analysis.

Entries were retained if the TPM count was greater than 0.1 in approximately 80\% (79.6\%) of samples. TPM counts were then $log_2$ transformed with a pseudocount of 1 and zero centred  \cite{sniekers2017genome},\cite{taskesen20162d},\cite{mele2015human}, see \cite{zhao2017gene}\footnote{\url{source('~/RProjects/gtex/R/exploratory_gtex/df_with_toppgene_log_normalise_plots.R')}}.



\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/Rplot_rough_qq.png}
    \caption{QQ plot log2 transformed and mean centred gene expression fronal cortex all genes passing quality control 13,787 genes}
    \label{fig:qqplot frontal cortex}
\end{figure}

The QQ plot of log2 frontal cortex expression is shown for illustration in figure~\ref{fig:qqplot frontal cortex}. The plot does not seem to deviate significantly from normal so Pearson's product moment correlation was calculated across brain regions. In the absence of checks of all brain regions, I carried out a non-parametric test (Wilcoxon rank sum test) to see if expression was higher in the PSP than other regions (multiple testing correction Bonferroni using R - see figure~\ref{fig:gtex_boxplot}).


\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/chapter2/ggplot/gtex/Rplot_boxplot_expression_add_theme.png}
    \caption{Box plot of log2 transformed and mean centred gene expression of PSP genes versus non PSP genes across 12 brain regions. p $< 2 \times 10^{-16}$ all comparisons. Wilcoxon test.}
    \tiny\url{source('~/RProjects/gtex/R/graph_gtex/multi_boxplot.R')}
    \label{fig:gtex_boxplot}
\end{figure}



% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/Rplot_compare_expression2.png}
%     \caption{Box plot of log2 transformed and mean centred gene expression of PSP genes versus non PSP genes across 12 brain regions. p $< 2 \times 10^{-16}$ all comparisons. Wilcoxon test. True and False on x axis refer to whether genes are PSP in origin. Code at \url{source('~/RProjects/gtex/R/graph_gtex/multi_boxplot.R')}}
%     \label{fig:my_label}
% \end{figure}

\subsubsection{Results Brain Expression of PSP genes}
\label{sec:gtex_results}
3,055 genes with GTEx data passing the above quality control were available for the PSP and 10,732 non synaptic genes. 

Expression is significantly higher in the PSP genes across all 12 brain regions ($p<2.2\times10^{-16}$ Wilcoxon rank sum test;spinal cervical1 has been removed as is known to be an outlier; see figure~\ref{fig:gtex_boxplot}).  There is strong correlation between gene expression counts in all brain regions(table~\ref{tab:Correlation of PSP gene expression across brain regions. Expression values $log_2$ transformed with pseudocount of 1}). Expression across brain regions was closely correlated as reported in \cite{gtex2015genotype}. The results above indicate that the PSP genes have higher expression levels than other genes and that the expanded PSP continues to show increased CNS expression. 

An alternate way to calculate brain expression of these genes is implemented in FUMA using GTEx v 8 data. This is to create a list of differentially expressed genes and see if a test set is over represented amongst these. Results for the complete PSP are shown in figure~\ref{fig:gtex all PSP}, the PSP and Consensus PSD are compared in figure~\ref{fig:deg_gtex_psp}, and the expression in the PSP, Consensus PSD, Non PSP genes and those genes found in the PSP but not Consensus PSD are shown in figure~\ref{fig:compare all PSP gtex multipanel FUMA output}. 

% \paragraph{Summary of QC}
% MAGMA studies compare with summary data

% MAGMA GSA correlation between studies

% missing synaptic genes

% those not in LCC
% Welch Two Sample t-test Cortex 
% t = 46.441, df = 4558, p-value < 2.2e-16
% alternative hypothesis: true difference in means is not equal to 0
% 95 percent confidence interval:
%  1.663539 1.810180
% sample estimates:
%  mean of x  mean of y 
%  1.3519962 -0.3848629 
 
%  	Welch Two Sample t-test

% frontal cortex `Brain - Frontal Cortex (BA9)`
% t = 47.291, df = 4547.7, p-value < 2.2e-16
% alternative hypothesis: true difference in means is not equal to 0
% 95 percent confidence interval:
%  1.750205 1.901594
% sample estimates:
%  mean of x  mean of y 
%  1.4213062 -0.4045929 


%\subsubsection{Correlation of genes across brain areas}
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sat Mar  7 15:12:52 2020

%%
% \subsection{Comparison of expression in PSP and rest of genome}
% Only 19 PSP genes have not recorded expressi


% Median expression in PSP brain cortex was 21.85 and mean 100.07. The median in non PSP genes was 3.61 and mean 11.82 (see table~\ref{tab:Summary statistics for gene expression for PSP and non PSP genes}

% Mean expression in PSP genes brain cortex is higher than in non PSP. The data appear right skewed and do not appear normally distributed. T test 	Welch Two Sample t-test
% t = 3.3731, df = 3391.5. 95 percent confidence interval of difference in mean :36.96 -139.56, sample estimates:mean of PSP 100.07 mean of non PSP  11.82, p-value = 0.0007517

% A non parametric test Wilcoxon rank sum test with continuity correction W = 51340370, p-value $< 2.2 \times 10^{-16}$

% I also log 10 transformed the expression values for brain cortex and performed a t test to get an estimate of the difference in means. As there were a large number of values at 0 (which would have infinite log) is set these values log10 result to 0.

% 	Welch Two Sample t-test
% t = 60.809, df = 5310.4, alternative hypothesis: true difference in means is not equal to 0. 95 percent confidence interval: 0.826 - 0.881
% sample estimates:mean of PSP 1.288  mean of non PSP 0.434 , p-value $< 2.2 \times 10^{-16}$. 

% The distribution of log transformed median exp ression for cerebral cortex for PSP genes and non PSP genes included in the cohorts are shown in table~\ref{fig:all_gtex_panels}.\todo{ensure GTEx is correct ie check case throughout} \todo{Gene expression levels with centrality measures and across communities}
 
%  \textcolor{red}{want to do expression PSP compared with average expression in other tissues per Sniekers}

%  % latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Sat Mar  7 17:12:57 2020
% \begin{table}[ht]
% \centering
% \begin{tabular}{lrrrrrrr}
%   \hline
% group & n & Min. & 1st Qu. & Median & Mean & 3rd Qu. & Max. \\ 
%   \hline
% PSP & 3392 & 0 & 8.43 & 21.85 & 100.07 & 55.87 & 62949.60 \\ 
%   non PSP genes & \textcolor{red}{138980} & 0 & 0.38 & 3.61 & 11.82 & 12.10 & 1288.95 \\ 
%   \hline
% \end{tabular}
% \caption{Summary statistics for gene expression for PSP and non PSP genes calculated from GTEx project data. Expression is median normalised by TPM.}
% \label{tab:Summary statistics for gene expression for PSP and non PSP genes}
% \end{table}


%     See figure~\ref{fig:all_gtex_panels} for a graphical comparison of expression of PSP and non PSP genes in the cortex. \footnote{code \url{source('~/RProjects/gtex/R/graph_gtex/histogram_boxplot.R')} } \footnote{code using only genes in study and removing synaptic and rest of genome duplicates \url{source('~/RProjects/gtex/R/graph_gtex/graphs_study_genes.R')} }. Table~\ref{tab:Summary statistics for gene expression for PSP and non PSP genes} shows gene expression for PSPS and non PSP genes \footnote{\url{source('~/RProjects/gtex/R/graph_gtex/graphs_study_genes.R')}}
    
%   \todo{the non PSP genes number cant be right in table \ref{tab:Summary statistics for gene expression for PSP and non PSP genes}}
    
% \begin{figure}
%     \centering
%     \begin{subfigure}[t]{0.45\textwidth}
%         \centering
%         \includegraphics[width=\linewidth]{images/Rplot_corrected_gtex_histogram_log.png} 
%         \caption{Histogram of expression PSP and non PSP. Log10 scale} \label{fig:gtex_histogram_log}
%     \end{subfigure}
%     \hfill
%     \begin{subfigure}[t]{0.45\textwidth}
%         \centering
%         \includegraphics[width=\linewidth]{images/Rplot_corrected_gtex_boxplot_log10.png} 
%         \caption{Gtex expression boxplot PSP against not PSP. Log10 scale} \label{fig:gtex_log_boxplot}
%     \end{subfigure}
%     \caption{Plots of GTEx expression \textcolor{red}{may want to do this in subfigure packages rather than subcaption to line up edges and tidy up images in ggplot}}
%     \label{fig:all_gtex_panels}
% \end{figure}
\subsubsection{Correlation of GTEx expression across brain}

Table~\ref{tab:Correlation of PSP gene expression across brain regions. Expression values $log_2$ transformed with pseudocount of 1} shows the  correlation of expression of PSP genes between brain regions (Pearson product moment correlation). Anterior cingulate cortex ( mean correlation 0.94), was the most correlated brain region and cerebellum the least (mean correlation $r$=0.85). The minimum pairwise correlation was found between the cerebellar hemisphere and substantia nigra (0.792)\footnote{\url{source('~/RProjects/gtex/R/graph_gtex/gtex_correlation_table.R')}}. The highest correlation is between caudate and putamen ($r=$0.996), which are both in the basal ganglia. The overall correlation was high (median 0.948, Mean 0.914).

PSP gene expression is high in the brain and correlated with the lowest correlations found in the cerebellar hemisphere as reported by Mele et al.\cite{mele2015human}, suggesting that the PSP group is not markedly different from CNS expressed genes overall. 

Expression for genes in the PSP but not in the consensus PSD used by Hill et al\cite{hill2014human} ($n$=1323)  is increased compared to all other genes (which includes the consensus genes). The consensus genes are more highly expressed than the PSP genes not in the consensus set but the effect is much more modest (figure~\ref{fig:deg_gtex_psp})\footnote{\url{source('~/RProjects/gtex/R/graph_gtex/gtex_correlation_table_hPSD_too.R')}}.

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sat Aug  8 07:51:39 2020source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/EA2_FUMA2clipboard.R')


\begin{table}[ht]
\centering
    \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrrrrrrrr}
  \toprule
 &   Amy &   ACC &   Cau &   Ce H &   Cer &   Cor &   FC  &   Hip &   Hyp &   NA &   Put &  SN \\ 
  \midrule
  Amygdala & 1.00 & 0.98 & 0.97 & 0.80 & 0.81 & 0.96 & 0.95 & 0.99 & 0.97 & 0.97 & 0.97 & 0.95 \\ 
    Anterior cingulate cortex  & 0.98 & 1.00 & 0.96 & 0.83 & 0.83 & 0.99 & 0.99 & 0.97 & 0.96 & 0.97 & 0.95 & 0.91 \\ 
    Caudate$^*$ & 0.97 & 0.96 & 1.00 & 0.80 & 0.81 & 0.95 & 0.94 & 0.97 & 0.95 & 0.99 & 1.00 & 0.93 \\ 
    Cerebellar Hemisphere & 0.80 & 0.83 & 0.80 & 1.00 & 0.99 & 0.84 & 0.84 & 0.81 & 0.83 & 0.81 & 0.79 & 0.78 \\ 
    Cerebellum & 0.81 & 0.83 & 0.81 & 0.99 & 1.00 & 0.86 & 0.84 & 0.82 & 0.84 & 0.82 & 0.80 & 0.79 \\ 
    Cortex & 0.96 & 0.99 & 0.95 & 0.84 & 0.86 & 1.00 & 0.99 & 0.96 & 0.93 & 0.95 & 0.94 & 0.88 \\ 
    Frontal Cortex  & 0.95 & 0.99 & 0.94 & 0.84 & 0.84 & 0.99 & 1.00 & 0.95 & 0.94 & 0.95 & 0.93 & 0.88 \\ 
    Hippocampus & 0.99 & 0.97 & 0.97 & 0.81 & 0.82 & 0.96 & 0.95 & 1.00 & 0.97 & 0.96 & 0.97 & 0.95 \\ 
    Hypothalamus & 0.97 & 0.96 & 0.95 & 0.83 & 0.84 & 0.93 & 0.94 & 0.97 & 1.00 & 0.95 & 0.95 & 0.97 \\ 
    Nucleus accumbens$^*$ & 0.97 & 0.97 & 0.99 & 0.81 & 0.82 & 0.95 & 0.95 & 0.96 & 0.95 & 1.00 & 0.98 & 0.91 \\ 
    Putamen$^*$  & 0.97 & 0.95 & 1.00 & 0.79 & 0.80 & 0.94 & 0.93 & 0.97 & 0.95 & 0.98 & 1.00 & 0.94 \\ 
     Substantia nigra & 0.95 & 0.91 & 0.93 & 0.78 & 0.79 & 0.88 & 0.88 & 0.95 & 0.97 & 0.91 & 0.94 & 1.00 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Correlation of PSP gene expression across brain regions]{Correlation of PSP gene expression across brain regions. Expression values $log_2$ transformed with pseudocount of 1. $n$=3055 genes passing quality control. *=basal ganglia. Frontal cortex = Brain area 9, Anterior cingulate cortex = 24.}
\tiny\url{source('~/RProjects/gtex/R/graph_gtex/multi_boxplot.R')}
\label{tab:Correlation of PSP gene expression across brain regions. Expression values $log_2$ transformed with pseudocount of 1}
\end{table}




\section{Genome Wide Association Studies}

Previous network analyses have shown that synaptic topological models are differentially enriched for disease and gene ontology terms. There is also evidence that central nodes may be more likely to be disease nodes. These analyses have not however made use of the complex polygenetic signal available in genome wide association studies. Chapter 1 introduced the reasons for using intelligence as a phenotype.

To test the association between synaptic proteome network properties and structures and population genetic variation associated with intelligence, we will need to calculate a statistic measuring the association of genetic variants with the phenotype for each gene that is represented by a node in the network. These values will be calculated using MAGMA in four samples, two of intelligence and two of educational attainment\cite{de2015magma}.


\subsection{Population genetic samples}
\label{sec:cohorts from paper section}
%\textcolor{red}{from paper section}
To investigate whether topological communities are enriched with genetic variants associated with cognitive ability and educational attainment, and to reduce type II  error, I will use a discovery and replication cohort design (see section~\ref{sec:Study design topology based GSA} and figure~\ref{fig:study design community detection}). 

For consistency I will refer throughout the text to the GWA study of genetic variants associated with intelligence in UK Biobank\cite{bycroft2018uk} as Intelligence\textsubscript{Discovery}. and the Phase II Educational attainment GWA study conducted using UK Biobank as Education\textsubscript{Discovery}.

We refer to the meta-analytic GWAS of intelligence carried out by Sniekers et al. from the Complex Trait Genetics (CTG) lab as Intelligence\textsubscript{Replication} \cite{sniekers2017genome}.   The meta-analysis of GWA studies of educational attainment excluding UK Biobank and 23andMe by Okbay et al.\cite{okbay2016genome} (often called EA2 - Educational Attainment 2) is referred to as Education\textsubscript{Replication}. Details of the generation and composition of these samples are found in section~\ref{sec:samples from paper section}. 

Two further samples became available after the completion of the primary analysis. These are the largest intelligence and education cohorts to date \cite{savage2018genome}, \cite{lee2018gene} but overlap with the samples above. They will be used to check the findings from earlier sections in chapter~\ref{chap:checking our findings}.

\subsection{Samples}

\label{sec:samples from paper section}
The information in this section was prepared by Dr Hill (section~\ref{sec:ack}) for a manuscript we are preparing. Dr Hill calculated the $\chi^2$ estimates of heritability. I have repeated these calculations using LD hub,\footnote{LD hub \url{http://ldsc.broadinstitute.org/ldhub/} code to prepare files for LD hub at \url{source('~/RProjects/format_hub/R/format_summary_hub.R')}} a web interface for LD score regression \cite{zheng2017ld}.

 The Intelligence\textsubscript{Discovery} sample was formed using the ``fluid'' cognitive test from UK Biobank;  verbal-numerical reasoning (VNR). The VNR test contains 13 multiple choice items, six of which are verbal with the remaining seven being numerical in nature. Each individual’s VNR score was the sum of correct responses attained within a two-minute time period. This test has a large genetic correlation with intelligence scores derived from psycho-metrically validated cognitive tests \cite{hill2019combined}. A total of 120,934 participants (mean $\chi^2$=1.52) were available for analysis who did not contribute data to the Intelligence\textsubscript{Replication}\cite{sniekers2017genome} data set. A full description of how data from these participants was analysed can be found in Hill et al. \cite{hill2019combined}. 
 
UK Biobank was also used as the Education\textsubscript{Discovery} sample. Summary statistics were taken from Hill et al. on 273,274 participants (mean $\chi^2$= 1.67)\cite{hill2019combined}.  Education was defined as a binary choice of whether or not the individual had a University/College degree. A full description of how these data were analysed can be found in Hill et al. \cite{hill2019combined}.

The Intelligence\textsubscript{Replication} data set was formed using summary statistics obtained from the Sniekers et al. GWAS meta-analysis of intelligence (n = 78,308, mean $\chi^2$= 1.30).  \cite{sniekers2017genome}  An Education\textsubscript{Replication} data set was formed using summary GWAS data based on the the GWAS meta-analysis by Okbay et al (n = 217,569, mean $\chi^2$= 1.43) with UK Biobank participants excluded. \cite{okbay2016genome}. The summary statistics with UK Biobank removed were kindly provided by Dr Okbay and colleagues who generated the new summary statistics.    (See tables~\ref{tab:supplementary dh table 1 datasets} and \ref{tab:supplementary LD regression hill}). 
%\todo[inline]{convert these tables to latex DONE}
For genotype, meta-analysis methods and association analyses see the publications reporting the original GWAS\cite{sniekers2017genome},\cite{okbay2016genome},\cite{hill2019combined}.

%\textcolor{red}{end from paper section} \todo{add the calculation of chi sq for these studies was performed by Dr Hill in the course of preparation of a manuscript based on the community detection results}

%textcolor{red}{this is done using LD regression from LD hub repeat this yourself}


 





\subsection{MAGMA Gene level statistics: Validation}
\label{sec:MAGMA Gene level statistics validation from other studies}

To ensure the accuracy of the analysis of the gene level significance statistic I compared the gene level scores obtained using MAGMA to those reported in available studies\cite{de2015magma}. Potential sources of variation or bias include variations in command line options for software such as MAGMA,  different calculation of gene scores for raw genotype and summary data (see section~\ref{sec:MAGMA_gene_scores}) and different software versions, different reference populations to account for LD and variation in low level libraries across platforms\cite{kim2018experimenting}. There were no published papers available containing a MAGMA gene-level analysis of the Intelligence\textsubscript{Discovery} sample or Education\textsubscript{Discovery} as they were generated for this analysis.

The study by Okbay et al. (EA2)\cite{okbay2016genome}, providing the Education\textsubscript{Replication} sample did not report MAGMA gene scores, and differs from the Education\textsubscript{Replication} sample by including UK Biobank Educational attainment data. The only published reports of MAGMA gene scores from any of the exact discovery and replication samples was, therefore, the Intelligence\textsubscript{Discovery} sample \cite{sniekers2017genome}\footnote{supplementary table 8 at \url{https://static-content.springer.com/esm/art\%3A10.1038\%2Fng.3869/MediaObjects/41588_2017_BFng3869_MOESM2_ESM.xlsx})}.

Summary data has recently been made available from the current largest GWAS of intelligence to date by Savage et al. \cite{savage2018genome} ($n=269,867$) this reports MAGMA Gene level analysis results although only for significant genes\footnote{table S15 in supplementary materials at \url{https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6411041/bin/NIHMS1006639-supplement-Supplementary_Tables.xlsx})}.

The pairwise correlation between the gene scores reported in these studies and the results I calculated using MAGMA and their summary statistics  are shown in table~\ref{tab:Correlation between MAGMA gene scores obtained from summary statistics and those published in supplementary material to papers}. Magma 1.06b for MacOS was used with 1000 Genomes Phase 3 data to account for linkage disequilibrium. NCBI 37 was used for gene boundaries. Summary GWAS statistics were used to calculate gene scores using default settings. No gene window was used.



\begin{table}[ht]
    \centering
        \setlength{\extrarowheight}{2pt}
    \begin{tabular}{llllllll}
    \toprule
        Study & $n$ & $t$ & d.f. & $p$ & Pearson $r$ & CI   \\
        \midrule
       Savage (2018) \cite{savage2018genome} & 269,687 & 122.26& 352  & $<2.2 \times 10^{-16}$ & 0.99 & 0.986-0.991  \\ 
%       Hill \cite{hill2019combined} & 248482 & 368.27 & 17475 & $<2.2 \times 10^{-16}$& 0.94 & 0.939-0.942 \\
 %      Davies (2016) \cite{davies2016genome} & 112151 & 22.08 & 93 & $<2.2 \times 10^{-16}$ & 0.92 & 0.88-0.94\\
       Sniekers (2017)\cite{sniekers2017genome}& 78,308 & 466.41 & 18246 & $<2.2 \times 10^{-16}$ & 0.96 & 0.959-0.962\\
       \bottomrule
    \end{tabular}
    \caption[MAGMA scores benchmarked against publications]{Correlation between MAGMA gene scores obtained from summary statistics and those published in supplementary material to papers. CI = Confidence interval}
    \label{tab:Correlation between MAGMA gene scores obtained from summary statistics and those published in supplementary material to papers}
\end{table}

\subsubsection{Results}
The correlation coefficient varied in the range 0.96-0.99 despite slight differences in software version, gene windows and the potential use of genotyped versus summary data. I conclude that apart from small variants that there is no systematic inaccuracy in the calculation of MAGMA gene scores using the plaform and pipeline. 


\subsection{Synaptic genes absent from NCBI 37}

All of the primary GWA studies from which summary statistics were derived used NCBI 37 to provide genetic co-ordinates. Seventeen PSP genes were not in the reference NCBI 37 boundaries text file used by MAGMA to assign SNPs to genes. Files with NCBI 36,37 and 38 gene boundaries are supplied on the MAGMA project website,  five of the genes not in the NCBI 37 file are found in NCBI 38 (table~\ref{tab:Synaptic PSP genes  found in NCBI 38 but not in 37 listing included with MAGMA}\footnote{\tiny\url{source('~/RProjects/annotations_phd/R/missing_NCBI_genes.R')}}).


\begin{table}[ht]
\centering
\setlength{\extrarowheight}{2pt}
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{lllll}
  \toprule
 Gene ID & Symbol & Chr & Gene type & Full name from nomenclature authority \\ 
  \midrule
1804 & DPP6 & 7 & protein-coding & dipeptidyl peptidase like 6 \\ 
  3635 & INPP5D & 2 & protein-coding & inositol polyphosphate-5-phosphatase D \\ 
  3800 & KIF5C & 2 & protein-coding & kinesin family member 5C \\ 
  9554 & SEC22B & 1 & protein-coding & SEC22 homolog B, vesicle trafficking protein \\ 
  23380 & SRGAP2 & 1 & protein-coding & SLIT-ROBO Rho GTPase activating protein 2 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Synaptic PSP genes with boundaries found only in NCBI 38]{Synaptic PSP genes  found in NCBI 38 gene boundary file supplied with MAGMA but not in NCBI 37 file. Gene ID = Entrez Gene Gene ID, chr = chromosome number.}
\label{tab:Synaptic PSP genes  found in NCBI 38 but not in 37 listing included with MAGMA}
\end{table}


Ten PSP genes are missing from both NCBI 37 and NCBI 38 gene boundary files but are annotated in the \textit{Homo sapiens} gene information at Entrez Gene\footnote{\url{https://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Homo_sapiens.gene_info.gz} } (table~\ref{tab:10 synaptic genes not in NCBI 38 included with MAGMA but present in entrez gene gene info}).

\begin{table}[ht]
\centering
\setlength{\extrarowheight}{2pt}
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{lllll}
  \toprule
 Gene ID & Symbol & Chr & Gene type & Full name from nomenclature authority \\ 
  \midrule
293 & SLC25A6 & X$|$Y & protein-coding & solute carrier family 25 member 6 \\ 
  3493 & IGHA1 & 14 & other & immunoglobulin heavy constant alpha 1 \\ 
  3500 & IGHG1 & 14 & other & immunoglobulin heavy constant gamma 1 \\ 
  3502 & IGHG3 & 14 & other & immunoglobulin heavy constant gamma 3  \\ 
  3503 & IGHG4 & 14 & other & immunoglobulin heavy constant gamma 4  \\ 
  3507 & IGHM & 14 & other & immunoglobulin heavy constant mu \\ 
  3514 & IGKC & 2 & other & immunoglobulin kappa constant \\ 
  4509 & ATP8 & MT & protein-coding & mitochondrially encoded ATP synthase 8 \\ 
  4513 & COX2 & MT & protein-coding & mitochondrially encoded cytochrome c oxidase II \\ 
  4538 & ND4 & MT & protein-coding & mitochondrially encoded NADH dehydrogenase 4 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Synaptic genes not in NCBI boundary files in MAGMA]{PSP genes ($n=10$) not in MAGMA n NCBI gene boundary files (NCBI36,37,38) annotated in Entrez Gene \textit{Homo sapiens} gene info files.  GeneID= NCBI Entrez Gene GeneID, Chr = Chromosome number.  }
\tiny\url{https://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Homo_sapiens.gene_info.gz}
\label{tab:10 synaptic genes not in NCBI 38 included with MAGMA but present in entrez gene gene info}
\end{table}


Two further pseudo genes were identified using the ToppGene API\cite{chen2009toppgene}:  RAB1C, Entrez id:	441400, member RAS oncogene family pseudogene, UniProt Q92928, Protein transport. Probably involved in vesicular traffic (By similarity); POTEKP\footnote{POTE expressed in Prostate ovary testis and placenta}, POTE ankyrin domain family member K, Entrez id:440915, pseudogene\cite{bera2002pote}, Uniprot Putative beta-actin-like protein 3, Q9BYX7. 


Gene ontology enrichment analysis using ToppFun (part of the ToppGene suite) (section~\ref{sec:ToppGene GO enrichment}) did not find any phenotype or ontology suggestive of being associated with cognition  (GO:0034987,	immunoglobulin receptor binding, 	p Bonferroni $6.224\times10^{-9}$,	six terms; 	GO:0097478,	leaflet of membrane bilayer,	Bonferroni p=$1.296\times10^{-3}$,	six	terms). 

The potentially concerning gene is SRGAP2, Entrez 23380,SLIT-ROBO Rho GTPase activating protein 2 which is associated with the cellular component dendritic spine head GO:0044327 and associated with neural morphogenesis\cite{guerrier2009f}. The gene list was also analysed using the Gene2Func interface in FUMA \cite{watanabe2017functional} (see section~\ref{sec:FUMA go enrichment}) and no enriched gene sets or hits in GWAS catalog\cite{welter2014nhgri} were found.

 While it would be possible to make a custom annotation file for these genes this would complicate our analysis and cause it to differ methodologically from the rest of the literature using MAGMA and addressing the genetics of intelligence and educational attainment.
 
% latex table generated in R 3.6.1 by xtable 1.8-4 package
% Wed Feb 19 14:14:05 2020
% latex table generated in R 3.6.1 by xtable 1.8-4 package
% Wed Feb 19 14:15:51 2020

   




\section{MAGMA gene level statistic methods}
\label{sec:MAGMA gene level methods}
MAGMA version 1.06 for MacOS was used to calculate gene level statistics from summary data and 1000 Genome European Ancestry Phase 3 data\footnote{\url{https://ctg.cncr.nl/software/MAGMA/ref_data/g1000_eur.zip}} was used to account for linkage  disequilibrium. Gene boundaries were defined by the Genome Reference Consortium Human Build 37 (NCBI 37.3) data file provided on the MAGMA website\footnote{\url{https://ctg.cncr.nl/software/MAGMA/aux_files/NCBI37.3.zip}}.

Output logs and output files are included in the supplemental material.  Command line arguments were supplied using a bash shell script and default settings were used for summary GWAS statistic data (SNPwise sum)\footnote{ \url{/Users/grantrobertson/Programs/copy_paperMagma_draft-master/magma_script.sh}	}. No gene window was used around gene boundaries (see section~\ref{sec:Gene windows}). Where the number of individuals tested per SNP was available in the summary data this was supplied as a command line argument - otherwise the overall sample size was used. SNPs that were analysed in less than 50 individuals were discarded (default setting). 

\subsubsection{Gene Windows}
\label{sec:Gene windows}

 In order to include gene regulatory elements in gene level analysis a number of ``windows''
around the gene boundary have been used. VEGAS \cite{liu2010versatile} used a set 50kB boundary and this is incorporated in studies that used this method (e.g Hill et al. (2014)\cite{hill2014human})


%WYN
Reported window sizes vary. Zhao and Nyholt \cite{zhao2017gene} in a study of gene level statistics across psychiatric disorders using the GATES method \cite{li2011gates}  used a 15kb window reasoning that 90\% \cite{pickrell2010understanding} of eQTL are found in this region. 

The predominant practice at present appears to be either not use a gene window or only to use a small one (for example 5kb as reported in Savage et al (2018. \cite{savage2018genome}). In the absence of a widely agreed upon window size I have not used one believing this to be the simplest and most conservative option. 




\subsection{Correlation of MAGMA GWGAS between samples}

Educational attainment and Intelligence are genetically correlated\cite{plomin2018new} (see table~\ref{tab:supplementary LD regression hill}). It is likely that gene level scores in samples between and within these phenotypes are also correlated although this should be less than the overall SNP correlation because of inter-genic SNPs and a loss of information in the aggregation of SNPs to genes. It is worthwhile checking that the value is not extreme or unexpected (for example higher between low powered samples with different phenotypes) as part of overall quality control. 

\begin{table}[]
    \centering
   
  
    \setlength{\extrarowheight}{2pt}
     \begin{adjustbox}{width=\textwidth}
    \begin{tabular}{llllll}
    \toprule
        Phenotype &  sample & n & mean $\chi^2$ & LD intercept (SE) & Reference \\
        \midrule
       
        Intelligence& Discovery&120,934 & 1.53 & 1.02 (0.01)& NA\\
        & Replication &78,308 & 1.30 & 1.02 (0.01) & Sniekers et al.(2017)\cite{sniekers2017genome}\\
        \\
        Education& Discovery &273,274 & 1.67 & 1.042 (0.01) & NA\\
      & Replication & 217,569&1.43&1.02 (0.01) & Okbay et al.(2016)\cite{okbay2016genome}\\
         \bottomrule
    \end{tabular}
      \end{adjustbox}
    \caption[Size and heritability for each data sample]{References and heritability of each data set. Table adapted from table generated by W.D Hill for publication. Calculations and arrangement are work of Dr W.D. Hill}
    \label{tab:supplementary dh table 1 datasets}
\end{table}


Correlation was calculated between the -log10 transform of gene level p values between samples using Pearsons product moment correlation test in R. Where a gene did not have a p value in a study this pairwise comparison was omitted. The correlation is greatest between the two educational attainment samples (0.506; CI 0.495-0.516, t = 78.473, df = 17928, p-value $< 2.2 \times 10^{-16}$) (table~\ref{tab:correlation of MAGMA GWGAS scores}). For each sample other than Intelligence\textsubscript{Discovery} the correlation is highest with the matching phenotype.

The two discovery samples are similar in composition and the Intelligence\textsubscript{Replication} sample is likely to have least power (lowest sample size, meta-analysis and lowest $\chi^2$ on testing heritability using LD score regression (section~\ref{sec:samples from paper section}). Intelligence\textsubscript{Replication} is most highly correlated with Intelligence\textsubscript{Discovery} (0.378) despite its generally lower correlation levels. 

\begin{table}[]
    \centering
   \setlength{\extrarowheight}{2pt}
    \begin{tabular}{lllll}
    \toprule
      &  \multicolumn{2}{c}{\textit{Replication}} & \multicolumn{2}{c}{\textit{Discovery}} \\
\cmidrule{2-5}
    &  Intelligence  & Education &
    Intelligence & Education \\
    \midrule
     Intelligence\textsubscript{Replication}    & & 0.249 &0.378  & 0.308 \\
     Education\textsubscript{Replication} &0.249 &&0.325&0.506 \\
     Intelligence\textsubscript{Discovery}&0.378 &0.325&&0.446 \\
     Education\textsubscript{Discovery}&0.308&0.506 &0.446 & \\
\bottomrule
    \end{tabular}
    
    \caption[Correlation of MAGMA GWGAS scores between samples]{Correlation of MAGMA GWGAS scores between samples. Values are pearson product moment correlation (p$<2.2\times10^{-16}$ all). Correlation is between -log\textsubscript{10}(p) transformation of gene level p values. Cases where a gene score is present in one study are excluded}
    \tiny\url{source('~/RProjects/paper_xls_output/R/chapter_2/MAGMA/pairwise_gene_comparison.R')}
    \label{tab:correlation of MAGMA GWGAS scores}
\end{table}
%\footnote{REMOVE FINAL: Result details commented out search \%DETAILS in tex} 
%%%%% COMPARE DETAILS
% t = 34.404, df = 17937, p-value < 2.2e-16
% alternative hypothesis: true correlation is not equal to 0
% 95 percent confidence interval:
%  0.2350242 0.2624803
% sample estimates:
%       cor 
% 0.2488022 

% ed and ctg

% 	Pearson's product-moment correlation

% t = 43.632, df = 18158, p-value < 2.2e-16
% alternative hypothesis: true correlation is not equal to 0
% 95 percent confidence interval:
%  0.2948248 0.3211538
% sample estimates:
%       cor 
% 0.3080483 


% 	Pearson's product-moment correlation

% ukbb int and ctg
% t = 55.029, df = 18158, p-value < 2.2e-16
% alternative hypothesis: true correlation is not equal to 0
% 95 percent confidence interval:
%  0.3655291 0.3904609
% sample estimates:
%       cor 
% 0.3780636 

% 	Pearson's product-moment correlation

% ukbbed and ctg
% t = 43.632, df = 18158, p-value < 2.2e-16
% alternative hypothesis: true correlation is not equal to 0
% 95 percent confidence interval:
%  0.2948248 0.3211538
% sample estimates:
%       cor 
% 0.3080483 


% ukbbint and ea2
% 	Pearson's product-moment correlation

% 	Pearson's product-moment correlation

% t = 46.068, df = 17928, p-value < 2.2e-16
% alternative hypothesis: true correlation is not equal to 0
% 95 percent confidence interval:
%  0.3121876 0.3383644
% sample estimates:
%       cor 
% 0.3253383  
% 	Pearson's product-moment correlation
% ed and ea2

% t = 78.473, df = 17928, p-value < 2.2e-16
% alternative hypothesis: true correlation is not equal to 0
% 95 percent confidence interval:
%  0.4946612 0.5164525
% sample estimates:
%       cor 
% 0.5056375 

% 	Pearson's product-moment correlation

% ed and int  
% t = 67.416, df = 18285, p-value < 2.2e-16
% alternative hypothesis: true correlation is not equal to 0
% 95 percent confidence interval:
%  0.4344972 0.4577150
% sample estimates:
%       cor 
% 0.4461812 



\begin{table}[]
    \centering
      \setlength{\extrarowheight}{2pt}
    \begin{tabular}{llllll}
    \toprule
       &  &\multicolumn{2}{c}{\textit{Intelligence}} & \multicolumn{2}{c}{\textit{Education}} \\
       \cmidrule{3-6}
       Phenotype & Samples & Intelligence & & Education &    \\
       \midrule
         & & Discovery & Replication & Okbay & Hill\\
         \midrule
         Intelligence & Hill & 21.19\%(0.91\%) & & & \\
         & Sniekers & 0.99(0.02) & 19.45\%(1.09) & & \\
         \\
         Education & Okbay & 0.62(0.03) & 0.75(0.02) & 9.78(0.44\%) &\\
         & Hill & 0.66(0.02) & 0.72(0.02) & 0.96(0.02)&11.67\%(0.91\%)\\
         \bottomrule
    \end{tabular}
    \caption[Heritability of samples]{Using LD score regression heritability within, and genetic correlations between, each of the data sets used is shown. Heritability on the observed scale is shown on the diagonal and genetic correlations are shown on the off diagonal with standard error in brackets. Table contents and caption from Dr W.D. Hill~\ref{sec:ack}. }
    \label{tab:supplementary LD regression hill}
\end{table}



\subsection{Intelligence Replication GWGAS}
\label{sec:Intelligence Replication GWGAS}

\subsubsection{Methods}
The Intelligence\textsubscript{Replication} sample is the 2017 ``Intelligence wave 1'' meta-analysis study by Sniekers et al.\cite{sniekers2017genome} (table~\ref{tab:supplementary dh table 1 datasets}). 

GWAS summary data were downloaded from the authors Complex Trait Genomics (CTG) lab\footnote{ \url{https://ctg.cncr.nl/documents/p1651/sumstats.txt.gz}}. The meta-analysis of 78,308 individuals combined data from 13 cohorts with 29,610 individuals reported for the first time. UK Biobank participants took either a web based ($n=17,852$, raw genotype data used, reported for first time) or touch screen test ($n=35,257$). The Childhood Intelligence Consortium (CHIC) consortium supplied 12,441 participants\cite{benyamin2014childhood}. Five additional cohorts contributed $n=11,749$ participants.

The individuals from this meta-analysis do not contribute to any of the other three samples used in this thesis: individuals from UK Biobank were excluded from the Intelligence\textsubscript{Discovery} sample summary statistics provided by Dr Hill\footnote{see contributors/acknowledgments}. The authors report finding 47 genome wide significant genes using MAGMA to perform a Genome Wide Gene Association (GWGAS)\cite{sniekers2017genome},\cite{de2015magma}.

Gene level scores were calculated using MAGMA v1.06b (Mac OS)(section~\ref{sec:MAGMA gene level methods}) \footnote{Code to produce the following tables is at \url{source('~/RProjects/paper_xls_output/R/0_write_MAGMA_gene_level_results_PhD.R')} }. The number of individuals tested for each SNP was available and used for sample size. The European panel of the 1000 Genomes Phase 3 was used to account for linkage disequilibrium
 (updated 19/09/2018) and downloaded from CTG MAGMA website\footnote{\url{https://ctg.cncr.nl/software/magma}}. SNPs were assigned to genes using the NCBI37.3 Gene boundary file from the CTG website and no window was used around gene boundaries. Default options for summary statistics were used (SNPwise). All subsequent GWGAS performed using MAGMA use identical methods unless otherwise stated. 
% Welcome to MAGMA v1.06b (mac)

% Using flags:
%         --bfile g1000_eur
%         --gene-annot ./annotations/annotated_bim.genes.annot
%         --pval /Users/grantrobertson/Programs/GWAS_studydata/sumstats.txt use=3,11 ncol=6
%         --out ./gene_results/ctg

% Start time is 11:28:37, Wednesday 12 Sep 2018

% Loading PLINK-format data... 
% Reading file g1000_eur.fam... 503 individuals read
% Reading file g1000_eur.bim... 22665064 SNPs read
% Preparing file g1000_eur.bed... 

% Reading SNP synonyms from file g1000_eur.synonyms (auto-detected)
%         read 5966573 synonyms from file mapping to 3894269 SNPs in the data
%         WARNING: detected 102 synonymous SNP pairs in the data
%                  skipped all synonym entries involved, synonymous SNPs are kept in analysis
%                  writing list of detected synonyms in data to supp lementary log file
% Reading SNP p-values from file /Users/grantrobertson/Programs/GWAS_studydata/sumstats.txt... 
%         detected 12 variables in file
%         using variable: rsid (SNP id)
%         using variable: p_value (p-value)
%         using variable: N (sample size; discarding SNPs with N < 50)
%         read 12104295 lines from file, containing valid SNP p-values for 10674310 SNPs in data (88.19% of lines, 47.1% of SNPs in data)
%         WARNING: file contained 3 duplicate SNPs (same IDs or synonyms)
%                  dropped all occurrences of each from analysis
%                  writing list of duplicated IDs to supplementary log file
% Filtering phenotype/covariate missing values... 503 individuals remaining

% Loading gene annotation from file ./annotations/annotated_bim.genes.annot... 
%         19248 gene definitions read from file
%         found 18262 genes containing SNPs in genotype data

% Starting gene analysis... 
%         using model: SNPwise-sum

%         writing gene analysis results to file ./gene_results/ctg.genes.out
%         writing intermediate output to file ./gene_results/ctg.genes.raw


% End time is 11:40:04, Wednesday 12 Sep 2018 (elapsed: 00:11:27)


\subsubsection{Results }
MAGMA v1.06 calculated gene level scores for 18,262 genes using the supplied summary statistics.
The Bonferroni correction for multiple comparisons was used to calculate a genome wide significance level ($\alpha$ = 0.05/18262; p=$2.74 \times 10^{-6}$). Forty seven genes achieved genome wide significance using the adjusted $\alpha$ level.

This is identical to the number reported by Sniekers et al. \cite{sniekers2017genome}. Sixteen significant genes were found in the PSP (34.0\% of significant genes)  (table~\ref{tab:significant genes})\footnote{Code for Gene level statistics at \url{source('~/RProjects/paper_xls_output/R/0_write_MAGMA_gene_level_results.R')}}.  


Significant synaptic genes are shown in table~\ref{tab:Significant synaptic genes in ctg MAGMA GWAS2}. All significant genes ($n$=47) are listed in the supplemental material. Due to the number of significant genes in subsequent sections their details are recorded in the supplementary material. 

\begin{table}[ht]
    \centering
      \setlength{\extrarowheight}{2pt}
    \begin{tabular}{llll}
    \toprule
         Sample & $n$ significant PSP & $n$ significant genes & percent (\%) in PSP \\
      \midrule  
         Intelligence\textsubscript{Replication}\cite{sniekers2017genome} & 16 (7) & 47 & 34.0 \\
         Education\textsubscript{Replication}\cite{okbay2016genome} & 25 (12) & 99 & 25.2 \\
         Intelligence\textsubscript{Discovery}\cite{hill2019combined} & 51 (17) & 196 & 26.0\\
         Education\textsubscript{Discovery}\cite{hill2019combined} & 58 (27) & 235 & 24.7\\
         \bottomrule
    \end{tabular}
    
    \caption[Number of Genome wide significant genes and number of genes in PSP at GWGAS in Intelligence and Educational Attainment samples]{Number of genome wide significant genes (using Bonferroni correction of 0.05/$n$ genes)   using MAGMA Genome Wide Gene Association Analysis Study (GWGAS) for Educational attainment and Intelligence discovery and replication samples. The references in the study column refer to the publication from which the majority of the sample are taken.  . Number in parenthesis are number of significant genes in 1312 gene consensus hPSD.}
    \tiny\url{source('~/RProjects/paper_xls_output/R/chapter_2/exploratory_data_analysis_core_PSD.R')}
    \label{tab:significant genes}
\end{table}




% latex table generated in R 3.6.1 by xtable 1.8-4 package
% Wed Feb 19 11:25:28 2020
\begin{table}[ht]
\centering
   \setlength{\extrarowheight}{2pt}
\begin{tabular}{llllll}
  \toprule
 GeneID & Symbol & chr & $n$ SNPs & z-score & MAGMA $p$ \\ 
  \midrule
1434 & CSE1L &  20 & 176 & 5.96 & $1.26 \times 10^{-9}$ \\ 
  60412 & EXOC4 &   7 & 2184 & 5.88 & $1.99 \times 10^{-9}$ \\ 
  55964 & SEPT3 &  22 &  69 & 5.75 & $4.35 \times 10^{-9}$ \\ 
  6138 & RPL15 &   3 &  19 & 5.40 & $3.35 \times 10^{-8}$ \\ 
  23109 & DDN &  12 &   8 & 5.35 & $4.41 \times 10^{-8}$ \\ 
  85358 & SHANK3 &  22 & 199 & 5.31 & $5.49 \times 10^{-8}$ \\ 
  487 & ATP2A1 &  16 &  47 & 5.23 & $8.37 \times 10^{-8}$ \\ 
  11273 & ATXN2L &  16 &  36 & 5.17 & $1.17 \times 10^{-7}$ \\ 
  2870 & GRK6 &   5 &  39 & 5.07 & $2.03 \times 10^{-7}$ \\ 
  28512 & NKIRAS1 &   3 &  73 & 5.04 & $2.29 \times 10^{-7}$ \\ 
  4700 & NDUFA6 &  22 &  16 & 5.04 & $2.32 \times 10^{-7}$ \\ 
  
  4733 & DRG1 &  22 &  99 & 4.83 & $6.91 \times 10^{-7}$ \\ 
  10564 & ARFGEF2 &  20 & 406 & 4.80 & $7.83 \times 10^{-7}$ \\ 
  7284 & TUFM &  16 &   7 & 4.79 & $8.49 \times 10^{-7}$ \\ 
  553115 & PEF1 &   1 &  30 & 4.77 & $9.17 \times 10^{-7}$ \\ 
  8411 & EEA1 &  12 & 475 & 4.61 & $2.03 \times 10^{-6}$ \\ 
   \bottomrule
\end{tabular}
\caption[Intelligence\textsubscript{Replication} Sample Genes in the PSP meeting genome wide significance levels for MAGMA GWGAS] {MAGMA GWGAS Intelligence\textsubscript{Replication} PSP genes meeting genome wide significance at GWGAS using MAGMA for the Intelligence\textsubscript{Replication} sample. chr = chromosome number.  Sixteen of genes are found in the PSP (34.0\%, total significant genes = 47), Bonferroni corrected $\alpha=2.74 \times 10^{-06}$. $n$ SNPs = number of SNPs found within NCBI 37 
defined gene boundaries in sample. GeneID refers to Entrez Gene GeneID.} 
\label{tab:Significant synaptic genes in ctg MAGMA GWAS2}
\end{table}



\subsection{Education Replication MAGMA GWGAS}
\label{sec:Edcuation replication gwgas}
\subsubsection{Methods}

The Education\textsubscript{Replication} sample is a subset of individuals from the EA2 (Educational Attainment 2) meta-analytic GWA study of educational attainment\cite{okbay2016genome}. Educational attainment is measured as the number of years of schooling completed (EduYears).  The study comprised 293,723 individuals with 111,349 independent samples from UK Biobank used for out of sample replication \cite{sudlow2015uk}. A combined meta-analysis was carried out using the discovery and replication cohorts ($n$ = 405,702) allowing an increase in the number of genome wide loci reported from 74 to 162. Due to Institutional Review Board (IRB) restrictions, which did not allow reporting of over 10,000 SNPs in the summary statistics for participants from 23andme, the summary statistics provided on the SSGAC website are of $n$=328,917 individuals excluding 76,155 individuals from 23andme\footnote{\url{http://ssgac.org/documents/EduYears_Main.txt.gz}}.

The authors kindly made summary statistics available to us (see acknowledgements~\ref{sec:ack}) excluding both 23andme and UK Biobank creating a new sample that does not overlap with Education\textsubscript{Discovery}; $n$= 217,568 (293,723 - 76,155 = 217,568)\cite{okbay2016genome}\footnote{See Okbay et al. supplemental information \url{http://ssgac.org/documents/SI_74_loci_educational_attainment.pdf} p 16}. The summary statistics used were unsorted by sex. The modified summary statistics excluding UK Biobank and 23andMe  are deposited with the University of Edinburgh Research Data Service \footnote{https://www.ed.ac.uk/information-services/research-support/research-data-service}.
The number of individuals tested for each SNP was available and used. Twenty duplicate SNPs were identified. 

\subsubsection{Results}


 Using SNPs from the summary data MAGMA 1.06b calculated a gene level significance scores for 17,942 genes. The Bonferroni correction for multiple comparisons was used to calculate a genome wide significance level ($\alpha$ = 0.05/17942; p = $2.79 \times 10^{-06}$). At this $\alpha$ level 99 genes were identified of genome wide significance. Twenty five (25.3\%) of these were found in the PSP (see table~\ref{tab:significant genes}). All significant and synaptic significant genes are recorded in the supplementary material. 



% latex table generated in R 3.6.2 by xtable 1.8-4 package
% Thu Jan  2 13:31:54 2020






\subsection{Intelligence Discovery Phase II GWGAS}

\subsubsection{Methods}
The Intelligence\textsubscript{Discovery} sample comprise UK Biobank Phase II participants from an analysis reported by Hill et al.\cite{hill2019combined}. The authors used MTAG (Multi-Trait Analysis of GWAS) a technique that allows meta-analyses to increase their effective sample size using genetically related traits \cite{turley2018multi}. The published meta-analysis\cite{hill2019combined} combines the Intelligence\textsubscript{Replication} sample data \cite{sniekers2017genome} and UK Biobank Intelligence Phase II (Intelligence\textsubscript{Discovery} sample) combined with the genetically related educational attainment phenotype (the SSGAC EA2 study)\cite{okbay2016genome}. Summary GWAS statistics including only participants in UK Biobank Intelligence Phase II ($n$ = 120,934) were obtained from the Centre for Cognitive Ageing and Cognitive Epidemiology (CCACE) from the author Dr W.D. Hill (section~\ref{sec:ack}).  A column is provided with the summary data of numbers tested per SNP but is constant and equal to the sample size and was used as the input for the GWGAS. The participants had taken the VNR test in UK Biobank which is closely related to intelligence\cite{hill2019combined}. For further details on the cohorts, genotyping and testing see Hill et al.\cite{hill2019combined}.


\subsubsection{Results}
18,287 genes were assigned gene level statistics using MAGMA v1.06 to provide GWGAS.  With a Bonferroni $\alpha$ level of  $
2.73\times 10^{-06}$, 196 genome wide significant genes were found at GWGAS using MAGMA. Of these 51 genes were in the PSP synaptic set (26.0\% of total; table~\ref{tab:significant genes}).








\subsection{Education Discovery sample GWGAS}
\label{sec:UKBB Education discovery GWGAS}
\subsubsection{Methods}
The GWAS summary statistics for the Education\textsubscript{Discovery} Phase II are taken from UK Biobank Phase II. The data were obtained from the Centre for Cognitive Ageing and Cognitive Epidemiology (CCACE) from Dr W.D Hill (see acknowledgements~\ref{sec:ack}). The sample size is $n$=273,274. The outcome measure is the achievement of a college or university degree which has been shown to correlated closely with years of education\cite{rietveld2013gwas}.

\subsubsection{Results}
18,287 genes contained SNPs. Numbers of individuals tested for each SNP were available and used in the command line argument. At a Bonferroni adjusted $\alpha$ level of $2.73 \times 10^{-06}$, 235 significant genes were identified. Of these, 58 were members of the PSP (24.7\%).  

Significant genes are recorded in the supplemental material. 

% latex table generated in R 3.6.1 by xtable 1.8-4 package
% Fri Feb 21 12:10:53 2020
% latex table generated in R 3.6.2 by xtable 1.8-4 package
% Thu Jan  2 13:49:56 2020

\subsection{PASCAL and MAGMA}
\label{sec:compare pascal and magma}
Due to the importance of gene level p values to the network analysis I have used two different popular methods to calculate gene level p values for summary GWA data. PASCAL\footnote{version 0.1 15 April 2016} uses methods similar to the popular VEGAS package and is discussed in more detail in section~\ref{sec:PASCAL_gene_score} \cite{lamparter2016fast}. There is a strong linear correlation between MAGMA gene level statistics and those calculated using PASCAL see table~\ref{tab:correlation MAGMA and PASCAL}. 

\subsubsection{PASCAL software}
\label{sec:PASCAL methods}
Pascal was used using the default options and no gene boundary. The empirical method was used for calculating $p$ values. No window was used around the gene boundaries defined using NCBI 37. 1000 Genomes Phase 3 was used to account for linkage disequilibrium. The sole release version  Alpha 0.1 was used and instructions were recorded as a shell script. 

%%% Table comparison of MAGMA and PASCAL %%%
\begin{table}[ht]
    \centering
        \setlength{\extrarowheight}{2pt}
    \begin{tabular}{lllccc}
    \toprule
    Sample & $t$ & df & CI & $r$ & $p$ \\
    \midrule
      Education\textsubscript{Discovery}   & 274.9 & 17,241 & 0.890 - 0.905 & 0.902 & $<2.2 \times 10^{-16}$  \\
       Intelligence\textsubscript{Discovery} &286.01 & 17,164 & 0.907 - 0.912 & 0.909 & $<2.2 \times 10^{-16}$\\
      Education\textsubscript{Replication} & 375.95 & 17,079 & 0.943 - 0.946 & 0.945 & $<2.2 \times 10^{-16}$ \\
       
        Intelligence\textsubscript{Replication} &281.83 & 17,200 & 0.904 - 0.909 & 0.907 &$<2.2 \times 10^{-16}$ \\
        \bottomrule
        \end{tabular}
    \caption[Correlation of gene level scores MAGMA and Pascal]{Correlation between gene level $p$-values calculated in MAGMA and Pascal (-log\textsubscript{10} transform of $p$-values). Correlation and p for Pearson's product-moment correlation. }
    \label{tab:correlation MAGMA and PASCAL}
\end{table}
\clearpage



\section{Gene Ontology Analysis of significant GWGAS genes}
\label{sec:Gene Ontology Analysis of significant GWGAS genes}

Pathway analysis of the GWA studies making up the discovery and replication samples have been published\cite{sniekers2017genome},\cite{okbay2016genome},\cite{hill2019combined}, but use more than one method and can be difficult to compare. For example Okbay et. al\cite{okbay2016genome} used DEPICT for pathway analysis\cite{pers2015biological}, and reported many more involved pathways than the studies using MAGMA GSA\cite{sniekers2017genome},\cite{hill2019combined}. Although the samples making up these studies have been combined to produce powerful meta-analyses, analysis as pairs of non overlapping samples allows comparisons between samples and phenotypes has not previously been reported and may have implications for subsequent analysis.

The discovery cohorts are likely to have more power being more homogeneous (all UK Biobank), having larger samples and using a single test of intelligence in the Intelligence\textsubscript{Discovery} sample. Gene ontology analysis of the genome wide significant genes (both for the whole genome and those found in the PSP) may show if there systematic differences across the samples, for example features common to a particular phenotype or sample (more significant sets in Education\textsubscript{Discovery}).

The results reported using MAGMA GSA provide an estimate of the significance levels to be found using this method and can inform study design\cite{sniekers2017genome}\cite{hill2019combined}. The non-adjusted p values represent a likely lower bound for the significance levels in these samples given the number of gene sets examined (6,166 GO terms in Sniekers et al\cite{sniekers2017genome}, 10,891 in Hill et al.\cite{hill2019combined}). The fact that so few significant gene sets are reported (using MAGMA GSA) in some of the most well powered  population studies  in genetics limitation by type II error, or a very heterogeneous set of genetic variants \cite{sniekers2017genome},\cite{hill2019combined}. 

In the Intelligence\textsubscript{Replication} sample, the authors reported only one significant set using MAGMA GSA after multiple testing\cite{sniekers2017genome}. Hill et al. found 8 significant gene sets using MAGMA GSA (table~\ref{tab:MAGMA GSA results from Hill et al.}), but the sample size ($n$=248,482)  combines both Intelligence\textsubscript{Discovery} and Intelligence{Replication} samples, and is further increased using MTAG to incorporate data from a genetically related trait (educational attainment)\cite{hill2019combined},\cite{turley2018multi}.



Finally, it will provide a practical use of various gene ontologies enrichment platforms. The methods vary to the extent that they take into account the structure of the DAG, and hence the correlation between related terms in their implementation, how they deal with multiple comparisons, their frequency of update and variation in the results obtained\cite{khatri2005ontological},\cite{rhee2008use}.  I have therefore used more than one method in this section to discover to what extent these results are comparable, and any advantages and disadvantages the methods may have, in addition to those discussed in section~\ref{sec: gene ontology analysis}.
% The initial analysis here is an over representation analysis of genome wide significant genes using GWGAS. This does not test the competitive gene set hypothesis but rather answers a simpler question: what are these genes and what common features do they have? Although there is evidence implicating the synapse in intelligence and educational attainment \cite{hill2014human} Is there evidence that a sufficient number of important genes are synaptic to justify a synaptic network analysis.

% Finally it will provide the first practical use of gene ontologies which are the underpinning of nearly all set based approaches. One of the aims of this thesis is to learn practical lessons in the implementation of the combination genomic and network analysis methods. Gene ontologies have a structure and form a directed acyclic graph (see figure~\ref{fig:the DAG of Collateral sprouting}). Gene ontology structures are beyond the scope of this thesis and a number of methods of carrying out functional set or over representation analysis have been reported. The methods vary to the extent that they maintain the structure of the DAG and their implementation and for that reason I have had reason to use more than one method. 

% PANTHER introduced in section~\ref{sec:panther methods1} is a member of the gene ontology consortium and frequently updated. The results obtained contain links to easily view the GO tree and also present results in terms of the hierarchy. Mi et al\cite{mi2019protocol} in the most recent update acknowledge that given the relatedness of GO terms tests are non independent and therefore the Bonferroni correction is too conservative. As an alternative they recommend using the FDR method. They also implement SLIM ontologies which reduce the number of tests. Finally
% PANTHER allows the use of a custom background gene set to test enrichment against. The main drawback of PANTHER is that it can occasionally fail to recognise common gene identifiers even when presented in their recognised form. For that reason and given the fact that there are well described differences between the results for different enrichment methods I have also used ToppFun in the ToppGene package which provides convenient links to the complete gene list of terms which is useful in chapter 4. It also has a very clean interface and excellent identifier recognition (see discussion for further comments on ease of use). The main drawback of ToppFun is that it does not provide for an easy implementation of a custom background gene set. For importance of frequent updates see \cite{tomczak2018interpretation}.

% The \textit{elim} method described by Alexa \cite{alexa2006improved} and implemented in the popular topGO R package\cite{alexa2009gene} reduces to some degree unnecessary testing of parent terms when child terms have been found to be significant. It does not fully correct for multiple testing the authors stating that most multiple comparison methods are too strict given the connected structure of GO and they prefer to regard the \textit{elim} results as already corrected. topGO has the advantage of being a longstanding R package and part of the Bioconductor\cite{gentleman2004bioconductor} universe. It is well suited as part of a computational workflow but suffers from slower updates than PANTHER (and ToppGene) and misses the annotation of certain genes. 
% \paragraph{Multiple testing}
% The issue of multiple testing in GO enrichment is complex and beyond the scope of this thesis. The interested reader is directed to the excellent and complete summary by Rhee et al. (2008) \cite{rhee2008use}. Rhee et al. recommend the following approach:

% `An unbiased search for significant GO associations can be done with a bottom up approach as follows: for every leaf term, caclulate p-values with the genes directly associated to it. If any term is significant do not propagate its genes above. This would provide the most specific node that is significant in that particular branch. If a term is not significant, propagate the annotations to its parent and recalculate with the parent term. The genes will propagate upwards until a significant node is found or until the root is reached. A careful analysis is still necessary to properly correct for multiple comparisons.''\cite{rhee2008use}

% That careful analysis would presumably be that the number of tests is the number of significance tests carried out which is modified by the level of propagation. To my knowledge there is no widely available implementation of this method designed for genomic analysis.  Rhee et al. state that the Boneferroni correction is too conservative and Holm and FDR offer better alternatives and recommends minimising the number of tests wherever possible.
% \subparagraph{Clipped ontology}
% One way to do this is to use a restricted or clipped Ontology. This can be daunting to implement (see Xi) and I am not aware of any application providing a neural ontology enrichment with a simple interface or API (see Mitrea and discussion). A simple implementation of a restricted synaptic ontology is the SynGO project which provides enrichment for a synaptic restricted set, it provides a background enrichment set of brain expressed genes (which will be more conservative) but allows the use of a custom set. In the absence of a complete implementation of the strategy suggested by Rhee an alternative is to use gProfiler which uses sgs designed to increase power and respected DAG topology. 

% In practical terms one should be aware that multiple testing using Bonferroni correction (as implemented in MAGMA) is too strict, second that the results of enrichment in hypothesis generating applications do not truly represent an absolute value of the likelihood of a test under repetition but a measure of confidence in specific findings. I will return to this in the discussion. A more recent and technical analysis of the issue of  multiple testing for gene ontology is found in Meijer et al (2016)\footnote{this actually isn't the most technical there is a general treatment for inference in DAG by the same author}. \cite{meijer2016multiple}.

% OK I have just reread the MI paper. They don't talk about using the ontology tree but because of the non independence of tests they use FDR. The GO slim method by alexa et al only propogates up the tree to a certain level but does not include multiple testing

% The p values therefore are best regarded as a statement in the level of confidence that a gene set has enriched go terms rather than an exact


% Panther respects the DAG in so far as it allows visualisation of the graph structure and suggests using the FDR rather than previous correction of Bonferroni for multiple comparisons. The elim method of Alexa reduces excess testing but does not apply a test for multiple comparisons, the authors state correctly that multiple comparisons are too strict and choose to view the elim results as already corrected. Another alternative is to reduce the number of tests by using a limited or slim ontology. The GO slim ones are the ones that there is direct experimental evidence for and that are conserved but does not take into account the tree, an alternative is to use a tissue specific approach such as SynGO. It is well recognised that there is a lack of concordance across tools. Clearly the fold change is also significant but there is not a clear way it can be balanced however see chapter x. 

% The best way round this is to report multiple methods. This issue is complex and beyond the scope of the PhD. The interested reader is directed to Rhee, new one and maybe Xis paper. 

% So what - remove the Bonferroni

% MAGMA is certainly too conseervative in using Bonferroni. 

% Rhee suggests what is surely the correct approach which is worth quoting directly `

% There is a bias to large gene sets which I think is mentioned in the deLeuww paper. 



% Greater SynGO enrichment both educational attainment cohorts.


% \subsection{Methods}

% In addition to Panther described earlier I used the ToppFun functional enrichment service which is part of the toppgene suite. The web interface was used but data entry was carried out using a pipeline from GWGAS significance tests to the system clipboard to avoid transcription errors.
% Method - for each generated list of significant genes mapped to 
% Entrez, Symbol, Ensembl, Protein
% \subsubsection{from earlier}

% \todo{Other comments rhee} Rhee\cite{rhee2008use} see notes.rtf on Windows10 path commented out below 
% %%C:\Users\gxrob\OneDrive\Documents
% \textcolor{red}{The issue of multiple testing complicated for gene ontology terms as they form a directed acyclic graph where a child node is derived from a parent term (for example GO:008066 Molecular function glutamate synthesis is a child term of GO:0004888 transmembrane signalling receptor ). The recommended practice in calculating significance is to propogate up the tree until a node achieves significance and not to propagate to ever more general terms\cite{rhee2008use}. It is not clear to me that this is implemented in the enrichment packages using Gene ontology terms reported in the literature for the GWA studies for intelligence and educational attainment used here (eg MAGMA, FUMA). Variations in GO enrichment scores between packages are well recognised and as I have used both PANTHER and topGO below which respect DAG topology it does not change the analysis (however see discussion section)}


% For computational calculations of gene ontology where rapid output is required as part of a pipeline I have used topGO version 2.38.1 an R package t
% hat provides gene ontology enrichment \cite{alexa2006improved},\cite{alexa2009gene}\footnote{code \url{source('~/RProjects/graph2community/R/topgo/modified_exampleGoObject_groups.R')}}.
% specific to it will be described when first used as will be the case with other methods. 

% For limited numbers of tests where quality of results is the most important thing I have used the panther database as it is regularly updated, uses high standard corrections that respect the GO tree and is part of the gene ontology project. Results are also available as a hierarchy in gene ontology. 

% \subsubsection{PANTHER}

% Using PANTHER there were identifiers that were not recognised using Ensembl ID, Entrez ID and GeneID. The most robust appeared to be gene symbol. I used translation from Entrez ID to Gene Symbol including synonyms until all genes were identified. In addition to the code documenting the data transformation \footnote{code \url{ source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/EA2_FUMA2clipboard.R')}}, the gene mapping compatible with PANTHER is included in the supplementary materials.    \footnote{with ENSEMBL ID ENSG00000256762 was unmapped as was Uniprotand Q8IWL8 (STH)Entrez translation using FUMA resulted in 2 unrecognised gene symbols MGEA5 and C10orf76. Correcting these manually using lookup for synonyms the table was ommended to OGA and ARMH3 resulting in all 99 genes being correctly mapped. A full gene mapping compatible with PANTHER is available in the supplemental material}
%  \todo{CHANGE IMPORTANT}
%  The PANTHER enrichment respects the hierarchy of the ontology terms although the FDR values are lower than ToppGene the very specific term related to neurogenesis positive regulation of collateral sprouting (FDR p 0.041) is prominent in comparison to the same data ordered by FDR shown in table~\ref{tab:GO.biological.process.complete Panther Gene Ontology Enrichment significant genes in Education Replication}. There is also a considerable difference in actual enrichment values including the raw $p$ value as noted by ToppGene widely variant scores noted in Rhee (2008) \cite{rhee2008use} and see Khatri et al. for a detailed discussion \cite{khatri2005ontological}.\footnote{Douglas: I have included this as a screenshot. Parsing the xml and getting it into latex took longer than I thou
%  ght so I went straight to screenshot as a last resort could do the indentation by hand. Thought worth showing (as DEPICT, MAGMA etc as far as I can see ignore heirarchy) but interested in what you thought. The standard table output for panther doesn't have the heirarchy. I thought the differences in approach to the GO DAG were worth discussing  - as part of pointing out limitations to how enrichment has been approached in genomics. }
 
% \subsubsection{ToppGene}
% \label{sec:ToppGene GO erncihement}
%  Toppgene is a gene enrichment service which also includes murine and human phenotypes and disease enrichment\cite{chen2009toppgene}. It has a regularly updated database. There are differences in absolute enrichment significance values between different methods as found reported by Rhee \cite{rhee2008use} but fewer differences in annotation. Data entry to toppgene was done using Entrez ID as default. The ToppGene translation API implements  HTTP POST and HTTP PUT methods and was used for translation of identifiers and where this has been done it is noted in the manuscript. To reduce transcription errors where the text entry web interface was used this was done using direct writing to the clipboard as described in section~\ref{sec:panther methods1}. The output tables for significant ToppGene analyses including terms searched are included as tab separated text files  in the supplementary materials. ToppGene now allow the choice of a background set. Database update 28-07-20. GO BP 16465 annotations 20148 genes. CC  2043 annotations 20584 genes MF 4745 Genes 20584 (for full details see supplemental methods). GO ontology from Panther DB.  

% \subsubsection{topOnto}
% \label{sec:toponto GO enrichment preliminary}
% topOnto is an R package widely used to perform Gene Ontology enrichment and is thus very suitable for integration into a computational workflow abd implements the elim method. It is updated less frequently than PANTHER. As it is not used in this chapter a detailed discussion of the methods are contained in section~\ref{sec:Gene ontology topgo}). 
%  \subsubsection{SynGO} SynGO (Synaptic Gene ontology and annotations) is available as a web interface for Gene Ontology enrichment\footnote{\url{https://www.syngoportal.org/}}xls files in supplementary material\cite{koopmans2019syngo}. It provides an expert curated evidence based ontology mapping of synaptic genes to Gene Ontology terms and provides a subset of the complete GO ontology. It is provided as a web interface with text or file based entry. It provides a default background list of gene expressed genes. It covers the cellular component and biological function GO clades. Fishers exact test is used for testing over representation with the FDR method\cite{benjamini1995controlling} used to correct for multiple comparisons. The detailed supplemental data outputs in xls format are included in the supplemental materials. 
% \subsubsection{FUMA Gene2Func package} 
% \label{sec:FUMA go enrichment}

% FUMA (Functional Mapping and Annotation of Genome-Wide Association Studies)\footnote{\url{fuma.ctglab.nl}} is a comprehensive web based annotation and analysis packaged.\cite{watanabe2017functional} It has been widely used in the secondary analysis of GWA studies\cite{hill2019combined}\cite{savage2018genome} and implements Gene level GWGAS using MAGMA and provides extensive annotation services. The system divides into a SNP2Gene module takes summary SNP data as an input and Gene2Func module which takes a list of significant genes.  I have used the GTEx expression enrichment analysis from the Gene2Func module. This differs from the results and methods presented in section~\ref{sec:GTEx methods} in using a predetermined set of differentially expressed genes and testing over representation using the hypergeometric test and Bonferroni correction for multiple comparison (\footnote{for technical details see\url{https://fuma.ctglab.nl/tutorialgene2func} note the address is tutorial HASH gene2func}. This to provides a quick, graphical representation of the differences in tissue expression of genes in the PSP and genome wide significant genes at GWGAS in the discovery and replication samples.\todo{? add the fuma methods}. I have also used the Gene Set Analysis in the Gene2Func module which uses hypergeometric tests to assess over representation of terms. Data sets are provided ( including gene ontology terms, Wiki Pathways and GWAS catalogue) with Bonferroni correction per category. I have used the GWAS catalogue set enrichment to ensure that genes excluded from analysis are not potential sources of bias (ie reported in intelligence GWA or other neurocognitive phenotypes). FUMA v1.3.6 was used. 

% \subsubsection{g:Profiler} 
% \label{sec:gProfiler GO enrichment}
% g:Profiler, is a longstanding Gene Ontology Consortium (GOC) backed Gene Ontology functional enrichment package. recently updated modern web interface and an R API and an ELIXIR member. Over representation of ontology terms is tested using the cumulative hypergeometric test \cite{raudvere2019g} and the g:SCS (set counts and sizes) method which takes into account the overlapping between sets based on GO terms \cite{reimand2007g} used to correct for multiple comparisons. In addition it provides a nice Manhattan like plot of over representation and a comprehensive translation service that can be accessed from the R API. Although it includes data from KEGG and wiki pathways I have limited the analysis to GO enrichment terms (these are tested using ToppGene). 

% Functional enrichment was carried out using g:Profiler version e100\_eg47\_p14\_7733820 (database updated on 07/07/2020). Multiple testing was controlled using g:SCS method applying a significance threshold of 0.05\footnote{recommended citation format from website} 
% show under-representation false 

\subsection{Intelligence Replication GO analysis}
%textcolor{red}{Put in MAGMA GSA from Sniekers}
Sniekers et al. tested 6,166 Gene Ontology terms and 674 Reactome terms using MAGMA GSA
\footnote{\url{https://static-content.springer.com/esm/art\%3A10.1038\%2Fng.3869/MediaObjects/41588_2017_BFng3869_MOESM2_ESM.xlsx}}. 

 One significant gene set was identified after correction for multiple comparisons using MAGMA GSA: regulation of cell development ($p=3.5 \times 10^{-6}$; corrected p=0.03)\cite{sniekers2017genome}. This is a large and general Biological Process (GO:0060284 - lookup at Quick GO)  with 962 Homo sapiens genes annotated (using PANTHER at Gene Ontology Consortium site for lookup).

I carried out pathway analysis of the 47 genome-wide significant genes identified by GWGAS (section~\ref{sec:Intelligence Replication GWGAS}) using PANTHER\cite{mi2013large}, toppGene\cite{chen2009toppgene}, FUMA\cite{watanabe2017functional}, g:Profiler\cite{raudvere2019g} and SynGO\cite{koopmans2019syngo} (section~\ref{sec:GO enrichment methods}). Little consistent enrichment was found. 


\subsubsection{Results}
I found no significant gene ontology terms using PANTHER over representation analysis for the genome wide significant genes from MAGMA GWGAS for Biological Process or Molecular Function. Two cellular component terms were enriched: cytoplasm, GO:0005737 (41 of 11,717 genes in annotation; FDR adjusted $q$=0.0268) and synapse, (GO:0045202, 12 of 1383 in annotation, fold change 3.77, FDR adjusted $q$=0.0366). 

Using the PANTHER GO SLIM ontology, no enrichment was found for biological process or cellular component. ORA analysis of molecular function revealed three nested terms, the most specific being nucleic acid binding, GO:0003676  (11 of 1325 genes in annotation, Fold enrichment 3.68, $p$=$1.44\times10^{-4}$, FDR adjusted $q$ = 0.026)\footnote{\url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/CTG_add_uniprot.R')}}).

PANTHER failed to identify correctly formatted valid Ensembl ID and Entrez IDs as previously noted and I used a combination of HGNC symbols and synonyms to maximise the number of genes identified. This was a consistent issue in using PANTHER led me to use ToppGene once it enabled custom background sets. The exact formatting of the genes is found in supplemental material (see section~\ref{sec:panther methods1})%\footnote{ENSG did not identify ENSG00000251322 SHANK3, table from FUMA which has shank3 uniprot missing amended and added Q9BYB0.}
\footnote{Code for sets at \url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/CTG_add_uniprot.R')}\url{source('~/RProjects/paper_xls_output/R/chapter_2/translate_sig_genes/ctg_NCBI_translate2.R')}}. There was no enrichment using PANTHER pathways or protein class. 

\begin{table}[]
    \centering
        \setlength{\extrarowheight}{2pt}
    \begin{tabular}{llllll}
    \toprule
    ID & 	Cellular component &	 	p& 	FDR BH &	n tested & 	n Annotation\\ 
    \midrule
    GO:0032584 &	growth cone membrane &		$1.416\times10^{-4}$&0.038 	&	2& 	8       \\
        \bottomrule
    \end{tabular}
    \caption{Intelligence\textsubscript{Replication} pathway enrichment in ToppGene for gene ontology terms. Default background set used. FDR BH false discovery rate corrected using Benjamini and Hochberg method}
    \label{tab:toppgene ctg cc}
\end{table}

There was no evidence of increased enrichment of expression in CNS using FUMA (fig~\ref{fig:deg_upref_sample_gtex_gener} and \ref{fig:FUMA gtex deg samples multiple}). No pathway enrichment was found for biological function, molecular function or cellular component using g:Profiler. Only one term was found enriched in ToppFun a  cellular component (see table~\ref{tab:toppgene ctg cc}). No enrichment was found of gene ontology terms in ToppFun using the PSP as a background set.



 

% \begin{table}[]
%     \centering
%     \begin{tabular}{c|c}
%     ID & 	Name &	 	pValue& 	FDR BH &	FDR BY &	Bonferroni& 	Genes from Input& 	Genes in Annotation\\ 
%     GO:0032584 &	growth cone membrane &		1.416E-4 	&3.838E-2 	&2.373E-1& 3.838E-2 &	2& 	8       \\
        
%     \end{tabular}
%     \caption{Caption}
%     \label{tab:my_label}
% \end{table}
% ID 	Name 	Source 	pValue 	FDR BH 	FDR BY 	Bonferroni 	Genes from Input 	Genes in Annotation 
% latex table generated in R 3.6.3 by xtable 1.8-4 package
	

% BP panther
% All genes were singly mapped using HGNC symbols. eventually HGNC appears to be a primary key for PANTHER along with Uniprot from looking at matching but using HGNC to translate the NCBI37 symbols matching the significant entrez (from MAGMA output) you get 48 symbols

% Why?:  NRDRG1 comes as an alias for DRG1 input DRG1 output alias NDRG1

% if you don't however check aliases you don't detect all terms

% So find this by translating from output symbol HGNC to entrez using toppgene annotation service and finding the entrez id not in the magma output == 10397

% \url{source('~/RProjects/paper_xls_output/R/chapter_2/translate_sig_genes/ctg_NCBI_translate2.R')}



% CC Enriched for Synapse GO:0045202 1383 	12 	3.18 	3.77 	+ 	5.47E-05 	3.66E-02

% GO:0005737 cytoplasm 	11717 	41 	26.97 	1.52 	+ 	2.67E-05 	2.68E-02 
% intracellular 	14704 	46 	33.85 	1.36 	+ 	1.95E-05 	3.92E-02


% GO SLIM MFnucleic acid binding 	1325 	11 	3.05 	3.61 	+ 	1.76E-04 	3.12E-02
% organic cyclic compound binding 	1677 	13 	3.86 	3.37 	+ 	7.92E-05 	2.11E-02
% heterocyclic compound binding 	1646 	13 	3.79 	3.43 	+ 	6.55E-05 	3.48E-02


% GO SLIM CC
%  	Homo sapiens (REF) 	upload\_1 ( Hierarchy  NEW! Tips)
% PANTHER GO-Slim Cellular Component 	num num 	expected 	Fold Enrichment 	+/- 	raw P value 	FDR
% cell part 	8223 	32 	18.93 	1.69 	+ 	1.66E-04 	8.63E-02
% cell 	8223 	32 	18.93 	1.69 	+ 	1.66E-04 	4.32E-02

% GO SLIM BP
% None

% PANTHER pathway None

% PANTHER protein class None


% Using the PANTHER SLIM ontology no enrichment was found for biological process or cellular component. ORA analysis using PANTHER SLIM for molecular function revealed three nested terms the most specific Nucleic acid binding GO:0003676 11/1325 Fold enrichment 3.68 p $1.44\times10^{-4}$ FDR 0.0255.\footnote{\url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/CTG_add_uniprot.R')}}. PANTHER failed to identify some valid Ensembl ID and Entrez ID and so Uniprot ID was used \todo{cross ref to methods PANTHER}\footnote{ENSG did not identify ENSG00000251322 SHANK3, table from FUMA which has shank3 uniprot missing amended and added Q9BYB0.\url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/CTG_add_uniprot.R')}}

% Using FUMA for gene set analysis of these genes there was no enrichment for any Gene Ontology term. There was no statistically significant evidence of enrichment in gene expression in the central nervous system\todo{add image ? to supplementary}. 

% There was no evidence of enrichment of expression in CNS using FUMA (fig~\ref{fig:GTEX FUMA CTG}).

% No enrichment using gProfiler
% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/FUMA_plots/ctg_upregulated30tissues_46_genes.png}
%     \caption{GTEx upregulation in significant genes in Intelligence\textsubscript{Replication} sample. $-log_10(0.05)=1.301$}
%     \label{fig:GTEX FUMA CTG}
% \end{figure}
% \paragraph{toppfun}
% CC
% ID 	Name 	Source 	pValue 	FDR BH 	FDR BY 	Bonferroni 	Genes from Input 	Genes in Annotation 
% GO:0032584 	growth cone membrane 		1.416E-4 	3.838E-2 	
% 2.373E-1
% 	3.838E-2 	2 	8% latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Tue Sep 15 11:43:35 2020

% \paragraph{topp fun PSP background}
% Nil MF,BP,CC Nothing at all


\subsection{Education Replication Gene Ontology analysis}
Okbay et al.\cite{okbay2016genome}  used DEPICT\cite{pers2015biological} to investigate pathway enrichment in genes near significant SNPs. 283 gene sets (far more with DEPICT than other methods) were determined to be significant and clustered into five groups: ``neural progenitor cells, migration of new neurons to cortex, projection of axons to target, sprouting of dendrites and spines, and neuronal signalling and synaptic plasticity throughout the lifespan''\cite{okbay2016genome}. An association with genetic variants identified and  genes involved in intellectual disability was also reported. The Education\textsubscript{Replication} sample differs from that reported by Okbay et al. \cite{okbay2016genome} as results obtained from UK Biobank and the individuals from 23andme are excluded (see section~\ref{sec:Edcuation replication gwgas}).



 \subsection{PANTHER Gene Ontology analysis Education Replication}
 
 The significant genes were again formatted to maximise the number identified in PANTHER\footnote{HGNC id from Symbols from NCBI from MAGMA\url{source('~/RProjects/paper_xls_output/R/chapter_2/translate_sig_genes/ea2_NCBI_translate2.R')}}.
 
 Using the default background and FDR to correct for multiple comparisons, nineteen biological process terms were found to be enriched. No molecular function or cellular component terms were enriched. No enrichment was seen in PANTHER protein class, pathways (including Reactome pathways) or for any of the SLIM ontologies. 
 
%  Pathways nil
%  SLIM BP,MF,CC nil
 
%  Protein class nil
 
%  Reactome pathways nil
 
  %   \url{source('~/RProjects/paper_xls_output/R/chapter_2/translate_sig_genes/ea2_NCBI_translate2.R')}
     
   %  HGNC id from Symbols from NCBI from MAGMA
     
%      Fails to recognise 18839 but recognised STH which is HGNC18839. With HGNC symbols. 2 dual mapped MST1 to MST1	2 	HUMAN|HGNC=11408|UniProtKB=Q13043,
% HUMAN|HGNC=7380|UniProtKB=P26927
% but has the correct entrez 4485 (MAGMA output) P26927 MST1 (hepatocyte growth factor) other is Q13043 stk4 
%  99 entrez genes. Using approved HGNC symbols from NCBI get 1000 as maps MST1 to STK4
 
%  Using HGNC symbols as input does not recognise 11408 (comes from same output) which is STH which it always has difficulty with inputing STH is recognised and finds HGNC18839 don't know how to get round this
 The PANTHER enrichment displays the hierarchy of the ontology terms; although the FDR value is just less than nominal significance a very specific term related to neurogenesis, "positive regulation of collateral sprouting" (FDR p 0.041) is prominent in comparison to the same data ordered by FDR shown in table~\ref{tab:GO biological process complete Education Replication FDR}, figure~\ref{fig:the DAG of Collateral sprouting}

No significant enrichment was seen using PANTHER for molecular function or cellular function. Biological function was enriched for 19 terms using FDR to correct for multiple comparisons. The most enriched term is "nervous system development" (GO:0007399), FDR q = 0.006, but the term is the largest and most general with 2,430 members in the annotation (table~\ref{tab:GO biological process complete Education Replication FDRover represenation only}). "Positive regulation of collateral sprouting",GO:0048672, is just below nominal significance after correction for multiple comparisons using FDR but is a very specific term with over 52 times fold enrichment and 3 of the 12 genes annotated to this term are significant genes in GWGAS. This balance between significance and specificity is an issue to some extent for all GO pathway analyses (see fig~\ref{tab:GO biological process complete Education Replication FDR} and table~\ref{tab:GO biological process complete Education Replication FDRover represenation only}). No PANTHER SLIM ontology term showed significant enrichment. 

 \begin{figure}
     \centering
     \includegraphics[width=\textwidth]{images/chapter2/large_screenshots/edu_replication_large_bp_panther.png}
     \caption[GO enrichment PANTHER Biological Process Education\textsubscript{Replication}]{GO enrichment of genome wide significant genes at GWGAS in the Education\textsubscript{Replication} sample. Enrichment calculated
     using PANTHER with Fisher's exact test for the over representation test and using the False Discovery Rate correction for multiple comparisons. The term with the lowest FDR is GO:0007399 nervous system development (FDR=0.006). The figure shows the hierarchy of Gene Ontology terms, the most general terms are the least indented, the deepest terms in the directed acyclic graph are the most indented. Parent terms are below and more indented than child terms. Terms with identical levels of indentation are at identical layers in the hierarchy.}
     \tiny{Text output in \url{/home/grant/RProjects/paper_xls_output/data/PANTHER_txt_output/checked_for_thesis/ea2/BP_all.txt}}%previous image images/screenshots/EA2_BP_Panther_all_genes_FDR.png}
     
     \label{tab:GO biological process complete Education Replication FDR}
 \end{figure}
 % latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Sep  1 11:07:38 2020


\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/screenshots/quick_go_collateral.png}
    \caption{Gene Ontology Hierarchy of Biological Process term Collateral Sprouting from Quick GO. Although FDR q level is modest, in Education\textsubscript{Replication}, the fold change for the term is marked. Image source Quick GO. The term related to other terms such as growth cone development and neuron projection development (GO:0031175) found in both Education samples and in the paper by \cite{okbay2016genome}.How to prioritise such specific terms over general terms is an issue faced by GO enrichment methods.}
    \label{fig:the DAG of Collateral sprouting}
\end{figure}

% \begin{table}[ht]
% \resizebox{\textwidth}{!}{%
% \centering

% \begin{tabular}{llrrrrrr}
%   \hline
% GO & description & Ref & Test & E & Fold & P & FDR \\ 
%   \hline
% GO:0048672 & positive regulation of collateral sprouting  & 12 & 3 & 0.1 & 52.13 & $4.60 \times 10^{-5}$ & 0.049 \\ 
%   GO:0050919 & negative chemotaxis  & 46 & 5 & 0.2 & 22.66 & $4.42 \times 10^{-6}$ & 0.014 \\ 
%   GO:0033619 & membrane protein proteolysis  & 39 & 4 & 0.2 & 21.39 & $5.23 \times 10^{-5}$ & 0.046 \\ 
%   GO:0050771 & negative regulation of axonogenesis  & 72 & 5 & 0.3 & 14.48 & $3.37 \times 10^{-5}$ & 0.045 \\ 
%   GO:0010771 & negative regulation of cell morphogenesis involved in differentiation  & 100 & 6 & 0.5 & 12.51 & $1.18 \times 10^{-5}$ & 0.031 \\ 
%   GO:0061387 & regulation of extent of cell growth  & 115 & 6 & 0.6 & 10.88 & $2.51 \times 10^{-5}$ & 0.044 \\ 
%   GO:0008361 & regulation of cell size  & 187 & 7 & 0.9 & 7.81 & $4.05 \times 10^{-5}$ & 0.046 \\ 
%   GO:0050770 & regulation of axonogenesis  & 192 & 7 & 0.9 & 7.60 & $4.77 \times 10^{-5}$ & 0.045 \\ 
%   GO:0050807 & regulation of synapse organization  & 229 & 8 & 1.1 & 7.28 & $1.81 \times 10^{-5}$ & 0.041 \\ 
%   GO:0050803 & regulation of synapse structure or activity  & 240 & 8 & 1.1 & 6.95 & $2.51 \times 10^{-5}$ & 0.040 \\ 
%   GO:0010975 & regulation of neuron projection development  & 521 & 11 & 2.5 & 4.40 & $4.70 \times 10^{-5}$ & 0.047 \\ 
%   GO:0048666 & neuron development  & 848 & 17 & 4.1 & 4.18 & $6.68 \times 10^{-7}$ & 0.011 \\ 
%   GO:0031175 & neuron projection development  & 697 & 13 & 3.3 & 3.89 & $3.29 \times 10^{-5}$ & 0.048 \\ 
%   GO:0030182 & neuron differentiation  & 1046 & 18 & 5.0 & 3.59 & $2.58 \times 10^{-6}$ & 0.014 \\ 
%   GO:0050767 & regulation of neurogenesis  & 847 & 14 & 4.1 & 3.45 & $5.84 \times 10^{-5}$ & 0.049 \\ 
%   GO:0022008 & neurogenesis  & 1690 & 23 & 8.1 & 2.84 & $4.42 \times 10^{-6}$ & 0.018 \\ 
%   GO:0048699 & generation of neurons  & 1585 & 21 & 7.6 & 2.76 & $1.88 \times 10^{-5}$ & 0.037 \\ 
%   GO:0048468 & cell development  & 1654 & 21 & 7.9 & 2.65 & $3.51 \times 10^{-5}$ & 0.043 \\ 
%   GO:0007399 & nervous system development  & 2430 & 30 & 11.7 & 2.57 & $7.62 \times 10^{-7}$ & 0.006 \\ 
%   \hline
% \end{tabular}}
% \caption{GO biological process complete Education Replication FDRover represenation only  Ref reference set, test:number of genes being tested present in ontology termE expected number of genes being tested present in ontology term, Fold= Fold change P = raw p value, FDR = false discovery rate} 
% \label{tab:GO biological process complete Education Replication FDRover represenation only}
% \end{table}
 
%  % latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Sat Aug 29 13:31:48 2020
% \begin{table}[ht]
% \centering
% \begin{tabular}{llrrrlrr}
%   \hline
% GO & description & Ref & Test & E & OU & Fold & P Bnf \\ 
%   \hline
% GO:0050919 & negative chemotaxis  & 46 & 5 & 0.2  & + & 22.66 & 0.040 \\ 
%   GO:0048666 & neuron development  & 848 & 17 & 4.1 & + & 4.18 & 0.006 \\ 
%   GO:0030182 & neuron differentiation  & 1046 & 18 & 5.0 & + & 3.59 & 0.023 \\ 
%   GO:0022008 & neurogenesis  & 1690 & 23 & 8.1 & + & 2.84 & 0.040 \\ 
%   GO:0007399 & nervous system development  & 2430 & 30 & 11.7 & + & 2.57 & 0.007 \\ 
%   UNCLASSIFIED & Unclassified  & 2924 & 7 & 14.0 & - & 0.50 & 0.000 \\ 
%   \hline
% \end{tabular}
% \caption{GO biological process complete Education Replication Bonferroni} 
% \label{tab:GO biological process complete Education Replication Bonferroni}
% \end{table}

 % latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sat Aug 29 14:04:17 2020
\begin{table}[ht]
\centering
 \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{llllllll}
  \toprule
GO & description & Ref & Test & E & Fold & $p$ & FDR $q$ \\ 
  \midrule
GO:0048672 & positive regulation of collateral sprouting  & 12 & 3 & 0.1 & 52.13 & $4.600 \times 10^{-5}$ & 0.049 \\ 
  GO:0050919 & negative chemotaxis  & 46 & 5 & 0.2 & 22.66 & $4.420 \times 10^{-6}$ & 0.014 \\ 
  GO:0033619 & membrane protein proteolysis  & 39 & 4 & 0.2 & 21.39 & $5.230 \times 10^{-5}$ & 0.046 \\ 
  GO:0050771 & negative regulation of axonogenesis  & 72 & 5 & 0.3 & 14.48 & $3.370 \times 10^{-5}$ & 0.045 \\ 
  GO:0010771 & \makecell{negative regulation of cell morphogenesis\\ involved in differentiation}  & 100 & 6 & 0.5 & 12.51 & $1.180 \times 10^{-5}$ & 0.031 \\ 
  GO:0061387 & regulation of extent of cell growth  & 115 & 6 & 0.6 & 10.88 & $2.510 \times 10^{-5}$ & 0.044 \\ 
  GO:0008361 & regulation of cell size  & 187 & 7 & 0.9 & 7.81 & $4.050 \times 10^{-5}$ & 0.046 \\ 
  GO:0050770 & regulation of axonogenesis  & 192 & 7 & 0.9 & 7.60 & $4.770 \times 10^{-5}$ & 0.045 \\ 
  GO:0050807 & regulation of synapse organization  & 229 & 8 & 1.1 & 7.28 & $1.810 \times 10^{-5}$ & 0.041 \\ 
  GO:0050803 & regulation of synapse structure or activity  & 240 & 8 & 1.1 & 6.95 & $2.510 \times 10^{-5}$ & 0.040 \\ 
  GO:0010975 & regulation of neuron projection development  & 521 & 11 & 2.5 & 4.40 & $4.700 \times 10^{-5}$ & 0.047 \\ 
  GO:0048666 & neuron development  & 848 & 17 & 4.1 & 4.18 & $6.680 \times 10^{-7}$ & 0.011 \\ 
  GO:0031175 & neuron projection development  & 697 & 13 & 3.3 & 3.89 & $3.290 \times 10^{-5}$ & 0.048 \\ 
  GO:0030182 & neuron differentiation  & 1046 & 18 & 5.0 & 3.59 & $2.580 \times 10^{-6}$ & 0.014 \\ 
  GO:0050767 & regulation of neurogenesis  & 847 & 14 & 4.1 & 3.45 & $5.840 \times 10^{-5}$ & 0.049 \\ 
  GO:0022008 & neurogenesis  & 1690 & 23 & 8.1 & 2.84 & $4.420 \times 10^{-6}$ & 0.018 \\ 
  GO:0048699 & generation of neurons  & 1585 & 21 & 7.6 & 2.76 & $1.880 \times 10^{-5}$ & 0.037 \\ 
  GO:0048468 & cell development  & 1654 & 21 & 7.9 & 2.65 & $3.510 \times 10^{-5}$ & 0.043 \\ 
  GO:0007399 & nervous system development  & 2430 & 30 & 11.7 & 2.57 & $7.620 \times 10^{-7}$ & 0.006 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption{Gene ontology enrichment Biological Process using PANTHER Education\textsubscript{Replication} over representation analysis, with background all genome (PANTHER default). Ref: $n$ in reference set, test:number of genes being tested present in ontology term, E expected number of genes being tested present in ontology term, Fold= Fold change, $p$ = raw p value, FDR = false discovery rate} \tiny\url{source('~/RProjects/paper_xls_output/R/chapter_2/make_PANTHER_tables/make_Panther_table_GO_col_nounderFDR.R')}
\label{tab:GO biological process complete Education Replication FDRover represenation only}
\end{table}


Using FUMA GTex enrichment I found no statistically significant differential enrichment for the genes in the central nervous system (figure~\ref{fig:FUMA gtex deg samples multiple} and fig~\ref{fig:deg_upref_sample_gtex_gener})



% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/gprofiler/gprofiler_ea2_clip.png}
%     \caption{GO enrichment for significant genes using gProfiler multiple testing control SGS. Educational\textsubscript{Replication} sample}
    
%     \label{fig:gprofiler_ea2}
% \end{figure}

% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/gprofiler/gprofiler_ukbbint_clip.png}
%     \caption{GO enrichment for significant genes using gProfiler multiple testing control SGS. Intelligence\textsubscript{Discovery} sample}
%     \label{fig:gprofiler_ukbbint}
% \end{figure}

% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/gprofiler/gprofiler_eukbbed_clip.png}
%     \caption{GO enrichment for significant genes using gProfiler multiple testing control SGS. Education\textsubscript{Discovery} sample}
%     \label{fig:gprofiler_ukbbed}
% \end{figure}

% \begin{figure}
%     \centering

%     \includegraphics[width=\textwidth]{images/screenshots/EA2_BP_Panther_all_genes_Bonferroni.png}
%     \caption{Education\textsubscript{Replication} Gene Ontology enrichment using PANTHER. Bonferroni correction. Hierarchy of ontology terms shown.}
%     \label{fig:EA2_Panther_BP_Bonf_Hierarchy}
% \end{figure}

% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/FUMA_plots/gtex_v8_ts_FUMA_30_99_genes.png}
%     \caption{GTEx differential enrichment ordered by upregulated genes in Education\textsubscript{Replication} cohort}
%     \label{fig:GTEX_EA2_DEG}
% \end{figure}



\subsubsection{ToppGene:Results Education Replication}

The 99 significant genes were enriched compared to the rest of the genome  using ToppFun and the FDR method, Benjamini and Hochberg (FDR BH) to correct for multiple comparisons. Seventy seven significant terms were found for Biological Process using the FDR method Benjamini and Hochberg (10 with Bonferroni, 22 using FDR Q value Benjamini and Yekutieli, FDR BY). Twenty annotations for cellular component were significantly over represented (2 Bonferroni, 2 FDR BY). No molecular function annotations were found to be enriched in the 99 genes found at GWGAS. Enrichment was performed using default background and settings. 


The significant biological process terms are shown in  table~\ref{tab:BP EA2 all significant toppgene top}. The top two terms are neurogenesis (GO:0022008,$q$ FDR BH= 0.002) and gliogenesis (GO:0042063,$q$ FDR BH= 0.002) both reported by Okbay et al. using DEPICT in an expanded sample\cite{okbay2016genome}. The significant genes are not enriched for molecular function but show enrichment for two cellular component terms, the top being ``post synapse'', GO:0098794.

 There is also a considerable difference in actual enrichment values, including the raw $p$ value calculated in ToppGene and PANTHER (compare tables~\ref{tab:BP EA2 all significant toppgene top} and table~\ref{tab:GO biological process complete Education Replication FDRover represenation only}) . Widely variant scores are noted in Rhee et al.\cite{rhee2008use} and also see Khatri et al. for a detailed discussion \cite{khatri2005ontological}.

Given that differences in significance values have been reported between enrichment packages\cite{khatri2005ontological}(here differences can be seen in the number annotated for identical inputs), I calculated  the correlation of the -log10 transform of the uncorrected $p$ values for the seventeen annotations found in both PANTHER and ToppGene (17 of 19 for PANTHER, 17 of 77 for ToppGene). Pearson's product-moment correlation was 0.594 (95\%CI 0.158-0.84); t=2.861,df=15, $p$-value= 0.01189 (see figure~\ref{fig:correlation of gene ontology terms between toppgene and panther}). 

\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/chapter2/strontium/Rplot_compare_toppgene_panther.png}
    \caption{Correlation between gene ontology term enrichment values in ToppGene and PANTHER for Education\textsubscript{Discovery} sample. Scatterplot shows -log10 transform of uncorrected p value for PANTHER on x-axis and -log10 transform of ToppGene values on y axis. There appears to be a linear relationship between the two although the number of samples is small.}
    \label{fig:correlation of gene ontology terms between toppgene and panther}
\end{figure}


% t = 2.8612, df = 15, p-value = 0.01189
% alternative hypothesis: true correlation is not equal to 0
% 95 percent confidence interval:
%  0.1589549 0.8360662
% sample estimates:
%       cor 
% 0.5942021 

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Mon Sep 21 14:31:22 2020
\begin{table}[ht]
\centering
 \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{llrrrrrr}
  \toprule
ID & Name & p & q Bonf & q BH & q BY & n query & n genome \\ 
  \midrule
GO:0022008 & neurogenesis & $5.969 \times 10^{-7}$ & 0.002 & 0.002 & 0.014 & 26 & 1866 \\ 
  GO:0042063 & gliogenesis & $1.143 \times 10^{-6}$ & 0.003 & 0.002 & 0.014 & 11 & 352 \\ 
  GO:0007417 & central nervous system development & $1.812 \times 10^{-6}$ & 0.005 & 0.002 & 0.015 & 19 & 1129 \\ 
  GO:0048666 & neuron development & $3.528 \times 10^{-6}$ & 0.010 & 0.002 & 0.021 & 20 & 1297 \\ 
  GO:0010001 & glial cell differentiation & $5.159 \times 10^{-6}$ & 0.015 & 0.002 & 0.021 & 9 & 261 \\ 
  GO:0050808 & synapse organization & $5.386 \times 10^{-6}$ & 0.016 & 0.002 & 0.021 & 12 & 498 \\ 
  GO:0048667 & cell morphogenesis involved in neuron dif. & $5.919 \times 10^{-6}$ & 0.017 & 0.002 & 0.021 & 14 & 688 \\ 
  GO:0048699 & generation of neurons & $8.435 \times 10^{-6}$ & 0.025 & 0.003 & 0.027 & 23 & 1751 \\ 
  GO:0048812 & neuron projection morphogenesis & $1.389 \times 10^{-5}$ & 0.041 & 0.004 & 0.031 & 14 & 742 \\ 
  GO:0010771 & negative regulation of cell morphogenesis  dif. & $1.534 \times 10^{-5}$ & 0.045 & 0.004 & 0.031 & 6 & 109 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption{Top 10 terms biological process Topp Gene Education\textsubscript{Replication}. dif. = involved in differentiation. BH FDR q value Benjaminin and Hochberg, BY FDR Benjamini and Yekutieli, Bonf = Bonferroni, n query = number of genes in sample (GWGAS significant genes in Education\textsubscript{Replication}) annotated to this term, n genome = number of genes in genome annotated to this term}
\tiny\url{source('~/RProjects/paper_xls_output/R/chapter_2/panther_and_toppgene/panther_and_toppgene_ea2.R')}
\label{tab:BP EA2 all significant toppgene top}
\end{table}




% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Mon Sep 21 14:47:23 2020
\begin{table}[ht]
\centering
 \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{llllllll}
  \toprule
ID & Name & $p$ & $q$ Bonf & $q$ BH & $q$ BY & $n$ query & $n$ genome \\ 
  \midrule
GO:0098794 & postsynapse & $1.458 \times 10^{-6}$ & 0.001 & 0.001 & 0.003 & 16 & 825 \\ 
  GO:0045202 & synapse & $1.580 \times 10^{-5}$ & 0.005 & 0.003 & 0.018 & 20 & 1482 \\ 
  GO:0005794 & Golgi apparatus & $2.099 \times 10^{-4}$ & 0.073 & 0.015 & 0.097 & 19 & 1643 \\ 
  GO:0097418 & neurofibrillary tangle & $2.178 \times 10^{-4}$ & 0.075 & 0.015 & 0.097 & 2 & 5 \\ 
  GO:0035658 & Mon1-Ccz1 complex & $2.178 \times 10^{-4}$ & 0.075 & 0.015 & 0.097 & 2 & 5 \\ 
  GO:0031252 & cell leading edge & $2.705 \times 10^{-4}$ & 0.094 & 0.016 & 0.100 & 9 & 449 \\ 
  GO:0071458 & integral component of cytoplasmic side of ERM & $3.256 \times 10^{-4}$ & 0.113 & 0.016 & 0.103 & 2 & 6 \\ 
  GO:0044297 & cell body & $4.697 \times 10^{-4}$ & 0.163 & 0.020 & 0.131 & 12 & 820 \\ 
  GO:0036477 & somatodendritic compartment & $6.068 \times 10^{-4}$ & 0.210 & 0.023 & 0.146 & 14 & 1096 \\ 
  GO:0043025 & neuronal cell body & $6.590 \times 10^{-4}$ & 0.228 & 0.023 & 0.146 & 11 & 732 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption{ Top 10 Terms Cellular component ToppGene Education\textsubscript{Replication}. ERM=Endoplasmic reticulum membrane BH FDR q value Benjaminin and Hochberg, BY FDR Benjamini and Yekutieli, Bonf = Bonferroni, n query = number of genes in sample (GWGAS significant genes in Education	extsubscript{Replication}) annotated to this term, n genome = number of genes in genome annotated to this term} 
\label{tab:CC EA2 all significant toppgene}
\end{table}

% \begin{table}[ht]
% \centering
% \begin{tabular}{rllrrrr}
%   \hline
%  & ID & Name & test & Genome & p & Bon \\ 
%   \hline
% 1 & GO:0022008 & neurogenesis & 26 & 1866 & $5.97 \times 10^{-7}$ & 0.00176 \\ 
%   2 & GO:0042063 & gliogenesis & 11 & 352 & $1.14 \times 10^{-6}$ & 0.00337 \\ 
%   3 & GO:0007417 & central nervous syst & 19 & 1129 & $1.81 \times 10^{-6}$ & 0.00535 \\ 
%   4 & GO:0048666 & neuron development & 20 & 1297 & $3.53 \times 10^{-6}$ & 0.01041 \\ 
%   5 & GO:0010001 & glial cell different & 9 & 261 & $5.16 \times 10^{-6}$ & 0.01522 \\ 
%   6 & GO:0050808 & synapse organization & 12 & 498 & $5.39 \times 10^{-6}$ & 0.01589 \\ 
%   7 & GO:0048667 & cell morphogenesis i & 14 & 688 & $5.92 \times 10^{-6}$ & 0.01747 \\ 
%   8 & GO:0048699 & generation of neuron & 23 & 1751 & $8.43 \times 10^{-6}$ & 0.02489 \\ 
%   9 & GO:0048812 & neuron projection mo & 14 & 742 & $1.39 \times 10^{-5}$ & 0.04100 \\ 
%   10 & GO:0010771 & negative regulation  & 6 & 109 & $1.53 \times 10^{-5}$ & 0.04527 \\ 
%   \hline
% \end{tabular}
% \caption{BP EA2 all significant Toppgene 99 genes in total \url{source('~/RProjects/paper_xls_output/R/chapter_2/eda_toppgene_all_significant_ea2.R')}} 
% \label{tab:BP EA2 all significant toppgene}
% \end{table}


 %$p$ corrected 0.005 (see table~\ref{tab:CC EA2 all significant Toppgene}).
 
 
% \todo{CHANGE IMPORTANT}
\paragraph{SynGO}

Sixteen genes were annotated ontology terms using SynGO and over representation was carried out using a background set, for which 1104 genes are SynGO annotated genes. The results are shown in table~\ref{tab:enrichment of gene ontology terms in syngo}. The terms are more significant, with lower raw p values, due to a smaller background set (1104 SynGO annotated genes) and a decreased number of terms. 

\paragraph{g:Profiler}

One molecular function term, two cellular component terms and eight biological process terms were found to be significant after corrections for multiple comparisons using the g:SCS method (see section~\ref{sec:gProfiler GO enrichment}). The two cellular component terms are the same as those discovered using ToppGene with the Bonferroni or Benjamini-Yekutieli methods for multiple comparisons (synapse and post synapse). Synapse organisation (GO:0050808) is the most enriched biological process term (p adjusted g:SCS $5.78\times10^{-4}$). Terms related to neurogenesis (GO:0022008) p adjusted = 0.0154) and neuron projection development (GO:0031175, p adjusted g:SCS=0.0024) are enriched as reported by Okbay et al\cite{okbay2016genome} using DEPICT (see figure~\ref{fig:gProfiler EA2}).



\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/gprofiler/all_terms/ea2_all_terms.png}
    \caption{gProfiler Education\textsubscript{Replication}. p adj is corrected p value using g:SGS method}
    \label{fig:gProfiler EA2}
\end{figure}

% Dataset name
% ea
% ID mapping
% 16 / 99 (unique) genes from your gene list were mapped to SynGO annotated genes.
% Settings
% experimental-evidence filtering is not enabled (default setting)
% Genes
% 14 genes have a Cellular Component annotation, 13 for Biological Processes. Note: a gene may have multiple annotations.
% Annotations
% 18 annotations against a Cellular Component, 25 for Biological Processes.
% Enrichment
% 1 Cellular Component terms are significantly enriched at 1% FDR (testing terms with at least three matching input genes), 4 for Biological Processes.
% Background
% "brain expressed" background set was selected, contains 18035 unique genes in total of which 1104 overlap with SynGO annotated genes.
% Readme commented out





\begin{table}[]
    \centering
    
    \begin{tabular}{llll}
    \toprule
    Biological Process & $n$ & $p$ & FDR $q$\\
    \midrule
synapse organization*&	12& $6.94\times10^{-8}$	& $3.47\times10^{-7}$\\
process in the synapse&	13&	$4.14\times10^{-4}$&$1.03\times10^{-3}$\\
regulation of synapse organization	&3&	$6.98\times10^{-4}$&	$1.16\times10^{-3}$\\
synapse adhesion between pre- and post-synapse&	3	&$1.35\times10^{-3}$&$1.68\times10^{-3}$\\
trans-synaptic signaling	&3	&0.0644&	\\
\\
\bottomrule
\toprule
 Cellular component & $n$ & $p$ & FDR $q$\\
 \midrule
    synapse&14	&$8.68\times10^{-4}$	&$3.47\times10^{-3}$\\
postsynapse&	9	&$5.50\times10^{-3}$	&0.010       \\
         \bottomrule
\end{tabular}
    \caption[GO enrichment SynGO Education Replication]{Enrichment of Gene Ontology terms Education Replication using SynGO. SynGO only tests for biological process and cellular comparment. FDR $q$ = adjusted significance level using FDR. Note the increased significance level despite correcting for multiple comparisons. For term Synapse Organisation (GO:0050808)* 12 genes (including child terms) in annotation, 286 in SynGO. }
    \label{tab:enrichment of gene ontology terms in syngo}
\end{table}
% SynGO data downloaded on 2020-9-8 12:2 @ https://www.syngoportal.org
% SynGO dataset version/release: 20180731

% Summary of your gene list
% ea
% 26 / 233 (unique) genes from your gene list were mapped to SynGO annotated genes.
% experimental-evidence filtering is not enabled (default setting)
% 23 genes have a Cellular Component annotation, 20 for Biological Processes. Note: a gene may have multiple annotations.
% 32 annotations against a Cellular Component, 39 for Biological Processes.
% 0 Cellular Component terms are significantly enriched at 1% FDR (testing terms with at least three matching input genes), 1 for Biological Processes.
% "brain expressed" background set was selected, contains 18035 unique genes in total of which 1104 overlap with SynGO annotated genes.

% "user_genelist_map_to_syngo_genes.xlsx" contains your input genelist (first column) together with the matching gene ID available in SynGO. If your input could not be matched to any SynGO annotated gene, the latter will be empty.
% "syngo_annotations_matching_user_input.xlsx" contains all available SynGO annotations that match your input genelist.
% "syngo_ontologies_with_annotations_matching_user_input.xlsx" contains all SynGO ontology terms, together with the Gene Set Enrichment Analysis (GSEA) results and all genes from these terms that match your input genelist.

% The JSON folder contains data in a format convenient for bioinformatic analysis.

% The SynGO_geneset_CC/BP.json files contain all SynGO annotated genes for each synaptic ontology term. On the first level of this nested list you can find 'direct' and 'aggregate' annotations, you will want to use the latter for geneset analyses as these contain (for each term) both the annotated genes for an ontology term and all genes annotated against (recursive) child terms. For instance, in the aggregate dataset the term 'presynapse' also contains all genes annotated against its child term 'active zone'. Further, note that we originally mapped the annotated proteins from various species to HGNC identifiers. The ensembl/entrez/MGI/RGD mappings (from HGNC ID) in these JSON files were provided by the genenames.org webservice, so if you focus on Ensembl genes consider mapping the HGNC IDs to the exact Ensembl build that you are using.

  \paragraph{Disease Enrichment}
  
  Disease enrichment in DisGeNet BeFree\cite{pinero2020disgenet} searched by ToppFun (part of the ToppGene suite) is found for intellectual disability (ID:C3714756) which is the top term. Twenty genes are found from 1219 in the annotation ($p=8.88\times10^{-7}$, FDR BH=0.00165). Intellectual disability was reported to be enriched by Okbay et al \cite{okbay2016genome} for a larger related sample using DEPICT. Six of the genes are found in the PSP (See fig~\ref{tab:Intellectual disability genes significant in EA2 found in PSP} for intellectual disability associated genes significant at GWGAS in Education\textsubscript{Replication} sample. ) Testing enrichment with PSP as background found no significantly enriched disease terms other than inflammatory bowel disease. (\todo{See supplemental material}).See fig~\ref{tab:Intellectual disability genes significant in EA2 found in PSP} for intellectual disability associated genes significant at GWGAS in Education\textsubscript{Replication} sample. 
        % latex table generated in R 3.6.3 by xtable 1.8-4 package
        
% Sun Aug 23 12:35:02 2020
\begin{table}[ht]
\centering
 \setlength{\extrarowheight}{2pt}
\begin{tabular}{lll}
  \toprule
Entrez Gene ID & Gene Symbol & Gene Name \\
  \midrule
5144 & PDE4D & phosphodiesterase 4D  \\
  5662 & PSD & pleckstrin and Sec7 domain containing \\
  
  4137 & MAPT & microtubule associated protein tau \\ 
  4208 & MEF2C & myocyte enhancer factor 2C \\ 
  4744 & NEFH & neurofilament heavy \\
  257194 & NEGR1 & neuronal growth regulator 1 \\ 
   \bottomrule
\end{tabular}
\caption[Intellectual disability genes in significant in GWGAS of Education Replication]{Genes in PSP and associated with intellectual disability found amongst  genome-wide significant  Education Replication genes.} 
\label{tab:Intellectual disability genes significant in EA2 found in PSP}
\end{table}
 
%  ToppFun with background PSP (new)
%  No GO terms
%  No phenotype
%  (4 out of 5 inflammatory bowel are in PSP)
        
        % GTEx
\subsection{Intelligence Discovery: Gene Ontology enrichment}


Gene set analysis of this sample, combined with the Intelligence\textsubscript{Discovery} sample and augmented by educational attainment data using MTAG is reported by Hill et al.  \cite{hill2019combined}. MAGMA GSA was carried out using 10,891 gene sets from Gene ontology, Reactome and MSigDB and using the Bonferroni correction for multiple comparisons (raw p required = 0.05/10891 = $4.59 \times 10^{-6}$)\footnote{Actual footnote: Bonferroni $\alpha$ from supplementary table 5 $2.75\times10^{-6}$ which I calculate as 18,182 comparisons the same as the correction for multiple comparisons in the gene level statistics. The paper however says a correction of 10891 which I calculate means the corrections is $4.59\times10^{-6}$. This does not however result in any extra significant sets.  27 gene sets are obtained using the FDR correction }. GSA revealed eight quite large gene sets\footnote{See \url{source('~/RProjects/chapter2_checks/R/expected_gsa/ctg_size_plots.R')} for correlation between raw p value and gene size.} including neurogenesis (1,355 genes), genes associated with the synapse (717 genes), regulation of nervous system development (722 genes), neuron projection (989 genes), neuron differentiation (842 genes), CNS neuron differentiation (160 genes) and cell development (808 genes) (see table~\ref{tab:MAGMA GSA results from Hill et al.}). 


\begin{table}[ht]
\centering
 \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrr}
  \toprule
Gene set name & N genes in gene set & Beta & SE($\beta$) & p \\ 
  \midrule
Neurogenesis  & 1355 & 0.20 & 0.05 & $5.59 \times 10^{-10}$ \\ 
  Regulation of nervous system development  & 722 & 0.23 & 0.05 & $4.02 \times 10^{-8}$ \\ 
  Regulation of cell development  & 808 & 0.22 & 0.04 & $7.38 \times 10^{-8}$ \\ 
  Neuron projection  & 898 & 0.20 & 0.04 & $2.07 \times 10^{-7}$ \\ 
  Central nervous system neuron differentiation  & 160 & 0.47 & 0.04 & $5.33 \times 10^{-7}$ \\ 
  Synapse  & 717 & 0.21 & 0.04 & $1.43 \times 10^{-6}$ \\ 
  Neuron differentiation  & 842 & 0.19 & 0.04 & $1.62 \times 10^{-6}$ \\ 
  Oligodendrocyte differentiation  & 1037 & 0.17 & 0.04 & $1.75 \times 10^{-6}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[MAGMA GSA significant sets from Hill(2019)]{MAGMA GSA significant sets from Hill et al. 2019 Bonferroni $\alpha$=$2.75\times10^{-6}$ from supplementary table 5   \cite{hill2019combined}.}
\label{tab:MAGMA GSA results from Hill et al.}
\end{table}

% % latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Tue Sep 15 11:43:35 2020
% \begin{table}[ht]
% \centering
% \begin{tabular}{lrrrr}
%   \hline
% Gene.set.Name & n genes & Beta & SE & p \\ 
%   \hline
% Neurogenesis  & 1355 & 0.20 & 0.05 & $5.59 \times 10^{-10}$ \\ 
%   Regulation of nervous system development  & 722 & 0.23 & 0.05 & $4.02 \times 10^{-8}$ \\ 
%   Regulation of cell development  & 808 & 0.22 & 0.04 & $7.38 \times 10^{-8}$ \\ 
%   Neuron projection  & 898 & 0.20 & 0.04 & $2.07 \times 10^{-7}$ \\ 
%   Central nervous system neuron differentiation  & 160 & 0.47 & 0.04 & $5.33 \times 10^{-7}$ \\ 
%   Synapse  & 717 & 0.21 & 0.04 & $1.43 \times 10^{-6}$ \\ 
%   Neuron differentiation  & 842 & 0.19 & 0.04 & $1.62 \times 10^{-6}$ \\ % latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Tue Sep 15 11:43:35 2020

% \end{tabular}
% \caption{MAGMA GSA significant sets from Hill et al. 2019 Bonferroni $\alpha$ from supplementary table 5 $2.75\times10^{-6}$ from \cite{hill2019combined}}
% \end{table}









\subsubsection{Intelligence Discovery Gene Ontology analysis PANTHER}
A modified list of gene symbols was again used to ensure registration of genes using PANTHER \footnote{code at \url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/UKBB_Int_FUMA2clipboard.R')} gene list at supplemental \url{/home/grant/Dropbox/paper_xls_data/Panther/input.txt}}.
% 197 genes - it is actually 196 genes saved actual genes in paperxls on dropbox

% 191 mapped with NCBI symbol
% Using only approved you get 179 (ie a lot of NCBI are not the approved ones)

% 194 uniquely mapped with symbol

% 196 of 192 uniquely mapped with hgnc id ie all are uniquely mapped and there are multiples but they are all there
% \url{source('~/RProjects/paper_xls_output/R/chapter_2/translate_sig_genes/ukbbint_NCBI_translate2.R')}\footnote{have put stop in at end of script and also there is function to toppgene which needs remov}
% Checked this again this is best using HGNC
% other is \url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/UKBB_Int_FUMA2clipboard.R')}
% may find more DNA binding with this above 194 terms of 195

% This is confused the correct one is this one which uses gene symbol
% \url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/UKBB_Int_FUMA2clipboard.R')}
% 11 missing using ENSG annotations
% Gene ID 3 unmapped
% GeneID:100132074 FOXO6
% GeneID:9753
% GeneID:3107

% MGEA5   OGA
% C10orf76    ARMH3
% CCDC101 SGF29
% PET112 GATB
% HIST1H3C H3C3 one of 25 synonyms
% HIST1H3I HIST1H3J one of 24 synonyms
% BAI1 ADGRB1 ADRGB1 Entrez to panther to id
% RP5-874C20.3 pseudogene entrz 7741 ZSCAN26 Entrez to panther to id\footnote{\url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/UKBB_Int_FUMA2clipboard.R')}}
In contrast with the findings at MAGMA GSA analysis of the significant genes revealed terms associated with DNA packaging although terms associated with the synapse (GO:0045202) and vesicle membrane (GO:0012506) are enriched.

Only one biological process term was enriched: GO:0010468, regulation of gene expression (p=$4.78\times10^{-6}$, FDR BH=0.038; n in annotation 4907, n in set = 74). 
Seven molecular function terms were enriched the most specific and smallest being DNA binding GO:0003677 (n in annotation 2506, test 48, $p=1.01\times10^{-6}$ FDR=0.0016 (table~\ref{tab:GO molecular function complete Intelligence Discovery FDRover represenation only}))


\begin{table}[ht]
\centering
 \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{llrrrrrr}
  \toprule
GO & description & Ref & Test & E & Fold & $p$ & FDR \\ 
  \midrule
GO:0003677 & DNA binding  & 2506 & 48 & 23.1 & 2.08 & $1.01 \times 10^{-6}$ & 0.0016 \\ 
  GO:0003676 & nucleic acid binding  & 4034 & 64 & 37.1 & 1.72 & $5.51 \times 10^{-6}$ & 0.0052 \\ 
  GO:1901363 & heterocyclic compound binding  & 5999 & 83 & 55.2 & 1.50 & $2.73 \times 10^{-5}$ & 0.0185 \\ 
  GO:0097159 & organic cyclic compound binding  & 6086 & 83 & 56.0 & 1.48 & $4.27 \times 10^{-5}$ & 0.0254 \\ 
  GO:0005515 & protein binding  & 14109 & 159 & 129.9 & 1.22 & $3.77 \times 10^{-6}$ & 0.0045 \\ 
  GO:0005488 & binding  & 16469 & 174 & 151.7 & 1.15 & $2.39 \times 10^{-5}$ & 0.0190 \\ 
  GO:0003674 & molecular\_function  & 18321 & 188 & 168.7 & 1.11 & $9.41 \times 10^{-7}$ & 0.0022 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[GO enrichment molecular function Intelligence Discovery]{GO molecular function complete Intelligence Discovery FDR over representation only shown.  Ref=reference set, test:number of genes being tested present in ontology term, E=expected number of genes, Fold= Fold change, P = raw p value, FDR = false discovery rate Benjamini and Hochberg. The most enriched term is DNA binding and all of the annotations are large (smallest is 2506 genes).}
\tiny\url{source('~/RProjects/paper_xls_output/R/chapter_2/make_PANTHER_tables/UKBB_int/make_Panther_table_GO_col_nounderFDR_int.R')} 
\label{tab:GO molecular function complete Intelligence Discovery FDRover represenation only}
\end{table}

Twenty three cellular component terms were enriched. ``DNA packaging complex'' is the most enriched term using FDR BH 85 terms in annotation, 9 appear in the significant genes found at GWGAS, (GO:0044815), FDR=0.0002. The second most enriched term is the child term nucleosome, GO:000786, FDR 0.0007 (see figure~\ref{fig:UKBiobank int significant genes at GWGAS. PANTHER GO enrichment Cellular component. FDR correction. Screenshot for hierarchy} and table~\ref{tab:GO cellular component complete Intelligence Discovery FDRover represenation only}). Twenty seven genes (from 1383) are enriched for the cellular component term synapse (GO:0045202;FDR 0.0199).
% \paragraph{MF}   3 terms most significant is Very general term GO:0003677 DNA binding 0.0067 p Bonf 0.0029 p FDR Fold change 2.02

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Sep 15 17:42:51 2020


\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/chapter2/strontium/ukbbint_cc.png}
        \caption{Intelligence Discovery significant genes at GWGAS. PANTHER GO enrichment Cellular component. FDR correction. Screenshot for hierarchy. Matches table table~\ref{tab:GO cellular component complete Intelligence Discovery FDRover represenation only} }
    \label{fig:UKBiobank int significant genes at GWGAS. PANTHER GO enrichment Cellular component. FDR correction. Screenshot for hierarchy}
\end{figure}

%Removed table tab:GO molecular function complete Intelligence Discovery FDRover represenation only replaces more accurate id changes

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Aug 30 16:02:36 2020
% \begin{table}[ht]
% \centering
% \begin{tabular}{llrrrrrr}
%   \hline
% GO  & description MF & Ref & Test & E & Fold & P & FDR \\ 
%   \hline
% GO:0003677 & DNA binding  & 2486 & 47 & 23.2 & 2.02 & $2.350 \times 10^{-6}$ & 0.003 \\ 
%   \hspace{2mm}GO:0003676 & \hspace{2mm}nucleic acid binding  & 4024 & 63 & 37.6 & 1.67 & $1.630 \times 10^{-5}$ & 0.015 \\ 
%   \hspace{4mm}GO:0097159 & organic cyclic compound binding  & 6075 & 83 & 56.8 & 1.46 & $7.060 \times 10^{-5}$ & 0.042 \\ 
%   \hspace{4mm}GO:1901363 & heterocyclic compound binding  & 5987 & 83 & 56.0 & 1.48 & $4.490 \times 10^{-5}$ & 0.030 \\ 
  
%   GO:0005515 & protein binding  & 14110 & 162 & 132.0 & 1.23 & $2.070 \times 10^{-6}$ & 0.003 \\ 
   
% %   GO:0003674 & molecular\_function  & 18318 & 191 & 171.3 & 1.11 & $6.440 \times 10^{-7}$ & 0.002 \\ 
% %   UNCLASSIFIED & Unclassified  & 2533 & 4 & 23.7 & 0.17 & $6.440 \times 10^{-7}$ & 0.003 \\ 
%   \hline
% \end{tabular}
% \caption{GO molecular function complete Intelligence\textsubscript{Discovery} FDR over represenation only  Ref reference set, test:number of genes being tested present in ontology termE expected number of genes being tested present in ontology term, Fold= Fold change, = raw p value, FDR = false discovery rate} 
% \label{tab:GO molecular function complete Intelligence DIscovery FDRover represenation only}
% \end{table}



% \paragraph{CC}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Sep 15 17:39:45 2020

\begin{table}[ht]
\centering
 \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{llrrrrrr}
\toprule
GO & Description & Ref & Test & E & Fold & $p$ value & FDR \\ 
  \midrule
GO:0044815 & DNA packaging complex  & 85 & 9 & 0.8 & 11.50 & $1.98 \times 10^{-7}$ & 0.0002 \\ 
  GO:0000786 & nucleosome  & 77 & 8 & 0.7 & 11.28 & $1.10 \times 10^{-6}$ & 0.0007 \\ 
  GO:0032993 & protein-DNA complex  & 175 & 9 & 1.6 & 5.59 & $5.03 \times 10^{-5}$ & 0.0056 \\ 
  GO:0000785 & chromatin  & 1504 & 36 & 13.8 & 2.60 & $1.50 \times 10^{-7}$ & 0.0003 \\ 
  GO:0030659 & cytoplasmic vesicle membrane  & 788 & 18 & 7.3 & 2.48 & $4.39 \times 10^{-4}$ & 0.0367 \\ 
  GO:0012506 & vesicle membrane  & 810 & 18 & 7.5 & 2.41 & $6.01 \times 10^{-4}$ & 0.0483 \\ 
  GO:0000790 & nuclear chromatin  & 1316 & 29 & 12.1 & 2.39 & $1.66 \times 10^{-5}$ & 0.0030 \\ 
  GO:0005694 & chromosome  & 2012 & 40 & 18.5 & 2.16 & $3.59 \times 10^{-6}$ & 0.0018 \\ 
  GO:0045202 & synapse  & 1383 & 27 & 12.7 & 2.12 & $2.28 \times 10^{-4}$ & 0.0199 \\ 
  GO:0000228 & nuclear chromosome  & 1542 & 30 & 14.2 & 2.11 & $1.36 \times 10^{-4}$ & 0.0124 \\ 
  GO:0031981 & nuclear lumen  & 4993 & 73 & 46.0 & 1.59 & $1.84 \times 10^{-5}$ & 0.0031 \\ 
  GO:0043232 & intracellular non-membrane-bounded organelle  & 5274 & 77 & 48.6 & 1.59 & $8.30 \times 10^{-6}$ & 0.0021 \\ 
  GO:0043228 & non-membrane-bounded organelle  & 5283 & 77 & 48.6 & 1.58 & $8.50 \times 10^{-6}$ & 0.0019 \\ 
  GO:0043233 & organelle lumen  & 6114 & 82 & 56.3 & 1.46 & $8.88 \times 10^{-5}$ & 0.0094 \\ 
  GO:0070013 & intracellular organelle lumen  & 6114 & 82 & 56.3 & 1.46 & $8.88 \times 10^{-5}$ & 0.0089 \\ 
  GO:0031974 & membrane-enclosed lumen  & 6114 & 82 & 56.3 & 1.46 & $8.88 \times 10^{-5}$ & 0.0085 \\ 
  GO:0005634 & nucleus  & 7578 & 101 & 69.8 & 1.45 & $5.29 \times 10^{-6}$ & 0.0018 \\ 
  GO:0043231 & intracellular membrane-bounded organelle  & 11041 & 132 & 101.7 & 1.30 & $1.14 \times 10^{-5}$ & 0.0023 \\ 
  GO:0043229 & intracellular organelle  & 12856 & 148 & 118.4 & 1.25 & $6.65 \times 10^{-6}$ & 0.0019 \\ 
  GO:0043227 & membrane-bounded organelle  & 12748 & 146 & 117.4 & 1.24 & $1.91 \times 10^{-5}$ & 0.0029 \\ 
  GO:0043226 & organelle  & 13868 & 155 & 127.7 & 1.21 & $2.01 \times 10^{-5}$ & 0.0029 \\ 
  GO:0005622 & intracellular  & 14704 & 163 & 135.4 & 1.20 & $4.77 \times 10^{-6}$ & 0.0019 \\ 
  GO:0110165 & cellular anatomical entity  & 18767 & 188 & 172.8 & 1.09 & $2.91 \times 10^{-5}$ & 0.0039 \\ 
  GO:0005575 & cellular\_component  & 18951 & 189 & 174.5 & 1.08 & $3.56 \times 10^{-5}$ & 0.0045 \\ 
  UNCLASSIFIED & Unclassified  & 1900 & 3 & 17.5 & 0.17 & $3.56 \times 10^{-5}$ & 0.0042 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
    \caption[GO enrichment cellular component Intelligence Discovery]{GO cellular component complete Intelligence Discovery FDR over represenation only.  Ref reference set, test:number of genes being tested present in ontology term, E expected number of genes being tested present in ontology term, Fold= Fold change, P = raw p value, FDR = false discovery rate}
    \tiny\url{source('~/RProjects/paper_xls_output/R/chapter_2/make_PANTHER_tables/UKBB_int/make_Panther_table_GO_col_nounderFDR_int.R')}
\label{tab:GO cellular component complete Intelligence Discovery FDRover represenation only}
\end{table}
% % Sun Aug 30 16:17:46 2020
% \begin{table}[ht]
% \centering
% \begin{tabular}{llrrrrll}
%   \toprule
% GO ID& Description & Ref & Test & E & Fold & P & FDR \\ 
%   \midrule
%     GO:0000786 & nucleosome  & 77 & 8 & 0.7 & 11.11 & $1.23 \times 10^{-6}$ & 0.0012 \\ 
%   \hspace{2mm}\ding{213}  GO:0032993 & protein-DNA complex  & 175 & 9 & 1.6 & 5.50 & $5.67 \times 10^{-5}$ & 0.0095 \\ 
%  \hspace{2mm}\ding{213}  GO:0044815 & DNA packaging complex  & 85 & 9 & 0.8 & 11.32 & $2.26 \times 10^{-7}$ & 0.0005 \\ 

%   \hspace{2mm}\ding{213}   GO:0000785 & chromatin  & 1229 & 25 & 11.5 & 2.18 & $3.29 \times 10^{-4}$ & 0.0472 \\ 
%   \hspace{4mm}\ding{221}   GO:0043229 & intracellular organelle  & 12843 & 150 & 120.1 & 1.25 & $7.58 \times 10^{-6}$ & 0.0030 \\ 
%   \hspace{6mm}\ding{235}   GO:0005622 & intracellular  & 14693 & 165 & 137.4 & 1.20 & $5.80 \times 10^{-6}$ & 0.0039 \\ 
%   \hspace{6mm}\ding{235}  GO:0043226 & organelle  & 13859 & 157 & 129.6 & 1.21 & $2.35 \times 10^{-5}$ & 0.0059 \\ 
%   \hspace{8mm}\ding{233} GO:0110165 & cellular anatomical entity  & 18761 & 191 & 175.4 & 1.09 & $3.060 \times 10^{-5}$ & 0.0056 \\ 
   
   
%   GO:0031981 & nuclear lumen  & 4786 & 68 & 44.8 & 1.52 & $1.60 \times 10^{-4}$ & 0.0247 \\ 
%   \hspace{2mm}\ding{213} GO:0005634 & nucleus  & 7567 & 102 & 70.8 & 1.44 & $6.21 \times 10^{-6}$ & 0.0031 \\ 
%   \hspace{4mm}\ding{221} GO:0043231 & \makecell{intracellular membrane-bounded\\ organelle}  & 11028 & 134 & 103.1 & 1.30 & $9.470 \times 10^{-6}$ & 0.0032 \\ 
  
  
%   \hspace{6mm}\ding{235} GO:0043227 & membrane-bounded organelle  & 12734 & 148 & 119.1 & 1.24 & $1.620 \times 10^{-5}$ & 0.0046 \\ 
  
 
  
% %   GO:0005575 & cellular\_component  & 18946 & 192 & 177.2 & 1.08 & $2.420 \times 10^{-5}$ & 0.0054 \\ 
  
% %   UNCLASSIFIED & Unclassified  & 1905 & 3 & 17.8 & 0.17 & $2.420 \times 10^{-5}$ & 0.0049 \\ 
%   \bottomrule
% \end{tabular}
% \caption{Gene Ontology Enrichment in PANTHER using cellular component complete for significant genes at GWGAS for Intelligence\textsubscript{Discovery FDR} sample. Over represenation of terms only shown. Indendation shows depth of term in GO DAG, least indented deepest term.   Ref reference set, test:number of genes being tested present in ontology term E expected number of genes being tested present in ontology term, Fold= Fold changeP = raw p value, FDR = false discovery rate,} 
% \label{tab:GO cellular component complete Intelligence Discovery FDRover represenation only}
% \end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
Two SLIM molecular function terms were enriched: transcription factor binding (FDR 0.0145) and DNA binding (FDR 0.0189)
table~\ref{tab:ukbbint mf slim panther} continuing the finding of terms related to transcription.
  	\begin{table}[]
  	    \centering
  	     \setlength{\extrarowheight}{2pt}
  	    \begin{tabular}{llllllll}
  	    \toprule
  	GO-Slim Molecular Function &	$n$ annotation & $n$ &		expected 	&fold  &	 $p$ value& 	FDR\\
  	   \midrule
transcription factor binding &	203 &	10& 	1.87 &	5.35  &$2.73 \times 10^{-5}$  &	0.0145\\
DNA binding &	806 &	20& 	7.42& 	2.69& 		$7.11 \times 10^{-5}$& 	0.0189\\
\bottomrule
  	    \end{tabular}
  	    \caption{Intelligence Discovery Molecular Function SLIM PANTHER. n annotation= number of genes in annotation; n= number of significant genes; fold = fold change; FDR = false discovery rate Benjamini and Hochberg}
  	    \label{tab:ukbbint mf slim panther}
  	\end{table}
  	
Eighteen SLIM Biological process terms are over represented figure~\ref{fig:ukbb_int_BP_GO_SLIM} for a screenshot that shows the hierarchy of terms. The terms are again related to transcription, including those related to nucleosome assembly, DNA packaging and those regulating RNA polymerase. No SLIM cellular component was enriched. 

\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/chapter2/large_screenshots/int_discovery_large_bp_slim_panther.png}
    \caption{Intelligence Discovery large screenshot Biological Process SLIM.}
    \tiny\url{/home/grant/Dropbox/paper_xls_data/Panther/output_txt/ukbbint/bp_all_slim_checked_with_image_matches_current_image_overleaf.txt}
    \label{fig:ukbb_int_BP_GO_SLIM}
\end{figure}

PANTHER protein classes were enriched for histone and DNA binding protein (see table~\ref{tab:intelligence discovery panther protein class}.

\begin{table}[]
    \centering 
    \setlength{\extrarowheight}{2pt}
    \begin{tabular}{llllllll}
    \toprule
      PANTHER Protein Class &	n annot & n sample & 	expected& 	Fold &  P & 	FDR\\
      \midrule
histone &	55& 	7 &	.51& 	13.82 &	 $1.52 \times 10^{-6}$ &$2.96 \times 10^{-4}$\\
DNA binding protein &	227& 	9 &	2.09 &	4.31 &$3.30 \times 10^{-4}$ &	0.032     \\
         \bottomrule
    \end{tabular}
    \caption{PANTHER protein class enrichment Intelligence\textsubscript{Discovery}}
    \label{tab:intelligence discovery panther protein class}
\end{table}


 Chromatin, RNA polymerase and Pre-Notch expression and processing were among the most enriched of the 98 Reactome pathways tested using PANTHER with 98 terms in total with FDR $<$ 0.05(See figure~\ref{fig:UKBB int topp reac}).
\footnote{table at \url{source('~/RProjects/paper_xls_output/R/chapter_2/make_PANTHER_tables/UKBB_int/make_Panther_table_GO_col_nounderFDRreactome.R')} includes reactome iids}



% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/chapter2/large_screenshots/int_discovery_large_reactome_panther1.png}
%     \caption{UKBB int top reactome pathways Panther}
%     \label{fig:UKBB int topp reac}
% \end{figure}

\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/chapter2/large_screenshots/int_discovery_large_reactome_panther3.png}
    \caption{Education Discovery most significantly enriched Reactome pathways PANTHER. NOTCH is a cell signalling pathway involved in neurogenesis}
    \label{fig:UKBB int topp reac}
\end{figure}

g:Profiler results are shown in figure~\ref{fig:gprofiler intelligence discovery} and are a good summary of the main sources of enrichment found in ToppGene and PANTHER (Transcription related and nucleosome assembly). 

\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/gprofiler/all_terms/ukbbint_all_terms.png}
    \caption{g:profiler GO enrichment results for significant genes found at GWGAS for the Intelligence\textsubscript{Discovery} sample}
    \label{fig:gprofiler intelligence discovery}
\end{figure}

\subsubsection{SynGO Intelligence Discovery}

No significant terms were enriched for cellular component. The terms enriched for biological process are in figure~\ref{tab:SynGO Enrichment UKBB intelligence significant genes. FDR}. Although the FDR values are higher than found in the Education\textsubscript{Discovery}, some synaptic terms are found to be enriched by using a limited ontology. 

\begin{table}[]
    \centering
     \setlength{\extrarowheight}{2pt}
    \begin{tabular}{llll}
    \toprule
      Term   & n & $p$ & FDR $q$  \\
      \midrule
       synapse organization&	11&	$1.81\times10^{-4}$&	$1.81\times10^{-3}$\\
process in the synapse&	17&	$3.01\times10^{-3}$&	0.0107\\
regulation of synapse organization&	3&	$3.93\times10^{-3}$&	0.0107\\
modification of postsynaptic structure&	3	&$4.29\times10^{-3}$	&0.0107\\
regulation of presynaptic membrane potential&	3&$9.62\times10^{-3}$&	0.0192  \\ 
\bottomrule
    \end{tabular}
    \caption[SynGO Ontology enrichment Intelligence Discovery]{SynGO Enrichment Intelligence\textsubscript{Discovery} sample of genome wide significant genes at GWGAS. Correction for multiple comparisons using FDR. Background gene set is all brain expressed genes.}
    \label{tab:SynGO Enrichment UKBB intelligence significant genes. FDR}
\end{table}



% % latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Sun Aug 30 16:17:46 2020
% \begin{table}[ht]
% \centering
% \begin{tabular}{llrrrrll}
%   \toprule
% GO ID& Description & Ref & Test & E & Fold & P & FDR \\ 
%   \midrule
%     GO:0000786 & nucleosome  & 77 & 8 & 0.7 & 11.11 & $1.23 \times 10^{-6}$ & 0.0012 \\ 
%   \hspace{2mm}\ding{213}  GO:0032993 & protein-DNA complex  & 175 & 9 & 1.6 & 5.50 & $5.67 \times 10^{-5}$ & 0.0095 \\ 
%  \hspace{2mm}\ding{213}  GO:0044815 & DNA packaging complex  & 85 & 9 & 0.8 & 11.32 & $2.26 \times 10^{-7}$ & 0.0005 \\ 

%   \hspace{2mm}\ding{213}   GO:0000785 & chromatin  & 1229 & 25 & 11.5 & 2.18 & $3.29 \times 10^{-4}$ & 0.0472 \\ 
%   \hspace{4mm}\ding{221}   GO:0043229 & intracellular organelle  & 12843 & 150 & 120.1 & 1.25 & $7.58 \times 10^{-6}$ & 0.0030 \\ 
%   \hspace{6mm}\ding{235}   GO:0005622 & intracellular  & 14693 & 165 & 137.4 & 1.20 & $5.80 \times 10^{-6}$ & 0.0039 \\ 
%   \hspace{6mm}\ding{235}  GO:0043226 & organelle  & 13859 & 157 & 129.6 & 1.21 & $2.35 \times 10^{-5}$ & 0.0059 \\ 
%   \hspace{8mm}\ding{233} GO:0110165 & cellular anatomical entity  & 18761 & 191 & 175.4 & 1.09 & $3.060 \times 10^{-5}$ & 0.0056 \\ 
   
   
%   GO:0031981 & nuclear lumen  & 4786 & 68 & 44.8 & 1.52 & $1.60 \times 10^{-4}$ & 0.0247 \\ 
%   \hspace{2mm}\ding{213} GO:0005634 & nucleus  & 7567 & 102 & 70.8 & 1.44 & $6.21 \times 10^{-6}$ & 0.0031 \\ 
%   \hspace{4mm}\ding{221} GO:0043231 & \makecell{intracellular membrane-bounded\\ organelle}  & 11028 & 134 & 103.1 & 1.30 & $9.470 \times 10^{-6}$ & 0.0032 \\ 
  
  
%   \hspace{6mm}\ding{235} GO:0043227 & membrane-bounded organelle  & 12734 & 148 & 119.1 & 1.24 & $1.620 \times 10^{-5}$ & 0.0046 \\ 
  
 
  
% %   GO:0005575 & cellular\_component  & 18946 & 192 & 177.2 & 1.08 & $2.420 \times 10^{-5}$ & 0.0054 \\ 
  
% %   UNCLASSIFIED & Unclassified  & 1905 & 3 & 17.8 & 0.17 & $2.420 \times 10^{-5}$ & 0.0049 \\ 
%   \bottomrule
% \end{tabular}
% \caption{Gene Ontology Enrichment in PANTHER using cellular component complete for significant genes at GWGAS for Intelligence\textsubscript{Discovery FDR} sample. Over represenation of terms only shown. Indendation shows depth of term in GO DAG, least indented deepest term.   Ref reference set, test:number of genes being tested present in ontology term E expected number of genes being tested present in ontology term, Fold= Fold changeP = raw p value, FDR = false discovery rate,} 
% \label{tab:GO cellular component complete Intelligence Discovery FDRover represenation only}
% \end{table}


% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/screenshots/ukbb_int_panther_cc_bonf.png}
%     \caption{UKBiobank intelligence significant genes at GWGAS. PANTHER GO enrichment Cellular component. Bonferroni correction. Screenshot for hierarchy}
%     \label{fig:UKBiobank significant genes at GWGAS. PANTHER GO enrichment Cellular component. Bonferroni correction. Screenshot for hierarchy}
% \end{figure}



% Previous to above did not match table
% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/screenshots/ukbb_int_panther_cc_fdr.png}
%     \caption{UKBiobank intelligence significant genes at GWGAS. PANTHER GO enrichment Cellular component. FDR correction. Screenshot for hierarchy}
%     \label{fig:UKBiobank int significant genes at GWGAS. PANTHER GO enrichment Cellular component. FDR correction. Screenshot for hierarchy}
% \end{figure}


\paragraph{ToppGene}
 
Five biological processes terms, twenty nine molecular function terms and five cellular components had FDR $<$ 0.05. See figure~\ref{fig:toppgene pic}. The enrichment is that is present is also predominantly terms such as nucleosome function, DNA packaging and chromatin binding. Synaptic terms and neurogenesis terms are absent compared with the education samples.\footnote{int discovery PSP as BGNo GOBeta catenin pathway} 

\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/chapter2/strontium/images_toppgene_all_groups.png}
    \caption{Screen shot Gene ontology ToppGene Biological Process, Molecular Function and Cellular component Intelligence Discovery. Terms related to DNA binding, nucleosomes and chromatin are prominent}
    \label{fig:toppgene pic}
\end{figure}



%\todo{use bg PSP}


% \todo[inline]{Important note to self, although there are other results they are not going to change the overall analysis therefore do not add any more just get this chapter done. If they are ready (ie formatted) and add to the PhD after completing the discussion they can be added. Potential data commented out in tex below}


There is no enrichment for genes differentially enriched in the central nervous system areas using GTEx version 8 analysed using FUMA (figure~\ref{fig:FUMA gtex deg samples multiple} and fig~\ref{fig:deg_upref_sample_gtex_gener}).

% Removed 20th Sep as now appears in multiple view
% \begin{figure}        
%     \centering
%     \includegraphics[width=\textwidth]{images/FUMA_plots/gtex_30_ukbbed.png}
%     \caption{GTEx FUMA UKBBEd}
%     \label{fig:GTEx FUMA UKBBEd}
% \end{figure}




% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/FUMA_plots/closeupdeg_ukbbed.png}
%     \caption{Caption}
%     \label{fig:my_label}
% \end{figure}
% \subsection{Education\textsubscript{Discovery} Ontology enrichment}

% \subsection{Intelligence Discovery}
% 197 genes
% 11 missing using ENSG annotations


% Gene ID 3 unmappedZZ
% GeneID:100132074 FOXO6
% GeneID:9753
% GeneID:3107

% Symbol
% MGEA5
% C10orf76
% CCDC101 SGF29
% PET112 GATB
% HIST1H3C H3C3 one of 25 synonyms
% HIST1H3I HIST1H3J one of 24 synonyms
% BAI1 ADGRB1
% RP5-874C20.3 pseudogene entrz 7741 ZSCAN26

% REMAINING FRAMEWORK BELOW
%     \subsection{Intelligence Discovery}
%         \subsubsection{Authors findings Intelligence Discovery}
% %         \todo[inline]{by Hill et all ? move to samples. Points to make big gene sets}
% Gene set analysis of this set augmented by mgtag \cite{hill2019combined} was carried out using 10891 gene sets from Gene ontology, Reactome and MSigDB (raw p required = 0.05/10891 = $4.59 \times 10^{-6}$. GSA revealed 7 sets including neurogenesis 1355 genes and genes expressed in synapse 717 genes , regulation of nervous system development 722 genes neuron projection 989 neuron differentation 842 and CNS neuron differentiation 160 genes cell development 808 genes (\textcolor{red}{quote p or do table})

% For gene set size analysis see\footnote{\url{source('~/RProjects/paper_xls_output/R/exploratory_data_analysis/gene_size_and_enrichment_magma_sets.R')}} table~\ref{tab:group size and enrichment}

%         \subsubsection{Ontology ORA Genome Intelligence discovery}
%         196 genes 
%         \paragraph{panther Intelligence discovery}3 unmapped GeneID:100132074
% GeneID:9753
% GeneID:3107
% \subparagraph{panther BP all Intelligence Discovery} Nil
% \subparagraph{panther MF all Intelligence Discovery}DNA binding 	2486 	44 	22.53 	1.95 	+ 	1.40E-05 	1.66E-02
% nucleic acid binding 	4024 	60 	36.47 	1.64 	+ 	5.93E-05 	4.70E-02
% binding 	16464 	171 	149.23 	1.15 	+ 	4.44E-05 	4.23E-02
% protein binding 	14110 	156 	127.90 	1.22 	+ 	6.81E-06 	1.08E-02
% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/screenshots/GO_MF_Panther_UKBBInt_Heirarchy.png}
%     \caption{Panther MF UKBB Intelligence}
%     \label{fig:panther mf ukbb intelligence}
% \end{figure}
% \subparagraph{panther CC all Intelligence Discovery}
% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/screenshots/panther_cc_ukbb_int_all.png}
%     \caption{Panther hierarchy cc ukbb int all}
%     \label{fig:panther hierarchy cc ukbb int all}
% \end{figure}
%          \subsubsection{Ontology ORA Synaptic}
%          \subsubsection{Others}
%         Path to code
        
%         GTEx
   

\subsection{Education Discovery Ontology Over representation analysis}
      The 235 genome-wide significant genes identified using MAGMA GWGAS for the Education\textsubscript{Discovery} sample (section~\ref{sec:UKBB Education discovery GWGAS})and  were tested for over-representation of ontology terms. 
      Using PANTHER, I was unable to have all of the significant genes output by MAGMA identified, and used the gene identifier with the most uniquely identified genes  (233 terms, HGNC symbols with synonyms\footnote{ Script at \url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/edit_gene_table/UKBB_Ed_FUMA2clipboard.R')}}).
    Thirty-three biological process ontology terms, were over-represented (FDR corrected $q<$0.05) amongst the 233 significant genes (see figure~\ref{fig:panther bp ukbb edu fdr}). There are several enriched terms related to neurogenesis and axonal development, similar to the Education\textsubscript{Replication} sample but distinct from the Intelligence samples. The most significant term (FDR=$8.51\times10^{-4}$) was ``nervous system development" (GO:0007399, 57 genes found amongst 2,432 in the annotation). Many of the sets are broad and non-specific; the lowest FDR for annotations with less than 500 genes was ``negative regulation of cell projection" (GO:0031345);11 of 183 genes;5.4 fold change; FDR=0.0089.  Considering only those terms with greater than mean fold change (2.61), the most enriched is ``regulation of nervous system development" (GO:0051960,FDR 0.003). There was no significant over-representation of molecular function terms. 
    
    \begin{figure}
                \centering
                \includegraphics[width=\textwidth]{images/chapter2/large_screenshots/edu_discovery_bp_allbg_panther.png}
                \caption[PANTHER GO Enrichment Education Discovery Biological Process]{PANTHER Gene Ontology Enrichment Biological Process Education\textsubscript{Discovery} FDR. Terms related to neurogenesis are prominent.}
                \label{fig:panther bp ukbb edu fdr}
            \end{figure}
            
            
    Forty cellular component terms were over-represented.  Five of the ten most enriched terms had over 10,000 genes in their annotation. This includes the most significant term ``Intracellular" (GO:0005622, 14,707 genes, FDR=$4.9\times10^{-5}$) although only six of the forty significant terms contained over 10,000 genes (see figure~\ref{fig:panther cc ukbb edu } and table \label{tab:GO cellular component complete Education Discovery FDRover represenation only}). Four terms have under 100 annotations amongst the ten most enriched, three are related to the endoplasmic reticulum  including  ``lumenal side of membrane" (GO:0098576 FDR =   $3.84\times10^{-4}$)), one is annotated to GO:0042613 ``MHC class II protein complex" ;(5 of 19 genes in annotation found;  fold change 23.1, FDR=0.00126).``Synapse" ( GO:0045202  ;   36 of 1,383 genes in annotation,    ; fold change 15.8,  FDR = 0.00116) was also amongst the ten most enriched terms.     No SLIM ontologies showed significant over-representation and there was no enrichment of PANTHER pathways. Only the protein class PC00149 ``major histocompatibility complex protein" was over-represented  5 of 18 terms; 24.3 fold change; FDR=0.0010).

            

            % \subparagraph{CC Panther Education Discovery}
            % 40 terms. Of the 10 terms with the lowest FDR 5 have more than 10000 genes in the annotation (Lowest FDR: GO:0005622 intracellular 14704 genes in annotation FDR=$4.9\times10^{-5}$, 2nd GO:0043231 intracellular membrane-bounded organelle                              11041 in annotation  166 Genes from significant genes annotated, 4th GO:0043229 intracellular organelle                                               12856 in annotation , 183 in significant genes; 5th Organelle GO:0043226 organelle                                                             13868 in annotation,  193 in test genes and 8thGO:0043227 membrane-bounded organelle                                            12748  in annotation. 4 terms have under 100 annotations in the top 10, 3 are related to the endoplasmic reticulum (top GO:0098576 lumenal side of membrane                                             ;in annotation    37 genes, number of significant genes     7;  fold change 16.6,  FDR =   $3.84\times10^{-4}$), one is annotated to GO:0042613 MHC class II protein complex                                     , ;genes in annotation        19, significant genes with annotation     5;  fold change 23.1, FDR=0.00126. The final top 10 cellular component annotation is  GO:0045202 synapse                                                            ;    1383genes in annotation,    36 significant genes with annotation; fold change 15.8,  FDR= 0.00116. Post synapse is also enriched 22 significant genesw in 651 in annotation, fold change 2.96, FDR=0.00157.
            % See figure~\ref{fig:panther cc ukbb edu } for results displayed with terms sorted to show their position in the hierarchy.
            
       
            
            \begin{figure}
                \centering
                \includegraphics[width=\textwidth]{images/chapter2/large_screenshots/edu_discovery_large_cc_panther.png}
                \caption{PANTHER Gene Ontology Enrichment Cellular Component Education\textsubscript{Discovery}. The lowest FDR again is found in the most general and largest gene ontology terms with correspondingly small fold enrichment. }
                \label{fig:panther cc ukbb edu }
            \end{figure}
            
            
\begin{table}[ht]
\centering
  \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{llllllll}

  \toprule
GO & Description & Ref & Test & E & Fold & $p$ & FDR \\ 
  \midrule
GO:0042613 & MHC class II protein complex  & 19 & 5 & 0.2 & 23.06 & $6.26 \times 10^{-6}$ & 0.0013 \\ 
  GO:0071556 & integral component of lumenal side of ER membrane  & 29 & 6 & 0.3 & 18.13 & $2.39 \times 10^{-6}$ & 0.0008 \\ 
  GO:0098553 & lumenal side of ER membrane  & 29 & 6 & 0.3 & 18.13 & $2.39 \times 10^{-6}$ & 0.0007 \\ 
  GO:0098576 & lumenal side of membrane  & 37 & 7 & 0.4 & 16.57 & $5.73 \times 10^{-7}$ & 0.0004 \\ 
  GO:0042611 & MHC protein complex  & 28 & 5 & 0.3 & 15.64 & $3.22 \times 10^{-5}$ & 0.0043 \\ 
  GO:0035097 & histone methyltransferase complex  & 81 & 7 & 0.9 & 7.57 & $6.23 \times 10^{-5}$ & 0.0069 \\ 
  GO:0012507 & ER to Golgi transport vesicle membrane  & 61 & 5 & 0.7 & 7.18 & $8.96 \times 10^{-4}$ & 0.0473 \\ 
  GO:0098982 & GABA-ergic synapse  & 79 & 6 & 0.9 & 6.65 & $4.03 \times 10^{-4}$ & 0.0253 \\ 
  GO:0030134 & COPII-coated ER to Golgi transport vesicle  & 96 & 7 & 1.1 & 6.39 & $1.69 \times 10^{-4}$ & 0.0141 \\ 
  GO:0034708 & methyltransferase complex  & 110 & 8 & 1.3 & 6.37 & $5.89 \times 10^{-5}$ & 0.0070 \\ 
  GO:0032588 & trans-Golgi network membrane  & 95 & 6 & 1.1 & 5.53 & $1.01 \times 10^{-3}$ & 0.0493 \\ 
  GO:0030660 & Golgi-associated vesicle membrane  & 112 & 7 & 1.3 & 5.48 & $4.10 \times 10^{-4}$ & 0.0249 \\ 
  GO:0005798 & Golgi-associated vesicle  & 184 & 10 & 2.1 & 4.76 & $7.46 \times 10^{-5}$ & 0.0079 \\ 
  GO:0030176 & integral component of ER membrane  & 164 & 8 & 1.9 & 4.27 & $7.71 \times 10^{-4}$ & 0.0442 \\ 
  GO:0031227 & intrinsic component of ER membrane  & 172 & 8 & 2.0 & 4.07 & $1.04 \times 10^{-3}$ & 0.0495 \\ 
  GO:0030658 & transport vesicle membrane  & 212 & 9 & 2.4 & 3.72 & $9.49 \times 10^{-4}$ & 0.0476 \\ 
  GO:0098852 & lytic vacuole membrane  & 383 & 14 & 4.4 & 3.20 & $1.80 \times 10^{-4}$ & 0.0144 \\ 
  GO:0005765 & lysosomal membrane  & 383 & 14 & 4.4 & 3.20 & $1.80 \times 10^{-4}$ & 0.0139 \\ 
  GO:0014069 & postsynaptic density  & 345 & 12 & 3.9 & 3.05 & $7.87 \times 10^{-4}$ & 0.0439 \\ 
  GO:0005774 & vacuolar membrane  & 436 & 15 & 5.0 & 3.01 & $2.00 \times 10^{-4}$ & 0.0149 \\ 
  GO:0032279 & asymmetric synapse  & 351 & 12 & 4.0 & 3.00 & $9.09 \times 10^{-4}$ & 0.0468 \\ 
  GO:0098794 & postsynapse  & 651 & 22 & 7.4 & 2.96 & $8.58 \times 10^{-6}$ & 0.0016 \\ 
  GO:0045202 & synapse  & 1383 & 36 & 15.8 & 2.28 & $5.18 \times 10^{-6}$ & 0.0012 \\ 
  GO:0000139 & Golgi membrane  & 769 & 20 & 8.8 & 2.28 & $7.70 \times 10^{-4}$ & 0.0455 \\ 
  GO:0043005 & neuron projection  & 1395 & 33 & 15.9 & 2.07 & $7.85 \times 10^{-5}$ & 0.0079 \\ 
  GO:0030054 & cell junction  & 2122 & 44 & 24.2 & 1.82 & $1.39 \times 10^{-4}$ & 0.0121 \\ 
  GO:0005654 & nucleoplasm  & 3990 & 67 & 45.5 & 1.47 & $8.77 \times 10^{-4}$ & 0.0476 \\ 
  GO:0005829 & cytosol  & 5314 & 88 & 60.7 & 1.45 & $9.63 \times 10^{-5}$ & 0.0092 \\ 
  GO:0032991 & protein-containing complex  & 5532 & 89 & 63.1 & 1.41 & $2.83 \times 10^{-4}$ & 0.0203 \\ 
  GO:0043233 & organelle lumen  & 6114 & 96 & 69.8 & 1.38 & $3.29 \times 10^{-4}$ & 0.0228 \\ 
  GO:0070013 & intracellular organelle lumen  & 6114 & 96 & 69.8 & 1.38 & $3.29 \times 10^{-4}$ & 0.0220 \\ 
  GO:0031974 & membrane-enclosed lumen  & 6114 & 96 & 69.8 & 1.38 & $3.29 \times 10^{-4}$ & 0.0213 \\ 
  GO:0005634 & nucleus  & 7578 & 116 & 86.5 & 1.34 & $1.07 \times 10^{-4}$ & 0.0097 \\ 
  GO:0043231 & intracellular membrane-bounded organelle  & 11041 & 166 & 126.0 & 1.32 & $1.94 \times 10^{-7}$ & 0.0002 \\ 
  GO:0043229 & intracellular organelle  & 12856 & 183 & 146.7 & 1.25 & $8.00 \times 10^{-7}$ & 0.0004 \\ 
  GO:0043227 & membrane-bounded organelle  & 12748 & 180 & 145.5 & 1.24 & $3.31 \times 10^{-6}$ & 0.0008 \\ 
  GO:0005737 & cytoplasm  & 11717 & 165 & 133.7 & 1.23 & $4.24 \times 10^{-5}$ & 0.0053 \\ 
  GO:0005622 & intracellular  & 14704 & 205 & 167.8 & 1.22 & $2.44 \times 10^{-8}$ & 0.0000 \\ 
  GO:0043226 & organelle  & 13  868 & 193 & 158.3 & 1.22 & $1.04 \times 10^{-6}$ & 0.0004 \\ 
%   GO:0110165 & cellular anatomical entity  & 18767 & 232 & 214.2 & 1.08 & $1.56 \times 10^{-5}$ & 0.0026 \\ 
%   GO:0005575 & cellular\_component  & 18951 & 233 & 216.3 & 1.08 & $1.96 \times 10^{-5}$ & 0.0028 \\ 
%   UNCLASSIFIED & Unclassified  & 1900 & 5 & 21.7 & 0.23 & $1.96 \times 10^{-5}$ & 0.0030 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[GO cellular component PANTHER Education Discovery]{GO cellular component complete Education Discovery FDRover represenation only  Ref reference set, test:number of genes being tested present in ontology termE expected number of genes being tested ,present in ontology term, Fold= Fold changeP = raw p value, FDR = false discovery rate. ER = endoplasmic reticulum Ordered by fold change.GO:0110165 Cellular anatomical entity and GO:0005575 cellular component omitted along with one unclassified term. } 
\label{tab:GO cellular component complete Education Discovery FDRover represenation only}
\end{table}
        
Seven reactome pathways were enriched (table~\ref{tab:Reactome pathways Education Discovery FDRover represenation only}).
\begin{table}[ht]
\centering
 \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{llrrrrrr}
  \toprule
GO & description & Ref & Test & E & Fold & P & FDR \\ 
  \midrule
R-HSA-202430 & Translocation of ZAP-70 to Immunological synapse  & 25 & 5 & 0.3 & 17.52 & $1.99 \times 10^{-5}$ & 0.0454 \\ 
  R-HSA-202427 & Phosphorylation of CD3 and TCR zeta chains  & 28 & 5 & 0.3 & 15.64 & $3.22 \times 10^{-5}$ & 0.0368 \\ 
  R-HSA-389948 & PD-1 signaling  & 29 & 5 & 0.3 & 15.11 & $3.74 \times 10^{-5}$ & 0.0285 \\ 
  R-HSA-202433 & Generation of second messenger molecules  & 40 & 5 & 0.5 & 10.95 & $1.48 \times 10^{-4}$ & 0.0484 \\ 
  R-HSA-3247509 & Chromatin modifying enzymes  & 240 & 11 & 2.7 & 4.02 & $1.40 \times 10^{-4}$ & 0.0639 \\ 
  R-HSA-4839726 & Chromatin organization  & 240 & 11 & 2.7 & 4.02 & $1.40 \times 10^{-4}$ & 0.0533 \\ 
  R-HSA-194315 & Signaling by Rho GTPases  & 417 & 15 & 4.8 & 3.15 & $1.26 \times 10^{-4}$ & 0.0717 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption{Reactome pathways Education Discovery FDRover represenation only  Ref reference set, test:number of genes being tested present in ontology termE expected number of genes being tested ,present in ontology term, Fold= Fold changeP = raw p value, FDR = false discovery rate} 
\label{tab:Reactome pathways Education Discovery FDRover represenation only}
\end{table}

           
            
            
%             \begin{table}[]
%                 \centering
%                 \begin{tabular}{llll}
%                 \toprule
%                 Term & n & p & fdr q\\
%                 \midrule
% synapse organization&	15&	$4.42\times10^{-6}$&	$3.97\times10^{-5}$\\
% process in the synapse&	20&	$2.49\times10^{-3}$&	0.0112\\
% trans-synaptic signaling&	6&	0.0194	&0.0378\\
% synapse adhesion between pre- and post-synapse&	3	&0.0130	&0.0378\\
% synapse assembly&	4	&0.0252	&0.0378\\
% regulation of synapse assembly&	3	&0.0213&	0.0378\\
%                 \bottomrule
%                 \end{tabular}
%                 \caption[SynGO Biological Process Education\textsubscript{Discovery} GWGAS significant genes.]{SynGO Biological Process Education\textsubscript{Discovery} GWGAS significant genes. It shows more significant enrichment for synaptic specific elements than enrichment analysis using topGO despite the fact that the background set of brain expressed genes will tend to be more conservative than using the entire genome or all protein encoding genes. \textcolor{red}{However I can't find a term corresponding to synapse organisation in AmiGO It a biological process CHANGE Duplcated below now too}}
%                 \label{tab:syngo cc ukbb ed}
%             \end{table}
            
            \subsubsection{ToppGENE}
%                       Two duplicates come from magma 1201	Duplicated
% 5296	Duplicated Not duplicated one is 
% Recognises as different in toppgene do again

%             Nil compared to background PSP
     
           
            
            Seventy six biological process terms were over-represented using toppGene (FDR Benjamini and Hochberg - FDR-BH $<0.05$). Using either the Bonferroni correction or FDR correction of Benjamini and Yekeutil (FDR-BY), seventeen terms are discovered.  Terms related to neuron development, neurogenesis, synapse organisation, axon development and cell projection are prominent ( Table~\ref{tab:Biological Process Education Discovery ToppGene top10 results BY}). No terms related to molecular function were enriched. 
             Forty five cellular component terms were enriched using FDR-BH table~\ref{tab:Cellular component Education Discovery ToppGene top10 results Bonfe} with 10 significant using the Bonferroni correction or FDR-BY. The synapse and post synapse are over-represented as is MHC class II and terms related to the endoplasmic reticulum.
            
            % latex table generated in R 3.6.3 by xtable 1.8-4 package
% Wed Sep 16 11:18:31 2020


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Wed Sep 16 11:29:35 2020
% \begin{table}[ht]
% \centering
% \begin{adjustbox}{width=\textwidth}
% \begin{tabular}{llrrrr}
%   \hline
% ID & Name & n & n annot & p & FDR BH \\ 
%   \hline
% GO:0048666 & neuron development & 38 & 1297 & $6.045 \times 10^{-8}$ & $2.629 \times 10^{-4}$ \\ 
%   GO:0030182 & neuron differentiation & 42 & 1578 & $1.580 \times 10^{-7}$ & $3.299 \times 10^{-4}$ \\ 
%   GO:0031175 & neuron projection development & 34 & 1150 & $2.686 \times 10^{-7}$ & $3.299 \times 10^{-4}$ \\ 
%   GO:0022008 & neurogenesis & 46 & 1866 & $3.362 \times 10^{-7}$ & $3.299 \times 10^{-4}$ \\ 
%   GO:0048699 & generation of neurons & 44 & 1751 & $3.792 \times 10^{-7}$ & $3.299 \times 10^{-4}$ \\ 
%   GO:0007417 & central nervous system development & 33 & 1129 & $5.320 \times 10^{-7}$ & $3.760 \times 10^{-4}$ \\ 
%   GO:0051129 & negative regulation of cellular component org. & 27 & 819 & $6.635 \times 10^{-7}$ & $3.760 \times 10^{-4}$ \\ 
%   GO:0051960 & regulation of nervous system development & 32 & 1087 & $6.916 \times 10^{-7}$ & $3.760 \times 10^{-4}$ \\ 
%   GO:0060284 & regulation of cell development & 32 & 1132 & $1.655 \times 10^{-6}$ & $7.231 \times 10^{-4}$ \\ 
%   GO:0120036 & plasma membrane bounded cell projection org. & 43 & 1789 & $1.721 \times 10^{-6}$ & $7.231 \times 10^{-4}$ \\ 
%   \hline
% \end{tabular}
% \end{adjustbox}
% \caption{Biological Process Education Discovery ToppGene top10 results \url{source('~/RProjects/paper_xls_output/R/chapter_2/toppgene/do_tables/recent/topp_gene_bp_education_discovery.R')}}
% \label{tab:Biological Process Education Discovery ToppGene top10 results}
% \end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Sep 22 12:38:46 2020
\begin{table}[ht]
\centering
 \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lllllll}
  \toprule
ID & Name & n & n annot & p & FDR BH & Bonferroni \\ 
  \midrule
GO:0048666 & neuron development & 38 & 1297 & $6.045 \times 10^{-8}$ & $2.629 \times 10^{-4}$ & 0.0003 \\ 
  GO:0030182 & neuron differentiation & 42 & 1578 & $1.580 \times 10^{-7}$ & $3.299 \times 10^{-4}$ & 0.0007 \\ 
  GO:0031175 & neuron projection development & 34 & 1150 & $2.686 \times 10^{-7}$ & $3.299 \times 10^{-4}$ & 0.0012 \\ 
  GO:0022008 & neurogenesis & 46 & 1866 & $3.362 \times 10^{-7}$ & $3.299 \times 10^{-4}$ & 0.0015 \\ 
  GO:0048699 & generation of neurons & 44 & 1751 & $3.792 \times 10^{-7}$ & $3.299 \times 10^{-4}$ & 0.0016 \\ 
  GO:0007417 & central nervous system development & 33 & 1129 & $5.320 \times 10^{-7}$ & $3.760 \times 10^{-4}$ & 0.0023 \\ 
  GO:0051129 & negative regulation of cellular component organization & 27 & 819 & $6.635 \times 10^{-7}$ & $3.760 \times 10^{-4}$ & 0.0029 \\ 
  GO:0051960 & regulation of nervous system development & 32 & 1087 & $6.916 \times 10^{-7}$ & $3.760 \times 10^{-4}$ & 0.0030 \\ 
  GO:0060284 & regulation of cell development & 32 & 1132 & $1.655 \times 10^{-6}$ & $7.231 \times 10^{-4}$ & 0.0072 \\ 
  GO:0120036 & plasma membrane bounded cell projection organization & 43 & 1789 & $1.721 \times 10^{-6}$ & $7.231 \times 10^{-4}$ & 0.0075 \\ 
  GO:0050767 & regulation of neurogenesis & 29 & 971 & $1.829 \times 10^{-6}$ & $7.231 \times 10^{-4}$ & 0.0080 \\ 
  GO:0030030 & cell projection organization & 43 & 1827 & $2.968 \times 10^{-6}$ & $1.075 \times 10^{-3}$ & 0.0129 \\ 
  GO:0050808 & \textbf{synapse organization} & 19 & 498 & $4.261 \times 10^{-6}$ & $1.425 \times 10^{-3}$ & 0.0185 \\ 
  GO:0120035 & regulation of plasma membrane bounded cell projection organization & 25 & 821 & $7.140 \times 10^{-6}$ & $2.218 \times 10^{-3}$ & 0.0311 \\ 
  GO:0031344 & regulation of cell projection organization & 25 & 831 & $8.787 \times 10^{-6}$ & $2.548 \times 10^{-3}$ & 0.0382 \\ 
  GO:0031345 & negative regulation of cell projection organization & 12 & 226 & $1.084 \times 10^{-5}$ & $2.921 \times 10^{-3}$ & 0.0471 \\ 
  GO:0061564 & axon development & 20 & 583 & $1.142 \times 10^{-5}$ & $2.921 \times 10^{-3}$ & 0.0496 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption{Biological Process Education Discovery ToppGene Bonferroni and BY significant results n=17} \tiny\url{source('~/RProjects/paper_xls_output/R/chapter_2/toppgene/do_tables/recent/topp_gene_bp_education_discovery_and_bonfe.R'))}
\label{tab:Biological Process Education Discovery ToppGene top10 results BY}
\end{table}
         
            
 
           
            % latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Wed Sep 16 11:34:55 2020
% \begin{table}[ht]
% \centering
% \begin{adjustbox}{width=\textwidth}
% \begin{tabular}{llrrrr}
%   \hline
% ID & Name & n & n annot & p & FDR BH \\ 
%   \hline
% GO:0045202 & synapse & 40 & 1482 & $1.881 \times 10^{-7}$ & $9.876 \times 10^{-5}$ \\ 
%   GO:0098794 & postsynapse & 26 & 825 & $2.158 \times 10^{-6}$ & $5.666 \times 10^{-4}$ \\ 
%   GO:0098553 & lumenal side of ER membrane & 5 & 29 & $1.625 \times 10^{-5}$ & $2.132 \times 10^{-3}$ \\ 
%   GO:0071556 & integral component of lumenal side of ER membrane & 5 & 29 & $1.625 \times 10^{-5}$ & $2.132 \times 10^{-3}$ \\ 
%   GO:0042613 & MHC class II protein complex & 4 & 16 & $2.530 \times 10^{-5}$ & $2.462 \times 10^{-3}$ \\ 
%   GO:0043005 & neuron projection & 37 & 1624 & $2.814 \times 10^{-5}$ & $2.462 \times 10^{-3}$ \\ 
%   GO:0098576 & lumenal side of membrane & 5 & 34 & $3.637 \times 10^{-5}$ & $2.728 \times 10^{-3}$ \\ 
%   GO:0005798 & Golgi-associated vesicle & 10 & 186 & $4.982 \times 10^{-5}$ & $3.269 \times 10^{-3}$ \\ 
%   GO:0034708 & methyltransferase complex & 8 & 121 & $6.751 \times 10^{-5}$ & $3.938 \times 10^{-3}$ \\ 
%   GO:0035097 & histone methyltransferase complex & 7 & 94 & $9.152 \times 10^{-5}$ & $4.400 \times 10^{-3}$ \\ 
%   \hline
% \end{tabular}
% \end{adjustbox}
% \caption{Cellular component Education Discovery ToppGene top10.  results\url{source('~/RProjects/paper_xls_output/R/chapter_2/toppgene/do_tables/toppgene_ukbbed_cc.R')}. ER = endoplasmic reticulum}
% \label{tab:Cellular component Education Discovery ToppGene top10 results}
% \end{table}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Sep 22 12:42:20 2020
\begin{table}[ht]
\centering
 \setlength{\extrarowheight}{2pt}
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{llrrccc}
  \toprule
ID & Name & n & n annot & p & FDR BH & Bonferroni \\ 
  \midrule
GO:0045202 & \textbf{synapse} & 40 & 1482 & $1.881 \times 10^{-7}$ & $9.876 \times 10^{-5}$ & 0.0001 \\ 
  GO:0098794 & postsynapse & 26 & 825 & $2.158 \times 10^{-6}$ & $5.666 \times 10^{-4}$ & 0.0011 \\ 
  GO:0098553 & lumenal side of endoplasmic reticulum membrane & 5 & 29 & $1.625 \times 10^{-5}$ & $2.132 \times 10^{-3}$ & 0.0085 \\ 
  GO:0071556 & integral component of lumenal side of ER membrane & 5 & 29 & $1.625 \times 10^{-5}$ & $2.132 \times 10^{-3}$ & 0.0085 \\ 
  GO:0042613 & MHC class II protein complex & 4 & 16 & $2.530 \times 10^{-5}$ & $2.462 \times 10^{-3}$ & 0.0133 \\ 
  GO:0043005 & neuron projection & 37 & 1624 & $2.814 \times 10^{-5}$ & $2.462 \times 10^{-3}$ & 0.0148 \\ 
  GO:0098576 & lumenal side of membrane & 5 & 34 & $3.637 \times 10^{-5}$ & $2.728 \times 10^{-3}$ & 0.0191 \\ 
  GO:0005798 & Golgi-associated vesicle & 10 & 186 & $4.982 \times 10^{-5}$ & $3.269 \times 10^{-3}$ & 0.0261 \\ 
  GO:0034708 & methyltransferase complex & 8 & 121 & $6.751 \times 10^{-5}$ & $3.938 \times 10^{-3}$ & 0.0354 \\ 
  GO:0035097 & histone methyltransferase complex & 7 & 94 & $9.152 \times 10^{-5}$ & $4.400 \times 10^{-3}$ & 0.0481 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption{Cellular component Education Discovery ToppGene top10 these are those of bonferroni significance. Identical to graph above other than including bonferroni  resultsER = endoplasmic reticulum}
\tiny\url{source('~/RProjects/paper_xls_output/R/chapter_2/toppgene/do_tables/toppgene_ukbbed_cc.R')}
\label{tab:Cellular component Education Discovery ToppGene top10 results Bonfe}
\end{table}

            \paragraph{Phenotype}
            40 genes were associated with intellectual disability on searching DisGeNET using ToppGene. This was found in the Education\textsubscript{Replication} sample but in none of the Intelligence samples. Eleven of these genes are found in PSP and are shown in table~\ref{tab:Genes associated with intellectual disability found in PSP Education Discovery}. 
            Of the 29 not in the PSP no molecular function or cellular component enrichment was found but the biological process
            GO:0007417 	central nervous system development 	was enriched (0.02 FDR BH). 
            % latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Sep 22 12:48:03 2020
\begin{table}[ht]
\centering
\begin{tabular}{lll}
  \toprule
Gene ID & Gene Symbol & Gene Name \\ 
  \midrule
5144 & PDE4D & phosphodiesterase 4D \\ 
  5662 & PSD & pleckstrin and Sec7 domain containing \\ 
  4137 & MAPT & microtubule associated protein tau \\ 
  7248 & TSC1 & TSC complex subunit 1 \\ 
  4208 & MEF2C & myocyte enhancer factor 2C \\ 
  120 & ADD3 & adducin 3 \\ 
  9378 & NRXN1 & neurexin 1 \\ 
  257194 & NEGR1 & neuronal growth regulator 1 \\ 
  23334 & SZT2 & SZT2 subunit of KICSTOR complex \\ 
  22907 & DHX30 & DExH-box helicase 30 \\ 
  8085 & KMT2D & lysine methyltransferase 2D \\ 
   \bottomrule
\end{tabular}
\caption{Genes associated with intellectual disability found in PSP C3714756 DisGeNET. ORA in ToppGene. Education Discovery. Gene ID = Entrez Gene Gene ID} 
\label{tab:Genes associated with intellectual disability found in PSP Education Discovery}
\end{table}
            \subsubsection{g:Profiler}
         Twenty four biological process terms were significantly over represented in GWGAS significant genes in the Education\textsubscript{Discovery} sample using g:profiler after correction for multiple comparisons. Six cellular component terms are enriched. The terms include neurogenesis and nervous system development and cellular components include synapse and post synapse and are clearly similar to those in PANTHER and ToppGene although exact adjusted significance levels differ (See figure~\ref{fig:table all terms Education Discovery gProfiler}). Figure~\ref{fig:gProfiler 3 samples} shows a summary of enrichment in g:profiler across the three samples in which enrichment for gene ontology terms was detected. 
            % \begin{figure}
            %     \centering
            %     \includegraphics[width=\textwidth]{images/gprofiler/gProfiler_hsapiens_2020-09-16_09-45-55_ukbbed.png}
            %     \caption{Gene ontology enrichment Education\textsubscript{Discovery g profiler} Selected genes}
            %     \label{fig:gprofiler education discovery}
            % \end{figure}
        
            \begin{figure}
                \centering
                \includegraphics[width=\textwidth]{images/gprofiler/all_terms/ukbbed_allterms.png}
                \caption{Table of all terms Education Discovery gProfiler}
                \label{fig:table all terms Education Discovery gProfiler}
            \end{figure}

  \subsubsection{SynGO}
  \begin{table}[]
    \centering
    \begin{tabular}{llll}
    \toprule
    Cellular component & n & p & FDRq\\
    \midrule
    synapse organization&	15&	$7.44\times10^{-6}$&	$6.69\times10^{-5}$\\

process in the synapse&	20	&$4.06\times10^{-3}$	&0.0183\\
trans-synaptic signaling&	6&	0.0234&	0.0428\\
synapse adhesion between pre- and post-synapse&	3	&0.0146&	0.0428\\
regulation of synapse assembly&	3&	0.0238&	0.0428\\

synapse assembly&	4&	0.0289	&0.0433\\
      \bottomrule  
    \end{tabular}
    \caption{SynGO Biological Process Education\textsubscript{Discovery}}
    \label{tab:syngo biological process education discovery}
\end{table}
  
  \begin{table}[]
    \centering
    \begin{tabular}{llll}
    \toprule
    Cellular component & n & p & FDRq\\
    \midrule
 synapse&	23&	0.00451&	0.0316\\
postsynapse	&14	&0.0245&	0.0858\\
\bottomrule
\end{tabular}
\caption{syngo CC Education Discovery}
\label{tab:syngo CC Education discovery}
\end{table}
            % \paragraph{BP Education Discovery} One term synapse	 n= 23	p=2.64e-3	q=0.0185
            
            % \paragraph{CC Education Discovery}
            % 6 significant terms. See table~\ref{tab:syngo cc ukbb ed}
            
            
 
 
 
 Synapse, GO:0045202, was the only enriched cellular component term using SynGO (table~\ref{tab:syngo CC Education discovery}).
SynGO annotates only 23 of the significant genes to synapse, compared to 40 genes in ToppGene table~\ref{tab:Cellular component Education Discovery ToppGene top10 results Bonfe}. Six biological process terms are enriched,  including synapse organisation GO:0050807( table~\ref{tab:syngo biological process education discovery}). Fifteen terms are annotated synapse organisation by SynGO compared with 19 in toppgene (table~\ref{tab:Biological Process Education Discovery ToppGene top10 results BY} but the limited ontology results in a more significant FDR ($6.69\times10^{-5}$ compared to 0.018). These results show both the benefits and limitations of specialised ontologies.      \footnote{      NOte synapse organisation GO:0050807) has 15 annotated terms with FDR q $3.97\times10^{-5}$ compared to regulation of synapse organisation 11 of 229 P $0.943\times{10^-5}$ FDR 0.0483 in PANTHER GO:0050807
 See table~\ref{tab:syngo biological process education discovery}}

 
 Using FUMA to analyse gene expression the GWGAS significant Education\textsubscript{Replication} genes are over-represented amongst  differentially expressed genes up-regulated in the brain using GTEx (figure~\ref{fig:ukbbed gtex}). This is the only sample in which this is the case (see figure~\ref{fig:FUMA gtex deg samples multiple} and figure~\ref{fig:deg_upref_sample_gtex_gener} which shows only over-representation). 

% \subsubsection{Summary of Education Discovery}
% ToppGene and gprofiler No Molecular function

% Genes association with ID

% MF nil in PANTHER
% BP neurogenesis and synapse organisation and neuron projection

% CC MHC (which is assoc type II dm) ER PSD and post synapse, neuron projection
%         \subsubsection{Summary of above findings}

 \begin{figure}
 \centering
  \begin{subfigure}{12cm}
    \centering\includegraphics[width=11cm]{images/gprofiler/gprofiler_ea2_clip.png}
    \caption{gProfiler. Education Replication}
    \end{subfigure}
    
  \begin{subfigure}{12cm}
    \centering\includegraphics[width=11cm]{images/gprofiler/gprofiler_ukbbint_clip.png}
    \caption{gProfiler Intelligence Discovery}
  \end{subfigure}
 
  \begin{subfigure}{12cm}
    \centering\includegraphics[width=11cm]{images/gprofiler/gprofiler_eukbbed_clip.png}
    \caption{gProfiler Education Discovery}
  \end{subfigure}
  \caption{Gene Ontology enrichment of significant genes using gProfilers}
  \label{fig:gProfiler 3 samples}
  
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/FUMA_plots/gtex_v8_ts_general_FUMA_gene2func_UKBBEd_syn_protein_bp_sig.png}
    \caption{Gtex General UKBBEd Synaptic significant.png}
    \label{fig:ukbbed gtex}
\end{figure}


\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/FUMA_plots/gtex_v8_ts_FUMA_PSP_gtex.png}
    \caption[GTEx All PSP]{GTex all PSP. Note top row of upregulated genes. Shows that signal from expanded PSP is very specific to CNS expression. Red significant. Y axis -log10 of p value. Hypergeometric test of genes differentially expressed from pre-created list see FUMA \cite{watanabe2017functional} for details (two sided Bonferroni corrected p versus all labels with cut off fold change) }
    \label{fig:gtex all PSP}
\end{figure}
\clearpage


% \begin{figure}
%     \centering
%     \includegraphics[width=\textwidth]{images/chapter2/ggplot/Rplot_consensusgtex.png}
%     \caption{Differentially expressed genes over representation analysis in Gtex. Top panel genes in consensus PSD $n=1312$. Lower panel genes not in consensus PSD but in PSP.\url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/plot_gtex/plot_consensus_multiplot.R')}}
%     \label{fig:deg_gtex_psp}
% \end{figure}

\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/chapter2/ggplot/gtex/Rplot_boxplot_FUMA_hPDF_PSP_theme.png}
    \caption{Differentially expressed genes over representation analysis in Gtex. Top panel genes in consensus PSD $n=1312$. Lower panel genes not in consensus PSD but in PSP.}
    \tiny\url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/plot_gtex/plot_consensus_multiplot_theme.R')}
    \label{fig:deg_gtex_psp}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{images/chapter2/ggplot/ggplot_FUMA/Rplot_upregulated_general.png}
    \caption{Over representation of differentially expressed genes (up regulated) amongst significant genes at GWGAS for each of the samples clockwise from top left Intelligence Replication, Intelligence Discovery, Education Discovery and Education replication using FUMA to perform over representation. Gene expression data GTEx v8. x axis is -log10 transform of p value for cumulative hypergeometric test. Significant tissues are shown in cyan. The only sample with over representation of genes with up-regulated differential expression in the brain is Education\textsubscript{Replication} \url{source('~/RProjects/paper_xls_output/R/chapter_2/FUMA/plot_gtex/plot_study/multiplot_samples.R')}}
    \label{fig:deg_upref_sample_gtex_gener}
\end{figure}

%%%
% \subsection{Statistical analysis}\todo{move}

% Correction for multiple testing - by package or using adjust\_p in R. Statistical tests in R version . 

 \begin{figure}
  \begin{subfigure}{8cm}
    \centering\includegraphics[width=7cm]{images/FUMA_plots/deg_general_up/ctg_upreg_general_gtex_v8_ts_general_FUMA_gene2func44709.png}
    \caption{Gene expression Intelligence Replication}
    \end{subfigure}
  \begin{subfigure}{8cm}
    \centering\includegraphics[width=7cm]{images/FUMA_plots/deg_general_up/ea2_corrected_upreg_general_gtex_v8_ts_general_FUMA_gene2func44667.png}
    \caption{Education Replication}
  \end{subfigure}
 
  \begin{subfigure}{8cm}
    \centering\includegraphics[width=7cm]{images/FUMA_plots/deg_general_up/ukbb_int_upreg_general_gtex_v8_ts_general_FUMA_gene2func44709.png}
    \caption{DEG Up Regulated Intelligence Discovery}
  \end{subfigure}
  \begin{subfigure}{8cm}
    \centering\includegraphics[width=7cm]{images/FUMA_plots/deg_general_up/ukbbed_upreg_general_gtex_v8_ts_general_FUMA_gene2func44709.png}
    \caption{Education Discovery}
  \end{subfigure}
  \caption{FUMA output. Enrichment for differentially expressed genes ordered by up regulated genes. Only education discovery shows evidence of up regulated differentially expressed genes in CNS}
  \label{fig:FUMA gtex deg samples multiple}
\end{figure}

 \begin{figure}
  \begin{subfigure}{8cm}
    \centering\includegraphics[width=7cm]{images/FUMA_plots/gtex_v8_ts_nonPSP_FUMA_gene2func44651.png}
    \caption{Gene expression genes not in PSP}
    \end{subfigure}
  \begin{subfigure}{8cm}
    \centering\includegraphics[width=7cm]{images/FUMA_plots/gtex_v8_ts_FUMA_PSP_gtex.png}
    \caption{Genes in PSP}
  \end{subfigure}
 
  \begin{subfigure}{8cm}
    \centering\includegraphics[width=7cm]{images/FUMA_plots/gtex_v8_ts_psp_not_consensus_FUMA_gene2func44650.png}
    \caption{Genes in PSP but not in consensus PSD. ie new genes}
  \end{subfigure}
  \begin{subfigure}{8cm}
    \centering\includegraphics[width=7cm]{images/FUMA_plots/gtex_v8_ts_1313_consensus_FUMA_gene2func44650.png}
    \caption{Genes in consensus PSD}
  \end{subfigure}
  \caption[\textbf{Comparison of gene expression} in consensus PSD and PSP]{Comparison of gene expression in genes not in the PSP, genes in the PSP, genes in PSP but not in the consensus PSD (new genes), genes in the consensus PSD - that which is conserved in different vertebrate species. Images from FUMA. GTEx v8 54 tissue types. Ordered from left to right by up-regulated differentially expressed genes p values (hypergeometric test). Tissues with significant differential enrichment after bonferroni correction for multiple comparisons are shown in blue. Y axis -log10(p) of enrichment for gene set (eg PSP) }
  \label{fig:compare all PSP gtex multipanel FUMA output}
\end{figure}

 
 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%





\clearpage
\section{Synaptic component enrichment MAGMA}
\label{sec:Synaptic component enrichment MAGMA}

Functional synaptic group gene sets are curated  at the Complex Trait Genetics lab \footnote{\url{https://ctg.cncr.nl/software/GENESETS/synapse.zip}} based on stuies by Ruano et al. (2012)\cite{ruano2010functional}\footnote{paper looking at G protein receptors} and Lips et al. (2012)\cite{lips2012functional}. Using these sets with MAGMA will provide an estimate of how effective a purely functional annotation approach is and an estimate of the value of significance tests in these samples. 

MAGMA GSA was performed using the raw output of the GWGAS analyses in MAGMA as part of a standard two stage MAGMA GSA. Default options were used.

The annotated set was composed of 1,028 genes. 736 of these were in the PSP, 292 were not. A large number of the missing ones were non glutamatergic receptors and the  proteins associated with these. There are 18 gene sets each a functional group such as G-protein-coupled receptor signalling.

MAGMA accepts gene sets in gene matrix transposed format (gmt) where each gene set is a row in a text file that is tab separated. gmt files used for MAGMA enrichment are often those curated by MSigDB. I have used an Rscript to create a gmt file from the supplied data\footnote{\url{source('~/RProjects/paper_xls_output/R/magma_synapse/make_synapse_gmt.R')}}.

To reduce type II error, interesting sets are identified as those of nominal significance in the discovery samples and these selected sets are then tested using the replication samples. 

\paragraph{Results} For the Intelligence\textsubscript{Discovery} sets Cell adhesion and trans-synaptic signaling p=0.009 , Excitability 0.005 and Exocytosis p=0.048 are of potential interest having p$<$ 0.05 (supplementary table~\ref{tab:MAGMA enrichment of synaptic groups UKBBint}). Only exocytosis is of nominal significance in the Intelligence\textsubscript{Replication} cohort (p=0.016; p= 0.048 with Bonferroni correction see  table~\ref{tab:MAGMA enrichment of synaptic groups ctg}). 

For Education\textsubscript{Discovery} two sets achieve nominal levels of significance: Cell adhesion and trans-synaptic signaling $p$=0.001 and Excitability $n$=57 genes 0.003 (table~\ref{tab:MAGMA enrichment of synaptic groups UKBBEd}). None of these achieve nominal significance in the Education\textsubscript{Replication} cohort (table~\ref{tab:MAGMA enrichment of synaptic groups EA2}). 

The only significant group we have identified using the curated synaptic sets provided by the CTG lab and our samples are Exocytosis for the Intelligence sample. The test statistic in the discovery cohort (0.48) is close to the nominal significance (p$<$0.05) threshold. Enrichment by functional class has been of limited benefit, but has proven a useful guide to the likely significance levels found in MAGMA GSA of synaptic sets. See table~\ref{tab:curated synaptic discovery and replication} 


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Aug  9 12:39:58 2020


\begin{table}[]
    \centering
     \setlength{\extrarowheight}{2pt}
    \begin{tabular}{lllll}
    \toprule
      Term   & Phenotype & n & Discovery & Replication  \\
      \midrule
         Cell adhesion and trans-synaptic signalling & Intelligence& 76  & 0.009 & 0.155\\
         Excitability & Intelligence& 57  & 0.005 & 0.159\\
          Exocytosis & Intelligence&83  & \textbf{0.048} & \textbf{0.016}  \\ 
          \\
           Excitability & Education & 57 & 0.003 &  0.122\\ 
           Cell adhesion and trans-synaptic signalling & Education & 76 & 0.001 & 0.067 \\ 
           \bottomrule
           
    \end{tabular}
    \caption{Discovery and Replication cohorts of curated synaptic genes from CTG site. }
    \label{tab:curated synaptic discovery and replication}
\end{table}


\subsubsection{Gene size and enrichment intelligence studies MAGMA}

\begin{table}[]
    \centering
     \setlength{\extrarowheight}{2pt}
    \begin{tabular}{llllllll}
    \toprule
   Group  & $n$ & Min. &1Q & Median  &  Mean &3Q    &Max.\\ 
   \midrule
    all sets &   10,673  &3.0 &    17.0 &   34.0 &  100.4&    88.0&  1898.0 \\
      $p<0.05$ & 850 &4.0  &  20.0 &   52.0 &  180.2  & 155.8 & 1898.0 \\ 
       $p<0.001$ & 50 & 6.0&    20.5    &61.0&   320.7&   439.2  &1674.0     \\
       \bottomrule
     
    \end{tabular}
    \caption[Gene set group size and significance of enrichment]{Summary statistics for gene set size for different levels of significance from Hill et al. (2019)\cite{hill2019combined}}
    \label{tab:group size and enrichment}
\end{table}



Nominally significant gene sets are larger using 0.05 and size 50 as cutoff cor test 0.22 $p = 2.13\times10^{-6}$
\footnote{\url{source('~/RProjects/paper_xls_output/R/exploratory_data_analysis/gene_size_and_enrichment_magma_sets.R')}}

\begin{figure}
  \begin{subfigure}{15cm}
    \centering\includegraphics[width=12cm]{images/chapter2/ggplot/Rplot_corr_log10_set_p.png}
    \caption{Correlation of gene significance and set size with decreasing p. X axis maximum p value, y Pearsons correlation between -log10(p) of sets and set size. Data from supplementary material Sniekers et al\cite{sniekers2017genome}}
    \label{fig:correlation gene significance and set size Sniekers}
    \end{subfigure}
    
  \begin{subfigure}{15cm}
    \centering\includegraphics[width=12cm]{images/chapter2/ggplot/Rplot_minimum_p_mean_group_sizes.png}
    \caption{Changes in mean gene set size with decreasing maximum p values. Mean gene size increases. }
  \end{subfigure}
  \caption{Correlation between gene size and level of significance in MAGMA GSA and changes in mean gene size with increasingly significant gene sets (max p).Data from supplementary material Sniekers et al\cite{sniekers2017genome}}
   \label{fig:correlation gene significance and set size Sniekers with changes mean gene}
  \end{figure}


% \begin{figure}
%   \begin{subfigure}{8cm}
%     \centering\includegraphics[width=7cm]{images/FUMA_plots/gtex_v8_ts_general_FUMA_gene2func_UKBBEd_syn_protein_bp_sig.png}
%     \caption{Gene expression UKBBEd significant genes}
%     \end{subfigure}
%   \begin{subfigure}{8cm}
%     \centering\includegraphics[width=7cm]{images/FUMA_plots/gtex_v8_ts_FUMA_PSP_gtex.png}
%     \caption{Genes in PSP}
%   \end{subfigure}
 
%   \begin{subfigure}{8cm}
%     \centering\includegraphics[width=7cm]{images/FUMA_plots/gtex_v8_ts_psp_not_consensus_FUMA_gene2func44650.png}
%     \caption{Genes in PSP but not in consensus PSD. ie new genes}
%   \end{subfigure}
%   \begin{subfigure}{8cm}
%     \centering\includegraphics[width=7cm]{images/FUMA_plots/gtex_v8_ts_1313_consensus_FUMA_gene2func44650.png}
%     \caption{Genes in consensus PSD}
%   \end{subfigure}
% \end{figure}
%  \todo[inline]{the bottom two expression profiles look too similar redo these}
 

 
 
 

  


\clearpage


\section{Summary}
\label{sec:Methods Discussion}

The results obtained using MAGMA GSA correspond closely to those reported in the literature despite possible variations in versions, platforms and reference panels. In addition, results from the PASCAL scoring algorithm correlate closely with those from MAGMA. 

Most of the potential PSP genes identified by proteomics are in the large connected component. Isolated genes,  excluded from analysis, are few and show no evidence of possessing shared features that would bias results. Similarly, only a few PSP genes do not have boundaries defined in the reference gene boundaries used for MAGMA, and again, they show no evidence of common features. 

There are enriched disease terms related to intellectual disability in both education samples and neither intelligence sample. An association is reported in Sniekers et al. \cite{sniekers2017genome} using DEPICT but I am not aware that the difference with intelligence samples has been reported. It suggests a qualitative difference between enriched genes in the education and intelligence samples and that we may expect some subsequent findings to hold for only one of the phenotypes.  




PSP genes are expressed robustly in all brain regions using the GTEx data, and this suggests that the increase in gene number in the network has still resulted in a set of genes strongly expressed in the CNS. The pattern of correlation of gene expression between brain regions in the PSP is similar to that recorded of all genes literature \cite{mele2015human} suggesting that we do not have a very unusual sample.  

The difference between the findings for MAGMA GSA and ORA of significant genes in the intelligence discovery cohort show the limitations that network approaches limited to over-representation analysis face\cite{ghiassian2015disease}. I found ToppGene\cite{chen2009disease} had the best overall performance of a Gene Ontology enrichment package now that it accepts custom background sets. 

The relative lack of statistically significant gene associations in published studies using MAGMA GSA suggests it would be reasonable to reduce unnecessary comparisons. A reasonable way to achieve this is to use a discovery and replication cohort design and only test synaptic communities sets we have a reason to believe would be of interest. 

Sniekers et al.\cite{sniekers2017genome} tested 6,155 gene sets using MAGMA GSA, an extensive search over possible gene sets. The uncorrected p values should provide a rough guide to likely significance levels in using this sample (Intelligence\textsubscript{Replication} - this is the only sample with published MAGMA GSA and the summary data for the identical population available).

SynGO shows the value of using fewer tests and more specific terms (particularly for Education\textsubscript{Discovery} Biological Process).



Three hundred sixty-five gene sets are of greater than nominal significance (5.91\% of terms). There is no reason to expect a \textit{particular} network community defined by topology to be associated with a particular phenotype (intelligence or educational attainment). If there are between 15 and 120 communities (gene sets)\footnote{a rough estimate extrapolating from McLean et al. (2016)} in the PSP network (see chapter 4), and if Sniekers et al. is an \textit{approximate} guide to the distribution of MAGMA significance levels, then using nominal significance ($p<0.05$) to select interesting sets would result in around six genes being tested in the replication samples. 

The lowest $p$ value found in any of the samples using the 18 member functionally defined synaptic set was 0.001 (Cell adhesion and trans-synaptic signalling Education\textsubscript{Discovery}) and occurred in the best-powered cohort. This group also did not have the constraint of forming a topological unit.

 Few significant gene sets are reported in MAGMA GSA studies of intelligence or educational attainment. Those tended to be large (see table~\ref{tab:group size and enrichment}) and I find an association between group size and significance (figure~\label{fig:correlation gene significance and set size Sniekers} and figure~\ref{fig:correlation gene significance and set size Sniekers with changes mean gene}). The gene sets with very low unadjusted $p$ values in these studies would be larger than we would expect to find in the PSP.

Re-Analysis of the gene-level scores of significant genes in GWGAS studies of intelligence and education suggest that around 25\% of these genes are in the postsynaptic proteome. Therefore, we would expect that given the involvement of non PSP genes, the raw enrichment P values would be less significant than those reported in Sniekers et al. and other studies. 

The checks carried out in this chapter such as the correspondence of MAGMA scores in the analysis pipeline to those in the literature do not show any evidence of systematic bias and appear sound. Therefore in the next chapters I will move on to test the two hypotheses described in chapter~\ref{chap:introduction}. 




