\chapter{Supplementary tables chapter 3}
\subsection{Degree}
High degree nodes are enriched for terms associated with RNA binding, cell cycle and ubiquitin binding. This would be consistent with these high degree nodes being involved in central cellular processes\footnote{\url{source('~/RProjects/chapter3/R/sig_magma_genes/central_genes_ToppGene/degree/5_0_clip_centrality_degree_high_mf_n_10.R')}}.

\paragraph{Molecular function}
Molecular function table~\ref{tab:ToppGENE GO: Molecular Function. 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 186} shows enrichment for ubiquitin protein ligase, RNA binding and transcription factor binding. The q values are not as significant as those for kcoreness and eigenvector centrality.

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 15:28:01 2020

\subparagraph{aesthetics}
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 15:28:01 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}

\begin{tabular}{@{}clllcl@{}}
  \toprule
ID & Molecular Function & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 
  \midrule
GO:0031625 & ubiquitin protein ligase binding & 51 & 117 & $8.81 \times 10^{-22}$ & $9.12 \times 10^{-19}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 52 & 123 & $1.88 \times 10^{-21}$ & $9.74 \times 10^{-19}$ \\ 
  GO:0008134 & transcription factor binding & 58 & 156 & $1.53 \times 10^{-20}$ & $5.26 \times 10^{-18}$ \\ 
  GO:0019904 & protein domain specific binding & 87 & 329 & $3.13 \times 10^{-19}$ & $8.11 \times 10^{-17}$ \\ 
  GO:0003723 & RNA binding & 120 & 554 & $5.07 \times 10^{-19}$ & $1.05 \times 10^{-16}$ \\ 
  GO:0044877 & protein-containing complex binding & 111 & 513 & $2.27 \times 10^{-17}$ & $3.91 \times 10^{-15}$ \\ 
  GO:0003720 & telomerase activity & 21 & 30 & $5.04 \times 10^{-15}$ & $7.00 \times 10^{-13}$ \\ 
  GO:0003964 & RNA-directed DNA polymerase activity & 22 & 33 & $5.41 \times 10^{-15}$ & $7.00 \times 10^{-13}$ \\ 
  GO:0034061 & DNA polymerase activity & 23 & 37 & $1.23 \times 10^{-14}$ & $1.41 \times 10^{-12}$ \\ 
  GO:0019900 & kinase binding & 78 & 325 & $1.50 \times 10^{-14}$ & $1.55 \times 10^{-12}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology Molecular Function enrichment of genes above 90th centile of degree distribution]{\textbf{High degree genes. Molecular function - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular Function tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 186} 
\label{tab:ToppGENE GO: Molecular Function. 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 186}
\end{table}
\paragraph{Biological process}

Biological process table~\ref{tab:ToppGENE GO: Biological Process. 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 1584} shows enrichment for terms related to the cell cycle with low FDR values. 
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 15:30:52 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Biological Process & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0007049 & cell cycle & 139 & 489 & $1.98 \times 10^{-36}$ & $1.25 \times 10^{-32}$ \\ 
  GO:0022402 & cell cycle process & 119 & 414 & $7.10 \times 10^{-31}$ & $2.25 \times 10^{-27}$ \\ 
  GO:0000280 & nuclear division & 101 & 319 & $1.43 \times 10^{-29}$ & $3.02 \times 10^{-26}$ \\ 
  GO:0048285 & organelle fission & 103 & 337 & $9.65 \times 10^{-29}$ & $1.53 \times 10^{-25}$ \\ 
  GO:0000278 & mitotic cell cycle & 95 & 296 & $3.44 \times 10^{-28}$ & $3.10 \times 10^{-25}$ \\ 
  GO:0140014 & mitotic nuclear division & 95 & 296 & $3.44 \times 10^{-28}$ & $3.10 \times 10^{-25}$ \\ 
  GO:1903047 & mitotic cell cycle process & 95 & 296 & $3.44 \times 10^{-28}$ & $3.10 \times 10^{-25}$ \\ 
  GO:0044403 & symbiotic process & 106 & 364 & $1.23 \times 10^{-27}$ & $9.33 \times 10^{-25}$ \\ 
  GO:0051726 & regulation of cell cycle & 100 & 329 & $1.33 \times 10^{-27}$ & $9.33 \times 10^{-25}$ \\ 
  GO:0044419 & interspecies interaction between organisms & 106 & 368 & $3.48 \times 10^{-27}$ & $2.20 \times 10^{-24}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment Biological Process of genes above 90th centile of degree distribution]{\textbf{High degree genes. Biological Process - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological process tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 1584} 

\label{tab:ToppGENE GO: Biological Process. 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 1584}
\end{table}


\paragraph{Cellular component}


Cellular component table~\ref{tab:ToppGENE GO: Cellular Component. 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 159} shows enrichment for focal adhesion and cell junction terms, ribonucleoprotein and cytoskeletal fibre. 

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 15:33:48 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule

GO:1990904 & ribonucleoprotein complex & 81 & 303 & $2.75 \times 10^{-18}$ & $2.36 \times 10^{-15}$ \\ 
  GO:0015630 & microtubule cytoskeleton & 102 & 447 & $1.01 \times 10^{-17}$ & $4.33 \times 10^{-15}$ \\ 
  GO:0036464 & cytoplasmic ribonucleoprotein granule & 33 & 74 & $8.92 \times 10^{-15}$ & $2.55 \times 10^{-12}$ \\ 
  GO:0035770 & ribonucleoprotein granule & 33 & 78 & $5.84 \times 10^{-14}$ & $1.25 \times 10^{-11}$ \\ 
  GO:1902494 & catalytic complex & 86 & 392 & $1.07 \times 10^{-13}$ & $1.84 \times 10^{-11}$ \\ 
  GO:0030055 & cell-substrate junction & 62 & 244 & $7.12 \times 10^{-13}$ & $1.02 \times 10^{-10}$ \\ 
  GO:0005925 & focal adhesion & 61 & 239 & $9.13 \times 10^{-13}$ & $1.12 \times 10^{-10}$ \\ 
  GO:0000974 & Prp19 complex & 19 & 31 & $3.66 \times 10^{-12}$ & $3.92 \times 10^{-10}$ \\ 
  GO:0099513 & polymeric cytoskeletal fiber & 95 & 482 & $4.16 \times 10^{-12}$ & $3.96 \times 10^{-10}$ \\ 
  GO:0005815 & microtubule organizing center & 60 & 246 & $1.21 \times 10^{-11}$ & $1.04 \times 10^{-9}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment Cellular Component of genes above 90th centile of degree distribution]{\textbf{High degree genes. Cellular Component - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological process tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 159} 
\label{tab:ToppGENE GO: Cellular Component. 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 159}
\end{table}



\subsubsection{Low degree}
The only gene ontology class enriched was cellular component (table~\ref{tab:ToppGENE GO: Cellular Component. 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}) but the enriched terms are common in other low vaules of centrality measures in that it is enriched for components of the plasma membrane. Pathway enrichment shows enrichment of the extracellular matrix and ECM proteins which is found with other low centrality measure genes in PANTHER. The results for high and low degree genes appear qualitatively different even compared to other post synaptic protein genes (ie cell cycle processes verus ECM). The network architecture seems to be related to function but is it related to the phenotype?



% latex table generated in R 3.6.3 by xtable 1.8-4 package 
% Sat Oct  3 17:17:17 2020
\begin{table}[ht]
\centering

\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component &$p$ & $q$ FDR B H  & $n$ & $n$ PSP  \\ 
  \midrule
GO:0031226 & intrinsic component of plasma membrane & $1.564 \times 10^{-5}$ & $7.462 \times 10^{-3}$ & 51 & 331 \\ 
  GO:0005887 & integral component of plasma membrane & $4.228 \times 10^{-5}$ & $1.008 \times 10^{-2}$ & 47 & 307 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption{ToppGene GO: Cellular Component. 10 centile cwpsp.txtp Degree centrality = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. \textcolor{red}{columns in different order from above ? change}} 
\label{tab:ToppGENE GO: Cellular Component. 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}


Pathway table~\ref{tab:ToppGENE Pathway. 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP} 

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sat Oct  3 17:21:16 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}

  \toprule
ID & Pathway & p & q FDR B H & n & n PSP \\ 
  \midrule
M5889 & \makecell{Ensemble of genes encoding extracellular matrix and\\ extracellular matrix-associated proteins}  & $5.689 \times 10^{-6}$ & $5.831 \times 10^{-3}$ & 19 & 91 \\ 
  1269590 & Adenylate cyclase activating pathway & $9.603 \times 10^{-5}$ & $4.798 \times 10^{-2}$ & 4 & 5 \\ 
  1269923 &\makecell{ Transport of glucose and other sugars, bile salts and organic acids,\\ metal ions and amine compounds} & $2.551 \times 10^{-4}$ & $4.798 \times 10^{-2}$ & 5 & 10 \\ 
  1272485 & Aldosterone synthesis and secretion & $2.896 \times 10^{-4}$ & $4.798 \times 10^{-2}$ & 9 & 34 \\ 
  1269930 & Metal ion SLC transporters & $3.277 \times 10^{-4}$ & $4.798 \times 10^{-2}$ & 3 & 3 \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption{ToppGene Pathway. 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP} 
\tiny\url{source('~/RProjects/chapter3/R/sig_magma_genes/central_genes_toppgene/betweenness/5_0_clip_centrality_bet_hig_mf.R')}

\label{tab:ToppGENE Pathway. 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}



\subsection{Betweenness ToppGene}
\subsubsection{High betweenness}
\paragraph{Molecular function}
Molecular function see table~\ref{tab:ToppGENE GO: Molecular Function. bet 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 11:46:04 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Molecular Function & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0044877 & protein-containing complex binding & 120 & 513 & $1.74 \times 10^{-22}$ & $1.83 \times 10^{-19}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 51 & 117 & $5.84 \times 10^{-22}$ & $3.07 \times 10^{-19}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 52 & 123 & $1.24 \times 10^{-21}$ & $4.34 \times 10^{-19}$ \\ 
  GO:0019904 & protein domain specific binding & 89 & 329 & $1.02 \times 10^{-20}$ & $2.69 \times 10^{-18}$ \\ 
  GO:0019900 & kinase binding & 76 & 325 & $9.81 \times 10^{-14}$ & $2.06 \times 10^{-11}$ \\ 
  GO:0008234 & cysteine-type peptidase activity & 54 & 189 & $1.31 \times 10^{-13}$ & $2.29 \times 10^{-11}$ \\ 
  GO:0008134 & transcription factor binding & 48 & 156 & $1.64 \times 10^{-13}$ & $2.45 \times 10^{-11}$ \\ 
  GO:0005102 & signaling receptor binding & 87 & 405 & $2.09 \times 10^{-13}$ & $2.74 \times 10^{-11}$ \\ 
  GO:0050839 & cell adhesion molecule binding & 69 & 288 & $5.05 \times 10^{-13}$ & $5.88 \times 10^{-11}$ \\ 
  GO:0019901 & protein kinase binding & 69 & 295 & $1.77 \times 10^{-12}$ & $1.86 \times 10^{-10}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High betweenness genes Molecular Function of genes above 90th centile of distribution]{\textbf{High betweenness genes. Cellular Component - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological process tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 

\label{tab:ToppGENE GO: Molecular Function. bet 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}

\paragraph{Biological process}
Biological process table~\ref{tab:ToppGENE GO: Biological Process. bet 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 11:49:29 2020
  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Biological Process & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0007049 & cell cycle & 124 & 489 & $5.10 \times 10^{-27}$ & $3.33 \times 10^{-23}$ \\ 
  GO:0051247 & positive regulation of protein metabolic process & 117 & 463 & $4.11 \times 10^{-25}$ & $1.34 \times 10^{-21}$ \\ 
  GO:0042981 & regulation of apoptotic process & 108 & 416 & $5.38 \times 10^{-24}$ & $1.17 \times 10^{-20}$ \\ 
  GO:0043067 & regulation of programmed cell death & 108 & 423 & $2.44 \times 10^{-23}$ & $3.99 \times 10^{-20}$ \\ 
  GO:0009894 & regulation of catabolic process & 95 & 350 & $2.16 \times 10^{-22}$ & $2.82 \times 10^{-19}$ \\ 
  GO:0006508 & proteolysis & 105 & 417 & $4.31 \times 10^{-22}$ & $4.29 \times 10^{-19}$ \\ 
  GO:0010941 & regulation of cell death & 112 & 464 & $4.60 \times 10^{-22}$ & $4.29 \times 10^{-19}$ \\ 
  GO:0032270 & positive regulation of cellular protein metabolic process & 106 & 433 & $2.89 \times 10^{-21}$ & $2.36 \times 10^{-18}$ \\ 
  GO:0009057 & macromolecule catabolic process & 106 & 443 & $2.02 \times 10^{-20}$ & $1.47 \times 10^{-17}$ \\ 
  GO:0043066 & negative regulation of apoptotic process & 73 & 240 & $3.94 \times 10^{-20}$ & $2.36 \times 10^{-17}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High betweenness genes Biological Process of genes above 90th centile of distribution]{\textbf{High betweenness genes. Biological Process - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological process tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}\url{source('~/RProjects/chapter3/R/sig_magma_genes/central_genes_toppgene/betweenness/5_0_clip_centrality_bet_hig_bp.R')}} 
\label{tab:ToppGENE GO: Biological Process. bet 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}

\paragraph{Cellular component}
Cellular component table~\ref{tab:ToppGENE GO: Cellular Component. bet 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 11:50:08 2020

    \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0015630 & microtubule cytoskeleton & 98 & 447 & $4.56 \times 10^{-16}$ & $4.02 \times 10^{-13}$ \\ 
  GO:0005814 & centriole & 55 & 194 & $7.79 \times 10^{-14}$ & $3.43 \times 10^{-11}$ \\ 
  GO:0005813 & centrosome & 52 & 182 & $2.92 \times 10^{-13}$ & $8.59 \times 10^{-11}$ \\ 
  GO:0005815 & microtubule organizing center & 62 & 246 & $6.14 \times 10^{-13}$ & $1.35 \times 10^{-10}$ \\ 
  GO:0030990 & intraciliary transport particle & 51 & 212 & $5.95 \times 10^{-10}$ & $8.73 \times 10^{-8}$ \\ 
  GO:0005929 & cilium & 51 & 212 & $5.95 \times 10^{-10}$ & $8.73 \times 10^{-8}$ \\ 
  GO:0098589 & membrane region & 42 & 160 & $1.37 \times 10^{-9}$ & $1.72 \times 10^{-7}$ \\ 
  GO:0005819 & spindle & 35 & 120 & $1.74 \times 10^{-9}$ & $1.92 \times 10^{-7}$ \\ 
  GO:0099513 & polymeric cytoskeletal fiber & 87 & 482 & $3.13 \times 10^{-9}$ & $2.46 \times 10^{-7}$ \\ 
  GO:0045121 & membrane raft & 41 & 158 & $3.20 \times 10^{-9}$ & $2.46 \times 10^{-7}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High betweenness genes Cellular Component of genes above 90th centile of distribution]{\textbf{High betweenness genes. Cellular Component - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological process tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}}
\label{tab:ToppGENE GO: Cellular Component. bet 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}





\clearpage

\subsubsection{Low betweenness}

\paragraph{Cellular component}

Cellular component table~\ref{tab:ToppGENE GO: Cellular Component. bet 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 11:58:29 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0031226 & intrinsic component of plasma membrane & 61 & 331 & $1.85 \times 10^{-6}$ & $9.76 \times 10^{-4}$ \\ 
  GO:0005887 & integral component of plasma membrane & 56 & 307 & $7.36 \times 10^{-6}$ & $1.94 \times 10^{-3}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment Low betweenness genes Cellular Component of genes below 10th centile of distribution]{\textbf{Low betweenness genes. Cellular Component - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological process tested gene set is below 10th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}}
\label{tab:ToppGENE GO: Cellular Component. bet 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}

\paragraph{Pathway}

Extra cellular matrix table~\ref{tab:ToppGENE Pathway. bet 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}. Glycosylphosphatidylinositol proteins (GPI proteins) are directed to the endoplasmic reticulum. 




% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 12:01:20 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lcllcl}
  \hline
ID & Pathway & n & n PSP & p & q FDR B H \\ 
  \hline
M5889 &\makecell{ Ensemble of genes encoding extracellular matrix\\ and extracellular matrix-associated proteins}  & 21 & 91 & $7.23 \times 10^{-6}$ & $8.28 \times 10^{-3}$ \\ 
  M1268710 & \makecell{Post-translational modification: synthesis of GPI-anchored proteins} & 6 & 11 & $9.13 \times 10^{-5}$ & $4.00 \times 10^{-2}$ \\ 
  M5884 & \makecell{Ensemble of genes encoding core extracellular matrix including\\ ECM glycoproteins, collagens and proteoglycans} & 10 & 31 & $1.05 \times 10^{-4}$ & $4.00 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{ToppGene Pathway. bet 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP} 
\label{tab:ToppGENE Pathway. bet 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}

\clearpage
\subsection{Closeness Centrality ToppGene}



\paragraph{Molecular function}

Molecular function Gene Ontology Enrichment High Closeness centrality in table \ref{tab:ToppGENE GO: Molecular Function. clo 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 12:10:43 2020

  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Molecular Function & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0003723 & RNA binding & 167 & 554 & $2.11 \times 10^{-50}$ & $2.14 \times 10^{-47}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 57 & 117 & $1.38 \times 10^{-27}$ & $7.00 \times 10^{-25}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 58 & 123 & $4.10 \times 10^{-27}$ & $1.39 \times 10^{-24}$ \\ 
  GO:0051082 & unfolded protein binding & 31 & 56 & $2.20 \times 10^{-17}$ & $5.57 \times 10^{-15}$ \\ 
  GO:0044877 & protein-containing complex binding & 110 & 513 & $4.36 \times 10^{-17}$ & $8.84 \times 10^{-15}$ \\ 
  GO:0019904 & protein domain specific binding & 81 & 329 & $5.59 \times 10^{-16}$ & $9.46 \times 10^{-14}$ \\ 
  GO:0003729 & mRNA binding & 39 & 100 & $6.55 \times 10^{-15}$ & $9.50 \times 10^{-13}$ \\ 
  GO:0008134 & transcription factor binding & 50 & 156 & $8.56 \times 10^{-15}$ & $1.09 \times 10^{-12}$ \\ 
  GO:0005102 & signaling receptor binding & 86 & 405 & $7.31 \times 10^{-13}$ & $8.25 \times 10^{-11}$ \\ 
  GO:0140097 & catalytic activity, acting on DNA & 31 & 76 & $1.13 \times 10^{-12}$ & $1.15 \times 10^{-10}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High Closeness genes Molecular Function of genes above 90th centile of distribution]{\textbf{High Closeness genes. Molecular Function - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular function tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 
\label{tab:ToppGENE GO: Molecular Function. clo 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}

\paragraph{Biological Process}
Biological process table~\ref{tab:ToppGENE GO: Biological Process. clo 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 12:13:59 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Biological Process & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0044265 & cellular macromolecule catabolic process & 131 & 372 & $4.77 \times 10^{-46}$ & $2.98 \times 10^{-42}$ \\ 
  GO:0009057 & macromolecule catabolic process & 143 & 443 & $1.34 \times 10^{-45}$ & $4.18 \times 10^{-42}$ \\ 
  GO:0016071 & mRNA metabolic process & 106 & 260 & $2.87 \times 10^{-43}$ & $5.96 \times 10^{-40}$ \\ 
  GO:0044403 & symbiotic process & 125 & 364 & $3.12 \times 10^{-42}$ & $4.86 \times 10^{-39}$ \\ 
  GO:0044419 & interspecies interaction between organisms & 125 & 368 & $1.23 \times 10^{-41}$ & $1.53 \times 10^{-38}$ \\ 
  GO:0016032 & viral process & 118 & 348 & $6.85 \times 10^{-39}$ & $7.13 \times 10^{-36}$ \\ 
  GO:0006402 & mRNA catabolic process & 75 & 180 & $1.13 \times 10^{-30}$ & $1.00 \times 10^{-27}$ \\ 
  GO:0006401 & RNA catabolic process & 76 & 186 & $2.10 \times 10^{-30}$ & $1.63 \times 10^{-27}$ \\ 
  GO:0010608 & posttranscriptional regulation of gene expression & 82 & 225 & $1.11 \times 10^{-28}$ & $7.69 \times 10^{-26}$ \\ 
  GO:0034655 & nucleobase-containing compound catabolic process & 80 & 217 & $2.37 \times 10^{-28}$ & $1.48 \times 10^{-25}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High Closeness genes Biological Process of genes above 90th centile of distribution]{\textbf{High Closeness genes. Biological Process - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological process tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 

\label{tab:ToppGENE GO: Biological Process. clo 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}





\paragraph{Cellular component}
Cellular component table~\ref{tab:ToppGENE GO: Cellular Component. clo 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 12:15:50 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:1990904 & ribonucleoprotein complex & 102 & 303 & $5.76 \times 10^{-33}$ & $4.89 \times 10^{-30}$ \\ 
  GO:0005925 & focal adhesion & 77 & 239 & $4.94 \times 10^{-23}$ & $2.10 \times 10^{-20}$ \\ 
  GO:0030055 & cell-substrate junction & 77 & 244 & $2.18 \times 10^{-22}$ & $6.15 \times 10^{-20}$ \\ 
  GO:0005912 & adherens junction & 85 & 314 & $8.51 \times 10^{-20}$ & $1.80 \times 10^{-17}$ \\ 
  GO:0070161 & anchoring junction & 85 & 319 & $2.64 \times 10^{-19}$ & $4.47 \times 10^{-17}$ \\ 
  GO:0035770 & ribonucleoprotein granule & 36 & 78 & $1.03 \times 10^{-16}$ & $1.30 \times 10^{-14}$ \\ 
  GO:0036464 & cytoplasmic ribonucleoprotein granule & 35 & 74 & $1.07 \times 10^{-16}$ & $1.30 \times 10^{-14}$ \\ 
  GO:0000974 & Prp19 complex & 20 & 31 & $2.08 \times 10^{-13}$ & $2.21 \times 10^{-11}$ \\ 
  GO:0101031 & chaperone complex & 16 & 21 & $1.04 \times 10^{-12}$ & $9.79 \times 10^{-11}$ \\ 
  GO:0005681 & spliceosomal complex & 22 & 40 & $1.30 \times 10^{-12}$ & $1.11 \times 10^{-10}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High Closeness genes Cellular Component of genes above 90th centile of distribution]{\textbf{High Closeness genes. Cellular Component - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological process tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 

\label{tab:ToppGENE GO: Cellular Component. clo 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}






\subsubsection{Low closeness}

Note human phenotype epilepsy and disease enriched\footnote{\url{source('~/RProjects/chapter3/R/sig_magma_genes/central_genes_toppgene/closeness/low_closeness/5_0_clip_centrality_clo_low_cc_n_10.R'}}
\paragraph{Molecular function}
Molecular function table~\ref{tab:ToppGENE GO: Molecular Function. clo 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}. Closeness is more normally distributed it appears the low part of its distribution is also more discriminating than those with power law distribution that have a number of genes separated by small absolute values at the lower end of the distribution. Genes encoding proteins with low closeness centrality are enriched for molecular functions involving ion channels and trans-membrane transporters. 

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 12:19:53 2020

  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Molecular Function & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0005215 & transporter activity & 66 & 419 & $2.40 \times 10^{-5}$ & $1.02 \times 10^{-2}$ \\ 
  GO:0022857 & transmembrane transporter activity & 61 & 378 & $2.44 \times 10^{-5}$ & $1.02 \times 10^{-2}$ \\ 
  GO:0004930 & G protein-coupled receptor activity & 10 & 24 & $3.91 \times 10^{-5}$ & $1.09 \times 10^{-2}$ \\ 
  GO:0043176 & amine binding & 4 & 4 & $1.01 \times 10^{-4}$ & $2.10 \times 10^{-2}$ \\ 
  GO:0046873 & metal ion transmembrane transporter activity & 36 & 203 & $2.12 \times 10^{-4}$ & $3.19 \times 10^{-2}$ \\ 
  GO:0015075 & ion transmembrane transporter activity & 53 & 340 & $2.34 \times 10^{-4}$ & $3.19 \times 10^{-2}$ \\ 
  GO:0022803 & passive transmembrane transporter activity & 39 & 230 & $3.05 \times 10^{-4}$ & $3.19 \times 10^{-2}$ \\ 
  GO:0015267 & channel activity & 39 & 230 & $3.05 \times 10^{-4}$ & $3.19 \times 10^{-2}$ \\ 
  GO:0008324 & cation transmembrane transporter activity & 46 & 288 & $3.61 \times 10^{-4}$ & $3.36 \times 10^{-2}$ \\ 
  GO:0015318 & inorganic molecular entity transmembrane transporter activity & 50 & 324 & $4.60 \times 10^{-4}$ & $3.86 \times 10^{-2}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment Low Closeness genes Molecular Function of genes above 90th centile of distribution]{\textbf{Low Closeness genes. Molecular Function - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological process tested gene set is below 10th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 
 
\label{tab:ToppGENE GO: Molecular Function. clo 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}


\paragraph{Cellular component}
Cellular component table~\ref{tab:ToppGENE GO: Cellular Component. clo 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}. Again shows transmembrane transporters also integral part of synaptic and post synaptic membrane. The proteins encoded by these genes the ion channels and synaptic membrane components are also on the periphery of the network in the sense that they are more distant from most other nodes.




  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0031226 & intrinsic component of plasma membrane & 72 & 331 & $8.90 \times 10^{-12}$ & $4.95 \times 10^{-9}$ \\ 
  GO:0005887 & integral component of plasma membrane & 66 & 307 & $1.41 \times 10^{-10}$ & $3.91 \times 10^{-8}$ \\ 
  GO:1990351 & transporter complex & 30 & 132 & $7.50 \times 10^{-6}$ & $1.35 \times 10^{-3}$ \\ 
  GO:1902495 & transmembrane transporter complex & 29 & 127 & $9.72 \times 10^{-6}$ & $1.35 \times 10^{-3}$ \\ 
  GO:0034702 & ion channel complex & 27 & 117 & $1.62 \times 10^{-5}$ & $1.57 \times 10^{-3}$ \\ 
  GO:0034703 & cation channel complex & 24 & 98 & $1.69 \times 10^{-5}$ & $1.57 \times 10^{-3}$ \\ 
  GO:0099055 & integral component of postsynaptic membrane & 19 & 72 & $4.33 \times 10^{-5}$ & $3.22 \times 10^{-3}$ \\ 
  GO:0072534 & perineuronal net & 5 & 6 & $5.12 \times 10^{-5}$ & $3.22 \times 10^{-3}$ \\ 
  GO:0098936 & intrinsic component of postsynaptic membrane & 20 & 79 & $5.22 \times 10^{-5}$ & $3.22 \times 10^{-3}$ \\ 
  GO:0099699 & integral component of synaptic membrane & 22 & 96 & $1.13 \times 10^{-4}$ & $6.28 \times 10^{-3}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment Low Closeness genes Cellular Component of genes above 90th centile of distribution]{\textbf{Low Closeness genes. Cellular Component - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological process tested gene set is below 10th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 

\label{tab:ToppGENE GO: Cellular Component. clo 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}


\clearpage


\subsection{Eigenvector centrality}
\subsubsection{High eigenvector centrality}

\paragraph{Molecular function}

Molecular function table~\ref{tab:ToppGENE GO: Molecular Function. 90 centile cw psp eig.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP} shows over representation of ubiquitin, RNA binding and ribosomal components. Similar to other centrality measurs but very strong over representation\footnote{\url{source('~/RProjects/chapter3/R/sig_magma_genes/central_genes_toppgene/eigenvector/5_0_clip_centrality_eig_mfunc_n_10.R')}}.


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 12:32:37 2020
  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Molecular Function & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0003723 & RNA binding & 198 & 554 & $7.00 \times 10^{-79}$ & $7.01 \times 10^{-76}$ \\ 
  GO:0003735 & structural constituent of ribosome & 54 & 103 & $2.82 \times 10^{-28}$ & $1.41 \times 10^{-25}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 56 & 117 & $1.15 \times 10^{-26}$ & $3.83 \times 10^{-24}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 57 & 123 & $3.21 \times 10^{-26}$ & $8.03 \times 10^{-24}$ \\ 
  GO:0003729 & mRNA binding & 51 & 100 & $5.76 \times 10^{-26}$ & $1.15 \times 10^{-23}$ \\ 
  GO:0005198 & structural molecule activity & 82 & 303 & $5.13 \times 10^{-19}$ & $8.55 \times 10^{-17}$ \\ 
  GO:0019843 & rRNA binding & 25 & 34 & $1.26 \times 10^{-18}$ & $1.80 \times 10^{-16}$ \\ 
  GO:0003727 & single-stranded RNA binding & 21 & 31 & $1.19 \times 10^{-14}$ & $1.49 \times 10^{-12}$ \\ 
  GO:0045296 & cadherin binding & 59 & 224 & $4.31 \times 10^{-13}$ & $4.80 \times 10^{-11}$ \\ 
  GO:0003730 & mRNA 3'-UTR binding & 20 & 32 & $5.22 \times 10^{-13}$ & $5.22 \times 10^{-11}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}

\caption[Gene ontology enrichment High Eigenvector centrality genes Molecular Function of genes above 90th centile of distribution]{\textbf{High Eigenvector centrality genes. Molecular Function - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular function tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 
\label{tab:ToppGENE GO: Molecular Function. 90 centile cw psp eig.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}

\paragraph{Biological process}
Biological process table~\ref{tab:ToppGENE GO: Biological Process. 90 centile cw psp eig.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 12:34:28 2020
  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Biological Process & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0016071 & mRNA metabolic process & 143 & 260 & $1.30 \times 10^{-83}$ & $7.64 \times 10^{-80}$ \\ 
  GO:0009057 & macromolecule catabolic process & 159 & 443 & $1.09 \times 10^{-59}$ & $3.20 \times 10^{-56}$ \\ 
  GO:0044265 & cellular macromolecule catabolic process & 144 & 372 & $5.80 \times 10^{-58}$ & $9.34 \times 10^{-55}$ \\ 
  GO:0006402 & mRNA catabolic process & 101 & 180 & $6.34 \times 10^{-58}$ & $9.34 \times 10^{-55}$ \\ 
  GO:0006401 & RNA catabolic process & 102 & 186 & $3.28 \times 10^{-57}$ & $3.87 \times 10^{-54}$ \\ 
  GO:0044403 & symbiotic process & 139 & 364 & $1.05 \times 10^{-54}$ & $1.03 \times 10^{-51}$ \\ 
  GO:0044419 & interspecies interaction between organisms & 139 & 368 & $5.35 \times 10^{-54}$ & $4.50 \times 10^{-51}$ \\ 
  GO:0016032 & viral process & 134 & 348 & $6.30 \times 10^{-53}$ & $4.64 \times 10^{-50}$ \\ 
  GO:0034655 & nucleobase-containing compound catabolic process & 104 & 217 & $7.76 \times 10^{-51}$ & $5.08 \times 10^{-48}$ \\ 
  GO:0019439 & aromatic compound catabolic process & 104 & 232 & $2.67 \times 10^{-47}$ & $1.58 \times 10^{-44}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High Eigenvector centrality genes Biological Process of genes above 90th centile of distribution]{\textbf{High Eigenvector centrality genes. Biological Process - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular function tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 
\label{tab:ToppGENE GO: Biological Process. 90 centile cw psp eig.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}


\paragraph{Cellular component}
Cellular component table~\ref{tab:ToppGENE GO: Cellular Component. 90 centile cw psp eig.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}. PRP is pre-ribosomal rna processing complx 19.snRNP are small ribonuclear protein. Enrichment for ribosomes and for focal adhesion. U5 snRNP associates with Prp8 (ref wikipedia). The coverage of some of these ontology terms for cellular component (to coin a phrase where i am referring to how much n in the group represents of the whole annotation). It finds 28 of 34 U5 snRNP genes. 

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 12:38:30 2020
 \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:1990904 & ribonucleoprotein complex & 144 & 303 & $4.68 \times 10^{-73}$ & $3.68 \times 10^{-70}$ \\ 
  GO:0022626 & cytosolic ribosome & 60 & 97 & $3.21 \times 10^{-37}$ & $1.26 \times 10^{-34}$ \\ 
  GO:0044391 & ribosomal subunit & 61 & 124 & $4.99 \times 10^{-30}$ & $1.31 \times 10^{-27}$ \\ 
  GO:0005840 & ribosome & 65 & 141 & $6.93 \times 10^{-30}$ & $1.36 \times 10^{-27}$ \\ 
  GO:0000974 & Prp19 complex & 29 & 31 & $1.48 \times 10^{-27}$ & $2.33 \times 10^{-25}$ \\ 
  GO:0005681 & spliceosomal complex & 32 & 40 & $1.11 \times 10^{-25}$ & $1.46 \times 10^{-23}$ \\ 
  GO:0071013 & catalytic step 2 spliceosome & 27 & 29 & $1.49 \times 10^{-25}$ & $1.67 \times 10^{-23}$ \\ 
  GO:0097525 & spliceosomal snRNP complex & 29 & 35 & $3.56 \times 10^{-24}$ & $3.49 \times 10^{-22}$ \\ 
  GO:0005925 & focal adhesion & 78 & 239 & $7.67 \times 10^{-24}$ & $6.70 \times 10^{-22}$ \\ 
  GO:0005682 & U5 snRNP & 28 & 34 & $3.16 \times 10^{-23}$ & $2.48 \times 10^{-21}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High Eigenvector centrality genes Cellular Component of genes above 90th centile of distribution]{\textbf{High Eigenvector centrality genes. Cellular Component - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular function tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 

\label{tab:ToppGENE GO: Cellular Component. 90 centile cw psp eig.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}

\clearpage
\subsubsection{Low eigenvector centrality}
\paragraph{Molecular function}

Molecular function \ref{tab:ToppGENE GO: Molecular Function. 10 centile cw psp eig.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}. Low eigenvector genes are enriched for molecular function terms related to transmembrane transporter activity and voltage gated ion channels. 

No significant enrichment for biological process. 

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 14:17:27 2020
 \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Molecular Function & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0022857 & transmembrane transporter activity & 69 & 378 & $5.33 \times 10^{-8}$ & $2.75 \times 10^{-5}$ \\ 
  GO:0005215 & transporter activity & 74 & 419 & $6.69 \times 10^{-8}$ & $2.75 \times 10^{-5}$ \\ 
  GO:0022803 & passive transmembrane transporter activity & 47 & 230 & $3.54 \times 10^{-7}$ & $7.28 \times 10^{-5}$ \\ 
  GO:0015267 & channel activity & 47 & 230 & $3.54 \times 10^{-7}$ & $7.28 \times 10^{-5}$ \\ 
  GO:0015075 & ion transmembrane transporter activity & 61 & 340 & $7.16 \times 10^{-7}$ & $1.18 \times 10^{-4}$ \\ 
  GO:0015318 & inorganic molecular entity transmembrane transporter activity & 58 & 324 & $1.54 \times 10^{-6}$ & $1.67 \times 10^{-4}$ \\ 
  GO:0005244 & voltage-gated ion channel activity & 29 & 118 & $1.62 \times 10^{-6}$ & $1.67 \times 10^{-4}$ \\ 
  GO:0022832 & voltage-gated channel activity & 29 & 118 & $1.62 \times 10^{-6}$ & $1.67 \times 10^{-4}$ \\ 
  GO:0008324 & cation transmembrane transporter activity & 53 & 288 & $1.96 \times 10^{-6}$ & $1.79 \times 10^{-4}$ \\ 
  GO:0046873 & metal ion transmembrane transporter activity & 41 & 203 & $2.97 \times 10^{-6}$ & $2.44 \times 10^{-4}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment Low Eigenvector centrality genes Molecular Function of genes above 10th centile of distribution]{\textbf{Low Eigenvector centrality genes. Molecular Function - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular function tested gene set is 10th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 

\label{tab:ToppGENE GO: Molecular Function. 10 centile cw psp eig.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}

\paragraph{Cellular component}

Cellular component table~\ref{tab:ToppGENE GO: Cellular Component. 10 centile cw psp eig.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP} is enriched for transmembrane transporter complexes, ion channels and integral and intrinsic part of synaptic and post synaptic membrane. 
 \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0031226 & intrinsic component of plasma membrane & 75 & 331 & $3.06 \times 10^{-13}$ & $1.69 \times 10^{-10}$ \\ 
  GO:0005887 & integral component of plasma membrane & 69 & 307 & $5.24 \times 10^{-12}$ & $1.45 \times 10^{-9}$ \\ 
  GO:1990351 & transporter complex & 36 & 132 & $5.47 \times 10^{-9}$ & $9.30 \times 10^{-7}$ \\ 
  GO:1902495 & transmembrane transporter complex & 35 & 127 & $6.71 \times 10^{-9}$ & $9.30 \times 10^{-7}$ \\ 
  GO:0034702 & ion channel complex & 33 & 117 & $9.91 \times 10^{-9}$ & $1.10 \times 10^{-6}$ \\ 
  GO:0034703 & cation channel complex & 29 & 98 & $2.53 \times 10^{-8}$ & $2.33 \times 10^{-6}$ \\ 
  GO:0099055 & integral component of postsynaptic membrane & 21 & 72 & $3.01 \times 10^{-6}$ & $2.37 \times 10^{-4}$ \\ 
  GO:0099699 & integral component of synaptic membrane & 25 & 96 & $3.42 \times 10^{-6}$ & $2.37 \times 10^{-4}$ \\ 
  GO:0098936 & intrinsic component of postsynaptic membrane & 22 & 79 & $4.08 \times 10^{-6}$ & $2.51 \times 10^{-4}$ \\ 
  GO:0099240 & intrinsic component of synaptic membrane & 26 & 108 & $1.04 \times 10^{-5}$ & $5.78 \times 10^{-4}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment Low Eigenvector centrality genes Molecular Function of genes above 10th centile of distribution]{\textbf{Low Eigenvector centrality genes. Molecular Function - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular function tested gene set is 10th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): \textcolor{red}{n}} 

\label{tab:ToppGENE GO: Cellular Component. 10 centile cw psp eig.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP}
\end{table}


\paragraph{Other tables}
Pathway neuronal system compared with rest PSP  34,189 q = 0.0348

Human phenotype epilepsies 

Diseases top 10 epilepsies  
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 14:20:15 2020



\clearpage


\subsection{Transitivity}
Transitivity with isolates set to NA and NA removed. 

\textcolor{red}{Key findings}

Significance score lower

smaller groups
+

\subsubsection{High transitivity}
\paragraph{Molecular function}
Molecular function table~\ref{tab:ToppGENE GO: Molecular Function. tra 90 centile NA narm cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 2}. Only two significant terms, compare with other high centralities. Over representation of constituent of ribosome. 

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 14:35:45 2020
  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID &Molecular Function & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0003735 & structural constituent of ribosome & 29 & 103 & $1.45 \times 10^{-8}$ & $1.06 \times 10^{-5}$ \\ 
  GO:0005198 & structural molecule activity & 48 & 303 & $6.32 \times 10^{-5}$ & $2.31 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High Transitivity NaN genes. Molecular Function of genes above 90th centile of distribution]{\textbf{High Transitivity NaN genes. Molecular Function - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular function tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 2} 

\label{tab:ToppGENE GO: Molecular Function. tra 90 centile NA narm cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 2}
\end{table}
\paragraph{Biological process}
Biological process table~\ref{tab:ToppGENE GO: Biological Process. tra 90 centile NA narm cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 22}

Mitochondrial translation is enriched. 



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 14:37:31 2020
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 12:34:28 2020
  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Biological Process & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0006415 & translational termination & 14 & 35 & $8.46 \times 10^{-7}$ & $2.13 \times 10^{-3}$ \\ 
  GO:0070126 & mitochondrial translational termination & 13 & 32 & $1.71 \times 10^{-6}$ & $2.13 \times 10^{-3}$ \\ 
  GO:0070125 & mitochondrial translational elongation & 13 & 32 & $1.71 \times 10^{-6}$ & $2.13 \times 10^{-3}$ \\ 
  GO:0043624 & cellular protein complex disassembly & 25 & 108 & $7.84 \times 10^{-6}$ & $6.98 \times 10^{-3}$ \\ 
  GO:0032543 & mitochondrial translation & 15 & 47 & $9.37 \times 10^{-6}$ & $6.98 \times 10^{-3}$ \\ 
  GO:0006414 & translational elongation & 46 & 279 & $3.13 \times 10^{-5}$ & $1.66 \times 10^{-2}$ \\ 
  GO:0006412 & translation & 46 & 279 & $3.13 \times 10^{-5}$ & $1.66 \times 10^{-2}$ \\ 
  GO:0140053 & mitochondrial gene expression & 15 & 52 & $3.65 \times 10^{-5}$ & $1.70 \times 10^{-2}$ \\ 
  GO:0043043 & peptide biosynthetic process & 46 & 283 & $4.54 \times 10^{-5}$ & $1.72 \times 10^{-2}$ \\ 
  GO:0006521 & regulation of cellular amino acid metabolic process & 12 & 36 & $4.62 \times 10^{-5}$ & $1.72 \times 10^{-2}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High Transitivity NaN genes Biological Process of genes above 90th centile of distribution]{\textbf{High Transitivity NaN genes. Biological Process - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular function tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 22} 
\label{tab:ToppGENE GO: Biological Process. tra 90 centile NA narm cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 22}
\end{table}

\paragraph{Cellular component}

Cellular component table~\ref{tab:ToppGENE GO: Cellular Component. tra 90 centile NA narm cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 26}. Mitochondrial protein complex and ribosomal subunit are enriched consisent with findings of translation being enriched in biological process.

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 14:38:37 2020
  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0044391 & ribosomal subunit & 31 & 124 & $8.04 \times 10^{-8}$ & $3.86 \times 10^{-5}$ \\ 
  GO:0098798 & mitochondrial protein complex & 33 & 139 & $1.15 \times 10^{-7}$ & $3.86 \times 10^{-5}$ \\ 
  GO:0005840 & ribosome & 32 & 141 & $5.48 \times 10^{-7}$ & $1.23 \times 10^{-4}$ \\ 
  GO:0005761 & mitochondrial ribosome & 13 & 31 & $1.03 \times 10^{-6}$ & $1.39 \times 10^{-4}$ \\ 
  GO:0000313 & organellar ribosome & 13 & 31 & $1.03 \times 10^{-6}$ & $1.39 \times 10^{-4}$ \\ 
  GO:0005762 & mitochondrial large ribosomal subunit & 9 & 16 & $2.44 \times 10^{-6}$ & $2.34 \times 10^{-4}$ \\ 
  GO:0000315 & organellar large ribosomal subunit & 9 & 16 & $2.44 \times 10^{-6}$ & $2.34 \times 10^{-4}$ \\ 
  GO:0015934 & large ribosomal subunit & 18 & 64 & $8.18 \times 10^{-6}$ & $6.88 \times 10^{-4}$ \\ 
  GO:0000502 & proteasome complex & 13 & 38 & $1.50 \times 10^{-5}$ & $1.01 \times 10^{-3}$ \\ 
  GO:1905369 & endopeptidase complex & 13 & 38 & $1.50 \times 10^{-5}$ & $1.01 \times 10^{-3}$ \\ 
   \hline
\bottomrule

\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High Transitivity NaN genes Cellular Component of genes above 90th centile of distribution]{\textbf{High Transitivity NaN genes. Cellular Component - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular function tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 26} 
\label{tab:ToppGENE GO: Cellular Component. tra 90 centile NA narm cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 26}
\end{table}




\paragraph{Other}
Pathway related to mitochondria. No disease enrichment.




\subsubsection{Low transitivity}
No enrichment



\clearpage
\subsection{kcoreness}


\subsubsection{High kcoreness}
\paragraph{Molecular function}

Molecular function table~\ref{tab:ToppGENE GO: Molecular Function. kco 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 180}
Ubiquitin and RNA binding. Low adjusted p for RNA binding comparable to eigenvector\footnote{\url{source('~/RProjects/chapter3/R/sig_magma_genes/central_genes_toppgene/kcoreness/5_0_clip_centrality_kco_high_mf_n_10.R')}}.

  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Molecular Function & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0003723 & RNA binding & 188 & 554 & $2.64 \times 10^{-58}$ & $2.68 \times 10^{-55}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 60 & 123 & $2.82 \times 10^{-26}$ & $1.43 \times 10^{-23}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 58 & 117 & $7.13 \times 10^{-26}$ & $2.42 \times 10^{-23}$ \\ 
  GO:0003729 & mRNA binding & 48 & 100 & $1.05 \times 10^{-20}$ & $2.66 \times 10^{-18}$ \\ 
  GO:0003735 & structural constituent of ribosome & 43 & 103 & $8.30 \times 10^{-16}$ & $1.69 \times 10^{-13}$ \\ 
  GO:0101005 & ubiquitinyl hydrolase activity & 36 & 77 & $2.84 \times 10^{-15}$ & $4.12 \times 10^{-13}$ \\ 
  GO:0004843 & thiol-dependent ubiquitin-specific protease activity & 36 & 77 & $2.84 \times 10^{-15}$ & $4.12 \times 10^{-13}$ \\ 
  GO:0008134 & transcription factor binding & 53 & 156 & $1.02 \times 10^{-14}$ & $1.30 \times 10^{-12}$ \\ 
  GO:0044877 & protein-containing complex binding & 112 & 513 & $4.46 \times 10^{-14}$ & $5.04 \times 10^{-12}$ \\ 
  GO:0019843 & rRNA binding & 22 & 34 & $1.33 \times 10^{-13}$ & $1.35 \times 10^{-11}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment High kcoreness genes Molecular Function of genes above 90th centile of distribution]{\textbf{High kcoreness genes. Molecular Function - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Molecular function tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 180} 
\label{tab:ToppGENE GO: Molecular Function. kco 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 180}
\end{table}

\paragraph{Biological process}
Biological process table~\ref{tab:ToppGENE GO: Biological Process. kco 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 1141}.

Again mRNA and also viral processes and symbiotic showing that these are common and probably preserved across different organisms. Very low adjusted value and high coverage for terms.


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 15:02:29 2020
  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Biological Process & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0016071 & mRNA metabolic process & 142 & 260 & $3.73 \times 10^{-74}$ & $2.26 \times 10^{-70}$ \\ 
  GO:0009057 & macromolecule catabolic process & 166 & 443 & $4.75 \times 10^{-57}$ & $1.44 \times 10^{-53}$ \\ 
  GO:0044265 & cellular macromolecule catabolic process & 150 & 372 & $1.43 \times 10^{-55}$ & $2.88 \times 10^{-52}$ \\ 
  GO:0006402 & mRNA catabolic process & 100 & 180 & $2.06 \times 10^{-51}$ & $3.12 \times 10^{-48}$ \\ 
  GO:0006401 & RNA catabolic process & 100 & 186 & $1.34 \times 10^{-49}$ & $1.62 \times 10^{-46}$ \\ 
  GO:0044403 & symbiotic process & 140 & 364 & $2.98 \times 10^{-48}$ & $3.01 \times 10^{-45}$ \\ 
  GO:0044419 & interspecies interaction between organisms & 140 & 368 & $1.46 \times 10^{-47}$ & $1.27 \times 10^{-44}$ \\ 
  GO:0016032 & viral process & 135 & 348 & $8.48 \times 10^{-47}$ & $6.43 \times 10^{-44}$ \\ 
  GO:0034655 & nucleobase-containing compound catabolic process & 102 & 217 & $1.67 \times 10^{-43}$ & $1.13 \times 10^{-40}$ \\ 
  GO:0019439 & aromatic compound catabolic process & 102 & 232 & $3.61 \times 10^{-40}$ & $2.19 \times 10^{-37}$ \\ 
   \bottomrule
\end{tabular}
\end{adjustbox}

\caption[Gene ontology enrichment High kcoreness genes Biological Process of genes above 90th centile of distribution]{\textbf{High kcoreness genes. Biological Process - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Biological Process tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 1141} 
\label{tab:ToppGENE GO: Biological Process. kco 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 1141}
\end{table}

\paragraph{Cellular component}

Cellular component table~\ref{tab:ToppGENE GO: Cellular Component. kco 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 181} again ribonucleoprotein terms and splicasome terms as per eigenvector. Low FDR q values. Adherens junction which contains proteins at cell to cell junction including cadherins. 

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 15:05:27 2020
  \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:1990904 & ribonucleoprotein complex & 137 & 303 & $6.81 \times 10^{-58}$ & $5.48 \times 10^{-55}$ \\ 
  GO:0005925 & focal adhesion & 87 & 239 & $9.15 \times 10^{-27}$ & $3.68 \times 10^{-24}$ \\ 
  GO:0022626 & cytosolic ribosome & 53 & 97 & $1.86 \times 10^{-26}$ & $4.98 \times 10^{-24}$ \\ 
  GO:0000974 & Prp19 complex & 29 & 31 & $4.58 \times 10^{-26}$ & $8.40 \times 10^{-24}$ \\ 
  GO:0030055 & cell-substrate junction & 87 & 244 & $5.22 \times 10^{-26}$ & $8.40 \times 10^{-24}$ \\ 
  GO:0071013 & catalytic step 2 spliceosome & 27 & 29 & $3.59 \times 10^{-24}$ & $4.81 \times 10^{-22}$ \\ 
  GO:0005681 & spliceosomal complex & 32 & 40 & $4.61 \times 10^{-24}$ & $5.29 \times 10^{-22}$ \\ 
  GO:0097525 & spliceosomal snRNP complex & 29 & 35 & $1.04 \times 10^{-22}$ & $1.05 \times 10^{-20}$ \\ 
  GO:0005682 & U5 snRNP & 28 & 34 & $8.18 \times 10^{-22}$ & $7.31 \times 10^{-20}$ \\ 
  GO:0005912 & adherens junction & 94 & 314 & $1.07 \times 10^{-21}$ & $8.59 \times 10^{-20}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}

\caption[Gene ontology enrichment High kcoreness genes Cellular Component of genes above 90th centile of distribution]{\textbf{High kcoreness genes. Cellular Component - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Cellular Component tested gene set is 90th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 181} 
\label{tab:ToppGENE GO: Cellular Component. kco 90 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 181}
\end{table}

\paragraph{Others}
Mouse phenotype embroynic lethality

Human phenotype neoplasm and limb abnormalities. 

Diseases all very severe viral infections and neoplasms top neurological is number 7 C0338451  Frontotemporal dementia n in high coreness 44 n in annotation in PSP 119 p $3.75 \times 10^{-13}$ FDR q $3.87 \times 10^{-10}$  

Pub med integrin and beta catenin in neoplasms

Pathway strong enrichment gene expression
\clearpage
\subsubsection{Low kcoreness}

\paragraph{Cellular component}
Cellular component table~\ref{tab:ToppGENE GO: Cellular Component. kco 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 2}. Over representation of integral part of plasma membrane similar to low closeness the low kcore is also the periphery (although possibly a different periphery - ie core periphery rather than far away)


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 15:15:03 2020
 \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Cellular Component & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
GO:0031226 & intrinsic component of plasma membrane & 98 & 331 & $8.26 \times 10^{-9}$ & $5.35 \times 10^{-6}$ \\ 
  GO:0005887 & integral component of plasma membrane & 91 & 307 & $2.94 \times 10^{-8}$ & $9.51 \times 10^{-6}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
% \caption{ToppGene GO: Cellular Component. kco 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 2} 
\caption[Gene ontology enrichment Low kcoreness genes Cellular Component of genes above 90th centile of distribution]{\textbf{Low kcoreness genes. Cellular Component - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Cellular Component tested gene set is 10th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 2} 
\label{tab:ToppGENE GO: Cellular Component. kco 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 2}
\end{table}

\paragraph{Pathway}

Pathway shown in table~\ref{tab:ToppGENE Pathway. kco 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 8} is enriched for extra cellular matrix and matrix associated proteins and post translational modification of GPI. Good coverage (contains a lot of those found in the PSP by selecting this centrality measure also high for a low measure . 

\paragraph{Others}
Pub med pathway for regulation of habenula development 19906978  Brn3a and Nurr1 mediate a gene regulatory pathway for habenula development. n 22 n in PSP 44 p $8.18 \times 10^{-7}$ FDR q $2.36 \times 10^{-2}$
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 15:17:32 2020


Gene family is shown in table~\ref{tab:ToppGENE Gene Family. kco 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 5}. It is enriched for calcium voltage gate channel subunits, fibronectin and solute carriers.

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Oct  4 15:24:07 2020
 \begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\setlength{\extrarowheight}{2pt}
\begin{tabular}{@{}clllcl@{}}
  \toprule
  ID & Gene Family & $n$ & $n$ PSP & $p$ & $q$ FDR B H \\ 

  \midrule
752 & Solute carriers & 21 & 45 & $7.61 \times 10^{-7}$ & $1.58 \times 10^{-4}$ \\ 
  253 & Calcium voltage-gated channel subunits & 8 & 14 & $4.36 \times 10^{-4}$ & $4.53 \times 10^{-2}$ \\ 
  1210 & ATPase phospholipid transporting & 4 & 4 & $7.39 \times 10^{-4}$ & $4.77 \times 10^{-2}$ \\ 
  593 & \makecell{Fibronectin type III domain containing$|$I-set domain containing$|$\\Immunoglobulin like domain containing} & 15 & 42 & $1.11 \times 10^{-3}$ & $4.77 \times 10^{-2}$ \\ 
  594 & \makecell{Immunoglobulin like domain containing$|$Interleukin receptors$|$\\TIR domain containing} & 13 & 34 & $1.15 \times 10^{-3}$ & $4.77 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption[Gene ontology enrichment Low kcoreness genes Gene Family of genes above 90th centile of distribution]{\textbf{Low kcoreness genes. Gene Family - Gene ontology enrichment analysis} using the ToppGene web client and all PSP genes as the background set.  Gene Family tested gene set is 10th centile of degree distribution.  p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; $n$= $n$ genes annotated in test group; $n$ PSP= n genes annotated in PSP. $n$ ontology terms significantly (q FDR$>=0.05$) enriched for this centrality measure and ontology category (e.g. MF,CC,BP): 5} 
% \caption{ToppGene Gene Family. kco 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 5} 
\label{tab:ToppGENE Gene Family. kco 10 centile cwpsp.txtp = p value; q FDR B H = q adjusted significance level False Discovery Rate using Benjamini and Hochberg adjustment; n= n genes annotated in test group; n PSP= n genes annotated in PSP. n significant in category 5}
\end{table}
