\chapter{Supplementary tables}



% SIGNIFICANT MAGMA CTG GWAS

\section{Significant genes in GWAS}
Significant genes in CTG Intelligence Replication sample using MAGMA from summary statistics table~\ref{tab:Significant genes in ctg MAGMA GWAS}

Significant genes for education replication sample EA2 table~\ref{tab:Significant genes in EA2 MAGMA GWAS}




Significant genes for intelligence discovery (UKBB) sample table~\ref{tab:Significant genes in UKBB int MAGMA GWAS}

Significant genes for educational attainment discovery (UKBB) sample table~\ref{tab:Significant genes in UKBB edu MAGMA GWAS}


\begin{table}[ht]
\centering
\begin{tabular}{rlrrrrl}
  \hline
GENE & SYMBOL & CHR & NSNPS & ZSTAT & P & synaptic \\ 
  \hline
1434 & CSE1L &  20 & 176 & 5.96 & $1.26 \times 10^{-9}$ & TRUE \\ 
  2309 & FOXO3 &   6 & 315 & 5.96 & $1.26 \times 10^{-9}$ & FALSE \\ 
  60412 & EXOC4 &   7 & 2184 & 5.88 & $1.99 \times 10^{-9}$ & TRUE \\ 
  164684 & WBP2NL &  22 &  94 & 5.83 & $2.83 \times 10^{-9}$ & FALSE \\ 
  55964 & SEPT3 &  22 &  69 & 5.75 & $4.35 \times 10^{-9}$ & TRUE \\ 
  1565 & CYP2D6 &  22 &  36 & 5.73 & $5.11 \times 10^{-9}$ & FALSE \\ 
  27332 & ZNF638 &   2 & 657 & 5.72 & $5.22 \times 10^{-9}$ & FALSE \\ 
  4668 & NAGA &  22 &  34 & 5.64 & $8.41 \times 10^{-9}$ & FALSE \\ 
  320 & APBA1 &   9 & 777 & 5.61 & $1.03 \times 10^{-8}$ & FALSE \\ 
  6780 & STAU1 &  20 & 231 & 5.55 & $1.43 \times 10^{-8}$ & FALSE \\ 
  8816 & DCAF5 &  14 & 258 & 5.53 & $1.60 \times 10^{-8}$ & FALSE \\ 
  6138 & RPL15 &   3 &  19 & 5.40 & $3.35 \times 10^{-8}$ & TRUE \\ 
  79631 & EFTUD1 &  15 & 570 & 5.39 & $3.53 \times 10^{-8}$ & FALSE \\ 
  221037 & JMJD1C &  10 & 997 & 5.35 & $4.39 \times 10^{-8}$ & FALSE \\ 
  23109 & DDN &  12 &   8 & 5.35 & $4.41 \times 10^{-8}$ & TRUE \\ 
  85358 & SHANK3 &  22 & 199 & 5.31 & $5.49 \times 10^{-8}$ & TRUE \\ 
  55628 & ZNF407 &  18 & 1447 & 5.31 & $5.63 \times 10^{-8}$ & FALSE \\ 
  25970 & SH2B1 &  16 &  19 & 5.28 & $6.48 \times 10^{-8}$ & FALSE \\ 
  487 & ATP2A1 &  16 &  47 & 5.23 & $8.37 \times 10^{-8}$ & TRUE \\ 
  11273 & ATXN2L &  16 &  36 & 5.17 & $1.17 \times 10^{-7}$ & TRUE \\ 
  1630 & DCC &  18 & 6036 & 5.13 & $1.41 \times 10^{-7}$ & FALSE \\ 
  659 & BMPR2 &   2 & 442 & 5.07 & $1.94 \times 10^{-7}$ & FALSE \\ 
  2870 & GRK6 &   5 &  39 & 5.07 & $2.03 \times 10^{-7}$ & TRUE \\ 
  64116 & SLC39A8 &   4 & 358 & 5.06 & $2.07 \times 10^{-7}$ & FALSE \\ 
  28512 & NKIRAS1 &   3 &  73 & 5.04 & $2.29 \times 10^{-7}$ & TRUE \\ 
  4700 & NDUFA6 &  22 &  16 & 5.04 & $2.32 \times 10^{-7}$ & TRUE \\ 
  5137 & PDE1C &   7 & 2614 & 5.02 & $2.52 \times 10^{-7}$ & FALSE \\ 
  91445 & RNF185 &  22 & 158 & 4.98 & $3.26 \times 10^{-7}$ & FALSE \\ 
  6487 & ST3GAL3 &   1 & 652 & 4.83 & $6.83 \times 10^{-7}$ & FALSE \\ 
  4733 & DRG1 &  22 &  99 & 4.83 & $6.91 \times 10^{-7}$ & TRUE \\ 
  63891 & RNF123 &   3 &  71 & 4.82 & $7.19 \times 10^{-7}$ & FALSE \\ 
  3061 & HCRTR1 &   1 &  34 & 4.82 & $7.35 \times 10^{-7}$ & FALSE \\ 
  10564 & ARFGEF2 &  20 & 406 & 4.80 & $7.83 \times 10^{-7}$ & TRUE \\ 
  8729 & GBF1 &  10 & 269 & 4.79 & $8.31 \times 10^{-7}$ & FALSE \\ 
  7284 & TUFM &  16 &   7 & 4.79 & $8.49 \times 10^{-7}$ & TRUE \\ 
  553115 & PEF1 &   1 &  30 & 4.77 & $9.17 \times 10^{-7}$ & TRUE \\ 
  728741 & NPIPB6 &  16 &  13 & 4.74 & $1.06 \times 10^{-6}$ & FALSE \\ 
  9807 & IP6K1 &   3 & 129 & 4.73 & $1.11 \times 10^{-6}$ & FALSE \\ 
  285525 & YIPF7 &   4 & 155 & 4.72 & $1.16 \times 10^{-6}$ & FALSE \\ 
  6942 & TCF20 &  22 & 256 & 4.71 & $1.26 \times 10^{-6}$ & FALSE \\ 
  8631 & SKAP1 &  17 & 884 & 4.70 & $1.30 \times 10^{-6}$ & FALSE \\ 
  112869 & CCDC101 &  16 & 114 & 4.66 & $1.59 \times 10^{-6}$ & FALSE \\ 
  55843 & ARHGAP15 &   2 & 1943 & 4.65 & $1.66 \times 10^{-6}$ & FALSE \\ 
  1307 & COL16A1 &   1 & 158 & 4.63 & $1.80 \times 10^{-6}$ & FALSE \\ 
  8411 & EEA1 &  12 & 475 & 4.61 & $2.03 \times 10^{-6}$ & TRUE \\ 
  3985 & LIMK2 &  22 & 173 & 4.60 & $2.15 \times 10^{-6}$ & FALSE \\ 
  11078 & TRIOBP &  22 & 322 & 4.57 & $2.49 \times 10^{-6}$ & FALSE \\ 
   \hline
\end{tabular}
\caption{Significant genes in ctg MAGMA GWAS \cite{sniekers2017genome}. ? change last column to superscript or bold if synaptic} 
\label{tab:Significant genes in ctg MAGMA GWAS}
\end{table}



%% SIGNIFICANT GENES ALL EA2


\begin{table}[ht]
\centering
\begin{adjustbox}{totalheight=10in}


\begin{tabular}{rllrrrl}
  \hline
GENE & SYMBOL & CHR & NSNPS & ZSTAT & P & synaptic \\ 
  \hline
55534 & MAML3 & 4 & 1058 & 8.48 & $1.11 \times 10^{-17}$ & FALSE \\ 
  3899 & AFF3 & 2 & 1372 & 8.06 & $3.78 \times 10^{-16}$ & FALSE \\ 
  79012 & CAMKV & 3 &  17 & 7.16 & $3.95 \times 10^{-13}$ & TRUE \\ 
  84315 & MON1A & 3 &  27 & 7.10 & $6.31 \times 10^{-13}$ & FALSE \\ 
  8927 & BSN & 3 & 148 & 6.98 & $1.45 \times 10^{-12}$ & TRUE \\ 
  6405 & SEMA3F & 3 &  72 & 6.95 & $1.82 \times 10^{-12}$ & FALSE \\ 
  63891 & RNF123 & 3 &  49 & 6.88 & $2.94 \times 10^{-12}$ & FALSE \\ 
  10181 & RBM5 & 3 &  30 & 6.86 & $3.45 \times 10^{-12}$ & FALSE \\ 
  1605 & DAG1 & 3 & 109 & 6.82 & $4.69 \times 10^{-12}$ & FALSE \\ 
  10991 & SLC38A3 & 3 &  21 & 6.74 & $7.75 \times 10^{-12}$ & FALSE \\ 
  10180 & RBM6 & 3 & 252 & 6.56 & $2.63 \times 10^{-11}$ & FALSE \\ 
  84276 & NICN1 & 3 &   8 & 6.56 & $2.73 \times 10^{-11}$ & FALSE \\ 
  6988 & TCTA & 3 &   7 & 6.55 & $2.89 \times 10^{-11}$ & FALSE \\ 
  4486 & MST1R & 3 &  23 & 6.53 & $3.34 \times 10^{-11}$ & FALSE \\ 
  327 & APEH & 3 &   8 & 6.44 & $5.86 \times 10^{-11}$ & TRUE \\ 
  4485 & MST1 & 3 &   8 & 6.44 & $5.97 \times 10^{-11}$ & FALSE \\ 
  9807 & IP6K1 & 3 &  88 & 6.44 & $6.02 \times 10^{-11}$ & FALSE \\ 
  2065 & ERBB3 & 12 &  32 & 6.40 & $7.75 \times 10^{-11}$ & FALSE \\ 
  387 & RHOA & 3 &  90 & 6.38 & $8.67 \times 10^{-11}$ & TRUE \\ 
  275 & AMT & 3 &   9 & 6.34 & $1.15 \times 10^{-10}$ & FALSE \\ 
  10293 & TRAIP & 3 &  40 & 6.30 & $1.54 \times 10^{-10}$ & FALSE \\ 
  2876 & GPX1 & 3 &   2 & 6.25 & $2.01 \times 10^{-10}$ & TRUE \\ 
  1161 & ERCC8 & 5 & 179 & 6.12 & $4.54 \times 10^{-10}$ & FALSE \\ 
  389118 & CDHR4 & 3 &  16 & 6.11 & $5.13 \times 10^{-10}$ & FALSE \\ 
  57605 & PITPNM2 & 12 & 167 & 6.06 & $6.94 \times 10^{-10}$ & TRUE \\ 
  9682 & KDM4A & 1 &  93 & 6.00 & $9.60 \times 10^{-10}$ & FALSE \\ 
  53335 & BCL11A & 2 & 171 & 5.94 & $1.38 \times 10^{-9}$ & FALSE \\ 
  91574 & C12orf65 & 12 &  66 & 5.91 & $1.73 \times 10^{-9}$ & FALSE \\ 
  2779 & GNAT1 & 3 &   3 & 5.87 & $2.14 \times 10^{-9}$ & FALSE \\ 
  92017 & SNX29 & 16 & 2796 & 5.87 & $2.17 \times 10^{-9}$ & FALSE \\ 
  10198 & MPHOSPH9 & 12 & 199 & 5.84 & $2.55 \times 10^{-9}$ & FALSE \\ 
  80031 & SEMA6D & 15 & 1986 & 5.84 & $2.56 \times 10^{-9}$ & FALSE \\ 
  91942 & NDUFAF2 & 5 & 500 & 5.82 & $2.93 \times 10^{-9}$ & FALSE \\ 
  8729 & GBF1 & 10 & 208 & 5.81 & $3.17 \times 10^{-9}$ & FALSE \\ 
  164832 & LONRF2 & 2 & 198 & 5.80 & $3.38 \times 10^{-9}$ & FALSE \\ 
  55147 & RBM23 & 14 &  47 & 5.78 & $3.68 \times 10^{-9}$ & FALSE \\ 
  10419 & PRMT5 & 14 &  16 & 5.77 & $4.07 \times 10^{-9}$ & TRUE \\ 
  5792 & PTPRF & 1 & 260 & 5.67 & $7.09 \times 10^{-9}$ & TRUE \\ 
  5662 & PSD & 10 &  23 & 5.67 & $7.20 \times 10^{-9}$ & TRUE \\ 
  55729 & ATF7IP & 12 & 374 & 5.60 & $1.08 \times 10^{-8}$ & TRUE \\ 
  30819 & KCNIP2 & 10 &  28 & 5.57 & $1.28 \times 10^{-8}$ & TRUE \\ 
  54930 & HAUS4 & 14 &  20 & 5.53 & $1.64 \times 10^{-8}$ & FALSE \\ 
  4208 & MEF2C & 5 & 339 & 5.44 & $2.63 \times 10^{-8}$ & TRUE \\ 
  7003 & TEAD1 & 11 & 474 & 5.43 & $2.75 \times 10^{-8}$ & FALSE \\ 
  4864 & NPC1 & 18 & 141 & 5.42 & $2.91 \times 10^{-8}$ & FALSE \\ 
  257194 & NEGR1 & 1 & 1898 & 5.40 & $3.34 \times 10^{-8}$ & TRUE \\ 
  79591 & C10orf76 & 10 & 301 & 5.37 & $3.90 \times 10^{-8}$ & FALSE \\ 
  8099 & CDK2AP1 & 12 &  32 & 5.36 & $4.16 \times 10^{-8}$ & FALSE \\ 
  84962 & AJUBA & 14 &  18 & 5.36 & $4.20 \times 10^{-8}$ & FALSE \\ 
  26575 & RGS17 & 6 & 411 & 5.35 & $4.31 \times 10^{-8}$ & TRUE \\ 
  1946 & EFNA5 & 5 & 758 & 5.35 & $4.52 \times 10^{-8}$ & FALSE \\ 
  29919 & C18orf8 & 18 &  53 & 5.33 & $4.85 \times 10^{-8}$ & TRUE \\ 
  2823 & GPM6A & 4 & 1150 & 5.31 & $5.35 \times 10^{-8}$ & TRUE \\ 
  51329 & ARL6IP4 & 12 &   1 & 5.31 & $5.56 \times 10^{-8}$ & FALSE \\ 
  79993 & ELOVL7 & 5 & 278 & 5.30 & $5.68 \times 10^{-8}$ & FALSE \\ 
  7473 & WNT3 & 17 &  92 & 5.26 & $7.17 \times 10^{-8}$ & FALSE \\ 
  56987 & BBX & 3 & 615 & 5.25 & $7.64 \times 10^{-8}$ & FALSE \\ 
  1803 & DPP4 & 2 & 208 & 5.22 & $8.76 \times 10^{-8}$ & FALSE \\ 
  6487 & ST3GAL3 & 1 & 522 & 5.21 & $9.28 \times 10^{-8}$ & FALSE \\ 
  60686 & C14orf93 & 14 &  30 & 5.12 & $1.52 \times 10^{-7}$ & FALSE \\ 
  79676 & OGFOD2 & 12 &   5 & 5.09 & $1.77 \times 10^{-7}$ & FALSE \\ 
  55790 & CSGALNACT1 & 8 & 1172 & 5.09 & $1.78 \times 10^{-7}$ & FALSE \\ 
  221037 & JMJD1C & 10 & 666 & 5.08 & $1.91 \times 10^{-7}$ & FALSE \\ 
  6310 & ATXN1 & 6 & 1548 & 5.08 & $1.94 \times 10^{-7}$ & FALSE \\ 
  1394 & CRHR1 & 17 & 989 & 5.07 & $2.01 \times 10^{-7}$ & FALSE \\ 
  4781 & NFIB & 9 & 784 & 5.06 & $2.06 \times 10^{-7}$ & FALSE \\ 
  5144 & PDE4D & 5 & 3995 & 5.06 & $2.14 \times 10^{-7}$ & TRUE \\ 
  10724 & MGEA5 & 10 &  48 & 5.04 & $2.28 \times 10^{-7}$ & TRUE \\ 
  27253 & PCDH17 & 13 & 210 & 5.04 & $2.37 \times 10^{-7}$ & FALSE \\ 
  3615 & IMPDH2 & 3 &   2 & 5.01 & $2.68 \times 10^{-7}$ & FALSE \\ 
  7318 & UBA7 & 3 &   7 & 4.98 & $3.20 \times 10^{-7}$ & FALSE \\ 
  162540 & SPPL2C & 17 &  17 & 4.98 & $3.23 \times 10^{-7}$ & FALSE \\ 
  221035 & REEP3 & 10 & 258 & 4.98 & $3.23 \times 10^{-7}$ & FALSE \\ 
  6821 & SUOX & 12 &  10 & 4.97 & $3.34 \times 10^{-7}$ & FALSE \\ 
  8555 & CDC14B & 9 & 102 & 4.95 & $3.73 \times 10^{-7}$ & FALSE \\ 
  5309 & PITX3 & 10 &  19 & 4.88 & $5.35 \times 10^{-7}$ & FALSE \\ 
  51594 & NBAS & 2 & 941 & 4.84 & $6.64 \times 10^{-7}$ & FALSE \\ 
  4744 & NEFH & 22 &  31 & 4.81 & $7.47 \times 10^{-7}$ & TRUE \\ 
  121665 & SPPL3 & 12 & 438 & 4.81 & $7.62 \times 10^{-7}$ & FALSE \\ 
  401036 & ASB18 & 2 & 264 & 4.80 & $7.76 \times 10^{-7}$ & FALSE \\ 
  55843 & ARHGAP15 & 2 & 1540 & 4.80 & $7.82 \times 10^{-7}$ & FALSE \\ 
  25825 & BACE2 & 21 & 311 & 4.72 & $1.16 \times 10^{-6}$ & FALSE \\ 
  473 & RERE & 1 & 715 & 4.72 & $1.17 \times 10^{-6}$ & FALSE \\ 
  8706 & B3GALNT1 & 3 &  41 & 4.72 & $1.20 \times 10^{-6}$ & FALSE \\ 
  54790 & TET2 & 4 & 322 & 4.70 & $1.29 \times 10^{-6}$ & FALSE \\ 
  7375 & USP4 & 3 &  65 & 4.69 & $1.37 \times 10^{-6}$ & FALSE \\ 
  60412 & EXOC4 & 7 & 1394 & 4.69 & $1.37 \times 10^{-6}$ & TRUE \\ 
  652991 & SKOR2 & 18 &  92 & 4.68 & $1.43 \times 10^{-6}$ & FALSE \\ 
  4137 & MAPT & 17 & 696 & 4.68 & $1.47 \times 10^{-6}$ & TRUE \\ 
  246744 & STH & 17 &   1 & 4.67 & $1.47 \times 10^{-6}$ & FALSE \\ 
  5869 & RAB5B & 12 &  44 & 4.66 & $1.55 \times 10^{-6}$ & TRUE \\ 
  345222 & MSANTD1 & 4 &  94 & 4.65 & $1.65 \times 10^{-6}$ & FALSE \\ 
  55328 & RNLS & 10 & 803 & 4.64 & $1.73 \times 10^{-6}$ & FALSE \\ 
  9859 & CEP170 & 1 & 160 & 4.63 & $1.82 \times 10^{-6}$ & TRUE \\ 
  9486 & CHST10 & 2 &  90 & 4.62 & $1.96 \times 10^{-6}$ & FALSE \\ 
  4905 & NSF & 17 &  66 & 4.61 & $1.99 \times 10^{-6}$ & TRUE \\ 
  29119 & CTNNA3 & 10 & 7101 & 4.61 & $2.06 \times 10^{-6}$ & FALSE \\ 
  11122 & PTPRT & 20 & 3517 & 4.58 & $2.28 \times 10^{-6}$ & TRUE \\ 
  643155 & SMIM15 & 5 &   6 & 4.54 & $2.75 \times 10^{-6}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Significant genes in EA2 MAGMA GWAS} 
\label{tab:Significant genes in EA2 MAGMA GWAS}
\end{table}




%%% SIGNIFICANT GENES UKBB INT

% latex table generated in R 3.6.2 by xtable 1.8-4 package
% Thu Jan  2 13:35:37 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{totalheight=10in}
\begin{tabular}{rllrrrl}
  \hline
GENE & SYMBOL & CHR & NSNPS & ZSTAT & P & synaptic \\ 
  \hline
63891 & RNF123 & 3 & 100 & 9.01 & $1.06 \times 10^{-19}$ & FALSE \\ 
  9807 & IP6K1 & 3 & 171 & 8.95 & $1.86 \times 10^{-19}$ & FALSE \\ 
  10293 & TRAIP & 3 &  65 & 7.74 & $4.96 \times 10^{-15}$ & FALSE \\ 
  8341 & HIST1H2BN & 6 &  67 & 7.57 & $1.85 \times 10^{-14}$ & TRUE \\ 
  6780 & STAU1 & 20 & 274 & 7.37 & $8.37 \times 10^{-14}$ & FALSE \\ 
  5934 & RBL2 & 16 & 254 & 7.25 & $2.15 \times 10^{-13}$ & FALSE \\ 
  11118 & BTN3A2 & 6 & 126 & 7.19 & $3.27 \times 10^{-13}$ & FALSE \\ 
  389118 & CDHR4 & 3 &  30 & 7.17 & $3.74 \times 10^{-13}$ & FALSE \\ 
  10181 & RBM5 & 3 &  56 & 7.14 & $4.74 \times 10^{-13}$ & FALSE \\ 
  11120 & BTN2A1 & 6 & 116 & 7.12 & $5.30 \times 10^{-13}$ & FALSE \\ 
  1434 & CSE1L & 20 & 195 & 7.10 & $6.36 \times 10^{-13}$ & TRUE \\ 
  1630 & DCC & 18 & 7236 & 7.04 & $9.41 \times 10^{-13}$ & FALSE \\ 
  4208 & MEF2C & 5 & 603 & 7.04 & $9.75 \times 10^{-13}$ & TRUE \\ 
  10180 & RBM6 & 3 & 421 & 7.02 & $1.09 \times 10^{-12}$ & FALSE \\ 
  8927 & BSN & 3 & 312 & 7.00 & $1.29 \times 10^{-12}$ & TRUE \\ 
  257194 & NEGR1 & 1 & 3314 & 6.89 & $2.73 \times 10^{-12}$ & TRUE \\ 
  4248 & MGAT3 & 22 & 165 & 6.89 & $2.84 \times 10^{-12}$ & FALSE \\ 
  487 & ATP2A1 & 16 &  94 & 6.86 & $3.56 \times 10^{-12}$ & TRUE \\ 
  25970 & SH2B1 & 16 &  41 & 6.84 & $3.92 \times 10^{-12}$ & FALSE \\ 
  27044 & SND1 & 7 & 1381 & 6.81 & $5.01 \times 10^{-12}$ & TRUE \\ 
  11273 & ATXN2L & 16 &  60 & 6.78 & $5.95 \times 10^{-12}$ & TRUE \\ 
  8085 & KMT2D & 12 &  96 & 6.78 & $6.20 \times 10^{-12}$ & TRUE \\ 
  6799 & SULT1A2 & 16 &  59 & 6.75 & $7.18 \times 10^{-12}$ & FALSE \\ 
  79012 & CAMKV & 3 &  30 & 6.74 & $7.94 \times 10^{-12}$ & TRUE \\ 
  83698 & CALN1 & 7 & 3317 & 6.65 & $1.49 \times 10^{-11}$ & FALSE \\ 
  6817 & SULT1A1 & 16 & 101 & 6.56 & $2.61 \times 10^{-11}$ & FALSE \\ 
  7318 & UBA7 & 3 &  25 & 6.49 & $4.20 \times 10^{-11}$ & FALSE \\ 
  84315 & MON1A & 3 &  40 & 6.49 & $4.23 \times 10^{-11}$ & FALSE \\ 
  81697 & OR2B2 & 6 &   5 & 6.48 & $4.46 \times 10^{-11}$ & FALSE \\ 
  10564 & ARFGEF2 & 20 & 440 & 6.47 & $4.84 \times 10^{-11}$ & TRUE \\ 
  275 & AMT & 3 &  17 & 6.46 & $5.39 \times 10^{-11}$ & FALSE \\ 
  4486 & MST1R & 3 &  43 & 6.45 & $5.67 \times 10^{-11}$ & FALSE \\ 
  23109 & DDN & 12 &  11 & 6.43 & $6.38 \times 10^{-11}$ & TRUE \\ 
  5188 & GATB & 4 & 375 & 6.43 & $6.50 \times 10^{-11}$ & FALSE \\ 
  100132074 & FOXO6 & 1 &  85 & 6.37 & $9.34 \times 10^{-11}$ & FALSE \\ 
  6405 & SEMA3F & 3 & 119 & 6.37 & $9.52 \times 10^{-11}$ & FALSE \\ 
  1201 & CLN3 & 16 &  41 & 6.36 & $9.95 \times 10^{-11}$ & FALSE \\ 
  112869 & CCDC101 & 16 & 163 & 6.33 & $1.21 \times 10^{-10}$ & FALSE \\ 
  1605 & DAG1 & 3 & 219 & 6.32 & $1.33 \times 10^{-10}$ & FALSE \\ 
  27327 & TNRC6A & 16 & 380 & 6.28 & $1.70 \times 10^{-10}$ & FALSE \\ 
  28512 & NKIRAS1 & 3 & 104 & 6.28 & $1.71 \times 10^{-10}$ & TRUE \\ 
  5571 & PRKAG1 & 12 &  40 & 6.28 & $1.74 \times 10^{-10}$ & FALSE \\ 
  696 & BTN1A1 & 6 &  48 & 6.26 & $1.91 \times 10^{-10}$ & FALSE \\ 
  64400 & AKTIP & 16 &  62 & 6.25 & $2.08 \times 10^{-10}$ & FALSE \\ 
  164832 & LONRF2 & 2 & 306 & 6.24 & $2.15 \times 10^{-10}$ & FALSE \\ 
  6138 & RPL15 & 3 &  36 & 6.20 & $2.90 \times 10^{-10}$ & TRUE \\ 
  387032 & ZKSCAN4 & 6 &  58 & 6.18 & $3.14 \times 10^{-10}$ & FALSE \\ 
  55911 & APOBR & 16 &  20 & 6.17 & $3.42 \times 10^{-10}$ & FALSE \\ 
  5078 & PAX4 & 7 &  27 & 6.16 & $3.69 \times 10^{-10}$ & FALSE \\ 
  7284 & TUFM & 16 &  11 & 6.14 & $4.18 \times 10^{-10}$ & TRUE \\ 
  137868 & SGCZ & 8 & 10096 & 6.13 & $4.27 \times 10^{-10}$ & FALSE \\ 
  79874 & RABEP2 & 16 &  65 & 6.12 & $4.75 \times 10^{-10}$ & FALSE \\ 
  11074 & TRIM31 & 6 & 128 & 6.10 & $5.34 \times 10^{-10}$ & FALSE \\ 
  553115 & PEF1 & 1 &  41 & 6.08 & $6.09 \times 10^{-10}$ & TRUE \\ 
  4485 & MST1 & 3 &   9 & 6.08 & $6.15 \times 10^{-10}$ & FALSE \\ 
  387 & RHOA & 3 & 186 & 6.07 & $6.33 \times 10^{-10}$ & TRUE \\ 
  1307 & COL16A1 & 1 & 214 & 6.05 & $7.46 \times 10^{-10}$ & FALSE \\ 
  9753 & ZSCAN12 & 6 & 116 & 6.01 & $9.21 \times 10^{-10}$ & FALSE \\ 
  7745 & ZKSCAN8 & 6 &  86 & 5.96 & $1.30 \times 10^{-9}$ & FALSE \\ 
  64943 & NT5DC2 & 3 &  58 & 5.93 & $1.51 \times 10^{-9}$ & TRUE \\ 
  343263 & MYBPHL & 1 &  84 & 5.92 & $1.57 \times 10^{-9}$ & FALSE \\ 
  327 & APEH & 3 &  18 & 5.91 & $1.71 \times 10^{-9}$ & TRUE \\ 
  575 & ADGRB1 & 8 & 404 & 5.91 & $1.74 \times 10^{-9}$ & TRUE \\ 
  3061 & HCRTR1 & 1 &  40 & 5.88 & $2.09 \times 10^{-9}$ & FALSE \\ 
  8729 & GBF1 & 10 & 377 & 5.87 & $2.23 \times 10^{-9}$ & FALSE \\ 
  84547 & PGBD1 & 6 & 115 & 5.85 & $2.42 \times 10^{-9}$ & FALSE \\ 
  79692 & ZNF322 & 6 & 130 & 5.85 & $2.47 \times 10^{-9}$ & FALSE \\ 
  79571 & GCC1 & 7 &  22 & 5.80 & $3.25 \times 10^{-9}$ & FALSE \\ 
  8368 & HIST1H4L & 6 &   2 & 5.79 & $3.47 \times 10^{-9}$ & FALSE \\ 
  10454 & TAB1 & 22 & 133 & 5.78 & $3.73 \times 10^{-9}$ & FALSE \\ 
  30819 & KCNIP2 & 10 &  51 & 5.78 & $3.76 \times 10^{-9}$ & TRUE \\ 
  64288 & ZSCAN31 & 6 & 168 & 5.77 & $3.88 \times 10^{-9}$ & FALSE \\ 
  10786 & SLC17A3 & 6 & 152 & 5.72 & $5.42 \times 10^{-9}$ & FALSE \\ 
  80736 & SLC44A4 & 6 &  89 & 5.71 & $5.63 \times 10^{-9}$ & FALSE \\ 
  26053 & AUTS2 & 7 & 4444 & 5.68 & $6.54 \times 10^{-9}$ & FALSE \\ 
  7746 & ZSCAN9 & 6 &  56 & 5.68 & $6.66 \times 10^{-9}$ & FALSE \\ 
  2309 & FOXO3 & 6 & 456 & 5.67 & $7.02 \times 10^{-9}$ & FALSE \\ 
  80317 & ZKSCAN3 & 6 & 105 & 5.67 & $7.05 \times 10^{-9}$ & FALSE \\ 
  64151 & NCAPG & 4 & 105 & 5.67 & $7.06 \times 10^{-9}$ & FALSE \\ 
  7741 & ZSCAN26 & 6 &  37 & 5.67 & $7.18 \times 10^{-9}$ & FALSE \\ 
  29999 & FSCN3 & 7 &  30 & 5.67 & $7.35 \times 10^{-9}$ & FALSE \\ 
  23166 & STAB1 & 3 & 117 & 5.66 & $7.39 \times 10^{-9}$ & FALSE \\ 
  168850 & ZNF800 & 7 &  97 & 5.60 & $1.05 \times 10^{-8}$ & FALSE \\ 
  55193 & PBRM1 & 3 & 607 & 5.59 & $1.17 \times 10^{-8}$ & FALSE \\ 
  26354 & GNL3 & 3 &  36 & 5.58 & $1.17 \times 10^{-8}$ & FALSE \\ 
  8340 & HIST1H2BL & 6 &   2 & 5.52 & $1.69 \times 10^{-8}$ & TRUE \\ 
  2903 & GRIN2A & 16 & 2864 & 5.52 & $1.70 \times 10^{-8}$ & TRUE \\ 
  9975 & NR1D2 & 3 & 160 & 5.51 & $1.78 \times 10^{-8}$ & FALSE \\ 
  7840 & ALMS1 & 2 & 921 & 5.47 & $2.27 \times 10^{-8}$ & TRUE \\ 
  3017 & HIST1H2BD & 6 &  59 & 5.46 & $2.45 \times 10^{-8}$ & FALSE \\ 
  54790 & TET2 & 4 & 563 & 5.45 & $2.55 \times 10^{-8}$ & FALSE \\ 
  80034 & CSRNP3 & 2 & 794 & 5.42 & $3.03 \times 10^{-8}$ & FALSE \\ 
  5201 & PFDN1 & 5 & 186 & 5.41 & $3.10 \times 10^{-8}$ & FALSE \\ 
  3009 & HIST1H1B & 6 &  14 & 5.40 & $3.31 \times 10^{-8}$ & TRUE \\ 
  246778 & IL27 & 16 &  22 & 5.40 & $3.36 \times 10^{-8}$ & FALSE \\ 
  7003 & TEAD1 & 11 & 1046 & 5.39 & $3.50 \times 10^{-8}$ & FALSE \\ 
  134728 & IRAK1BP1 & 6 & 430 & 5.39 & $3.60 \times 10^{-8}$ & FALSE \\ 
  55023 & PHIP & 6 & 437 & 5.38 & $3.80 \times 10^{-8}$ & FALSE \\ 
  80345 & ZSCAN16 & 6 &  31 & 5.36 & $4.25 \times 10^{-8}$ & FALSE \\ 
  2774 & GNAL & 18 & 1131 & 5.34 & $4.53 \times 10^{-8}$ & TRUE \\ 
  11122 & PTPRT & 20 & 5717 & 5.33 & $4.78 \times 10^{-8}$ & TRUE \\ 
  254251 & LCORL & 4 & 672 & 5.33 & $4.81 \times 10^{-8}$ & FALSE \\ 
  8895 & CPNE3 & 8 & 231 & 5.30 & $5.73 \times 10^{-8}$ & TRUE \\ 
  55830 & GLT8D1 & 3 &  35 & 5.30 & $5.84 \times 10^{-8}$ & FALSE \\ 
  729830 & FAM160A1 & 4 & 1040 & 5.27 & $6.93 \times 10^{-8}$ & FALSE \\ 
  1499 & CTNNB1 & 3 & 143 & 5.27 & $7.00 \times 10^{-8}$ & TRUE \\ 
  22989 & MYH15 & 3 & 645 & 5.25 & $7.40 \times 10^{-8}$ & FALSE \\ 
  3107 & HLA-C & 6 &  96 & 5.25 & $7.46 \times 10^{-8}$ & FALSE \\ 
  5253 & PHF2 & 9 & 682 & 5.25 & $7.53 \times 10^{-8}$ & FALSE \\ 
  2876 & GPX1 & 3 &   5 & 5.25 & $7.79 \times 10^{-8}$ & TRUE \\ 
  116154 & PHACTR3 & 20 & 1561 & 5.22 & $8.91 \times 10^{-8}$ & FALSE \\ 
  57512 & GPR158 & 10 & 2497 & 5.22 & $8.91 \times 10^{-8}$ & FALSE \\ 
  3992 & FADS1 & 11 &  38 & 5.21 & $9.23 \times 10^{-8}$ & TRUE \\ 
  60412 & EXOC4 & 7 & 3112 & 5.21 & $9.24 \times 10^{-8}$ & TRUE \\ 
  10322 & SMYD5 & 2 &  49 & 5.17 & $1.16 \times 10^{-7}$ & FALSE \\ 
  10475 & TRIM38 & 6 & 123 & 5.17 & $1.20 \times 10^{-7}$ & FALSE \\ 
  8352 & HIST1H3C & 6 &   3 & 5.16 & $1.24 \times 10^{-7}$ & FALSE \\ 
  10473 & HMGN4 & 6 &  33 & 5.15 & $1.28 \times 10^{-7}$ & FALSE \\ 
  23312 & DMXL2 & 15 & 642 & 5.15 & $1.31 \times 10^{-7}$ & TRUE \\ 
  7718 & ZNF165 & 6 &  43 & 5.13 & $1.48 \times 10^{-7}$ & FALSE \\ 
  79591 & C10orf76 & 10 & 577 & 5.12 & $1.56 \times 10^{-7}$ & FALSE \\ 
  7726 & TRIM26 & 6 & 267 & 5.11 & $1.63 \times 10^{-7}$ & FALSE \\ 
  3899 & AFF3 & 2 & 2283 & 5.10 & $1.71 \times 10^{-7}$ & FALSE \\ 
  6787 & NEK4 & 3 & 246 & 5.10 & $1.71 \times 10^{-7}$ & FALSE \\ 
  5334 & PLCL1 & 2 & 1111 & 5.09 & $1.74 \times 10^{-7}$ & FALSE \\ 
  10274 & STAG1 & 3 & 1246 & 5.07 & $1.95 \times 10^{-7}$ & FALSE \\ 
  5309 & PITX3 & 10 &  33 & 5.07 & $1.95 \times 10^{-7}$ & FALSE \\ 
  3135 & HLA-G & 6 &  81 & 5.07 & $1.98 \times 10^{-7}$ & FALSE \\ 
  79154 & DHRS11 & 17 &  32 & 5.05 & $2.17 \times 10^{-7}$ & FALSE \\ 
  8359 & HIST1H4A & 6 &   2 & 5.05 & $2.26 \times 10^{-7}$ & TRUE \\ 
  54876 & DCAF16 & 4 &  26 & 5.04 & $2.36 \times 10^{-7}$ & FALSE \\ 
  777 & CACNA1E & 1 & 1484 & 4.99 & $3.09 \times 10^{-7}$ & TRUE \\ 
  6568 & SLC17A1 & 6 & 268 & 4.98 & $3.16 \times 10^{-7}$ & FALSE \\ 
  3697 & ITIH1 & 3 &  69 & 4.97 & $3.31 \times 10^{-7}$ & FALSE \\ 
  80821 & DDHD1 & 14 & 394 & 4.95 & $3.71 \times 10^{-7}$ & FALSE \\ 
  5702 & PSMC3 & 11 &  35 & 4.95 & $3.80 \times 10^{-7}$ & TRUE \\ 
  83985 & SPNS1 & 16 &  38 & 4.94 & $3.88 \times 10^{-7}$ & FALSE \\ 
  7486 & WRN & 8 & 551 & 4.94 & $3.97 \times 10^{-7}$ & TRUE \\ 
  29106 & SCG3 & 15 & 153 & 4.93 & $4.21 \times 10^{-7}$ & FALSE \\ 
  23196 & FAM120A & 9 & 543 & 4.92 & $4.23 \times 10^{-7}$ & TRUE \\ 
  11281 & POU6F2 & 7 & 2191 & 4.92 & $4.40 \times 10^{-7}$ & FALSE \\ 
  23301 & EHBP1 & 2 & 934 & 4.91 & $4.56 \times 10^{-7}$ & TRUE \\ 
  8624 & PSMG1 & 21 & 588 & 4.91 & $4.56 \times 10^{-7}$ & FALSE \\ 
  6988 & TCTA & 3 &  18 & 4.90 & $4.76 \times 10^{-7}$ & FALSE \\ 
  10499 & NCOA2 & 8 & 897 & 4.90 & $4.81 \times 10^{-7}$ & FALSE \\ 
  10360 & NPM3 & 10 &   7 & 4.90 & $4.82 \times 10^{-7}$ & FALSE \\ 
  4439 & MSH5 & 6 & 105 & 4.89 & $5.01 \times 10^{-7}$ & FALSE \\ 
  6487 & ST3GAL3 & 1 & 848 & 4.89 & $5.07 \times 10^{-7}$ & FALSE \\ 
  84132 & USP42 & 7 & 321 & 4.87 & $5.68 \times 10^{-7}$ & FALSE \\ 
  64129 & TINAGL1 & 1 &  25 & 4.84 & $6.36 \times 10^{-7}$ & FALSE \\ 
  84276 & NICN1 & 3 &  18 & 4.84 & $6.37 \times 10^{-7}$ & FALSE \\ 
  344022 & NOTO & 2 &  36 & 4.84 & $6.39 \times 10^{-7}$ & FALSE \\ 
  84279 & PRADC1 & 2 &  22 & 4.83 & $6.68 \times 10^{-7}$ & FALSE \\ 
  203068 & TUBB & 6 &  30 & 4.83 & $6.96 \times 10^{-7}$ & TRUE \\ 
  29777 & ABT1 & 6 &  18 & 4.82 & $7.02 \times 10^{-7}$ & FALSE \\ 
  10724 & MGEA5 & 10 &  98 & 4.82 & $7.32 \times 10^{-7}$ & TRUE \\ 
  83734 & ATG10 & 5 & 842 & 4.81 & $7.39 \times 10^{-7}$ & FALSE \\ 
  2065 & ERBB3 & 12 &  66 & 4.81 & $7.54 \times 10^{-7}$ & FALSE \\ 
  5987 & TRIM27 & 6 & 126 & 4.80 & $7.85 \times 10^{-7}$ & FALSE \\ 
  10544 & PROCR & 20 & 175 & 4.80 & $8.10 \times 10^{-7}$ & FALSE \\ 
  79893 & GGNBP2 & 17 & 157 & 4.78 & $8.71 \times 10^{-7}$ & FALSE \\ 
  10642 & IGF2BP1 & 17 & 182 & 4.77 & $9.30 \times 10^{-7}$ & FALSE \\ 
  81797 & OR12D3 & 6 &  15 & 4.77 & $9.38 \times 10^{-7}$ & FALSE \\ 
  6272 & SORT1 & 1 & 272 & 4.75 & $9.94 \times 10^{-7}$ & TRUE \\ 
  150726 & FBXO41 & 2 &  54 & 4.75 & $1.03 \times 10^{-6}$ & TRUE \\ 
  9887 & SMG7 & 1 & 217 & 4.75 & $1.04 \times 10^{-6}$ & FALSE \\ 
  8460 & TPST1 & 7 & 590 & 4.75 & $1.04 \times 10^{-6}$ & FALSE \\ 
  8356 & HIST1H3J & 6 &   1 & 4.75 & $1.04 \times 10^{-6}$ & FALSE \\ 
  284612 & SYPL2 & 1 &  69 & 4.74 & $1.05 \times 10^{-6}$ & FALSE \\ 
  1803 & DPP4 & 2 & 311 & 4.74 & $1.06 \times 10^{-6}$ & FALSE \\ 
  84418 & CYSTM1 & 5 & 242 & 4.74 & $1.07 \times 10^{-6}$ & FALSE \\ 
  124842 & TMEM132E & 17 & 238 & 4.74 & $1.07 \times 10^{-6}$ & FALSE \\ 
  3760 & KCNJ3 & 2 & 777 & 4.73 & $1.12 \times 10^{-6}$ & TRUE \\ 
  10590 & SCGN & 6 & 310 & 4.73 & $1.14 \times 10^{-6}$ & FALSE \\ 
  8332 & HIST1H2AL & 6 &   5 & 4.73 & $1.15 \times 10^{-6}$ & FALSE \\ 
  79922 & MRM1 & 17 &  29 & 4.72 & $1.16 \times 10^{-6}$ & FALSE \\ 
  1839 & HBEGF & 5 &  47 & 4.71 & $1.23 \times 10^{-6}$ & FALSE \\ 
  5686 & PSMA5 & 1 &  95 & 4.68 & $1.40 \times 10^{-6}$ & TRUE \\ 
  284613 & CYB561D1 & 1 &  26 & 4.67 & $1.48 \times 10^{-6}$ & FALSE \\ 
  55789 & DEPDC1B & 5 & 306 & 4.67 & $1.49 \times 10^{-6}$ & FALSE \\ 
  8354 & HIST1H3I & 6 &   5 & 4.67 & $1.51 \times 10^{-6}$ & FALSE \\ 
  51460 & SFMBT1 & 3 & 525 & 4.67 & $1.54 \times 10^{-6}$ & FALSE \\ 
  55534 & MAML3 & 4 & 1866 & 4.65 & $1.63 \times 10^{-6}$ & FALSE \\ 
  55765 & C1orf106 & 1 & 117 & 4.64 & $1.72 \times 10^{-6}$ & FALSE \\ 
  10919 & EHMT2 & 6 &  82 & 4.64 & $1.72 \times 10^{-6}$ & TRUE \\ 
  7324 & UBE2E1 & 3 & 371 & 4.63 & $1.83 \times 10^{-6}$ & FALSE \\ 
  8343 & HIST1H2BF & 6 &   6 & 4.62 & $1.91 \times 10^{-6}$ & FALSE \\ 
  53833 & IL20RB & 3 & 173 & 4.62 & $1.91 \times 10^{-6}$ & FALSE \\ 
  2893 & GRIA4 & 11 & 1378 & 4.61 & $1.97 \times 10^{-6}$ & TRUE \\ 
  29113 & C6orf15 & 6 &  23 & 4.61 & $2.04 \times 10^{-6}$ & FALSE \\ 
  27072 & VPS41 & 7 & 913 & 4.60 & $2.08 \times 10^{-6}$ & TRUE \\ 
  5898 & RALA & 7 & 288 & 4.58 & $2.37 \times 10^{-6}$ & TRUE \\ 
  8347 & HIST1H2BC & 6 &  86 & 4.57 & $2.41 \times 10^{-6}$ & FALSE \\ 
  4857 & NOVA1 & 14 & 609 & 4.56 & $2.51 \times 10^{-6}$ & TRUE \\ 
  2147 & F2 & 11 &  96 & 4.56 & $2.54 \times 10^{-6}$ & FALSE \\ 
  55604 & LRRC16A & 6 & 2282 & 4.55 & $2.64 \times 10^{-6}$ & TRUE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Significant genes in UKBB int MAGMA GWAS. This is obviously much too small type we can move it to supplementary tables. It would ideally go into a split table in portrait view but that is a bit harder than I thought in latex} 
\label{tab:Significant genes in UKBB int MAGMA GWAS}
\end{table}

%%% SIGNIFICANT UKBBED GENES 


% latex table generated in R 3.6.1 by xtable 1.8-4 package
% Fri Feb 21 12:10:53 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{totalheight=10in}
\begin{tabular}{rllrrrl}
  \hline
GENE & SYMBOL & CHR & NSNPS & ZSTAT & P & synaptic \\ 
  \hline
63891 & RNF123 & 3 & 100 & 12.57 & $1.60 \times 10^{-36}$ & FALSE \\ 
  9807 & IP6K1 & 3 & 171 & 11.60 & $2.07 \times 10^{-31}$ & FALSE \\ 
  8927 & BSN & 3 & 312 & 10.79 & $2.02 \times 10^{-27}$ & TRUE \\ 
  84315 & MON1A & 3 &  40 & 10.42 & $1.01 \times 10^{-25}$ & FALSE \\ 
  10293 & TRAIP & 3 &  65 & 10.25 & $6.16 \times 10^{-25}$ & FALSE \\ 
  10180 & RBM6 & 3 & 421 & 10.18 & $1.26 \times 10^{-24}$ & FALSE \\ 
  79012 & CAMKV & 3 &  30 & 10.03 & $5.67 \times 10^{-24}$ & TRUE \\ 
  10181 & RBM5 & 3 &  56 & 9.91 & $1.93 \times 10^{-23}$ & FALSE \\ 
  1605 & DAG1 & 3 & 219 & 9.63 & $3.10 \times 10^{-22}$ & FALSE \\ 
  4486 & MST1R & 3 &  43 & 9.59 & $4.51 \times 10^{-22}$ & FALSE \\ 
  6405 & SEMA3F & 3 & 119 & 9.21 & $1.70 \times 10^{-20}$ & FALSE \\ 
  275 & AMT & 3 &  17 & 9.20 & $1.72 \times 10^{-20}$ & FALSE \\ 
  387 & RHOA & 3 & 186 & 9.10 & $4.37 \times 10^{-20}$ & TRUE \\ 
  327 & APEH & 3 &  18 & 8.93 & $2.05 \times 10^{-19}$ & TRUE \\ 
  389118 & CDHR4 & 3 &  30 & 8.69 & $1.77 \times 10^{-18}$ & FALSE \\ 
  4485 & MST1 & 3 &   9 & 8.15 & $1.80 \times 10^{-16}$ & FALSE \\ 
  253559 & CADM2 & 3 & 5365 & 7.96 & $8.45 \times 10^{-16}$ & FALSE \\ 
  6487 & ST3GAL3 & 1 & 848 & 7.80 & $3.19 \times 10^{-15}$ & FALSE \\ 
  6988 & TCTA & 3 &  18 & 7.66 & $9.66 \times 10^{-15}$ & FALSE \\ 
  7318 & UBA7 & 3 &  25 & 7.49 & $3.33 \times 10^{-14}$ & FALSE \\ 
  2876 & GPX1 & 3 &   5 & 7.49 & $3.40 \times 10^{-14}$ & TRUE \\ 
  10659 & CELF2 & 10 & 2900 & 7.49 & $3.57 \times 10^{-14}$ & TRUE \\ 
  84276 & NICN1 & 3 &  18 & 7.46 & $4.44 \times 10^{-14}$ & FALSE \\ 
  9884 & LRRC37A & 17 & 104 & 7.16 & $4.08 \times 10^{-13}$ & FALSE \\ 
  91942 & NDUFAF2 & 5 & 740 & 7.10 & $6.39 \times 10^{-13}$ & FALSE \\ 
  54870 & QRICH1 & 3 & 166 & 7.09 & $6.70 \times 10^{-13}$ & FALSE \\ 
  2779 & GNAT1 & 3 &   7 & 7.05 & $8.74 \times 10^{-13}$ & FALSE \\ 
  1951 & CELSR3 & 3 &  51 & 7.04 & $9.56 \times 10^{-13}$ & TRUE \\ 
  4905 & NSF & 17 & 146 & 7.04 & $9.70 \times 10^{-13}$ & TRUE \\ 
  100506084 & ARL17B & 17 &  48 & 7.03 & $1.05 \times 10^{-12}$ & FALSE \\ 
  23261 & CAMTA1 & 1 & 4535 & 6.99 & $1.35 \times 10^{-12}$ & FALSE \\ 
  51517 & NCKIPSD & 3 &  64 & 6.98 & $1.45 \times 10^{-12}$ & TRUE \\ 
  5144 & PDE4D & 5 & 6306 & 6.94 & $1.96 \times 10^{-12}$ & TRUE \\ 
  2868 & GRK4 & 4 & 425 & 6.90 & $2.54 \times 10^{-12}$ & FALSE \\ 
  375341 & C3orf62 & 3 &  23 & 6.88 & $3.05 \times 10^{-12}$ & FALSE \\ 
  284058 & KANSL1 & 17 & 994 & 6.88 & $3.09 \times 10^{-12}$ & FALSE \\ 
  7375 & USP4 & 3 & 191 & 6.87 & $3.26 \times 10^{-12}$ & FALSE \\ 
  339834 & CCDC36 & 3 & 175 & 6.83 & $4.36 \times 10^{-12}$ & FALSE \\ 
  57688 & ZSWIM6 & 5 & 633 & 6.81 & $4.76 \times 10^{-12}$ & FALSE \\ 
  10991 & SLC38A3 & 3 &  42 & 6.80 & $5.29 \times 10^{-12}$ & FALSE \\ 
  1630 & DCC & 18 & 7236 & 6.75 & $7.22 \times 10^{-12}$ & FALSE \\ 
  1394 & CRHR1 & 17 & 1320 & 6.75 & $7.62 \times 10^{-12}$ & FALSE \\ 
  51447 & IP6K2 & 3 &  82 & 6.73 & $8.21 \times 10^{-12}$ & FALSE \\ 
  65010 & SLC26A6 & 3 &  37 & 6.72 & $9.19 \times 10^{-12}$ & FALSE \\ 
  79993 & ELOVL7 & 5 & 366 & 6.70 & $1.02 \times 10^{-11}$ & FALSE \\ 
  7473 & WNT3 & 17 & 210 & 6.70 & $1.03 \times 10^{-11}$ & FALSE \\ 
  257194 & NEGR1 & 1 & 3314 & 6.70 & $1.06 \times 10^{-11}$ & TRUE \\ 
  5792 & PTPRF & 1 & 473 & 6.69 & $1.13 \times 10^{-11}$ & TRUE \\ 
  646498 & C3orf84 & 3 &  46 & 6.66 & $1.33 \times 10^{-11}$ & FALSE \\ 
  4137 & MAPT & 17 & 891 & 6.62 & $1.84 \times 10^{-11}$ & TRUE \\ 
  164832 & LONRF2 & 2 & 306 & 6.61 & $1.96 \times 10^{-11}$ & FALSE \\ 
  53335 & BCL11A & 2 & 318 & 6.54 & $3.11 \times 10^{-11}$ & FALSE \\ 
  50999 & TMED5 & 1 & 124 & 6.52 & $3.59 \times 10^{-11}$ & FALSE \\ 
  8602 & NOP14 & 4 & 128 & 6.51 & $3.66 \times 10^{-11}$ & FALSE \\ 
  162540 & SPPL2C & 17 &  26 & 6.48 & $4.54 \times 10^{-11}$ & FALSE \\ 
  79591 & C10orf76 & 10 & 577 & 6.47 & $4.79 \times 10^{-11}$ & FALSE \\ 
  80031 & SEMA6D & 15 & 2735 & 6.40 & $7.91 \times 10^{-11}$ & FALSE \\ 
  26091 & HERC4 & 10 & 550 & 6.39 & $8.39 \times 10^{-11}$ & FALSE \\ 
  30819 & KCNIP2 & 10 &  51 & 6.39 & $8.50 \times 10^{-11}$ & TRUE \\ 
  9682 & KDM4A & 1 & 177 & 6.36 & $9.95 \times 10^{-11}$ & FALSE \\ 
  9842 & PLEKHM1 & 17 & 177 & 6.34 & $1.14 \times 10^{-10}$ & TRUE \\ 
  1161 & ERCC8 & 5 & 259 & 6.33 & $1.24 \times 10^{-10}$ & FALSE \\ 
  23411 & SIRT1 & 10 & 162 & 6.32 & $1.31 \times 10^{-10}$ & FALSE \\ 
  8898 & MTMR2 & 11 & 373 & 6.31 & $1.38 \times 10^{-10}$ & TRUE \\ 
  143684 & FAM76B & 11 &  97 & 6.30 & $1.48 \times 10^{-10}$ & FALSE \\ 
  343099 & CCDC18 & 1 & 421 & 6.28 & $1.69 \times 10^{-10}$ & TRUE \\ 
  3913 & LAMB2 & 3 &  30 & 6.26 & $1.95 \times 10^{-10}$ & FALSE \\ 
  3017 & HIST1H2BD & 6 &  59 & 6.26 & $1.98 \times 10^{-10}$ & FALSE \\ 
  27253 & PCDH17 & 13 & 359 & 6.24 & $2.12 \times 10^{-10}$ & FALSE \\ 
  200942 & KLHDC8B & 3 &  14 & 6.24 & $2.25 \times 10^{-10}$ & FALSE \\ 
  29119 & CTNNA3 & 10 & 9897 & 6.23 & $2.27 \times 10^{-10}$ & FALSE \\ 
  3831 & KLC1 & 14 & 310 & 6.23 & $2.40 \times 10^{-10}$ & TRUE \\ 
  8365 & HIST1H4H & 6 &  30 & 6.22 & $2.51 \times 10^{-10}$ & FALSE \\ 
  27086 & FOXP1 & 3 & 2455 & 6.21 & $2.66 \times 10^{-10}$ & FALSE \\ 
  100132074 & FOXO6 & 1 &  85 & 6.19 & $2.92 \times 10^{-10}$ & FALSE \\ 
  246744 & STH & 17 &   1 & 6.19 & $2.98 \times 10^{-10}$ & FALSE \\ 
  54790 & TET2 & 4 & 563 & 6.19 & $3.06 \times 10^{-10}$ & FALSE \\ 
  6799 & SULT1A2 & 16 &  59 & 6.19 & $3.08 \times 10^{-10}$ & FALSE \\ 
  3899 & AFF3 & 2 & 2283 & 6.13 & $4.29 \times 10^{-10}$ & FALSE \\ 
  112869 & CCDC101 & 16 & 163 & 6.13 & $4.40 \times 10^{-10}$ & FALSE \\ 
  64409 & WBSCR17 & 7 & 3079 & 6.13 & $4.43 \times 10^{-10}$ & FALSE \\ 
  83698 & CALN1 & 7 & 3317 & 6.12 & $4.66 \times 10^{-10}$ & FALSE \\ 
  9702 & CEP57 & 11 & 217 & 6.12 & $4.79 \times 10^{-10}$ & FALSE \\ 
  1806 & DPYD & 1 & 3626 & 6.11 & $5.13 \times 10^{-10}$ & FALSE \\ 
  201176 & ARHGAP27 & 17 & 170 & 6.09 & $5.56 \times 10^{-10}$ & FALSE \\ 
  8379 & MAD1L1 & 7 & 3155 & 6.05 & $7.05 \times 10^{-10}$ & FALSE \\ 
  6789 & STK4 & 20 & 504 & 6.03 & $8.12 \times 10^{-10}$ & FALSE \\ 
  1294 & COL7A1 & 3 &  78 & 6.01 & $9.47 \times 10^{-10}$ & FALSE \\ 
  10724 & MGEA5 & 10 &  98 & 6.00 & $9.88 \times 10^{-10}$ & TRUE \\ 
  80034 & CSRNP3 & 2 & 794 & 5.99 & $1.07 \times 10^{-9}$ & FALSE \\ 
  84334 & APOPT1 & 14 &  94 & 5.95 & $1.37 \times 10^{-9}$ & FALSE \\ 
  11273 & ATXN2L & 16 &  60 & 5.94 & $1.39 \times 10^{-9}$ & TRUE \\ 
  1810 & DR1 & 1 &  68 & 5.90 & $1.79 \times 10^{-9}$ & FALSE \\ 
  10425 & ARIH2 & 3 & 145 & 5.86 & $2.28 \times 10^{-9}$ & FALSE \\ 
  5576 & PRKAR2A & 3 & 212 & 5.79 & $3.58 \times 10^{-9}$ & TRUE \\ 
  487 & ATP2A1 & 16 &  94 & 5.78 & $3.81 \times 10^{-9}$ & TRUE \\ 
  8428 & STK24 & 13 & 656 & 5.75 & $4.46 \times 10^{-9}$ & TRUE \\ 
  5143 & PDE4C & 19 & 188 & 5.75 & $4.56 \times 10^{-9}$ & FALSE \\ 
  1201 & CLN3 & 16 &  41 & 5.71 & $5.53 \times 10^{-9}$ & FALSE \\ 
  6817 & SULT1A1 & 16 & 101 & 5.71 & $5.65 \times 10^{-9}$ & FALSE \\ 
  4208 & MEF2C & 5 & 603 & 5.69 & $6.25 \times 10^{-9}$ & TRUE \\ 
  55789 & DEPDC1B & 5 & 306 & 5.69 & $6.38 \times 10^{-9}$ & FALSE \\ 
  8729 & GBF1 & 10 & 377 & 5.68 & $6.63 \times 10^{-9}$ & FALSE \\ 
  25970 & SH2B1 & 16 &  41 & 5.64 & $8.73 \times 10^{-9}$ & FALSE \\ 
  83894 & TTC29 & 4 & 1048 & 5.62 & $9.70 \times 10^{-9}$ & FALSE \\ 
  440955 & TMEM89 & 3 &   5 & 5.58 & $1.20 \times 10^{-8}$ & FALSE \\ 
  79823 & CAMKMT & 2 & 2038 & 5.58 & $1.22 \times 10^{-8}$ & FALSE \\ 
  57597 & BAHCC1 & 17 & 335 & 5.57 & $1.27 \times 10^{-8}$ & FALSE \\ 
  10360 & NPM3 & 10 &   7 & 5.56 & $1.33 \times 10^{-8}$ & FALSE \\ 
  4134 & MAP4 & 3 & 578 & 5.56 & $1.36 \times 10^{-8}$ & TRUE \\ 
  256646 & NUTM1 & 15 &  66 & 5.56 & $1.36 \times 10^{-8}$ & FALSE \\ 
  152940 & C4orf45 & 4 & 655 & 5.56 & $1.37 \times 10^{-8}$ & FALSE \\ 
  56521 & DNAJC12 & 10 & 139 & 5.56 & $1.38 \times 10^{-8}$ & FALSE \\ 
  8028 & MLLT10 & 10 & 560 & 5.55 & $1.39 \times 10^{-8}$ & FALSE \\ 
  399818 & METTL10 & 10 & 169 & 5.55 & $1.44 \times 10^{-8}$ & FALSE \\ 
  55954 & ZMAT5 & 22 & 177 & 5.55 & $1.45 \times 10^{-8}$ & FALSE \\ 
  57452 & GALNT16 & 14 & 455 & 5.53 & $1.58 \times 10^{-8}$ & FALSE \\ 
  79874 & RABEP2 & 16 &  65 & 5.52 & $1.73 \times 10^{-8}$ & FALSE \\ 
  5095 & PCCA & 13 & 1504 & 5.50 & $1.85 \times 10^{-8}$ & TRUE \\ 
  115708 & TRMT61A & 14 &  38 & 5.50 & $1.90 \times 10^{-8}$ & FALSE \\ 
  7284 & TUFM & 16 &  11 & 5.46 & $2.34 \times 10^{-8}$ & TRUE \\ 
  57492 & ARID1B & 6 & 1853 & 5.46 & $2.35 \times 10^{-8}$ & FALSE \\ 
  55147 & RBM23 & 14 &  95 & 5.46 & $2.40 \times 10^{-8}$ & FALSE \\ 
  23031 & MAST3 & 19 & 273 & 5.43 & $2.82 \times 10^{-8}$ & TRUE \\ 
  22823 & MTF2 & 1 & 215 & 5.43 & $2.82 \times 10^{-8}$ & FALSE \\ 
  23671 & TMEFF2 & 2 & 1282 & 5.42 & $3.01 \times 10^{-8}$ & TRUE \\ 
  55911 & APOBR & 16 &  20 & 5.36 & $4.07 \times 10^{-8}$ & FALSE \\ 
  25788 & RAD54B & 8 & 471 & 5.36 & $4.09 \times 10^{-8}$ & FALSE \\ 
  8085 & KMT2D & 12 &  96 & 5.36 & $4.09 \times 10^{-8}$ & TRUE \\ 
  3127 & HLA-DRB5 & 6 &  41 & 5.36 & $4.24 \times 10^{-8}$ & FALSE \\ 
  23334 & SZT2 & 1 & 201 & 5.36 & $4.25 \times 10^{-8}$ & TRUE \\ 
  167227 & DCP2 & 5 & 236 & 5.35 & $4.29 \times 10^{-8}$ & FALSE \\ 
  4248 & MGAT3 & 22 & 165 & 5.35 & $4.38 \times 10^{-8}$ & FALSE \\ 
  84164 & ASCC2 & 22 & 219 & 5.30 & $5.66 \times 10^{-8}$ & FALSE \\ 
  4194 & MDM4 & 1 & 543 & 5.30 & $5.70 \times 10^{-8}$ & FALSE \\ 
  3123 & HLA-DRB1 & 6 &  60 & 5.28 & $6.47 \times 10^{-8}$ & FALSE \\ 
  7075 & TIE1 & 1 &  65 & 5.27 & $6.85 \times 10^{-8}$ & FALSE \\ 
  4685 & NCAM2 & 21 & 3092 & 5.25 & $7.46 \times 10^{-8}$ & FALSE \\ 
  246778 & IL27 & 16 &  22 & 5.25 & $7.58 \times 10^{-8}$ & FALSE \\ 
  3781 & KCNN2 & 5 & 586 & 5.25 & $7.70 \times 10^{-8}$ & TRUE \\ 
  56853 & CELF4 & 18 & 1442 & 5.24 & $7.84 \times 10^{-8}$ & TRUE \\ 
  5662 & PSD & 10 &  45 & 5.20 & $1.01 \times 10^{-7}$ & TRUE \\ 
  23172 & FAM175B & 10 & 127 & 5.18 & $1.12 \times 10^{-7}$ & FALSE \\ 
  6497 & SKI & 1 & 420 & 5.15 & $1.29 \times 10^{-7}$ & FALSE \\ 
  221935 & SDK1 & 7 & 7054 & 5.14 & $1.34 \times 10^{-7}$ & TRUE \\ 
  388662 & SLC6A17 & 1 & 192 & 5.14 & $1.39 \times 10^{-7}$ & FALSE \\ 
  100861412 & FSBP & 8 &  35 & 5.13 & $1.42 \times 10^{-7}$ & FALSE \\ 
  5296 & PIK3R2 & 19 &  59 & 5.12 & $1.52 \times 10^{-7}$ & FALSE \\ 
  3615 & IMPDH2 & 3 &  10 & 5.12 & $1.54 \times 10^{-7}$ & FALSE \\ 
  29106 & SCG3 & 15 & 153 & 5.12 & $1.55 \times 10^{-7}$ & FALSE \\ 
  91574 & C12orf65 & 12 & 103 & 5.12 & $1.57 \times 10^{-7}$ & FALSE \\ 
  100130370 & LOC100130370 & 17 &  42 & 5.11 & $1.61 \times 10^{-7}$ & FALSE \\ 
  5210 & PFKFB4 & 3 & 103 & 5.10 & $1.67 \times 10^{-7}$ & FALSE \\ 
  2099 & ESR1 & 6 & 2067 & 5.09 & $1.82 \times 10^{-7}$ & FALSE \\ 
  3119 & HLA-DQB1 & 6 & 163 & 5.08 & $1.84 \times 10^{-7}$ & FALSE \\ 
  80309 & SPHKAP & 2 & 1018 & 5.08 & $1.88 \times 10^{-7}$ & FALSE \\ 
  60412 & EXOC4 & 7 & 3112 & 5.08 & $1.92 \times 10^{-7}$ & TRUE \\ 
  5859 & QARS & 3 &  19 & 5.08 & $1.93 \times 10^{-7}$ & TRUE \\ 
  4857 & NOVA1 & 14 & 609 & 5.07 & $1.95 \times 10^{-7}$ & TRUE \\ 
  205428 & C3orf58 & 3 &  85 & 5.06 & $2.12 \times 10^{-7}$ & FALSE \\ 
  118 & ADD1 & 4 & 403 & 5.06 & $2.14 \times 10^{-7}$ & TRUE \\ 
  140733 & MACROD2 & 20 & 10230 & 5.05 & $2.16 \times 10^{-7}$ & FALSE \\ 
  2565 & GABRG1 & 4 & 425 & 5.05 & $2.18 \times 10^{-7}$ & FALSE \\ 
  8000 & PSCA & 8 &  79 & 5.04 & $2.28 \times 10^{-7}$ & FALSE \\ 
  4771 & NF2 & 22 & 332 & 5.04 & $2.37 \times 10^{-7}$ & FALSE \\ 
  54714 & CNGB3 & 8 & 820 & 5.03 & $2.49 \times 10^{-7}$ & FALSE \\ 
  54805 & CNNM2 & 10 & 637 & 5.03 & $2.49 \times 10^{-7}$ & FALSE \\ 
  533 & ATP6V0B & 1 &  12 & 5.02 & $2.60 \times 10^{-7}$ & FALSE \\ 
  10419 & PRMT5 & 14 &  26 & 5.01 & $2.76 \times 10^{-7}$ & TRUE \\ 
  389119 & FAM212A & 3 &   2 & 4.98 & $3.14 \times 10^{-7}$ & FALSE \\ 
  51115 & RMDN1 & 8 & 209 & 4.97 & $3.34 \times 10^{-7}$ & FALSE \\ 
  11216 & AKAP10 & 17 & 211 & 4.96 & $3.53 \times 10^{-7}$ & FALSE \\ 
  7248 & TSC1 & 9 & 204 & 4.95 & $3.74 \times 10^{-7}$ & TRUE \\ 
  5201 & PFDN1 & 5 & 186 & 4.95 & $3.75 \times 10^{-7}$ & FALSE \\ 
  27068 & PPA2 & 4 & 622 & 4.94 & $3.90 \times 10^{-7}$ & FALSE \\ 
  54930 & HAUS4 & 14 &  43 & 4.94 & $3.99 \times 10^{-7}$ & FALSE \\ 
  29919 & C18orf8 & 18 & 117 & 4.92 & $4.43 \times 10^{-7}$ & TRUE \\ 
  8344 & HIST1H2BE & 6 &   5 & 4.91 & $4.45 \times 10^{-7}$ & FALSE \\ 
  7003 & TEAD1 & 11 & 1046 & 4.91 & $4.48 \times 10^{-7}$ & FALSE \\ 
  57528 & KCTD16 & 5 & 1047 & 4.91 & $4.51 \times 10^{-7}$ & FALSE \\ 
  25791 & NGEF & 2 & 794 & 4.91 & $4.60 \times 10^{-7}$ & TRUE \\ 
  55802 & DCP1A & 3 & 224 & 4.88 & $5.19 \times 10^{-7}$ & FALSE \\ 
  55703 & POLR3B & 12 & 509 & 4.87 & $5.47 \times 10^{-7}$ & FALSE \\ 
  5934 & RBL2 & 16 & 254 & 4.87 & $5.58 \times 10^{-7}$ & FALSE \\ 
  696 & BTN1A1 & 6 &  48 & 4.86 & $5.76 \times 10^{-7}$ & FALSE \\ 
  10198 & MPHOSPH9 & 12 & 325 & 4.86 & $5.90 \times 10^{-7}$ & FALSE \\ 
  4864 & NPC1 & 18 & 326 & 4.85 & $6.23 \times 10^{-7}$ & FALSE \\ 
  343263 & MYBPHL & 1 &  84 & 4.84 & $6.46 \times 10^{-7}$ & FALSE \\ 
  4781 & NFIB & 9 & 1715 & 4.84 & $6.55 \times 10^{-7}$ & FALSE \\ 
  27132 & CPNE7 & 16 & 145 & 4.84 & $6.59 \times 10^{-7}$ & TRUE \\ 
  83855 & KLF16 & 19 &  45 & 4.83 & $6.74 \times 10^{-7}$ & FALSE \\ 
  8897 & MTMR3 & 22 & 574 & 4.83 & $6.98 \times 10^{-7}$ & FALSE \\ 
  9374 & PPT2 & 6 &  47 & 4.81 & $7.37 \times 10^{-7}$ & FALSE \\ 
  10473 & HMGN4 & 6 &  33 & 4.81 & $7.71 \times 10^{-7}$ & FALSE \\ 
  9048 & ARTN & 1 &  17 & 4.80 & $7.84 \times 10^{-7}$ & FALSE \\ 
  55500 & ETNK1 & 12 & 183 & 4.79 & $8.35 \times 10^{-7}$ & FALSE \\ 
  6660 & SOX5 & 12 & 5170 & 4.78 & $8.98 \times 10^{-7}$ & FALSE \\ 
  2119 & ETV5 & 3 & 185 & 4.77 & $9.13 \times 10^{-7}$ & FALSE \\ 
  2065 & ERBB3 & 12 &  66 & 4.76 & $9.72 \times 10^{-7}$ & FALSE \\ 
  23109 & DDN & 12 &  11 & 4.75 & $9.92 \times 10^{-7}$ & TRUE \\ 
  23338 & JADE2 & 5 & 255 & 4.75 & $1.02 \times 10^{-6}$ & FALSE \\ 
  55122 & AKIRIN2 & 6 & 136 & 4.75 & $1.04 \times 10^{-6}$ & FALSE \\ 
  27125 & AFF4 & 5 & 358 & 4.74 & $1.06 \times 10^{-6}$ & FALSE \\ 
  5571 & PRKAG1 & 12 &  40 & 4.72 & $1.15 \times 10^{-6}$ & FALSE \\ 
  284612 & SYPL2 & 1 &  69 & 4.72 & $1.16 \times 10^{-6}$ & FALSE \\ 
  51337 & THEM6 & 8 &  42 & 4.72 & $1.16 \times 10^{-6}$ & TRUE \\ 
  84962 & AJUBA & 14 &  40 & 4.72 & $1.20 \times 10^{-6}$ & FALSE \\ 
  4352 & MPL & 1 &  39 & 4.72 & $1.21 \times 10^{-6}$ & FALSE \\ 
  1152 & CKB & 14 &   9 & 4.71 & $1.22 \times 10^{-6}$ & TRUE \\ 
  54681 & P4HTM & 3 &  36 & 4.71 & $1.24 \times 10^{-6}$ & FALSE \\ 
  3117 & HLA-DQA1 & 6 & 160 & 4.70 & $1.32 \times 10^{-6}$ & FALSE \\ 
  145567 & TTC7B & 14 & 1354 & 4.68 & $1.40 \times 10^{-6}$ & FALSE \\ 
  2066 & ERBB4 & 2 & 6781 & 4.68 & $1.46 \times 10^{-6}$ & FALSE \\ 
  5686 & PSMA5 & 1 &  95 & 4.65 & $1.63 \times 10^{-6}$ & TRUE \\ 
  6272 & SORT1 & 1 & 272 & 4.65 & $1.65 \times 10^{-6}$ & TRUE \\ 
  9486 & CHST10 & 2 & 167 & 4.64 & $1.71 \times 10^{-6}$ & FALSE \\ 
  788 & SLC25A20 & 3 & 104 & 4.64 & $1.76 \times 10^{-6}$ & TRUE \\ 
  3769 & KCNJ13 & 2 &  46 & 4.64 & $1.78 \times 10^{-6}$ & FALSE \\ 
  3799 & KIF5B & 10 & 152 & 4.63 & $1.82 \times 10^{-6}$ & TRUE \\ 
  2870 & GRK6 & 5 &  52 & 4.62 & $1.89 \times 10^{-6}$ & TRUE \\ 
  643155 & SMIM15 & 5 &   9 & 4.61 & $1.99 \times 10^{-6}$ & FALSE \\ 
  9378 & NRXN1 & 2 & 6004 & 4.61 & $2.01 \times 10^{-6}$ & TRUE \\ 
  5314 & PKHD1 & 6 & 2364 & 4.61 & $2.03 \times 10^{-6}$ & FALSE \\ 
  63940 & GPSM3 & 6 &  17 & 4.59 & $2.17 \times 10^{-6}$ & FALSE \\ 
  158067 & AK8 & 9 & 738 & 4.59 & $2.20 \times 10^{-6}$ & FALSE \\ 
  54439 & RBM27 & 5 & 259 & 4.59 & $2.27 \times 10^{-6}$ & FALSE \\ 
  9679 & FAM53B & 10 & 487 & 4.58 & $2.28 \times 10^{-6}$ & FALSE \\ 
  7517 & XRCC3 & 14 &  83 & 4.58 & $2.30 \times 10^{-6}$ & FALSE \\ 
  991 & CDC20 & 1 &  12 & 4.57 & $2.41 \times 10^{-6}$ & FALSE \\ 
  22907 & DHX30 & 3 & 103 & 4.57 & $2.44 \times 10^{-6}$ & TRUE \\ 
  120 & ADD3 & 10 & 613 & 4.57 & $2.44 \times 10^{-6}$ & TRUE \\ 
  575 & ADGRB1 & 8 & 404 & 4.57 & $2.48 \times 10^{-6}$ & TRUE \\ 
  1993 & ELAVL2 & 9 & 729 & 4.56 & $2.55 \times 10^{-6}$ & TRUE \\ 
  23468 & CBX5 & 12 & 111 & 4.55 & $2.72 \times 10^{-6}$ & FALSE \\ 
  9529 & BAG5 & 14 &  17 & 4.55 & $2.72 \times 10^{-6}$ & TRUE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Significant genes in UKBB edu MAGMA GWAS} 
\label{tab:Significant genes in UKBB edu MAGMA GWAS}
\end{table}


\subsection{Duplicates removed from exac data}

See table~\ref{tab:Supplementary table- duplicate exac data}
% latex table generated in R 3.6.2 by xtable 1.8-4 package
% Fri Mar  6 16:23:36 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{lllrrrrrrrrrrrrrrrrrrrr}
  \hline
transcript & gene & chr & n\_exons & tx\_start & tx\_end & bp & mu\_syn & mu\_mis & mu\_lof & n\_syn & n\_mis & n\_lof & exp\_syn & exp\_mis & exp\_lof & syn\_z & mis\_z & lof\_z & pLI & n\_cnv & exp\_cnv & cnv\_z \\ 
  \hline
ENST00000290866.4 & ACE & 17 &  25 & 61554455 & 61574727 & 3921 & 0.00 & 0.00 & 0.00 & 283 & 546 &  34 & 255.36 & 497.51 & 46.68 & -1.07 & -1.06 & 1.84 & 0.00 & 6.00 & 5.18 & -0.12 \\ 
  ENST00000290866.4 & ACE & 17 &  25 & 61554455 & 61574727 & 3921 & 0.00 & 0.00 & 0.00 & 283 & 546 &  34 & 255.36 & 497.51 & 46.68 & -1.07 & -1.06 & 1.84 & 0.00 & 6.00 & 7.56 & 0.21 \\ 
  ENST00000531154.1 & ALG9 & 11 &  11 & 111657120 & 111731321 & 1344 & 0.00 & 0.00 & 0.00 &  50 & 112 &   8 & 55.10 & 131.14 & 18.65 & 0.43 & 0.82 & 2.44 & 0.00 & 4.00 & 6.06 & 0.32 \\ 
  ENST00000531154.1 & ALG9 & 11 &  11 & 111657120 & 111731321 & 1344 & 0.00 & 0.00 & 0.00 &  50 & 112 &   8 & 55.10 & 131.14 & 18.65 & 0.43 & 0.82 & 2.44 & 0.00 & 4.00 & 5.89 & 0.29 \\ 
  ENST00000311813.4 & AQP1 & 7 &   4 & 30951524 & 30963244 & 810 & 0.00 & 0.00 & 0.00 &  64 &  96 &   3 & 66.55 & 122.95 & 6.89 & 0.19 & 1.19 & 1.47 & 0.05 & 5.00 & 8.98 & 0.52 \\ 
  ENST00000311813.4 & AQP1 & 7 &   4 & 30951524 & 30963244 & 810 & 0.00 & 0.00 & 0.00 &  64 &  96 &   3 & 66.55 & 122.95 & 6.89 & 0.19 & 1.19 & 1.47 & 0.05 & 5.00 & 8.09 & 0.42 \\ 
  ENST00000287202.5 & CELF6 & 15 &  12 & 72579605 & 72612215 & 1446 & 0.00 & 0.00 & 0.00 &  50 &  82 &   3 & 84.99 & 153.15 & 13.16 & 2.35 & 2.81 & 2.78 & 0.38 & 1.00 & 3.71 & 0.56 \\ 
  ENST00000287202.5 & CELF6 & 15 &  12 & 72579605 & 72612215 & 1446 & 0.00 & 0.00 & 0.00 &  50 &  82 &   3 & 84.99 & 153.15 & 13.16 & 2.35 & 2.81 & 2.78 & 0.38 & 1.00 & 4.31 & 0.66 \\ 
  ENST00000425368.2 & CFB & 6 &  18 & 31913998 & 31919807 & 2295 & 0.00 & 0.00 & 0.00 &  93 & 200 &  11 & 119.61 & 274.94 & 31.25 & 1.51 & 2.21 & 3.59 & 0.00 & 11.00 & 11.77 & 0.07 \\ 
  ENST00000425368.2 & CFB & 6 &  18 & 31913998 & 31919807 & 2295 & 0.00 & 0.00 & 0.00 &  93 & 200 &  11 & 119.61 & 274.94 & 31.25 & 1.51 & 2.21 & 3.59 & 0.00 & 11.00 & 11.67 & 0.07 \\ 
  ENST00000330233.7 & CRIP1 & 14 &   4 & 105953596 & 105954848 & 234 & 0.00 & 0.00 & 0.00 &  11 &  36 &   4 & 17.04 & 29.49 & 3.56 & 0.91 & -0.59 & -0.23 & 0.00 & 4.00 & 3.46 & -0.09 \\ 
  ENST00000330233.7 & CRIP1 & 14 &   4 & 105953596 & 105954848 & 234 & 0.00 & 0.00 & 0.00 &  11 &  36 &   4 & 17.04 & 29.49 & 3.56 & 0.91 & -0.59 & -0.23 & 0.00 & 3.00 & 3.49 & 0.09 \\ 
  ENST00000368073.3 & DCAF8 & 1 &  12 & 160187381 & 160213798 & 1794 & 0.00 & 0.00 & 0.00 &  70 & 108 &   0 & 81.10 & 221.34 & 23.24 & 0.76 & 3.73 & 4.78 & 1.00 & 1.00 & 6.39 & 0.97 \\ 
  ENST00000368073.3 & DCAF8 & 1 &  12 & 160187381 & 160213798 & 1794 & 0.00 & 0.00 & 0.00 &  70 & 108 &   0 & 81.10 & 221.34 & 23.24 & 0.76 & 3.73 & 4.78 & 1.00 & 1.00 & 6.57 & 1.00 \\ 
  ENST00000368408.3 & EFNA3 & 1 &   5 & 155051417 & 155059019 & 717 & 0.00 & 0.00 & 0.00 &  27 &  59 &   2 & 52.99 & 92.60 & 6.23 & 2.21 & 1.71 & 1.68 & 0.18 & 1.00 & 2.83 & 0.40 \\ 
  ENST00000368408.3 & EFNA3 & 1 &   5 & 155051417 & 155059019 & 717 & 0.00 & 0.00 & 0.00 &  27 &  59 &   2 & 52.99 & 92.60 & 6.23 & 2.21 & 1.71 & 1.68 & 0.18 & 1.00 & 2.83 & 0.40 \\ 
  ENST00000368026.6 & F11R & 1 &  10 & 160968660 & 160990863 & 900 & 0.00 & 0.00 & 0.00 &  37 & 103 &   9 & 47.96 & 111.44 & 15.08 & 0.98 & 0.39 & 1.55 & 0.00 & 5.00 & 5.60 & 0.09 \\ 
  ENST00000368026.6 & F11R & 1 &  10 & 160968660 & 160990863 & 900 & 0.00 & 0.00 & 0.00 &  37 & 103 &   9 & 47.96 & 111.44 & 15.08 & 0.98 & 0.39 & 1.55 & 0.00 & 5.00 & 5.61 & 0.09 \\ 
  ENST00000378842.3 & GALT & 9 &  11 & 34646701 & 34650446 & 1140 & 0.00 & 0.00 & 0.00 &  58 & 104 &   8 & 65.65 & 157.51 & 18.57 & 0.59 & 2.09 & 2.43 & 0.00 & 25.00 & 5.27 & -1.71 \\ 
  ENST00000378842.3 & GALT & 9 &  11 & 34646701 & 34650446 & 1140 & 0.00 & 0.00 & 0.00 &  58 & 104 &   8 & 65.65 & 157.51 & 18.57 & 0.59 & 2.09 & 2.43 & 0.00 & 24.00 & 7.93 & -1.30 \\ 
  ENST00000609810.1 & HOXC4 & 12 &   2 & 54447706 & 54448989 & 795 & 0.00 & 0.00 & 0.00 &  42 &  79 &   2 & 69.84 & 123.84 & 8.04 & 2.06 & 1.97 & 2.11 & 0.31 & 0.00 & 2.71 & 0.64 \\ 
  ENST00000609810.1 & HOXC4 & 12 &   2 & 54447706 & 54448989 & 795 & 0.00 & 0.00 & 0.00 &  42 &  79 &   2 & 69.84 & 123.84 & 8.04 & 2.06 & 1.97 & 2.11 & 0.31 & 0.00 & 2.71 & 0.64 \\ 
  ENST00000256433.3 & IER3IP1 & 18 &   3 & 44682547 & 44702648 & 249 & 0.00 & 0.00 & 0.00 &  15 &  29 &   3 & 13.36 & 24.88 & 4.67 & -0.28 & -0.40 & 0.77 & 0.02 & 2.00 & 3.71 & 0.33 \\ 
  ENST00000256433.3 & IER3IP1 & 18 &   3 & 44682547 & 44702648 & 249 & 0.00 & 0.00 & 0.00 &  15 &  29 &   3 & 13.36 & 24.88 & 4.67 & -0.28 & -0.40 & 0.77 & 0.02 & 3.00 & 4.14 & 0.20 \\ 
  ENST00000399932.3 & ITFG3 & 16 &  11 & 304412 & 315021 & 1659 & 0.00 & 0.00 & 0.00 & 119 & 258 &  10 & 134.47 & 247.19 & 17.48 & 0.83 & -0.34 & 1.77 & 0.00 & 25.00 & 5.50 & -1.68 \\ 
  ENST00000399932.3 & ITFG3 & 16 &  11 & 304412 & 315021 & 1659 & 0.00 & 0.00 & 0.00 & 119 & 258 &  10 & 134.47 & 247.19 & 17.48 & 0.83 & -0.34 & 1.77 & 0.00 & 25.00 & 5.46 & -1.69 \\ 
  ENST00000559488.1 & ITGB3 & 17 &  15 & 45331227 & 45387570 & 2367 & 0.00 & 0.00 & 0.00 & 120 & 222 &   7 & 123.45 & 287.63 & 27.32 & 0.19 & 1.89 & 3.85 & 0.14 & 8.00 & 7.39 & -0.08 \\ 
  ENST00000559488.1 & ITGB3 & 17 &  15 & 45331227 & 45387570 & 2367 & 0.00 & 0.00 & 0.00 & 120 & 222 &   7 & 123.45 & 287.63 & 27.32 & 0.19 & 1.89 & 3.85 & 0.14 & 8.00 & 7.51 & -0.06 \\ 
  ENST00000430070.2 & KBTBD4 & 11 &   4 & 47594481 & 47600503 & 1605 & 0.00 & 0.00 & 0.00 &  68 & 159 &   5 & 73.57 & 186.26 & 12.69 & 0.40 & 0.98 & 2.14 & 0.02 & 0.00 & 3.02 & 0.70 \\ 
  ENST00000430070.2 & KBTBD4 & 11 &   4 & 47594481 & 47600503 & 1605 & 0.00 & 0.00 & 0.00 &  68 & 159 &   5 & 73.57 & 186.26 & 12.69 & 0.40 & 0.98 & 2.14 & 0.02 & 0.00 & 3.40 & 0.77 \\ 
  ENST00000534898.4 & KIAA0391 & 14 &   7 & 35592451 & 35742772 & 1752 & 0.00 & 0.00 & 0.00 &  66 & 173 &  10 & 66.99 & 173.64 & 16.46 & 0.08 & 0.02 & 1.58 & 0.00 & 25.00 & 4.10 & -1.90 \\ 
  ENST00000534898.4 & KIAA0391 & 14 &   7 & 35592451 & 35742772 & 1752 & 0.00 & 0.00 & 0.00 &  66 & 173 &  10 & 66.99 & 173.64 & 16.46 & 0.08 & 0.02 & 1.58 & 0.00 & 19.00 & 6.25 & -1.21 \\ 
  ENST00000376832.4 & KLK9 & 19 &   5 & 51506366 & 51512803 & 753 & 0.00 & 0.00 & 0.00 &  37 &  99 &   5 & 39.36 & 95.63 & 8.93 & 0.23 & -0.17 & 1.30 & 0.00 & 6.00 & 5.33 & -0.10 \\ 
  ENST00000376832.4 & KLK9 & 19 &   5 & 51506366 & 51512803 & 753 & 0.00 & 0.00 & 0.00 &  37 &  99 &   5 & 39.36 & 95.63 & 8.93 & 0.23 & -0.17 & 1.30 & 0.00 & 9.00 & 6.49 & -0.31 \\ 
  ENST00000394052.3 & KRT222 & 17 &   6 & 38812653 & 38821351 & 888 & 0.00 & 0.00 & 0.00 &  31 &  84 &   6 & 34.66 & 85.68 & 11.51 & 0.39 & 0.09 & 1.61 & 0.00 & 4.00 & 7.16 & 0.47 \\ 
  ENST00000394052.3 & KRT222 & 17 &   6 & 38812653 & 38821351 & 888 & 0.00 & 0.00 & 0.00 &  31 &  84 &   6 & 34.66 & 85.68 & 11.51 & 0.39 & 0.09 & 1.61 & 0.00 & 5.00 & 8.76 & 0.50 \\ 
  ENST00000341206.4 & LCN6 & 9 &   5 & 139639113 & 139642935 & 492 & 0.00 & 0.00 & 0.00 &  40 &  74 &   7 & 33.56 & 61.50 & 6.97 & -0.69 & -0.78 & -0.01 & 0.00 & 27.00 & 3.81 & -2.05 \\ 
  ENST00000341206.4 & LCN6 & 9 &   5 & 139639113 & 139642935 & 492 & 0.00 & 0.00 & 0.00 &  40 &  74 &   7 & 33.56 & 61.50 & 6.97 & -0.69 & -0.78 & -0.01 & 0.00 & 28.00 & 3.59 & -2.14 \\ 
  ENST00000381775.1 & LSP1 & 11 &  10 & 1887704 & 1908793 & 1404 & 0.00 & 0.00 & 0.00 &  56 & 117 &   3 & 58.61 & 108.17 & 11.72 & 0.21 & -0.42 & 2.52 & 0.26 & 2.00 & 2.14 & 0.03 \\ 
  ENST00000381775.1 & LSP1 & 11 &  10 & 1887704 & 1908793 & 1404 & 0.00 & 0.00 & 0.00 &  56 & 117 &   3 & 58.61 & 108.17 & 11.72 & 0.21 & -0.42 & 2.52 & 0.26 & 5.00 & 4.28 & -0.11 \\ 
  ENST00000543919.1 & LTB4R2 & 14 &   1 & 24779870 & 24780947 & 1077 & 0.00 & 0.00 & 0.00 & 103 & 140 &   6 & 113.49 & 204.56 & 7.35 & 0.61 & 2.21 & 0.49 & 0.00 & 0.00 & 1.39 & 0.35 \\ 
  ENST00000543919.1 & LTB4R2 & 14 &   1 & 24779870 & 24780947 & 1077 & 0.00 & 0.00 & 0.00 & 103 & 140 &   6 & 113.49 & 204.56 & 7.35 & 0.61 & 2.21 & 0.49 & 0.00 & 1.00 & 2.12 & 0.25 \\ 
  ENST00000426342.1 & MDGA2 & 14 &  13 & 47311133 & 47600947 & 2184 & 0.00 & 0.00 & 0.00 &  96 & 196 &   3 & 86.35 & 232.12 & 27.83 & -0.64 & 1.16 & 4.66 & 0.99 & 4.00 & 5.90 & 0.30 \\ 
  ENST00000426342.1 & MDGA2 & 14 &  13 & 47311133 & 47600947 & 2184 & 0.00 & 0.00 & 0.00 &  96 & 196 &   3 & 86.35 & 232.12 & 27.83 & -0.64 & 1.16 & 4.66 & 0.99 & 4.00 & 5.90 & 0.30 \\ 
  ENST00000555262.1 & MFRP & 11 &  13 & 119212257 & 119217223 & 1740 & 0.00 & 0.00 & 0.00 & 125 & 282 &  14 & 89.01 & 186.56 & 19.47 & -2.36 & -3.42 & 1.23 & 0.00 & 14.00 & 4.90 & -1.03 \\ 
  ENST00000555262.1 & MFRP & 11 &  13 & 119212257 & 119217223 & 1740 & 0.00 & 0.00 & 0.00 & 125 & 282 &  14 & 89.01 & 186.56 & 19.47 & -2.36 & -3.42 & 1.23 & 0.00 & 14.00 & 4.90 & -1.03 \\ 
  ENST00000338148.3 & MRPL30 & 2 &   5 & 99802666 & 99812168 & 486 & 0.00 & 0.00 & 0.00 &  15 &  54 &   1 & 18.58 & 48.13 & 6.23 & 0.51 & -0.41 & 2.08 & 0.54 & 98.00 & 4.08 & -3.94 \\ 
  ENST00000338148.3 & MRPL30 & 2 &   5 & 99802666 & 99812168 & 486 & 0.00 & 0.00 & 0.00 &  15 &  54 &   1 & 18.58 & 48.13 & 6.23 & 0.51 & -0.41 & 2.08 & 0.54 & 7.00 & 4.81 & -0.31 \\ 
  ENST00000285298.4 & MRPS17 & 7 &   2 & 56020888 & 56022871 & 393 & 0.00 & 0.00 & 0.00 &  16 &  54 &   0 & 19.82 & 43.37 & 4.22 & 0.53 & -0.79 & 2.04 & 0.74 & 2.00 & 18.82 & 1.92 \\ 
  ENST00000285298.4 & MRPS17 & 7 &   2 & 56020888 & 56022871 & 393 & 0.00 & 0.00 & 0.00 &  16 &  54 &   0 & 19.82 & 43.37 & 4.22 & 0.53 & -0.79 & 2.04 & 0.74 & 3.00 & 11.77 & 1.15 \\ 
  ENST00000407558.4 & NAA60 & 16 &   5 & 3526227 & 3534855 & 729 & 0.00 & 0.00 & 0.00 &  40 &  61 &   0 & 36.91 & 73.38 & 7.44 & -0.31 & 0.71 & 2.70 & 0.91 & 4.00 & 2.35 & -0.31 \\ 
  ENST00000407558.4 & NAA60 & 16 &   5 & 3526227 & 3534855 & 729 & 0.00 & 0.00 & 0.00 &  40 &  61 &   0 & 36.91 & 73.38 & 7.44 & -0.31 & 0.71 & 2.70 & 0.91 & 8.00 & 3.29 & -0.70 \\ 
  ENST00000301457.2 & NDUFA7 & 19 &   4 & 8376388 & 8386242 & 342 & 0.00 & 0.00 & 0.00 &  28 &  53 &   4 & 22.16 & 47.41 & 5.31 & -0.77 & -0.40 & 0.56 & 0.00 & 9.00 & 4.90 & -0.54 \\ 
  ENST00000301457.2 & NDUFA7 & 19 &   4 & 8376388 & 8386242 & 342 & 0.00 & 0.00 & 0.00 &  28 &  53 &   4 & 22.16 & 47.41 & 5.31 & -0.77 & -0.40 & 0.56 & 0.00 & 9.00 & 4.31 & -0.64 \\ 
  ENST00000299166.4 & NDUFB8 & 10 &   5 & 102283593 & 102289608 & 561 & 0.00 & 0.00 & 0.00 &  29 &  78 &   2 & 39.13 & 84.90 & 8.50 & 1.00 & 0.37 & 2.21 & 0.35 & 3.00 & 5.23 & 0.38 \\ 
  ENST00000299166.4 & NDUFB8 & 10 &   5 & 102283593 & 102289608 & 561 & 0.00 & 0.00 & 0.00 &  29 &  78 &   2 & 39.13 & 84.90 & 8.50 & 1.00 & 0.37 & 2.21 & 0.35 & 3.00 & 4.39 & 0.24 \\ 
  ENST00000254627.3 & OC90 & 8 &  13 & 133036727 & 133067273 & 1434 & 0.00 & 0.00 & 0.00 &  78 & 197 &  13 & 55.45 & 118.19 & 13.84 & -1.88 & -3.55 & 0.22 & 0.00 & 9.00 & 4.54 & -0.60 \\ 
  ENST00000254627.3 & OC90 & 8 &  13 & 133036727 & 133067273 & 1434 & 0.00 & 0.00 & 0.00 &  78 & 197 &  13 & 55.45 & 118.19 & 13.84 & -1.88 & -3.55 & 0.22 & 0.00 & 8.00 & 4.20 & -0.54 \\ 
  ENST00000609618.1 & PAGR1 & 16 &   3 & 29827846 & 29831075 & 765 & 0.00 & 0.00 & 0.00 &  29 &  74 &   1 & 48.34 & 98.41 & 5.80 & 1.72 & 1.20 & 1.97 & 0.50 & 47.00 & 10.04 & -2.02 \\ 
  ENST00000609618.1 & PAGR1 & 16 &   3 & 29827846 & 29831075 & 765 & 0.00 & 0.00 & 0.00 &  29 &  74 &   1 & 48.34 & 98.41 & 5.80 & 1.72 & 1.20 & 1.97 & 0.50 & 47.00 & 9.72 & -2.05 \\ 
  ENST00000370631.3 & PI4K2A & 10 &   9 & 99400499 & 99433499 & 1440 & 0.00 & 0.00 & 0.00 &  64 &  72 &   1 & 88.07 & 193.54 & 16.12 & 1.59 & 4.27 & 3.73 & 0.98 & 2.00 & 5.43 & 0.61 \\ 
  ENST00000370631.3 & PI4K2A & 10 &   9 & 99400499 & 99433499 & 1440 & 0.00 & 0.00 & 0.00 &  64 &  72 &   1 & 88.07 & 193.54 & 16.12 & 1.59 & 4.27 & 3.73 & 0.98 & 2.00 & 5.13 & 0.56 \\ 
  ENST00000222254.8 & PIK3R2 & 19 &  15 & 18266689 & 18280104 & 2187 & 0.00 & 0.00 & 0.00 & 108 & 178 &   3 & 142.12 & 281.92 & 21.84 & 1.77 & 3.03 & 3.99 & 0.94 & 3.00 & 4.40 & 0.25 \\ 
  ENST00000222254.8 & PIK3R2 & 19 &  15 & 18266689 & 18280104 & 2187 & 0.00 & 0.00 & 0.00 & 108 & 178 &   3 & 142.12 & 281.92 & 21.84 & 1.77 & 3.03 & 3.99 & 0.94 & 3.00 & 4.60 & 0.28 \\ 
  ENST00000418988.2 & PSMA1 & 11 &  10 & 14526737 & 14632532 & 810 & 0.00 & 0.00 & 0.00 &  45 &  61 &   1 & 32.34 & 86.00 & 13.45 & -1.38 & 1.32 & 3.36 & 0.95 & 5.00 & 6.05 & 0.15 \\ 
  ENST00000418988.2 & PSMA1 & 11 &  10 & 14526737 & 14632532 & 810 & 0.00 & 0.00 & 0.00 &  45 &  61 &   1 & 32.34 & 86.00 & 13.45 & -1.38 & 1.32 & 3.36 & 0.95 & 5.00 & 6.13 & 0.17 \\ 
  ENST00000223321.4 & PSMA2 & 7 &   8 & 42957172 & 42971757 & 705 & 0.00 & 0.00 & 0.00 &  37 &  33 &   1 & 32.82 & 72.42 & 12.92 & -0.45 & 2.27 & 3.29 & 0.94 & 3.00 & 4.92 & 0.33 \\ 
  ENST00000223321.4 & PSMA2 & 7 &   8 & 42957172 & 42971757 & 705 & 0.00 & 0.00 & 0.00 &  37 &  33 &   1 & 32.82 & 72.42 & 12.92 & -0.45 & 2.27 & 3.29 & 0.94 & 3.00 & 4.86 & 0.32 \\ 
  ENST00000610205.1 & RNASE11 & 14 &   1 & 21052033 & 21052633 & 600 & 0.00 & 0.00 & 0.00 &  31 &  67 &   4 & 25.00 & 57.35 & 3.11 & -0.74 & -0.62 & -0.50 & 0.00 & 2.00 & 4.46 & 0.46 \\ 
  ENST00000610205.1 & RNASE11 & 14 &   1 & 21052033 & 21052633 & 600 & 0.00 & 0.00 & 0.00 &  31 &  67 &   4 & 25.00 & 57.35 & 3.11 & -0.74 & -0.62 & -0.50 & 0.00 & 2.00 & 4.65 & 0.49 \\ 
  ENST00000375549.3 & SDHD & 11 &   4 & 111957631 & 111965694 & 480 & 0.00 & 0.00 & 0.00 &  14 &  50 &   1 & 25.31 & 51.46 & 8.05 & 1.39 & 0.10 & 2.46 & 0.71 & 16.00 & 7.70 & -0.80 \\ 
  ENST00000375549.3 & SDHD & 11 &   4 & 111957631 & 111965694 & 480 & 0.00 & 0.00 & 0.00 &  14 &  50 &   1 & 25.31 & 51.46 & 8.05 & 1.39 & 0.10 & 2.46 & 0.71 & 2.00 & 7.07 & 0.83 \\ 
  ENST00000321367.3 & SEPT1 & 16 &  12 & 30389754 & 30394125 & 1245 & 0.00 & 0.00 & 0.00 &  54 & 116 &   7 & 71.05 & 167.73 & 16.59 & 1.25 & 1.95 & 2.33 & 0.00 & 2.00 & 4.95 & 0.53 \\ 
  ENST00000321367.3 & SEPT1 & 16 &  12 & 30389754 & 30394125 & 1245 & 0.00 & 0.00 & 0.00 &  54 & 116 &   7 & 71.05 & 167.73 & 16.59 & 1.25 & 1.95 & 2.33 & 0.00 & 2.00 & 4.95 & 0.53 \\ 
  ENST00000225519.3 & SHPK & 17 &   7 & 3513853 & 3539513 & 1437 & 0.00 & 0.00 & 0.00 &  97 & 183 &  11 & 110.21 & 203.03 & 17.04 & 0.78 & 0.69 & 1.45 & 0.00 & 72.00 & 4.43 & -3.40 \\ 
  ENST00000225519.3 & SHPK & 17 &   7 & 3513853 & 3539513 & 1437 & 0.00 & 0.00 & 0.00 &  97 & 183 &  11 & 110.21 & 203.03 & 17.04 & 0.78 & 0.69 & 1.45 & 0.00 & 81.00 & 5.27 & -3.45 \\ 
  ENST00000331531.5 & SLC25A10 & 17 &  11 & 79679456 & 79687107 & 891 & 0.00 & 0.00 & 0.00 &  65 & 109 &   6 & 68.73 & 130.82 & 11.84 & 0.28 & 0.93 & 1.68 & 0.00 & 9.00 & 7.01 & -0.24 \\ 
  ENST00000331531.5 & SLC25A10 & 17 &  11 & 79679456 & 79687107 & 891 & 0.00 & 0.00 & 0.00 &  65 & 109 &   6 & 68.73 & 130.82 & 11.84 & 0.28 & 0.93 & 1.68 & 0.00 & 9.00 & 7.01 & -0.24 \\ 
  ENST00000381151.3 & SLC5A3 & 21 &   1 & 35467497 & 35469654 & 2157 & 0.00 & 0.00 & 0.00 & 114 & 157 &   1 & 84.81 & 223.83 & 10.32 & -1.96 & 2.18 & 2.87 & 0.85 & 2.00 & 1.38 & -0.14 \\ 
  ENST00000381151.3 & SLC5A3 & 21 &   1 & 35467497 & 35469654 & 2157 & 0.00 & 0.00 & 0.00 & 114 & 157 &   1 & 84.81 & 223.83 & 10.32 & -1.96 & 2.18 & 2.87 & 0.85 & 2.00 & 1.38 & -0.14 \\ 
  ENST00000338094.6 & SNURF & 15 &   3 & 25200195 & 25213184 & 216 & 0.00 & 0.00 & 0.00 &   8 &  27 &   0 & 10.65 & 29.63 & 4.76 & 0.50 & 0.24 & 2.16 & 0.78 & 10.00 & 6.69 & -0.39 \\ 
  ENST00000338094.6 & SNURF & 15 &   3 & 25200195 & 25213184 & 216 & 0.00 & 0.00 & 0.00 &   8 &  27 &   0 & 10.65 & 29.63 & 4.76 & 0.50 & 0.24 & 2.16 & 0.78 & 10.00 & 5.71 & -0.53 \\ 
  ENST00000304501.1 & SOX7 & 8 &   2 & 10583247 & 10587943 & 1167 & 0.00 & 0.00 & 0.00 & 111 & 183 &   2 & 102.80 & 180.80 & 5.32 & -0.50 & -0.08 & 1.43 & 0.13 & 10.00 & 2.41 & -1.09 \\ 
  ENST00000304501.1 & SOX7 & 8 &   2 & 10583247 & 10587943 & 1167 & 0.00 & 0.00 & 0.00 & 111 & 183 &   2 & 102.80 & 180.80 & 5.32 & -0.50 & -0.08 & 1.43 & 0.13 & 10.00 & 2.30 & -1.12 \\ 
  ENST00000374899.4 & TAP2 & 6 &  11 & 32790065 & 32806010 & 1962 & 0.00 & 0.00 & 0.00 &  89 & 176 &   9 & 109.53 & 238.31 & 23.93 & 1.22 & 1.97 & 3.02 & 0.00 & 10.00 & 5.24 & -0.60 \\ 
  ENST00000374899.4 & TAP2 & 6 &  11 & 32790065 & 32806010 & 1962 & 0.00 & 0.00 & 0.00 &  89 & 176 &   9 & 109.53 & 238.31 & 23.93 & 1.22 & 1.97 & 3.02 & 0.00 & 10.00 & 4.42 & -0.73 \\ 
  ENST00000254616.6 & TIMM10B & 11 &   3 & 6502746 & 6503451 & 312 & 0.00 & 0.00 & 0.00 &  17 &  53 &   5 & 20.80 & 48.60 & 6.51 & 0.52 & -0.31 & 0.59 & 0.00 & 2.00 & 5.37 & 0.60 \\ 
  ENST00000254616.6 & TIMM10B & 11 &   3 & 6502746 & 6503451 & 312 & 0.00 & 0.00 & 0.00 &  17 &  53 &   5 & 20.80 & 48.60 & 6.51 & 0.52 & -0.31 & 0.59 & 0.00 & 2.00 & 3.76 & 0.34 \\ 
  ENST00000360658.2 & TLR9 & 3 &   2 & 52255232 & 52259545 & 3099 & 0.00 & 0.00 & 0.00 & 199 & 327 &   8 & 197.86 & 448.55 & 16.43 & -0.05 & 2.81 & 2.06 & 0.00 & 0.00 & 2.51 & 0.60 \\ 
  ENST00000360658.2 & TLR9 & 3 &   2 & 52255232 & 52259545 & 3099 & 0.00 & 0.00 & 0.00 & 199 & 327 &   8 & 197.86 & 448.55 & 16.43 & -0.05 & 2.81 & 2.06 & 0.00 & 0.00 & 3.77 & 0.83 \\ 
  ENST00000273695.3 & TM4SF19 & 3 &   4 & 196050687 & 196054461 & 630 & 0.00 & 0.00 & 0.00 &  37 &  60 &   4 & 34.05 & 70.21 & 7.86 & -0.31 & 0.60 & 1.36 & 0.01 & 14.00 & 4.88 & -1.03 \\ 
  ENST00000273695.3 & TM4SF19 & 3 &   4 & 196050687 & 196054461 & 630 & 0.00 & 0.00 & 0.00 &  37 &  60 &   4 & 34.05 & 70.21 & 7.86 & -0.31 & 0.60 & 1.36 & 0.01 & 14.00 & 4.70 & -1.06 \\ 
  ENST00000261789.4 & TM9SF1 & 14 &   5 & 24658620 & 24664225 & 1821 & 0.00 & 0.00 & 0.00 &  98 & 177 &   9 & 87.83 & 234.60 & 17.77 & -0.67 & 1.84 & 2.06 & 0.00 & 5.00 & 7.89 & 0.40 \\ 
  ENST00000261789.4 & TM9SF1 & 14 &   5 & 24658620 & 24664225 & 1821 & 0.00 & 0.00 & 0.00 &  98 & 177 &   9 & 87.83 & 234.60 & 17.77 & -0.67 & 1.84 & 2.06 & 0.00 & 5.00 & 7.89 & 0.40 \\ 
  ENST00000358230.3 & TMBIM4 & 12 &   7 & 66531739 & 66563731 & 717 & 0.00 & 0.00 & 0.00 &  25 &  58 &   5 & 31.21 & 63.10 & 7.59 & 0.69 & 0.31 & 0.93 & 0.00 & 13.00 & 4.66 & -0.98 \\ 
  ENST00000358230.3 & TMBIM4 & 12 &   7 & 66531739 & 66563731 & 717 & 0.00 & 0.00 & 0.00 &  25 &  58 &   5 & 31.21 & 63.10 & 7.59 & 0.69 & 0.31 & 0.93 & 0.00 & 13.00 & 4.72 & -0.97 \\ 
  ENST00000503640.1 & UGT2A1 & 4 &   6 & 70455089 & 70513362 & 1584 & 0.00 & 0.00 & 0.00 &  75 & 215 &  16 & 59.31 & 149.78 & 15.41 & -1.26 & -2.61 & -0.15 & 0.00 & 13.00 & 4.41 & -1.02 \\ 
  ENST00000503640.1 & UGT2A1 & 4 &   6 & 70455089 & 70513362 & 1584 & 0.00 & 0.00 & 0.00 &  75 & 215 &  16 & 59.31 & 149.78 & 15.41 & -1.26 & -2.61 & -0.15 & 0.00 & 13.00 & 4.15 & -1.06 \\ 
  ENST00000591899.3 & UQCR11 & 19 &   2 & 1599438 & 1605408 & 171 & 0.00 & 0.00 & 0.00 &  15 &  28 &   2 & 12.52 & 26.17 & 2.14 & -0.43 & -0.18 & 0.10 & 0.03 & 3.00 & 3.76 & 0.14 \\ 
  ENST00000591899.3 & UQCR11 & 19 &   2 & 1599438 & 1605408 & 171 & 0.00 & 0.00 & 0.00 &  15 &  28 &   2 & 12.52 & 26.17 & 2.14 & -0.43 & -0.18 & 0.10 & 0.03 & 0.00 & 2.81 & 0.66 \\ 
  ENST00000330701.4 & ZFP41 & 8 &   1 & 144332013 & 144332610 & 597 & 0.00 & 0.00 & 0.00 &  41 & 101 &   4 & 42.09 & 87.86 & 2.58 & 0.10 & -0.69 & -0.88 & 0.00 & 2.00 & 1.78 & -0.05 \\ 
  ENST00000330701.4 & ZFP41 & 8 &   1 & 144332013 & 144332610 & 597 & 0.00 & 0.00 & 0.00 &  41 & 101 &   4 & 42.09 & 87.86 & 2.58 & 0.10 & -0.69 & -0.88 & 0.00 & 1.00 & 3.23 & 0.47 \\ 
  ENST00000357182.4 & ZMYM6 & 1 &  15 & 35452704 & 35496240 & 3978 & 0.00 & 0.00 & 0.00 & 109 & 302 &  19 & 121.20 & 317.71 & 36.71 & 0.69 & 0.43 & 2.90 & 0.00 & 2.00 & 5.72 & 0.65 \\ 
  ENST00000357182.4 & ZMYM6 & 1 &  15 & 35452704 & 35496240 & 3978 & 0.00 & 0.00 & 0.00 & 109 & 302 &  19 & 121.20 & 317.71 & 36.71 & 0.69 & 0.43 & 2.90 & 0.00 & 2.00 & 5.72 & 0.65 \\ 
  ENST00000434772.3 & ZNF223 & 19 &   4 & 44559329 & 44571430 & 1449 & 0.00 & 0.00 & 0.00 &  49 & 152 &  13 & 50.58 & 144.26 & 13.29 & 0.14 & -0.32 & 0.08 & 0.00 & 10.00 & 6.87 & -0.37 \\ 
  ENST00000434772.3 & ZNF223 & 19 &   4 & 44559329 & 44571430 & 1449 & 0.00 & 0.00 & 0.00 &  49 & 152 &  13 & 50.58 & 144.26 & 13.29 & 0.14 & -0.32 & 0.08 & 0.00 & 7.00 & 7.76 & 0.10 \\ 
  ENST00000539836.3 & ZNF668 & 16 &   3 & 31072388 & 31075939 & 1929 & 0.00 & 0.00 & 0.00 & 122 & 187 &   3 & 173.30 & 369.35 & 11.85 & 2.42 & 4.64 & 2.55 & 0.27 & 4.00 & 1.99 & -0.39 \\ 
  ENST00000539836.3 & ZNF668 & 16 &   3 & 31072388 & 31075939 & 1929 & 0.00 & 0.00 & 0.00 & 122 & 187 &   3 & 173.30 & 369.35 & 11.85 & 2.42 & 4.64 & 2.55 & 0.27 & 4.00 & 2.90 & -0.20 \\ 
  ENST00000397732.3 & ZNF709 & 19 &   4 & 12574809 & 12595471 & 1926 & 0.00 & 0.00 & 0.00 &  60 & 152 &  13 & 59.29 & 183.63 & 26.30 & -0.06 & 1.14 & 2.57 & 0.00 & 4.00 & 4.29 & 0.05 \\ 
  ENST00000397732.3 & ZNF709 & 19 &   4 & 12574809 & 12595471 & 1926 & 0.00 & 0.00 & 0.00 &  60 & 152 &  13 & 59.29 & 183.63 & 26.30 & -0.06 & 1.14 & 2.57 & 0.00 & 6.00 & 3.35 & -0.42 \\ 
  ENST00000252799.3 & ZNF747 & 16 &   2 & 30544379 & 30546000 & 576 & 0.00 & 0.00 & 0.00 &  28 &  68 &   6 & 39.79 & 74.07 & 4.21 & 1.16 & 0.35 & -0.86 & 0.00 & 0.00 & 1.36 & 0.35 \\ 
  ENST00000252799.3 & ZNF747 & 16 &   2 & 30544379 & 30546000 & 576 & 0.00 & 0.00 & 0.00 &  28 &  68 &   6 & 39.79 & 74.07 & 4.21 & 1.16 & 0.35 & -0.86 & 0.00 & 0.00 & 1.36 & 0.35 \\ 
  ENST00000343949.5 & ZNF763 & 19 &   4 & 12076023 & 12089924 & 1194 & 0.00 & 0.00 & 0.00 &  38 & 168 &   7 & 46.30 & 115.31 & 10.56 & 0.76 & -2.40 & 1.09 & 0.00 & 20.00 & 7.21 & -1.14 \\ 
  ENST00000343949.5 & ZNF763 & 19 &   4 & 12076023 & 12089924 & 1194 & 0.00 & 0.00 & 0.00 &  38 & 168 &   7 & 46.30 & 115.31 & 10.56 & 0.76 & -2.40 & 1.09 & 0.00 & 20.00 & 6.39 & -1.25 \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Duplicate exac data. Differing only in the number of copy number variation. Duplicate values were discarded}
\label{tab:Supplementary table- duplicate exac data}
\end{table}



\section{GO enrichment centrality measures}


\subsection{GO Degree centrality topgo}

Low degree Biological process 0.05 centile 304 nodes degree less than or equal to 1 no significant after correcting for multiple comparisons table~\ref{tab:304 degree centrality 0.05centile  1 BP background PSP. Alpha=4.5583006655119e-06}

High degree Biological process 0.95 centile 174 nodes degree centrality greater than or equal to 59 table~\ref{tab:Number of nodes 174 degree centrality 0.95centile  degree centrality >= 59 BP background PSP. Alpha = 4.5583006655119e-06}

Low degree molecular function 304 nodes degree less than or equal to 1  table~\ref{tab:Number of nodes 304 degree centrality 0.05 centile  degree centrality <= 1MF background PSP. Alpha=2.02265372168285e-05} none significant after correction for multiple comparisons.

High degree centrality molecular fucntion 174 nodes degree greater than or equal to 59 \ref{tab:Number of nodes 174 degree centrality 0.95centile  degree centrality >= 59 MF background PSP. Alpha = 2.02265372168285e-05}

Low degree cellular component topGO  table~\ref{tab:Number of nodes 304 degree centrality 0.05 centile  degree centrality <= 1CC background PSP. Alpha=3.33555703802535e-05}
High degree Cellular component
    table~\ref{tab:Number of nodes 174 degree centrality 0.95 centile  degree centrality >= 59 CC background PSP. Alpha = 3.33555703802535e-05}
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0009190 & cyclic nucleotide biosynthetic process & 9 & 5 & 0.8 & $3.90 \times 10^{-4}$ & FALSE \\ 
  GO:0052652 & cyclic purine nucleotide metabolic proce... & 9 & 5 & 0.8 & $3.90 \times 10^{-4}$ & FALSE \\ 
  GO:0097035 & regulation of membrane lipid distributio... & 9 & 5 & 0.8 & $3.90 \times 10^{-4}$ & FALSE \\ 
  GO:0038171 & cannabinoid signaling pathway & 3 & 3 & 0.2 & $5.90 \times 10^{-4}$ & FALSE \\ 
  GO:0071926 & endocannabinoid signaling pathway & 3 & 3 & 0.2 & $5.90 \times 10^{-4}$ & FALSE \\ 
  GO:0009187 & cyclic nucleotide metabolic process & 16 & 6 & 1.4 & $1.31 \times 10^{-3}$ & FALSE \\ 
  GO:0006171 & cAMP biosynthetic process & 4 & 3 & 0.3 & $2.22 \times 10^{-3}$ & FALSE \\ 
  GO:0021681 & cerebellar granular layer development & 8 & 4 & 0.7 & $2.62 \times 10^{-3}$ & FALSE \\ 
  GO:0046058 & cAMP metabolic process & 8 & 4 & 0.7 & $2.62 \times 10^{-3}$ & FALSE \\ 
  GO:0046339 & diacylglycerol metabolic process & 8 & 4 & 0.7 & $2.62 \times 10^{-3}$ & FALSE \\ 
  GO:1904321 & response to forskolin & 5 & 3 & 0.4 & $5.20 \times 10^{-3}$ & FALSE \\ 
  GO:1904322 & cellular response to forskolin & 5 & 3 & 0.4 & $5.20 \times 10^{-3}$ & FALSE \\ 
  GO:0006639 & acylglycerol metabolic process & 27 & 7 & 2.3 & $5.67 \times 10^{-3}$ & FALSE \\ 
  GO:0046461 & neutral lipid catabolic process & 10 & 4 & 0.8 & $6.87 \times 10^{-3}$ & FALSE \\ 
  GO:0046464 & acylglycerol catabolic process & 10 & 4 & 0.8 & $6.87 \times 10^{-3}$ & FALSE \\ 
  GO:0006638 & neutral lipid metabolic process & 28 & 7 & 2.4 & $7.03 \times 10^{-3}$ & FALSE \\ 
  GO:0001539 & cilium or flagellum-dependent cell motil... & 2 & 2 & 0.2 & $7.07 \times 10^{-3}$ & FALSE \\ 
  GO:0006741 & NADP biosynthetic process & 2 & 2 & 0.2 & $7.07 \times 10^{-3}$ & FALSE \\ 
  GO:0007168 & receptor guanylyl cyclase signaling path... & 2 & 2 & 0.2 & $7.07 \times 10^{-3}$ & FALSE \\ 
  GO:0007620 & copulation & 2 & 2 & 0.2 & $7.07 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{304 degree centrality 0.05centile degree $<=$ 1 BP background PSP. Alpha=4.5583006655119e-06} 
\label{tab:304 degree centrality 0.05centile  1 BP background PSP. Alpha=4.5583006655119e-06}
\end{table}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 14:48:36 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0007049 & cell cycle & 458 & 78 & 23.7 & $3.00 \times 10^{-25}$ & $3.29 \times 10^{-21}$ \\ 
  GO:0051246 & regulation of protein metabolic process & 677 & 90 & 35.1 & $1.20 \times 10^{-21}$ & $1.32 \times 10^{-17}$ \\ 
  GO:0009057 & macromolecule catabolic process & 431 & 70 & 22.3 & $6.00 \times 10^{-21}$ & $6.58 \times 10^{-17}$ \\ 
  GO:0060255 & regulation of macromolecule metabolic pr... & 1167 & 119 & 60.5 & $1.40 \times 10^{-20}$ & $1.54 \times 10^{-16}$ \\ 
  GO:0051171 & regulation of nitrogen compound metaboli... & 1070 & 113 & 55.5 & $3.00 \times 10^{-20}$ & $3.29 \times 10^{-16}$ \\ 
  GO:0080090 & regulation of primary metabolic process & 1111 & 115 & 57.6 & $5.10 \times 10^{-20}$ & $5.59 \times 10^{-16}$ \\ 
  GO:0010604 & positive regulation of macromolecule met... & 658 & 86 & 34.1 & $7.20 \times 10^{-20}$ & $7.90 \times 10^{-16}$ \\ 
  GO:0009893 & positive regulation of metabolic process & 733 & 90 & 38.0 & $4.50 \times 10^{-19}$ & $4.94 \times 10^{-15}$ \\ 
  GO:0022402 & cell cycle process & 362 & 61 & 18.8 & $8.90 \times 10^{-19}$ & $9.76 \times 10^{-15}$ \\ 
  GO:0051128 & regulation of cellular component organiz... & 849 & 96 & 44.0 & $4.70 \times 10^{-18}$ & $5.16 \times 10^{-14}$ \\ 
  GO:0006950 & response to stress & 872 & 97 & 45.2 & $9.00 \times 10^{-18}$ & $9.87 \times 10^{-14}$ \\ 
  GO:0051726 & regulation of cell cycle & 302 & 54 & 15.7 & $1.20 \times 10^{-17}$ & $1.32 \times 10^{-13}$ \\ 
  GO:0032268 & regulation of cellular protein metabolic... & 623 & 80 & 32.3 & $1.20 \times 10^{-17}$ & $1.32 \times 10^{-13}$ \\ 
  GO:0031323 & regulation of cellular metabolic process & 1167 & 114 & 60.5 & $1.80 \times 10^{-17}$ & $1.97 \times 10^{-13}$ \\ 
  GO:0033554 & cellular response to stress & 515 & 71 & 26.7 & $4.90 \times 10^{-17}$ & $5.37 \times 10^{-13}$ \\ 
  GO:0019222 & regulation of metabolic process & 1312 & 121 & 68.0 & $5.60 \times 10^{-17}$ & $6.14 \times 10^{-13}$ \\ 
  GO:0051704 & multi-organism process & 597 & 77 & 30.9 & $6.00 \times 10^{-17}$ & $6.58 \times 10^{-13}$ \\ 
  GO:0051173 & positive regulation of nitrogen compound... & 625 & 79 & 32.4 & $6.20 \times 10^{-17}$ & $6.80 \times 10^{-13}$ \\ 
  GO:0048523 & negative regulation of cellular process & 1062 & 107 & 55.1 & $7.00 \times 10^{-17}$ & $7.68 \times 10^{-13}$ \\ 
  GO:0009894 & regulation of catabolic process & 326 & 55 & 16.9 & $8.40 \times 10^{-17}$ & $9.21 \times 10^{-13}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 174 degree centrality 0.95centile  degree centrality $>=$ 59 BP background PSP. Alpha = 4.5583006655119e-06} 
\label{tab:Number of nodes 174 degree centrality 0.95centile  degree centrality >= 59 BP background PSP. Alpha = 4.5583006655119e-06}
\end{table}



%%%%%% MF %%%%% degree topgo

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 14:57:48 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0022857 & transmembrane transporter activity & 250 & 38 & 20.0 & $5.30 \times 10^{-5}$ & FALSE \\ 
  GO:0005215 & transporter activity & 270 & 39 & 21.6 & $1.30 \times 10^{-4}$ & FALSE \\ 
  GO:0016849 & phosphorus-oxygen lyase activity & 8 & 5 & 0.6 & $1.50 \times 10^{-4}$ & FALSE \\ 
  GO:0015075 & ion transmembrane transporter activity & 223 & 33 & 17.9 & $2.90 \times 10^{-4}$ & FALSE \\ 
  GO:0009975 & cyclase activity & 9 & 5 & 0.7 & $3.10 \times 10^{-4}$ & FALSE \\ 
  GO:0008514 & organic anion transmembrane transporter ... & 36 & 10 & 2.9 & $3.60 \times 10^{-4}$ & FALSE \\ 
  GO:0015081 & sodium ion transmembrane transporter act... & 33 & 9 & 2.6 & $8.20 \times 10^{-4}$ & FALSE \\ 
  GO:0008509 & anion transmembrane transporter activity & 55 & 12 & 4.4 & $1.05 \times 10^{-3}$ & FALSE \\ 
  GO:0015291 & secondary active transmembrane transport... & 29 & 8 & 2.3 & $1.48 \times 10^{-3}$ & FALSE \\ 
  GO:0004016 & adenylate cyclase activity & 4 & 3 & 0.3 & $1.91 \times 10^{-3}$ & FALSE \\ 
  GO:0046872 & metal ion binding & 706 & 76 & 56.6 & $2.00 \times 10^{-3}$ & FALSE \\ 
  GO:0015318 & inorganic molecular entity transmembrane... & 205 & 28 & 16.4 & $2.98 \times 10^{-3}$ & FALSE \\ 
  GO:0043169 & cation binding & 718 & 76 & 57.5 & $3.25 \times 10^{-3}$ & FALSE \\ 
  GO:0046873 & metal ion transmembrane transporter acti... & 123 & 19 & 9.8 & $3.61 \times 10^{-3}$ & FALSE \\ 
  GO:0022804 & active transmembrane transporter activit... & 83 & 14 & 6.7 & $5.43 \times 10^{-3}$ & FALSE \\ 
  GO:0001758 & retinal dehydrogenase activity & 2 & 2 & 0.2 & $6.40 \times 10^{-3}$ & FALSE \\ 
  GO:0005385 & zinc ion transmembrane transporter activ... & 2 & 2 & 0.2 & $6.40 \times 10^{-3}$ & FALSE \\ 
  GO:0008146 & sulfotransferase activity & 2 & 2 & 0.2 & $6.40 \times 10^{-3}$ & FALSE \\ 
  GO:0008332 & low voltage-gated calcium channel activi... & 2 & 2 & 0.2 & $6.40 \times 10^{-3}$ & FALSE \\ 
  GO:0031419 & cobalamin binding & 2 & 2 & 0.2 & $6.40 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 304 degree centrality 0.05 centile  degree centrality <= 1MF background PSP. Alpha=2.02265372168285e-05} 
\label{tab:Number of nodes 304 degree centrality 0.05 centile  degree centrality <= 1MF background PSP. Alpha=2.02265372168285e-05}
\end{table}

%%%% high degree MF topgo %%%

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 14:55:02 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0019899 & enzyme binding & 717 & 99 & 37.7 & $2.00 \times 10^{-25}$ & $4.94 \times 10^{-22}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 114 & 33 & 6.0 & $4.40 \times 10^{-17}$ & $1.09 \times 10^{-13}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 108 & 32 & 5.7 & $6.50 \times 10^{-17}$ & $1.61 \times 10^{-13}$ \\ 
  GO:0019904 & protein domain specific binding & 273 & 49 & 14.4 & $9.40 \times 10^{-16}$ & $2.32 \times 10^{-12}$ \\ 
  GO:0019900 & kinase binding & 250 & 43 & 13.2 & $4.20 \times 10^{-13}$ & $1.04 \times 10^{-9}$ \\ 
  GO:0019901 & protein kinase binding & 225 & 40 & 11.8 & $1.10 \times 10^{-12}$ & $2.72 \times 10^{-9}$ \\ 
  GO:0005515 & protein binding & 2796 & 173 & 147.1 & $2.90 \times 10^{-12}$ & $7.17 \times 10^{-9}$ \\ 
  GO:0008134 & transcription factor binding & 127 & 29 & 6.7 & $3.70 \times 10^{-12}$ & $9.15 \times 10^{-9}$ \\ 
  GO:0051219 & phosphoprotein binding & 32 & 15 & 1.7 & $9.30 \times 10^{-12}$ & $2.30 \times 10^{-8}$ \\ 
  GO:0042802 & identical protein binding & 487 & 59 & 25.6 & $5.10 \times 10^{-11}$ & $1.26 \times 10^{-7}$ \\ 
  GO:0044877 & protein-containing complex binding & 415 & 51 & 21.8 & $1.10 \times 10^{-9}$ & $2.72 \times 10^{-6}$ \\ 
  GO:0003676 & nucleic acid binding & 716 & 70 & 37.7 & $9.20 \times 10^{-9}$ & $2.27 \times 10^{-5}$ \\ 
  GO:0045296 & cadherin binding & 219 & 33 & 11.5 & $1.30 \times 10^{-8}$ & $3.21 \times 10^{-5}$ \\ 
  GO:0050839 & cell adhesion molecule binding & 272 & 37 & 14.3 & $2.60 \times 10^{-8}$ & $6.43 \times 10^{-5}$ \\ 
  GO:0003723 & RNA binding & 549 & 57 & 28.9 & $5.50 \times 10^{-8}$ & $1.36 \times 10^{-4}$ \\ 
  GO:0042826 & histone deacetylase binding & 25 & 10 & 1.3 & $2.10 \times 10^{-7}$ & $5.19 \times 10^{-4}$ \\ 
  GO:0001085 & RNA polymerase II transcription factor b... & 17 & 8 & 0.9 & $8.10 \times 10^{-7}$ & $2.00 \times 10^{-3}$ \\ 
  GO:0045309 & protein phosphorylated amino acid bindin... & 17 & 8 & 0.9 & $8.10 \times 10^{-7}$ & $2.00 \times 10^{-3}$ \\ 
  GO:0048156 & tau protein binding & 36 & 11 & 1.9 & $1.20 \times 10^{-6}$ & $2.97 \times 10^{-3}$ \\ 
  GO:0031072 & heat shock protein binding & 52 & 13 & 2.7 & $1.60 \times 10^{-6}$ & $3.96 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 174 degree centrality 0.95centile  degree centrality >= 59 MF background PSP. Alpha = 2.02265372168285e-05} 
\label{tab:Number of nodes 174 degree centrality 0.95centile  degree centrality >= 59 MF background PSP. Alpha = 2.02265372168285e-05}
\end{table}
  
 \begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0003723 & RNA binding & 549 & 97 & 28.7 & $1.00 \times 10^{-30}$ & $2.47 \times 10^{-27}$ \\ 
  GO:0003676 & nucleic acid binding & 716 & 107 & 37.4 & $1.00 \times 10^{-30}$ & $2.47 \times 10^{-27}$ \\ 
  GO:0097159 & organic cyclic compound binding & 1366 & 130 & 71.4 & $1.70 \times 10^{-20}$ & $4.20 \times 10^{-17}$ \\ 
  GO:1901363 & heterocyclic compound binding & 1355 & 129 & 70.9 & $3.30 \times 10^{-20}$ & $8.16 \times 10^{-17}$ \\ 
  GO:0019899 & enzyme binding & 717 & 86 & 37.5 & $4.90 \times 10^{-17}$ & $1.21 \times 10^{-13}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 108 & 32 & 5.7 & $5.40 \times 10^{-17}$ & $1.33 \times 10^{-13}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 114 & 32 & 6.0 & $3.20 \times 10^{-16}$ & $7.91 \times 10^{-13}$ \\ 
  GO:0003729 & mRNA binding & 87 & 25 & 4.5 & $4.40 \times 10^{-13}$ & $1.09 \times 10^{-9}$ \\ 
  GO:0019904 & protein domain specific binding & 273 & 44 & 14.3 & $1.80 \times 10^{-12}$ & $4.45 \times 10^{-9}$ \\ 
  GO:0008134 & transcription factor binding & 127 & 29 & 6.6 & $3.20 \times 10^{-12}$ & $7.91 \times 10^{-9}$ \\ 
  GO:0051082 & unfolded protein binding & 56 & 19 & 2.9 & $1.30 \times 10^{-11}$ & $3.21 \times 10^{-8}$ \\ 
  GO:0005515 & protein binding & 2796 & 171 & 146.2 & $5.90 \times 10^{-11}$ & $1.46 \times 10^{-7}$ \\ 
  GO:0003677 & DNA binding & 251 & 38 & 13.1 & $5.80 \times 10^{-10}$ & $1.43 \times 10^{-6}$ \\ 
  GO:0045296 & cadherin binding & 219 & 33 & 11.4 & $1.10 \times 10^{-8}$ & $2.72 \times 10^{-5}$ \\ 
  GO:0003727 & single-stranded RNA binding & 31 & 12 & 1.6 & $1.70 \times 10^{-8}$ & $4.20 \times 10^{-5}$ \\ 
  GO:0050839 & cell adhesion molecule binding & 272 & 37 & 14.2 & $2.20 \times 10^{-8}$ & $5.44 \times 10^{-5}$ \\ 
  GO:0043565 & sequence-specific DNA binding & 112 & 22 & 5.9 & $3.40 \times 10^{-8}$ & $8.40 \times 10^{-5}$ \\ 
  GO:0044877 & protein-containing complex binding & 415 & 47 & 21.7 & $7.40 \times 10^{-8}$ & $1.83 \times 10^{-4}$ \\ 
  GO:0019901 & protein kinase binding & 225 & 32 & 11.8 & $8.10 \times 10^{-8}$ & $2.00 \times 10^{-4}$ \\ 
  GO:0003730 & mRNA 3'-UTR binding & 29 & 11 & 1.5 & $8.90 \times 10^{-8}$ & $2.20 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242843 MF background PSP. Alpha = 2.02265372168285e-05} 
\label{tab:Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242843 MF background PSP. Alpha = 2.02265372168285e-05}
\end{table}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%55

%%%% Degree topgo CC %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0031224 & intrinsic component of membrane & 788 & 124 & 66.9 & $4.50 \times 10^{-15}$ & TRUE \\ 
  GO:0016021 & integral component of membrane & 747 & 117 & 63.4 & $7.10 \times 10^{-14}$ & TRUE \\ 
  GO:0044425 & membrane part & 1395 & 153 & 118.5 & $1.20 \times 10^{-5}$ & TRUE \\ 
  GO:0031226 & intrinsic component of plasma membrane & 308 & 48 & 26.2 & $1.20 \times 10^{-5}$ & TRUE \\ 
  GO:0005887 & integral component of plasma membrane & 288 & 45 & 24.5 & $2.30 \times 10^{-5}$ & TRUE \\ 
  GO:0099240 & intrinsic component of synaptic membrane & 67 & 14 & 5.7 & $1.20 \times 10^{-3}$ & FALSE \\ 
  GO:0060076 & excitatory synapse & 27 & 8 & 2.3 & $1.30 \times 10^{-3}$ & FALSE \\ 
  GO:0099699 & integral component of synaptic membrane & 61 & 13 & 5.2 & $1.40 \times 10^{-3}$ & FALSE \\ 
  GO:0031045 & dense core granule & 9 & 4 & 0.8 & $4.60 \times 10^{-3}$ & FALSE \\ 
  GO:0099055 & integral component of postsynaptic membr... & 47 & 10 & 4.0 & $5.10 \times 10^{-3}$ & FALSE \\ 
  GO:0005858 & axonemal dynein complex & 2 & 2 & 0.2 & $7.20 \times 10^{-3}$ & FALSE \\ 
  GO:0098936 & intrinsic component of postsynaptic memb... & 50 & 10 & 4.2 & $8.00 \times 10^{-3}$ & FALSE \\ 
  GO:0044459 & plasma membrane part & 742 & 79 & 63.0 & $1.18 \times 10^{-2}$ & FALSE \\ 
  GO:0031225 & anchored component of membrane & 38 & 8 & 3.2 & $1.26 \times 10^{-2}$ & FALSE \\ 
  GO:0032045 & guanyl-nucleotide exchange factor comple... & 7 & 3 & 0.6 & $1.64 \times 10^{-2}$ & FALSE \\ 
  GO:0098688 & parallel fiber to Purkinje cell synapse & 7 & 3 & 0.6 & $1.64 \times 10^{-2}$ & FALSE \\ 
  GO:0031012 & extracellular matrix & 82 & 13 & 7.0 & $1.93 \times 10^{-2}$ & FALSE \\ 
  GO:0098889 & intrinsic component of presynaptic membr... & 36 & 7 & 3.1 & $2.90 \times 10^{-2}$ & FALSE \\ 
  GO:0098533 & ATPase dependent transmembrane transport... & 9 & 3 & 0.8 & $3.46 \times 10^{-2}$ & FALSE \\ 
  GO:0031232 & extrinsic component of external side of ... & 4 & 2 & 0.3 & $3.84 \times 10^{-2}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 304 degree centrality 0.05 centile  degree centrality <= 1CC background PSP. Alpha=3.33555703802535e-05} 
\label{tab:Number of nodes 304 degree centrality 0.05 centile  degree centrality <= 1CC background PSP. Alpha=3.33555703802535e-05}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:02:42 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0044428 & nuclear part & 952 & 105 & 48.9 & $6.40 \times 10^{-20}$ & $9.59 \times 10^{-17}$ \\ 
  GO:0005634 & nucleus & 1439 & 129 & 73.8 & $3.20 \times 10^{-18}$ & $4.80 \times 10^{-15}$ \\ 
  GO:0005654 & nucleoplasm & 716 & 84 & 36.7 & $2.20 \times 10^{-16}$ & $3.30 \times 10^{-13}$ \\ 
  GO:0005829 & cytosol & 1643 & 135 & 84.3 & $6.40 \times 10^{-16}$ & $9.59 \times 10^{-13}$ \\ 
  GO:0031981 & nuclear lumen & 856 & 92 & 43.9 & $9.80 \times 10^{-16}$ & $1.47 \times 10^{-12}$ \\ 
  GO:0043232 & intracellular non-membrane-bounded organ... & 1252 & 113 & 64.2 & $1.30 \times 10^{-14}$ & $1.95 \times 10^{-11}$ \\ 
  GO:0043228 & non-membrane-bounded organelle & 1260 & 113 & 64.7 & $2.20 \times 10^{-14}$ & $3.30 \times 10^{-11}$ \\ 
  GO:0031974 & membrane-enclosed lumen & 1171 & 106 & 60.1 & $2.80 \times 10^{-13}$ & $4.20 \times 10^{-10}$ \\ 
  GO:0043233 & organelle lumen & 1171 & 106 & 60.1 & $2.80 \times 10^{-13}$ & $4.20 \times 10^{-10}$ \\ 
  GO:0070013 & intracellular organelle lumen & 1171 & 106 & 60.1 & $2.80 \times 10^{-13}$ & $4.20 \times 10^{-10}$ \\ 
  GO:0044446 & intracellular organelle part & 2308 & 158 & 118.4 & $3.20 \times 10^{-13}$ & $4.80 \times 10^{-10}$ \\ 
  GO:0032991 & protein-containing complex & 1540 & 125 & 79.0 & $4.00 \times 10^{-13}$ & $6.00 \times 10^{-10}$ \\ 
  GO:0044422 & organelle part & 2379 & 159 & 122.1 & $3.50 \times 10^{-12}$ & $5.25 \times 10^{-9}$ \\ 
  GO:0043231 & intracellular membrane-bounded organelle & 2301 & 155 & 118.1 & $1.90 \times 10^{-11}$ & $2.85 \times 10^{-8}$ \\ 
  GO:0044430 & cytoskeletal part & 615 & 66 & 31.6 & $1.90 \times 10^{-10}$ & $2.85 \times 10^{-7}$ \\ 
  GO:0005856 & cytoskeleton & 802 & 78 & 41.1 & $2.30 \times 10^{-10}$ & $3.45 \times 10^{-7}$ \\ 
  GO:0043229 & intracellular organelle & 2784 & 169 & 142.8 & $4.30 \times 10^{-10}$ & $6.45 \times 10^{-7}$ \\ 
  GO:0015630 & microtubule cytoskeleton & 407 & 49 & 20.9 & $2.30 \times 10^{-9}$ & $3.45 \times 10^{-6}$ \\ 
  GO:0005925 & focal adhesion & 237 & 35 & 12.2 & $3.90 \times 10^{-9}$ & $5.85 \times 10^{-6}$ \\ 
  GO:0005924 & cell-substrate adherens junction & 238 & 35 & 12.2 & $4.40 \times 10^{-9}$ & $6.60 \times 10^{-6}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 174 degree centrality 0.95 centile  degree centrality >= 59 CC background PSP. Alpha = 3.33555703802535e-05} 
\label{tab:Number of nodes 174 degree centrality 0.95 centile  degree centrality >= 59 CC background PSP. Alpha = 3.33555703802535e-05}
\end{table}


    
    
\subsection{Results topGO Eigenvector centrality}


Biological process low 0.05 centile table~\ref{tab:Number of nodes 173 eigen centrality 0.05 centile  eigen centrality $<=$ 0.00100651582133787 BP background PSP. Alpha=4.5583006655119e-06}

Biological process high 0.95 centile table~\ref{tab:Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242842 BP background PSP. Alpha = 4.5583006655119e-06}

Molecular function low 0.05 centile table~\ref{tab:Number of nodes 173 eigen centrality 0.05 centile  eigen centrality $<=$ 0.00100651582133787 MF background PSP. Alpha=2.02265372168285e-05}

Molecular function high 0.95 centile table~\ref{tab:Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242843 MF background PSP. Alpha = 2.02265372168285e-05}

Cellular component high eigenvector centrality
Gene Ontology topGO
tables~\ref{tab:Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242842 CC background PSP. Alpha = 3.33555703802535e-05}

Cellular component low eigenvector
tables~\ref{tab:Number of nodes 173 eigen centrality 0.05 centile  eigen centrality <= 0.00100651582133787CC background PSP. Alpha=3.33555703802535e-05}





% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:02:42 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0044428 & nuclear part & 952 & 105 & 48.9 & $6.40 \times 10^{-20}$ & $9.59 \times 10^{-17}$ \\ 
  GO:0005634 & nucleus & 1439 & 129 & 73.8 & $3.20 \times 10^{-18}$ & $4.80 \times 10^{-15}$ \\ 
  GO:0005654 & nucleoplasm & 716 & 84 & 36.7 & $2.20 \times 10^{-16}$ & $3.30 \times 10^{-13}$ \\ 
  GO:0005829 & cytosol & 1643 & 135 & 84.3 & $6.40 \times 10^{-16}$ & $9.59 \times 10^{-13}$ \\ 
  GO:0031981 & nuclear lumen & 856 & 92 & 43.9 & $9.80 \times 10^{-16}$ & $1.47 \times 10^{-12}$ \\ 
  GO:0043232 & intracellular non-membrane-bounded organ... & 1252 & 113 & 64.2 & $1.30 \times 10^{-14}$ & $1.95 \times 10^{-11}$ \\ 
  GO:0043228 & non-membrane-bounded organelle & 1260 & 113 & 64.7 & $2.20 \times 10^{-14}$ & $3.30 \times 10^{-11}$ \\ 
  GO:0031974 & membrane-enclosed lumen & 1171 & 106 & 60.1 & $2.80 \times 10^{-13}$ & $4.20 \times 10^{-10}$ \\ 
  GO:0043233 & organelle lumen & 1171 & 106 & 60.1 & $2.80 \times 10^{-13}$ & $4.20 \times 10^{-10}$ \\ 
  GO:0070013 & intracellular organelle lumen & 1171 & 106 & 60.1 & $2.80 \times 10^{-13}$ & $4.20 \times 10^{-10}$ \\ 
  GO:0044446 & intracellular organelle part & 2308 & 158 & 118.4 & $3.20 \times 10^{-13}$ & $4.80 \times 10^{-10}$ \\ 
  GO:0032991 & protein-containing complex & 1540 & 125 & 79.0 & $4.00 \times 10^{-13}$ & $6.00 \times 10^{-10}$ \\ 
  GO:0044422 & organelle part & 2379 & 159 & 122.1 & $3.50 \times 10^{-12}$ & $5.25 \times 10^{-9}$ \\ 
  GO:0043231 & intracellular membrane-bounded organelle & 2301 & 155 & 118.1 & $1.90 \times 10^{-11}$ & $2.85 \times 10^{-8}$ \\ 
  GO:0044430 & cytoskeletal part & 615 & 66 & 31.6 & $1.90 \times 10^{-10}$ & $2.85 \times 10^{-7}$ \\ 
  GO:0005856 & cytoskeleton & 802 & 78 & 41.1 & $2.30 \times 10^{-10}$ & $3.45 \times 10^{-7}$ \\ 
  GO:0043229 & intracellular organelle & 2784 & 169 & 142.8 & $4.30 \times 10^{-10}$ & $6.45 \times 10^{-7}$ \\ 
  GO:0015630 & microtubule cytoskeleton & 407 & 49 & 20.9 & $2.30 \times 10^{-9}$ & $3.45 \times 10^{-6}$ \\ 
  GO:0005925 & focal adhesion & 237 & 35 & 12.2 & $3.90 \times 10^{-9}$ & $5.85 \times 10^{-6}$ \\ 
  GO:0005924 & cell-substrate adherens junction & 238 & 35 & 12.2 & $4.40 \times 10^{-9}$ & $6.60 \times 10^{-6}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 174 degree centrality 0.95 centile  degree centrality $>=$ 59 CC background PSP. Alpha = 3.33555703802535e-05} 
\label{tab:Number of nodes 174 degree centrality 0.95 centile  degree centrality >= 59 CC background PSP. Alpha = 3.33555703802535e-05}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:10:09 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0031224 & intrinsic component of membrane & 788 & 84 & 37.9 & $7.10 \times 10^{-16}$ & TRUE \\ 
  GO:0016021 & integral component of membrane & 747 & 77 & 35.9 & $2.30 \times 10^{-13}$ & TRUE \\ 
  GO:0031226 & intrinsic component of plasma membrane & 308 & 41 & 14.8 & $4.10 \times 10^{-10}$ & TRUE \\ 
  GO:0005887 & integral component of plasma membrane & 288 & 38 & 13.8 & $2.70 \times 10^{-9}$ & TRUE \\ 
  GO:0044425 & membrane part & 1395 & 96 & 67.1 & $2.20 \times 10^{-6}$ & TRUE \\ 
  GO:0043202 & lysosomal lumen & 26 & 8 & 1.2 & $1.80 \times 10^{-5}$ & TRUE \\ 
  GO:0098936 & intrinsic component of postsynaptic memb... & 50 & 10 & 2.4 & $9.70 \times 10^{-5}$ & FALSE \\ 
  GO:0005796 & Golgi lumen & 12 & 5 & 0.6 & $1.50 \times 10^{-4}$ & FALSE \\ 
  GO:0034706 & sodium channel complex & 12 & 5 & 0.6 & $1.50 \times 10^{-4}$ & FALSE \\ 
  GO:1902495 & transmembrane transporter complex & 124 & 16 & 6.0 & $2.30 \times 10^{-4}$ & FALSE \\ 
  GO:0099240 & intrinsic component of synaptic membrane & 67 & 11 & 3.2 & $2.80 \times 10^{-4}$ & FALSE \\ 
  GO:0099055 & integral component of postsynaptic membr... & 47 & 9 & 2.3 & $3.10 \times 10^{-4}$ & FALSE \\ 
  GO:0034702 & ion channel complex & 115 & 15 & 5.5 & $3.20 \times 10^{-4}$ & FALSE \\ 
  GO:1990351 & transporter complex & 129 & 16 & 6.2 & $3.60 \times 10^{-4}$ & FALSE \\ 
  GO:0099699 & integral component of synaptic membrane & 61 & 10 & 2.9 & $5.40 \times 10^{-4}$ & FALSE \\ 
  GO:0030315 & T-tubule & 26 & 6 & 1.2 & $1.16 \times 10^{-3}$ & FALSE \\ 
  GO:0031225 & anchored component of membrane & 38 & 7 & 1.8 & $1.86 \times 10^{-3}$ & FALSE \\ 
  GO:0034703 & cation channel complex & 96 & 12 & 4.6 & $1.87 \times 10^{-3}$ & FALSE \\ 
  GO:0033268 & node of Ranvier & 12 & 4 & 0.6 & $1.88 \times 10^{-3}$ & FALSE \\ 
  GO:0001518 & voltage-gated sodium channel complex & 6 & 3 & 0.3 & $1.96 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 eigen centrality 0.05 centile  eigen centrality <= 0.00100651582133787CC background PSP. Alpha=3.33555703802535e-05} 
\label{tab:Number of nodes 173 eigen centrality 0.05 centile  eigen centrality <= 0.00100651582133787CC background PSP. Alpha=3.33555703802535e-05}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:17:36 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0005634 & nucleus & 1439 & 144 & 73.4 & $1.00 \times 10^{-29}$ & $1.50 \times 10^{-26}$ \\ 
  GO:1990904 & ribonucleoprotein complex & 293 & 67 & 14.9 & $1.50 \times 10^{-29}$ & $2.25 \times 10^{-26}$ \\ 
  GO:0044428 & nuclear part & 952 & 118 & 48.6 & $1.90 \times 10^{-29}$ & $2.85 \times 10^{-26}$ \\ 
  GO:0031981 & nuclear lumen & 856 & 109 & 43.7 & $4.00 \times 10^{-27}$ & $6.00 \times 10^{-24}$ \\ 
  GO:0031974 & membrane-enclosed lumen & 1171 & 123 & 59.7 & $8.10 \times 10^{-24}$ & $1.21 \times 10^{-20}$ \\ 
  GO:0043233 & organelle lumen & 1171 & 123 & 59.7 & $8.10 \times 10^{-24}$ & $1.21 \times 10^{-20}$ \\ 
  GO:0070013 & intracellular organelle lumen & 1171 & 123 & 59.7 & $8.10 \times 10^{-24}$ & $1.21 \times 10^{-20}$ \\ 
  GO:0005654 & nucleoplasm & 716 & 95 & 36.5 & $1.00 \times 10^{-23}$ & $1.50 \times 10^{-20}$ \\ 
  GO:0032991 & protein-containing complex & 1540 & 137 & 78.6 & $1.10 \times 10^{-20}$ & $1.65 \times 10^{-17}$ \\ 
  GO:0070062 & extracellular exosome & 886 & 101 & 45.2 & $3.10 \times 10^{-20}$ & $4.65 \times 10^{-17}$ \\ 
  GO:0043230 & extracellular organelle & 896 & 101 & 45.7 & $7.80 \times 10^{-20}$ & $1.17 \times 10^{-16}$ \\ 
  GO:1903561 & extracellular vesicle & 896 & 101 & 45.7 & $7.80 \times 10^{-20}$ & $1.17 \times 10^{-16}$ \\ 
  GO:0043232 & intracellular non-membrane-bounded organ... & 1252 & 120 & 63.9 & $6.20 \times 10^{-19}$ & $9.29 \times 10^{-16}$ \\ 
  GO:0005615 & extracellular space & 969 & 104 & 49.4 & $6.30 \times 10^{-19}$ & $9.44 \times 10^{-16}$ \\ 
  GO:0043228 & non-membrane-bounded organelle & 1260 & 120 & 64.3 & $1.10 \times 10^{-18}$ & $1.65 \times 10^{-15}$ \\ 
  GO:0044421 & extracellular region part & 1007 & 104 & 51.4 & $1.50 \times 10^{-17}$ & $2.25 \times 10^{-14}$ \\ 
  GO:0005576 & extracellular region & 1089 & 108 & 55.6 & $4.00 \times 10^{-17}$ & $6.00 \times 10^{-14}$ \\ 
  GO:0005925 & focal adhesion & 237 & 46 & 12.1 & $1.20 \times 10^{-16}$ & $1.80 \times 10^{-13}$ \\ 
  GO:0005924 & cell-substrate adherens junction & 238 & 46 & 12.1 & $1.40 \times 10^{-16}$ & $2.10 \times 10^{-13}$ \\ 
  GO:0030055 & cell-substrate junction & 241 & 46 & 12.3 & $2.30 \times 10^{-16}$ & $3.45 \times 10^{-13}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242842 CC background PSP. Alpha = 3.33555703802535e-05} 
\label{tab:Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242842 CC background PSP. Alpha = 3.33555703802535e-05}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:25:38 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0015075 & ion transmembrane transporter activity & 223 & 30 & 10.2 & $3.20 \times 10^{-8}$ & TRUE \\ 
  GO:0022857 & transmembrane transporter activity & 250 & 32 & 11.4 & $3.50 \times 10^{-8}$ & TRUE \\ 
  GO:0015081 & sodium ion transmembrane transporter act... & 33 & 11 & 1.5 & $1.00 \times 10^{-7}$ & TRUE \\ 
  GO:0005215 & transporter activity & 270 & 32 & 12.3 & $2.30 \times 10^{-7}$ & TRUE \\ 
  GO:0015318 & inorganic molecular entity transmembrane... & 205 & 27 & 9.4 & $2.80 \times 10^{-7}$ & TRUE \\ 
  GO:0046873 & metal ion transmembrane transporter acti... & 123 & 19 & 5.6 & $1.80 \times 10^{-6}$ & TRUE \\ 
  GO:0008324 & cation transmembrane transporter activit... & 177 & 22 & 8.1 & $1.10 \times 10^{-5}$ & TRUE \\ 
  GO:0015077 & monovalent inorganic cation transmembran... & 103 & 16 & 4.7 & $1.20 \times 10^{-5}$ & TRUE \\ 
  GO:0022890 & inorganic cation transmembrane transport... & 169 & 21 & 7.7 & $1.80 \times 10^{-5}$ & TRUE \\ 
  GO:0005272 & sodium channel activity & 16 & 6 & 0.7 & $4.50 \times 10^{-5}$ & FALSE \\ 
  GO:0005540 & hyaluronic acid binding & 7 & 4 & 0.3 & $1.30 \times 10^{-4}$ & FALSE \\ 
  GO:0008509 & anion transmembrane transporter activity & 55 & 10 & 2.5 & $1.40 \times 10^{-4}$ & FALSE \\ 
  GO:0005216 & ion channel activity & 113 & 15 & 5.2 & $1.50 \times 10^{-4}$ & FALSE \\ 
  GO:0022838 & substrate-specific channel activity & 114 & 15 & 5.2 & $1.60 \times 10^{-4}$ & FALSE \\ 
  GO:0015267 & channel activity & 119 & 15 & 5.4 & $2.60 \times 10^{-4}$ & FALSE \\ 
  GO:0022803 & passive transmembrane transporter activi... & 119 & 15 & 5.4 & $2.60 \times 10^{-4}$ & FALSE \\ 
  GO:0016831 & carboxy-lyase activity & 9 & 4 & 0.4 & $4.40 \times 10^{-4}$ & FALSE \\ 
  GO:0004930 & G protein-coupled receptor activity & 24 & 6 & 1.1 & $5.60 \times 10^{-4}$ & FALSE \\ 
  GO:0005452 & inorganic anion exchanger activity & 5 & 3 & 0.2 & $8.70 \times 10^{-4}$ & FALSE \\ 
  GO:0008514 & organic anion transmembrane transporter ... & 36 & 7 & 1.6 & $9.80 \times 10^{-4}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 eigen centrality 0.05 centile  eigen centrality $<=$ 0.00100651582133787MF background PSP. Alpha=2.02265372168285e-05} 
\label{tab:Number of nodes 173 eigen centrality 0.05 centile  eigen centrality $<=$ 0.00100651582133787 MF background PSP. Alpha=2.02265372168285e-05}
\end{table}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:25:40 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0003723 & RNA binding & 549 & 97 & 28.7 & $1.00 \times 10^{-30}$ & $2.47 \times 10^{-27}$ \\ 
  GO:0003676 & nucleic acid binding & 716 & 107 & 37.4 & $1.00 \times 10^{-30}$ & $2.47 \times 10^{-27}$ \\ 
  GO:0097159 & organic cyclic compound binding & 1366 & 130 & 71.4 & $1.70 \times 10^{-20}$ & $4.20 \times 10^{-17}$ \\ 
  GO:1901363 & heterocyclic compound binding & 1355 & 129 & 70.9 & $3.30 \times 10^{-20}$ & $8.16 \times 10^{-17}$ \\ 
  GO:0019899 & enzyme binding & 717 & 86 & 37.5 & $4.90 \times 10^{-17}$ & $1.21 \times 10^{-13}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 108 & 32 & 5.7 & $5.40 \times 10^{-17}$ & $1.33 \times 10^{-13}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 114 & 32 & 6.0 & $3.20 \times 10^{-16}$ & $7.91 \times 10^{-13}$ \\ 
  GO:0003729 & mRNA binding & 87 & 25 & 4.5 & $4.40 \times 10^{-13}$ & $1.09 \times 10^{-9}$ \\ 
  GO:0019904 & protein domain specific binding & 273 & 44 & 14.3 & $1.80 \times 10^{-12}$ & $4.45 \times 10^{-9}$ \\ 
  GO:0008134 & transcription factor binding & 127 & 29 & 6.6 & $3.20 \times 10^{-12}$ & $7.91 \times 10^{-9}$ \\ 
  GO:0051082 & unfolded protein binding & 56 & 19 & 2.9 & $1.30 \times 10^{-11}$ & $3.21 \times 10^{-8}$ \\ 
  GO:0005515 & protein binding & 2796 & 171 & 146.2 & $5.90 \times 10^{-11}$ & $1.46 \times 10^{-7}$ \\ 
  GO:0003677 & DNA binding & 251 & 38 & 13.1 & $5.80 \times 10^{-10}$ & $1.43 \times 10^{-6}$ \\ 
  GO:0045296 & cadherin binding & 219 & 33 & 11.4 & $1.10 \times 10^{-8}$ & $2.72 \times 10^{-5}$ \\ 
  GO:0003727 & single-stranded RNA binding & 31 & 12 & 1.6 & $1.70 \times 10^{-8}$ & $4.20 \times 10^{-5}$ \\ 
  GO:0050839 & cell adhesion molecule binding & 272 & 37 & 14.2 & $2.20 \times 10^{-8}$ & $5.44 \times 10^{-5}$ \\ 
  GO:0043565 & sequence-specific DNA binding & 112 & 22 & 5.9 & $3.40 \times 10^{-8}$ & $8.40 \times 10^{-5}$ \\ 
  GO:0044877 & protein-containing complex binding & 415 & 47 & 21.7 & $7.40 \times 10^{-8}$ & $1.83 \times 10^{-4}$ \\ 
  GO:0019901 & protein kinase binding & 225 & 32 & 11.8 & $8.10 \times 10^{-8}$ & $2.00 \times 10^{-4}$ \\ 
  GO:0003730 & mRNA 3'-UTR binding & 29 & 11 & 1.5 & $8.90 \times 10^{-8}$ & $2.20 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242843 MF background PSP. Alpha = 2.02265372168285e-05} 
\label{tab:Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242843 MF background PSP. Alpha = 2.02265372168285e-05}
\end{table}



\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0003723 & RNA binding & 549 & 97 & 28.7 & $1.00 \times 10^{-30}$ & $2.47 \times 10^{-27}$ \\ 
  GO:0003676 & nucleic acid binding & 716 & 107 & 37.4 & $1.00 \times 10^{-30}$ & $2.47 \times 10^{-27}$ \\ 
  GO:0097159 & organic cyclic compound binding & 1366 & 130 & 71.4 & $1.70 \times 10^{-20}$ & $4.20 \times 10^{-17}$ \\ 
  GO:1901363 & heterocyclic compound binding & 1355 & 129 & 70.9 & $3.30 \times 10^{-20}$ & $8.16 \times 10^{-17}$ \\ 
  GO:0019899 & enzyme binding & 717 & 86 & 37.5 & $4.90 \times 10^{-17}$ & $1.21 \times 10^{-13}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 108 & 32 & 5.7 & $5.40 \times 10^{-17}$ & $1.33 \times 10^{-13}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 114 & 32 & 6.0 & $3.20 \times 10^{-16}$ & $7.91 \times 10^{-13}$ \\ 
  GO:0003729 & mRNA binding & 87 & 25 & 4.5 & $4.40 \times 10^{-13}$ & $1.09 \times 10^{-9}$ \\ 
  GO:0019904 & protein domain specific binding & 273 & 44 & 14.3 & $1.80 \times 10^{-12}$ & $4.45 \times 10^{-9}$ \\ 
  GO:0008134 & transcription factor binding & 127 & 29 & 6.6 & $3.20 \times 10^{-12}$ & $7.91 \times 10^{-9}$ \\ 
  GO:0051082 & unfolded protein binding & 56 & 19 & 2.9 & $1.30 \times 10^{-11}$ & $3.21 \times 10^{-8}$ \\ 
  GO:0005515 & protein binding & 2796 & 171 & 146.2 & $5.90 \times 10^{-11}$ & $1.46 \times 10^{-7}$ \\ 
  GO:0003677 & DNA binding & 251 & 38 & 13.1 & $5.80 \times 10^{-10}$ & $1.43 \times 10^{-6}$ \\ 
  GO:0045296 & cadherin binding & 219 & 33 & 11.4 & $1.10 \times 10^{-8}$ & $2.72 \times 10^{-5}$ \\ 
  GO:0003727 & single-stranded RNA binding & 31 & 12 & 1.6 & $1.70 \times 10^{-8}$ & $4.20 \times 10^{-5}$ \\ 
  GO:0050839 & cell adhesion molecule binding & 272 & 37 & 14.2 & $2.20 \times 10^{-8}$ & $5.44 \times 10^{-5}$ \\ 
  GO:0043565 & sequence-specific DNA binding & 112 & 22 & 5.9 & $3.40 \times 10^{-8}$ & $8.40 \times 10^{-5}$ \\ 
  GO:0044877 & protein-containing complex binding & 415 & 47 & 21.7 & $7.40 \times 10^{-8}$ & $1.83 \times 10^{-4}$ \\ 
  GO:0019901 & protein kinase binding & 225 & 32 & 11.8 & $8.10 \times 10^{-8}$ & $2.00 \times 10^{-4}$ \\ 
  GO:0003730 & mRNA 3'-UTR binding & 29 & 11 & 1.5 & $8.90 \times 10^{-8}$ & $2.20 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242843 MF background PSP. Alpha = 2.02265372168285e-05} 
\label{tab:Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242843 MF background PSP. Alpha = 2.02265372168285e-05}
\end{table}

%%%%% GO Eigencentrality BP %%%%%%

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:35:43 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0044273 & sulfur compound catabolic process & 13 & 6 & 0.6 & $1.40 \times 10^{-5}$ & FALSE \\ 
  GO:1901136 & carbohydrate derivative catabolic proces... & 42 & 9 & 2.0 & $1.20 \times 10^{-4}$ & FALSE \\ 
  GO:0006029 & proteoglycan metabolic process & 12 & 5 & 0.6 & $1.40 \times 10^{-4}$ & FALSE \\ 
  GO:0006026 & aminoglycan catabolic process & 13 & 5 & 0.6 & $2.20 \times 10^{-4}$ & FALSE \\ 
  GO:0006027 & glycosaminoglycan catabolic process & 13 & 5 & 0.6 & $2.20 \times 10^{-4}$ & FALSE \\ 
  GO:0006790 & sulfur compound metabolic process & 89 & 13 & 4.3 & $2.50 \times 10^{-4}$ & FALSE \\ 
  GO:0006811 & ion transport & 476 & 39 & 22.8 & $3.40 \times 10^{-4}$ & FALSE \\ 
  GO:0030205 & dermatan sulfate metabolic process & 4 & 3 & 0.2 & $4.20 \times 10^{-4}$ & FALSE \\ 
  GO:0030206 & chondroitin sulfate biosynthetic process & 4 & 3 & 0.2 & $4.20 \times 10^{-4}$ & FALSE \\ 
  GO:0030208 & dermatan sulfate biosynthetic process & 4 & 3 & 0.2 & $4.20 \times 10^{-4}$ & FALSE \\ 
  GO:0006023 & aminoglycan biosynthetic process & 15 & 5 & 0.7 & $4.80 \times 10^{-4}$ & FALSE \\ 
  GO:0006024 & glycosaminoglycan biosynthetic process & 15 & 5 & 0.7 & $4.80 \times 10^{-4}$ & FALSE \\ 
  GO:0030166 & proteoglycan biosynthetic process & 9 & 4 & 0.4 & $5.30 \times 10^{-4}$ & FALSE \\ 
  GO:0097035 & regulation of membrane lipid distributio... & 9 & 4 & 0.4 & $5.30 \times 10^{-4}$ & FALSE \\ 
  GO:0009101 & glycoprotein biosynthetic process & 42 & 8 & 2.0 & $6.90 \times 10^{-4}$ & FALSE \\ 
  GO:0044272 & sulfur compound biosynthetic process & 44 & 8 & 2.1 & $9.50 \times 10^{-4}$ & FALSE \\ 
  GO:0034220 & ion transmembrane transport & 352 & 30 & 16.9 & $1.00 \times 10^{-3}$ & FALSE \\ 
  GO:0030207 & chondroitin sulfate catabolic process & 5 & 3 & 0.2 & $1.01 \times 10^{-3}$ & FALSE \\ 
  GO:0050650 & chondroitin sulfate proteoglycan biosynt... & 5 & 3 & 0.2 & $1.01 \times 10^{-3}$ & FALSE \\ 
  GO:0050651 & dermatan sulfate proteoglycan biosynthet... & 5 & 3 & 0.2 & $1.01 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 eigen centrality 0.05 centile  eigen centrality $<=$ 0.00100651582133787 BP background PSP. Alpha=4.5583006655119e-06} 
\label{tab:Number of nodes 173 eigen centrality 0.05 centile  eigen centrality $<=$ 0.00100651582133787 BP background PSP. Alpha=4.5583006655119e-06}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:35:59 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0090304 & nucleic acid metabolic process & 801 & 116 & 41.5 & $1.00 \times 10^{-30}$ & $1.10 \times 10^{-26}$ \\ 
  GO:0016071 & mRNA metabolic process & 254 & 65 & 13.2 & $1.00 \times 10^{-30}$ & $1.10 \times 10^{-26}$ \\ 
  GO:0060255 & regulation of macromolecule metabolic pr... & 1167 & 133 & 60.5 & $1.00 \times 10^{-30}$ & $1.10 \times 10^{-26}$ \\ 
  GO:0051704 & multi-organism process & 597 & 96 & 30.9 & $1.40 \times 10^{-30}$ & $1.54 \times 10^{-26}$ \\ 
  GO:0044403 & symbiont process & 359 & 74 & 18.6 & $2.10 \times 10^{-29}$ & $2.30 \times 10^{-25}$ \\ 
  GO:0044419 & interspecies interaction between organis... & 363 & 74 & 18.8 & $4.70 \times 10^{-29}$ & $5.16 \times 10^{-25}$ \\ 
  GO:0010468 & regulation of gene expression & 766 & 105 & 39.7 & $5.60 \times 10^{-28}$ & $6.14 \times 10^{-24}$ \\ 
  GO:0010467 & gene expression & 922 & 115 & 47.8 & $7.40 \times 10^{-28}$ & $8.12 \times 10^{-24}$ \\ 
  GO:0016032 & viral process & 344 & 70 & 17.8 & $3.40 \times 10^{-27}$ & $3.73 \times 10^{-23}$ \\ 
  GO:0043170 & macromolecule metabolic process & 1679 & 152 & 87.0 & $8.60 \times 10^{-27}$ & $9.43 \times 10^{-23}$ \\ 
  GO:0044260 & cellular macromolecule metabolic process & 1535 & 145 & 79.6 & $4.80 \times 10^{-26}$ & $5.27 \times 10^{-22}$ \\ 
  GO:0006139 & nucleobase-containing compound metabolic... & 1050 & 120 & 54.4 & $7.50 \times 10^{-26}$ & $8.23 \times 10^{-22}$ \\ 
  GO:0019222 & regulation of metabolic process & 1312 & 134 & 68.0 & $1.00 \times 10^{-25}$ & $1.10 \times 10^{-21}$ \\ 
  GO:0009057 & macromolecule catabolic process & 431 & 76 & 22.3 & $1.60 \times 10^{-25}$ & $1.76 \times 10^{-21}$ \\ 
  GO:0009059 & macromolecule biosynthetic process & 830 & 105 & 43.0 & $8.40 \times 10^{-25}$ & $9.21 \times 10^{-21}$ \\ 
  GO:0046483 & heterocycle metabolic process & 1076 & 120 & 55.8 & $9.20 \times 10^{-25}$ & $1.01 \times 10^{-20}$ \\ 
  GO:0034645 & cellular macromolecule biosynthetic proc... & 803 & 103 & 41.6 & $1.20 \times 10^{-24}$ & $1.32 \times 10^{-20}$ \\ 
  GO:0016070 & RNA metabolic process & 716 & 97 & 37.1 & $1.30 \times 10^{-24}$ & $1.43 \times 10^{-20}$ \\ 
  GO:0010556 & regulation of macromolecule biosynthetic... & 621 & 90 & 32.2 & $1.60 \times 10^{-24}$ & $1.76 \times 10^{-20}$ \\ 
  GO:2000112 & regulation of cellular macromolecule bio... & 596 & 88 & 30.9 & $2.00 \times 10^{-24}$ & $2.19 \times 10^{-20}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242842 BP background PSP. Alpha = 4.5583006655119e-06} 
\label{tab:Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242842 BP background PSP. Alpha = 4.5583006655119e-06}
\end{table}

\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0003723 & RNA binding & 549 & 97 & 28.7 & $1.00 \times 10^{-30}$ & $2.47 \times 10^{-27}$ \\ 
  GO:0003676 & nucleic acid binding & 716 & 107 & 37.4 & $1.00 \times 10^{-30}$ & $2.47 \times 10^{-27}$ \\ 
  GO:0097159 & organic cyclic compound binding & 1366 & 130 & 71.4 & $1.70 \times 10^{-20}$ & $4.20 \times 10^{-17}$ \\ 
  GO:1901363 & heterocyclic compound binding & 1355 & 129 & 70.9 & $3.30 \times 10^{-20}$ & $8.16 \times 10^{-17}$ \\ 
  GO:0019899 & enzyme binding & 717 & 86 & 37.5 & $4.90 \times 10^{-17}$ & $1.21 \times 10^{-13}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 108 & 32 & 5.7 & $5.40 \times 10^{-17}$ & $1.33 \times 10^{-13}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 114 & 32 & 6.0 & $3.20 \times 10^{-16}$ & $7.91 \times 10^{-13}$ \\ 
  GO:0003729 & mRNA binding & 87 & 25 & 4.5 & $4.40 \times 10^{-13}$ & $1.09 \times 10^{-9}$ \\ 
  GO:0019904 & protein domain specific binding & 273 & 44 & 14.3 & $1.80 \times 10^{-12}$ & $4.45 \times 10^{-9}$ \\ 
  GO:0008134 & transcription factor binding & 127 & 29 & 6.6 & $3.20 \times 10^{-12}$ & $7.91 \times 10^{-9}$ \\ 
  GO:0051082 & unfolded protein binding & 56 & 19 & 2.9 & $1.30 \times 10^{-11}$ & $3.21 \times 10^{-8}$ \\ 
  GO:0005515 & protein binding & 2796 & 171 & 146.2 & $5.90 \times 10^{-11}$ & $1.46 \times 10^{-7}$ \\ 
  GO:0003677 & DNA binding & 251 & 38 & 13.1 & $5.80 \times 10^{-10}$ & $1.43 \times 10^{-6}$ \\ 
  GO:0045296 & cadherin binding & 219 & 33 & 11.4 & $1.10 \times 10^{-8}$ & $2.72 \times 10^{-5}$ \\ 
  GO:0003727 & single-stranded RNA binding & 31 & 12 & 1.6 & $1.70 \times 10^{-8}$ & $4.20 \times 10^{-5}$ \\ 
  GO:0050839 & cell adhesion molecule binding & 272 & 37 & 14.2 & $2.20 \times 10^{-8}$ & $5.44 \times 10^{-5}$ \\ 
  GO:0043565 & sequence-specific DNA binding & 112 & 22 & 5.9 & $3.40 \times 10^{-8}$ & $8.40 \times 10^{-5}$ \\ 
  GO:0044877 & protein-containing complex binding & 415 & 47 & 21.7 & $7.40 \times 10^{-8}$ & $1.83 \times 10^{-4}$ \\ 
  GO:0019901 & protein kinase binding & 225 & 32 & 11.8 & $8.10 \times 10^{-8}$ & $2.00 \times 10^{-4}$ \\ 
  GO:0003730 & mRNA 3'-UTR binding & 29 & 11 & 1.5 & $8.90 \times 10^{-8}$ & $2.20 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242843 MF background PSP. Alpha = 2.02265372168285e-05} 
\label{tab:Number of nodes 173 eigen centrality 0.95 centile  eigen centrality $>=$ 0.179431213242843 MF background PSP. Alpha = 2.02265372168285e-05}
\end{table}














% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:56:23 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0009187 & cyclic nucleotide metabolic process & 16 & 7 & 1.6 & $5.00 \times 10^{-4}$ & FALSE \\ 
  GO:0009190 & cyclic nucleotide biosynthetic process & 9 & 5 & 0.9 & $9.00 \times 10^{-4}$ & FALSE \\ 
  GO:0052652 & cyclic purine nucleotide metabolic proce... & 9 & 5 & 0.9 & $9.00 \times 10^{-4}$ & FALSE \\ 
  GO:0097035 & regulation of membrane lipid distributio... & 9 & 5 & 0.9 & $9.00 \times 10^{-4}$ & FALSE \\ 
  GO:0038171 & cannabinoid signaling pathway & 3 & 3 & 0.3 & $1.00 \times 10^{-3}$ & FALSE \\ 
  GO:0071926 & endocannabinoid signaling pathway & 3 & 3 & 0.3 & $1.00 \times 10^{-3}$ & FALSE \\ 
  GO:0006171 & cAMP biosynthetic process & 4 & 3 & 0.4 & $3.70 \times 10^{-3}$ & FALSE \\ 
  GO:0015695 & organic cation transport & 4 & 3 & 0.4 & $3.70 \times 10^{-3}$ & FALSE \\ 
  GO:0021681 & cerebellar granular layer development & 8 & 4 & 0.8 & $5.10 \times 10^{-3}$ & FALSE \\ 
  GO:0046058 & cAMP metabolic process & 8 & 4 & 0.8 & $5.10 \times 10^{-3}$ & FALSE \\ 
  GO:0046339 & diacylglycerol metabolic process & 8 & 4 & 0.8 & $5.10 \times 10^{-3}$ & FALSE \\ 
  GO:0007186 & G protein-coupled receptor signaling pat... & 178 & 29 & 17.9 & $5.30 \times 10^{-3}$ & FALSE \\ 
  GO:0021696 & cerebellar cortex morphogenesis & 13 & 5 & 1.3 & $6.50 \times 10^{-3}$ & FALSE \\ 
  GO:0098742 & cell-cell adhesion via plasma-membrane a... & 62 & 13 & 6.2 & $7.40 \times 10^{-3}$ & FALSE \\ 
  GO:0015711 & organic anion transport & 113 & 20 & 11.4 & $7.90 \times 10^{-3}$ & FALSE \\ 
  GO:1904321 & response to forskolin & 5 & 3 & 0.5 & $8.70 \times 10^{-3}$ & FALSE \\ 
  GO:1904322 & cellular response to forskolin & 5 & 3 & 0.5 & $8.70 \times 10^{-3}$ & FALSE \\ 
  GO:0006820 & anion transport & 137 & 23 & 13.8 & $8.70 \times 10^{-3}$ & FALSE \\ 
  GO:0015893 & drug transport & 50 & 11 & 5.0 & $9.30 \times 10^{-3}$ & FALSE \\ 
  GO:0001539 & cilium or flagellum-dependent cell motil... & 2 & 2 & 0.2 & $1.01 \times 10^{-2}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 362 betweenness 0.05 centile  betweenness $<=$ 0 BP background PSP. Alpha=4.5583006655119e-06} 
\label{tab:Number of nodes 362 betweenness 0.05 centile  betweenness $<=$ 0 BP background PSP. Alpha=4.5583006655119e-06}
\end{table}



\subsection{Betweenness centrality topGO}

Biological process low 0.05 centile table~\ref{tab:Number of nodes 362 betweenness 0.05 centile  betweenness $<=$ 0 BP background PSP. Alpha=4.5583006655119e-06}

Biological process high 0.95 centile table~\ref{tab:Number of nodes 362 betweenness 0.05 centile  betweenness $<=$ 0 MF background PSP. Alpha=2.02265372168285e-05}

Low betweenness 0.05 centile Molecular function table~\ref{tab:Number of nodes 362 betweenness 0.05 centile  betweenness $<=$ 0 MF background PSP. Alpha=2.02265372168285e-05} None significant at adjusted alpha

High betweenness 0.95 centile molecular function table~\ref{tab:Number of nodes 173 betweenness 0.95 centile  betweenness $>=$ 11441.8526847438 MF background PSP. Alpha = 2.02265372168285e-05}


Low betweenness 0.05 centile cellular component table~\ref{tab:Number of nodes 362 betweenness 0.05 centile  betweenness $<=$ 0 CC background PSP. Alpha=3.33555703802535e-05}

High betweenness 0.95 centile cellular component table~\ref{tab:Number of nodes 173 betweenness 0.95 centile  betweenness $>=$ 11441.8526847438 CC background PSP. Alpha = 3.33555703802535e-05}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:56:38 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0051246 & regulation of protein metabolic process & 677 & 93 & 34.9 & $5.70 \times 10^{-24}$ & $6.25 \times 10^{-20}$ \\ 
  GO:0007049 & cell cycle & 458 & 74 & 23.6 & $2.10 \times 10^{-22}$ & $2.30 \times 10^{-18}$ \\ 
  GO:0051171 & regulation of nitrogen compound metaboli... & 1070 & 113 & 55.1 & $1.50 \times 10^{-20}$ & $1.65 \times 10^{-16}$ \\ 
  GO:0032268 & regulation of cellular protein metabolic... & 623 & 84 & 32.1 & $2.00 \times 10^{-20}$ & $2.19 \times 10^{-16}$ \\ 
  GO:0080090 & regulation of primary metabolic process & 1111 & 115 & 57.3 & $2.50 \times 10^{-20}$ & $2.74 \times 10^{-16}$ \\ 
  GO:0009057 & macromolecule catabolic process & 431 & 68 & 22.2 & $1.10 \times 10^{-19}$ & $1.21 \times 10^{-15}$ \\ 
  GO:0060255 & regulation of macromolecule metabolic pr... & 1167 & 117 & 60.1 & $1.30 \times 10^{-19}$ & $1.43 \times 10^{-15}$ \\ 
  GO:0006950 & response to stress & 872 & 99 & 45.0 & $3.10 \times 10^{-19}$ & $3.40 \times 10^{-15}$ \\ 
  GO:0009894 & regulation of catabolic process & 326 & 58 & 16.8 & $4.20 \times 10^{-19}$ & $4.61 \times 10^{-15}$ \\ 
  GO:0051704 & multi-organism process & 597 & 80 & 30.8 & $4.90 \times 10^{-19}$ & $5.37 \times 10^{-15}$ \\ 
  GO:0031323 & regulation of cellular metabolic process & 1167 & 115 & 60.1 & $2.20 \times 10^{-18}$ & $2.41 \times 10^{-14}$ \\ 
  GO:0031399 & regulation of protein modification proce... & 442 & 67 & 22.8 & $2.30 \times 10^{-18}$ & $2.52 \times 10^{-14}$ \\ 
  GO:0043412 & macromolecule modification & 911 & 100 & 47.0 & $2.40 \times 10^{-18}$ & $2.63 \times 10^{-14}$ \\ 
  GO:0006464 & cellular protein modification process & 899 & 99 & 46.3 & $3.40 \times 10^{-18}$ & $3.73 \times 10^{-14}$ \\ 
  GO:0036211 & protein modification process & 899 & 99 & 46.3 & $3.40 \times 10^{-18}$ & $3.73 \times 10^{-14}$ \\ 
  GO:0051247 & positive regulation of protein metabolic... & 409 & 64 & 21.1 & $3.50 \times 10^{-18}$ & $3.84 \times 10^{-14}$ \\ 
  GO:0010604 & positive regulation of macromolecule met... & 658 & 83 & 33.9 & $3.80 \times 10^{-18}$ & $4.17 \times 10^{-14}$ \\ 
  GO:0009893 & positive regulation of metabolic process & 733 & 88 & 37.8 & $4.80 \times 10^{-18}$ & $5.27 \times 10^{-14}$ \\ 
  GO:0048523 & negative regulation of cellular process & 1062 & 108 & 54.7 & $9.80 \times 10^{-18}$ & $1.07 \times 10^{-13}$ \\ 
  GO:0030163 & protein catabolic process & 263 & 50 & 13.6 & $1.40 \times 10^{-17}$ & $1.54 \times 10^{-13}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 betweenness 0.95 centile  betweenness $>=$ 11441.8526847438 BP background PSP. Alpha = 4.5583006655119e-06} 
\label{tab:Number of nodes 173 betweenness 0.95 centile  betweenness $>=$ 11441.8526847438 BP background PSP. Alpha = 4.5583006655119e-06}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:58:58 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0022857 & transmembrane transporter activity & 250 & 44 & 23.7 & $2.20 \times 10^{-5}$ & FALSE \\ 
  GO:0008509 & anion transmembrane transporter activity & 55 & 16 & 5.2 & $2.80 \times 10^{-5}$ & FALSE \\ 
  GO:0005215 & transporter activity & 270 & 45 & 25.6 & $7.30 \times 10^{-5}$ & FALSE \\ 
  GO:0015075 & ion transmembrane transporter activity & 223 & 39 & 21.2 & $8.00 \times 10^{-5}$ & FALSE \\ 
  GO:0008514 & organic anion transmembrane transporter ... & 36 & 11 & 3.4 & $3.20 \times 10^{-4}$ & FALSE \\ 
  GO:0016849 & phosphorus-oxygen lyase activity & 8 & 5 & 0.8 & $3.30 \times 10^{-4}$ & FALSE \\ 
  GO:0009975 & cyclase activity & 9 & 5 & 0.8 & $6.80 \times 10^{-4}$ & FALSE \\ 
  GO:0015318 & inorganic molecular entity transmembrane... & 205 & 32 & 19.5 & $2.68 \times 10^{-3}$ & FALSE \\ 
  GO:0015081 & sodium ion transmembrane transporter act... & 33 & 9 & 3.1 & $2.75 \times 10^{-3}$ & FALSE \\ 
  GO:0046872 & metal ion binding & 706 & 87 & 67.0 & $2.95 \times 10^{-3}$ & FALSE \\ 
  GO:0004016 & adenylate cyclase activity & 4 & 3 & 0.4 & $3.15 \times 10^{-3}$ & FALSE \\ 
  GO:0015101 & organic cation transmembrane transporter... & 4 & 3 & 0.4 & $3.15 \times 10^{-3}$ & FALSE \\ 
  GO:0015605 & organophosphate ester transmembrane tran... & 12 & 5 & 1.1 & $3.37 \times 10^{-3}$ & FALSE \\ 
  GO:0015291 & secondary active transmembrane transport... & 29 & 8 & 2.8 & $4.36 \times 10^{-3}$ & FALSE \\ 
  GO:0043169 & cation binding & 718 & 87 & 68.2 & $4.90 \times 10^{-3}$ & FALSE \\ 
  GO:0048037 & cofactor binding & 153 & 24 & 14.5 & $8.45 \times 10^{-3}$ & FALSE \\ 
  GO:0015238 & drug transmembrane transporter activity & 20 & 6 & 1.9 & $8.57 \times 10^{-3}$ & FALSE \\ 
  GO:0001758 & retinal dehydrogenase activity & 2 & 2 & 0.2 & $8.98 \times 10^{-3}$ & FALSE \\ 
  GO:0004993 & G protein-coupled serotonin receptor act... & 2 & 2 & 0.2 & $8.98 \times 10^{-3}$ & FALSE \\ 
  GO:0005385 & zinc ion transmembrane transporter activ... & 2 & 2 & 0.2 & $8.98 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 362 betweenness 0.05 centile  betweenness $<=$ 0 MF background PSP. Alpha=2.02265372168285e-05} 
\label{tab:Number of nodes 362 betweenness 0.05 centile  betweenness $<=$ 0 MF background PSP. Alpha=2.02265372168285e-05}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 15:59:01 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0019899 & enzyme binding & 717 & 98 & 37.5 & $5.70 \times 10^{-25}$ & $1.41 \times 10^{-21}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 114 & 36 & 6.0 & $4.00 \times 10^{-20}$ & $9.89 \times 10^{-17}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 108 & 35 & 5.7 & $5.50 \times 10^{-20}$ & $1.36 \times 10^{-16}$ \\ 
  GO:0019904 & protein domain specific binding & 273 & 51 & 14.3 & $2.50 \times 10^{-17}$ & $6.18 \times 10^{-14}$ \\ 
  GO:0019900 & kinase binding & 250 & 44 & 13.1 & $6.90 \times 10^{-14}$ & $1.71 \times 10^{-10}$ \\ 
  GO:0019901 & protein kinase binding & 225 & 39 & 11.8 & $4.20 \times 10^{-12}$ & $1.04 \times 10^{-8}$ \\ 
  GO:0005515 & protein binding & 2796 & 171 & 146.2 & $5.90 \times 10^{-11}$ & $1.46 \times 10^{-7}$ \\ 
  GO:0044877 & protein-containing complex binding & 415 & 53 & 21.7 & $7.70 \times 10^{-11}$ & $1.90 \times 10^{-7}$ \\ 
  GO:0051219 & phosphoprotein binding & 32 & 14 & 1.7 & $1.40 \times 10^{-10}$ & $3.46 \times 10^{-7}$ \\ 
  GO:0005102 & signaling receptor binding & 338 & 46 & 17.7 & $2.40 \times 10^{-10}$ & $5.93 \times 10^{-7}$ \\ 
  GO:0042802 & identical protein binding & 487 & 57 & 25.5 & $4.10 \times 10^{-10}$ & $1.01 \times 10^{-6}$ \\ 
  GO:0044325 & ion channel binding & 66 & 17 & 3.5 & $2.00 \times 10^{-8}$ & $4.94 \times 10^{-5}$ \\ 
  GO:0008022 & protein C-terminus binding & 85 & 19 & 4.5 & $3.50 \times 10^{-8}$ & $8.65 \times 10^{-5}$ \\ 
  GO:0008134 & transcription factor binding & 127 & 23 & 6.6 & $8.00 \times 10^{-8}$ & $1.98 \times 10^{-4}$ \\ 
  GO:0042826 & histone deacetylase binding & 25 & 10 & 1.3 & $2.00 \times 10^{-7}$ & $4.94 \times 10^{-4}$ \\ 
  GO:0047485 & protein N-terminus binding & 47 & 13 & 2.5 & $4.10 \times 10^{-7}$ & $1.01 \times 10^{-3}$ \\ 
  GO:0030235 & nitric-oxide synthase regulator activity & 6 & 5 & 0.3 & $2.10 \times 10^{-6}$ & $5.19 \times 10^{-3}$ \\ 
  GO:0097718 & disordered domain specific binding & 19 & 8 & 1.0 & $2.20 \times 10^{-6}$ & $5.44 \times 10^{-3}$ \\ 
  GO:0035258 & steroid hormone receptor binding & 25 & 9 & 1.3 & $2.40 \times 10^{-6}$ & $5.93 \times 10^{-3}$ \\ 
  GO:0019887 & protein kinase regulator activity & 50 & 12 & 2.6 & $6.10 \times 10^{-6}$ & $1.51 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 betweenness 0.95 centile  betweenness $>=$ 11441.8526847438 MF background PSP. Alpha = 2.02265372168285e-05} 
\label{tab:Number of nodes 173 betweenness 0.95 centile  betweenness $>=$ 11441.8526847438 MF background PSP. Alpha = 2.02265372168285e-05}
\end{table}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:00:33 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0031224 & intrinsic component of membrane & 788 & 144 & 79.9 & $3.40 \times 10^{-16}$ & TRUE \\ 
  GO:0016021 & integral component of membrane & 747 & 134 & 75.8 & $4.00 \times 10^{-14}$ & TRUE \\ 
  GO:0031226 & intrinsic component of plasma membrane & 308 & 58 & 31.2 & $8.60 \times 10^{-7}$ & TRUE \\ 
  GO:0005887 & integral component of plasma membrane & 288 & 54 & 29.2 & $2.50 \times 10^{-6}$ & TRUE \\ 
  GO:0044425 & membrane part & 1395 & 181 & 141.5 & $3.90 \times 10^{-6}$ & TRUE \\ 
  GO:0099240 & intrinsic component of synaptic membrane & 67 & 19 & 6.8 & $2.00 \times 10^{-5}$ & TRUE \\ 
  GO:0099699 & integral component of synaptic membrane & 61 & 17 & 6.2 & $7.00 \times 10^{-5}$ & FALSE \\ 
  GO:0098936 & intrinsic component of postsynaptic memb... & 50 & 15 & 5.1 & $7.50 \times 10^{-5}$ & FALSE \\ 
  GO:0099055 & integral component of postsynaptic membr... & 47 & 14 & 4.8 & $1.40 \times 10^{-4}$ & FALSE \\ 
  GO:0031225 & anchored component of membrane & 38 & 11 & 3.9 & $9.60 \times 10^{-4}$ & FALSE \\ 
  GO:0098889 & intrinsic component of presynaptic membr... & 36 & 10 & 3.6 & $2.30 \times 10^{-3}$ & FALSE \\ 
  GO:0060076 & excitatory synapse & 27 & 8 & 2.7 & $4.07 \times 10^{-3}$ & FALSE \\ 
  GO:0099146 & intrinsic component of postsynaptic dens... & 22 & 7 & 2.2 & $4.61 \times 10^{-3}$ & FALSE \\ 
  GO:0098948 & intrinsic component of postsynaptic spec... & 29 & 8 & 2.9 & $6.56 \times 10^{-3}$ & FALSE \\ 
  GO:0031045 & dense core granule & 9 & 4 & 0.9 & $8.67 \times 10^{-3}$ & FALSE \\ 
  GO:0045211 & postsynaptic membrane & 160 & 26 & 16.2 & $9.28 \times 10^{-3}$ & FALSE \\ 
  GO:0099056 & integral component of presynaptic membra... & 31 & 8 & 3.1 & $1.01 \times 10^{-2}$ & FALSE \\ 
  GO:0005858 & axonemal dynein complex & 2 & 2 & 0.2 & $1.03 \times 10^{-2}$ & FALSE \\ 
  GO:0097060 & synaptic membrane & 219 & 33 & 22.2 & $1.14 \times 10^{-2}$ & FALSE \\ 
  GO:0044459 & plasma membrane part & 742 & 92 & 75.3 & $1.40 \times 10^{-2}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 362 betweenness 0.05 centile  betweenness $<=$ 0 CC background PSP. Alpha=3.33555703802535e-05} 
\label{tab:Number of nodes 362 betweenness 0.05 centile  betweenness $<=$ 0 CC background PSP. Alpha=3.33555703802535e-05}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:00:37 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0005829 & cytosol & 1643 & 135 & 83.8 & $2.70 \times 10^{-16}$ & $4.05 \times 10^{-13}$ \\ 
  GO:0044428 & nuclear part & 952 & 96 & 48.6 & $6.80 \times 10^{-15}$ & $1.02 \times 10^{-11}$ \\ 
  GO:0005634 & nucleus & 1439 & 120 & 73.4 & $2.00 \times 10^{-13}$ & $3.00 \times 10^{-10}$ \\ 
  GO:0032991 & protein-containing complex & 1540 & 125 & 78.6 & $2.00 \times 10^{-13}$ & $3.00 \times 10^{-10}$ \\ 
  GO:0005654 & nucleoplasm & 716 & 76 & 36.5 & $3.40 \times 10^{-12}$ & $5.10 \times 10^{-9}$ \\ 
  GO:0031981 & nuclear lumen & 856 & 82 & 43.7 & $8.10 \times 10^{-11}$ & $1.21 \times 10^{-7}$ \\ 
  GO:0043228 & non-membrane-bounded organelle & 1260 & 105 & 64.3 & $9.90 \times 10^{-11}$ & $1.48 \times 10^{-7}$ \\ 
  GO:0044446 & intracellular organelle part & 2308 & 153 & 117.8 & $1.40 \times 10^{-10}$ & $2.10 \times 10^{-7}$ \\ 
  GO:0044422 & organelle part & 2379 & 155 & 121.4 & $3.50 \times 10^{-10}$ & $5.25 \times 10^{-7}$ \\ 
  GO:0043232 & intracellular non-membrane-bounded organ... & 1252 & 103 & 63.9 & $4.80 \times 10^{-10}$ & $7.20 \times 10^{-7}$ \\ 
  GO:0031982 & vesicle & 1350 & 108 & 68.9 & $6.00 \times 10^{-10}$ & $8.99 \times 10^{-7}$ \\ 
  GO:0005615 & extracellular space & 969 & 84 & 49.4 & $9.10 \times 10^{-9}$ & $1.36 \times 10^{-5}$ \\ 
  GO:0070062 & extracellular exosome & 886 & 79 & 45.2 & $9.90 \times 10^{-9}$ & $1.48 \times 10^{-5}$ \\ 
  GO:0031974 & membrane-enclosed lumen & 1171 & 95 & 59.7 & $1.40 \times 10^{-8}$ & $2.10 \times 10^{-5}$ \\ 
  GO:0043233 & organelle lumen & 1171 & 95 & 59.7 & $1.40 \times 10^{-8}$ & $2.10 \times 10^{-5}$ \\ 
  GO:0070013 & intracellular organelle lumen & 1171 & 95 & 59.7 & $1.40 \times 10^{-8}$ & $2.10 \times 10^{-5}$ \\ 
  GO:0043230 & extracellular organelle & 896 & 79 & 45.7 & $1.70 \times 10^{-8}$ & $2.55 \times 10^{-5}$ \\ 
  GO:1903561 & extracellular vesicle & 896 & 79 & 45.7 & $1.70 \times 10^{-8}$ & $2.55 \times 10^{-5}$ \\ 
  GO:0005856 & cytoskeleton & 802 & 73 & 40.9 & $2.30 \times 10^{-8}$ & $3.45 \times 10^{-5}$ \\ 
  GO:0044421 & extracellular region part & 1007 & 85 & 51.4 & $2.70 \times 10^{-8}$ & $4.05 \times 10^{-5}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 betweenness 0.95 centile  betweenness $>=$ 11441.8526847438 CC background PSP. Alpha = 3.33555703802535e-05} 
\label{tab:Number of nodes 173 betweenness 0.95 centile  betweenness $>=$ 11441.8526847438 CC background PSP. Alpha = 3.33555703802535e-05}
\end{table}



\subsection{GO closeness centrality}

GO enrichment calculated using topGO

Biological process low 0.05 centile closeness centrality table~\ref{tab:Number of nodes 173 closeness 0.05 centile  closeness $<=$ 7.85311545481246e-05 BP background PSP. Alpha=4.5583006655119e-06}

Biological process high 0.95 centile closeness centrality table~\ref{tab:Number of nodes 173 closeness 0.95 centile  closeness $>=$ 0.000116011972685393 BP background PSP. Alpha = 4.5583006655119e-06}

Molecular function low 0.05 centile closeness centrality table~\ref{tab:Number of nodes 173 closeness 0.05 centile  closeness $<=$ 7.85311545481246e-05 MF background PSP. Alpha=2.02265372168285e-05}

Molecular function high 0.05 centile closeness centrality table~\ref{tab:Number of nodes 173 closeness 0.95 centile  closeness $>=$ 0.000116011972685393 MF background PSP. Alpha = 2.02265372168285e-05}

Cellular component low 0.05 centile closeness centrality table~\ref{tab:Number of nodes 173 closeness 0.05 centile  closeness $<=$ 7.85311545481246e-05 CC background PSP. Alpha=3.33555703802535e-05}

Cellular component high 0.95 centile closeness centrality table~\ref{tab:Number of nodes 173 closeness 0.95 centile  closeness $>=$ 0.000116011972685393 CC background PSP. Alpha = 3.33555703802535e-05}


Code \url{source('~/RProjects/group_sizes/R/plotting/modified_exampleGoObject_summary_PSP_closeness_centrality_MF.R')}
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:05:04 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0044273 & sulfur compound catabolic process & 13 & 6 & 0.6 & $1.50 \times 10^{-5}$ & FALSE \\ 
  GO:0006811 & ion transport & 476 & 41 & 23.0 & $8.30 \times 10^{-5}$ & FALSE \\ 
  GO:0034220 & ion transmembrane transport & 352 & 33 & 17.0 & $9.60 \times 10^{-5}$ & FALSE \\ 
  GO:1901136 & carbohydrate derivative catabolic proces... & 42 & 9 & 2.0 & $1.30 \times 10^{-4}$ & FALSE \\ 
  GO:0006029 & proteoglycan metabolic process & 12 & 5 & 0.6 & $1.50 \times 10^{-4}$ & FALSE \\ 
  GO:0006026 & aminoglycan catabolic process & 13 & 5 & 0.6 & $2.30 \times 10^{-4}$ & FALSE \\ 
  GO:0006027 & glycosaminoglycan catabolic process & 13 & 5 & 0.6 & $2.30 \times 10^{-4}$ & FALSE \\ 
  GO:0009582 & detection of abiotic stimulus & 21 & 6 & 1.0 & $3.40 \times 10^{-4}$ & FALSE \\ 
  GO:0030205 & dermatan sulfate metabolic process & 4 & 3 & 0.2 & $4.30 \times 10^{-4}$ & FALSE \\ 
  GO:0030206 & chondroitin sulfate biosynthetic process & 4 & 3 & 0.2 & $4.30 \times 10^{-4}$ & FALSE \\ 
  GO:0030208 & dermatan sulfate biosynthetic process & 4 & 3 & 0.2 & $4.30 \times 10^{-4}$ & FALSE \\ 
  GO:0006023 & aminoglycan biosynthetic process & 15 & 5 & 0.7 & $5.00 \times 10^{-4}$ & FALSE \\ 
  GO:0006024 & glycosaminoglycan biosynthetic process & 15 & 5 & 0.7 & $5.00 \times 10^{-4}$ & FALSE \\ 
  GO:0030166 & proteoglycan biosynthetic process & 9 & 4 & 0.4 & $5.40 \times 10^{-4}$ & FALSE \\ 
  GO:0097035 & regulation of membrane lipid distributio... & 9 & 4 & 0.4 & $5.40 \times 10^{-4}$ & FALSE \\ 
  GO:0055085 & transmembrane transport & 473 & 38 & 22.8 & $7.00 \times 10^{-4}$ & FALSE \\ 
  GO:1901137 & carbohydrate derivative biosynthetic pro... & 179 & 19 & 8.6 & $7.90 \times 10^{-4}$ & FALSE \\ 
  GO:0006790 & sulfur compound metabolic process & 89 & 12 & 4.3 & $9.80 \times 10^{-4}$ & FALSE \\ 
  GO:0044272 & sulfur compound biosynthetic process & 44 & 8 & 2.1 & $9.90 \times 10^{-4}$ & FALSE \\ 
  GO:0030207 & chondroitin sulfate catabolic process & 5 & 3 & 0.2 & $1.03 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 closeness 0.05 centile  closeness $<=$ 7.85311545481246e-05 BP background PSP. Alpha=4.5583006655119e-06} 
\label{tab:Number of nodes 173 closeness 0.05 centile  closeness $<=$ 7.85311545481246e-05 BP background PSP. Alpha=4.5583006655119e-06}
\end{table}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:05:17 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0060255 & regulation of macromolecule metabolic pr... & 1167 & 132 & 60.5 & $4.30 \times 10^{-30}$ & $4.72 \times 10^{-26}$ \\ 
  GO:0051171 & regulation of nitrogen compound metaboli... & 1070 & 124 & 55.5 & $5.30 \times 10^{-28}$ & $5.81 \times 10^{-24}$ \\ 
  GO:0080090 & regulation of primary metabolic process & 1111 & 125 & 57.6 & $5.40 \times 10^{-27}$ & $5.92 \times 10^{-23}$ \\ 
  GO:0010604 & positive regulation of macromolecule met... & 658 & 96 & 34.1 & $6.00 \times 10^{-27}$ & $6.58 \times 10^{-23}$ \\ 
  GO:0009893 & positive regulation of metabolic process & 733 & 101 & 38.0 & $1.10 \times 10^{-26}$ & $1.21 \times 10^{-22}$ \\ 
  GO:0010468 & regulation of gene expression & 766 & 103 & 39.7 & $1.80 \times 10^{-26}$ & $1.97 \times 10^{-22}$ \\ 
  GO:0019222 & regulation of metabolic process & 1312 & 134 & 68.0 & $1.00 \times 10^{-25}$ & $1.10 \times 10^{-21}$ \\ 
  GO:0090304 & nucleic acid metabolic process & 801 & 104 & 41.5 & $1.80 \times 10^{-25}$ & $1.97 \times 10^{-21}$ \\ 
  GO:0009057 & macromolecule catabolic process & 431 & 75 & 22.3 & $1.00 \times 10^{-24}$ & $1.10 \times 10^{-20}$ \\ 
  GO:0044260 & cellular macromolecule metabolic process & 1535 & 143 & 79.6 & $1.90 \times 10^{-24}$ & $2.08 \times 10^{-20}$ \\ 
  GO:0043170 & macromolecule metabolic process & 1679 & 149 & 87.0 & $3.00 \times 10^{-24}$ & $3.29 \times 10^{-20}$ \\ 
  GO:0010467 & gene expression & 922 & 110 & 47.8 & $3.50 \times 10^{-24}$ & $3.84 \times 10^{-20}$ \\ 
  GO:0019219 & regulation of nucleobase-containing comp... & 629 & 90 & 32.6 & $4.50 \times 10^{-24}$ & $4.94 \times 10^{-20}$ \\ 
  GO:0031323 & regulation of cellular metabolic process & 1167 & 124 & 60.5 & $5.60 \times 10^{-24}$ & $6.14 \times 10^{-20}$ \\ 
  GO:0051173 & positive regulation of nitrogen compound... & 625 & 89 & 32.4 & $1.40 \times 10^{-23}$ & $1.54 \times 10^{-19}$ \\ 
  GO:0010556 & regulation of macromolecule biosynthetic... & 621 & 88 & 32.2 & $4.60 \times 10^{-23}$ & $5.05 \times 10^{-19}$ \\ 
  GO:0051704 & multi-organism process & 597 & 86 & 30.9 & $6.40 \times 10^{-23}$ & $7.02 \times 10^{-19}$ \\ 
  GO:0048518 & positive regulation of biological proces... & 1394 & 134 & 72.3 & $1.10 \times 10^{-22}$ & $1.21 \times 10^{-18}$ \\ 
  GO:0031325 & positive regulation of cellular metaboli... & 660 & 90 & 34.2 & $1.80 \times 10^{-22}$ & $1.97 \times 10^{-18}$ \\ 
  GO:2000112 & regulation of cellular macromolecule bio... & 596 & 85 & 30.9 & $2.90 \times 10^{-22}$ & $3.18 \times 10^{-18}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 closeness 0.95 centile  closeness $>=$ 0.000116011972685393 BP background PSP. Alpha = 4.5583006655119e-06} 
\label{tab:Number of nodes 173 closeness 0.95 centile  closeness $>=$ 0.000116011972685393 BP background PSP. Alpha = 4.5583006655119e-06}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:12:53 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0015075 & ion transmembrane transporter activity & 223 & 32 & 10.3 & $2.80 \times 10^{-9}$ & TRUE \\ 
  GO:0022857 & transmembrane transporter activity & 250 & 34 & 11.6 & $3.40 \times 10^{-9}$ & TRUE \\ 
  GO:0005215 & transporter activity & 270 & 35 & 12.5 & $7.00 \times 10^{-9}$ & TRUE \\ 
  GO:0008324 & cation transmembrane transporter activit... & 177 & 26 & 8.2 & $6.70 \times 10^{-8}$ & TRUE \\ 
  GO:0015318 & inorganic molecular entity transmembrane... & 205 & 28 & 9.5 & $9.80 \times 10^{-8}$ & TRUE \\ 
  GO:0022890 & inorganic cation transmembrane transport... & 169 & 24 & 7.8 & $4.40 \times 10^{-7}$ & TRUE \\ 
  GO:0046873 & metal ion transmembrane transporter acti... & 123 & 20 & 5.7 & $4.90 \times 10^{-7}$ & TRUE \\ 
  GO:0015081 & sodium ion transmembrane transporter act... & 33 & 10 & 1.5 & $1.20 \times 10^{-6}$ & TRUE \\ 
  GO:0015077 & monovalent inorganic cation transmembran... & 103 & 17 & 4.8 & $3.10 \times 10^{-6}$ & TRUE \\ 
  GO:0022804 & active transmembrane transporter activit... & 83 & 13 & 3.8 & $8.50 \times 10^{-5}$ & FALSE \\ 
  GO:0005540 & hyaluronic acid binding & 7 & 4 & 0.3 & $1.40 \times 10^{-4}$ & FALSE \\ 
  GO:0008514 & organic anion transmembrane transporter ... & 36 & 8 & 1.7 & $1.70 \times 10^{-4}$ & FALSE \\ 
  GO:0005261 & cation channel activity & 96 & 13 & 4.4 & $3.80 \times 10^{-4}$ & FALSE \\ 
  GO:0016831 & carboxy-lyase activity & 9 & 4 & 0.4 & $4.60 \times 10^{-4}$ & FALSE \\ 
  GO:0005272 & sodium channel activity & 16 & 5 & 0.7 & $5.70 \times 10^{-4}$ & FALSE \\ 
  GO:0005216 & ion channel activity & 113 & 14 & 5.2 & $5.80 \times 10^{-4}$ & FALSE \\ 
  GO:0004930 & G protein-coupled receptor activity & 24 & 6 & 1.1 & $6.00 \times 10^{-4}$ & FALSE \\ 
  GO:0022838 & substrate-specific channel activity & 114 & 14 & 5.3 & $6.30 \times 10^{-4}$ & FALSE \\ 
  GO:0005244 & voltage-gated ion channel activity & 65 & 10 & 3.0 & $6.60 \times 10^{-4}$ & FALSE \\ 
  GO:0022832 & voltage-gated channel activity & 65 & 10 & 3.0 & $6.60 \times 10^{-4}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 closeness 0.05 centile  closeness $<=$ 7.85311545481246e-05 MF background PSP. Alpha=2.02265372168285e-05} 
\label{tab:Number of nodes 173 closeness 0.05 centile  closeness $<=$ 7.85311545481246e-05 MF background PSP. Alpha=2.02265372168285e-05}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:12:56 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0003723 & RNA binding & 549 & 84 & 28.7 & $6.90 \times 10^{-24}$ & $1.71 \times 10^{-20}$ \\ 
  GO:0003676 & nucleic acid binding & 716 & 95 & 37.4 & $6.80 \times 10^{-23}$ & $1.68 \times 10^{-19}$ \\ 
  GO:0019899 & enzyme binding & 717 & 93 & 37.5 & $1.80 \times 10^{-21}$ & $4.45 \times 10^{-18}$ \\ 
  GO:1901363 & heterocyclic compound binding & 1355 & 126 & 70.9 & $2.70 \times 10^{-18}$ & $6.67 \times 10^{-15}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 114 & 34 & 6.0 & $3.90 \times 10^{-18}$ & $9.64 \times 10^{-15}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 108 & 33 & 5.7 & $5.70 \times 10^{-18}$ & $1.41 \times 10^{-14}$ \\ 
  GO:0097159 & organic cyclic compound binding & 1366 & 126 & 71.4 & $6.10 \times 10^{-18}$ & $1.51 \times 10^{-14}$ \\ 
  GO:0019904 & protein domain specific binding & 273 & 51 & 14.3 & $2.50 \times 10^{-17}$ & $6.18 \times 10^{-14}$ \\ 
  GO:0008134 & transcription factor binding & 127 & 29 & 6.6 & $3.20 \times 10^{-12}$ & $7.91 \times 10^{-9}$ \\ 
  GO:0005515 & protein binding & 2796 & 171 & 146.2 & $5.90 \times 10^{-11}$ & $1.46 \times 10^{-7}$ \\ 
  GO:0051082 & unfolded protein binding & 56 & 18 & 2.9 & $1.30 \times 10^{-10}$ & $3.21 \times 10^{-7}$ \\ 
  GO:0044877 & protein-containing complex binding & 415 & 52 & 21.7 & $2.60 \times 10^{-10}$ & $6.43 \times 10^{-7}$ \\ 
  GO:0003677 & DNA binding & 251 & 38 & 13.1 & $5.80 \times 10^{-10}$ & $1.43 \times 10^{-6}$ \\ 
  GO:0042802 & identical protein binding & 487 & 56 & 25.5 & $1.30 \times 10^{-9}$ & $3.21 \times 10^{-6}$ \\ 
  GO:0003727 & single-stranded RNA binding & 31 & 13 & 1.6 & $1.30 \times 10^{-9}$ & $3.21 \times 10^{-6}$ \\ 
  GO:0043565 & sequence-specific DNA binding & 112 & 23 & 5.9 & $6.50 \times 10^{-9}$ & $1.61 \times 10^{-5}$ \\ 
  GO:0003729 & mRNA binding & 87 & 20 & 4.5 & $8.90 \times 10^{-9}$ & $2.20 \times 10^{-5}$ \\ 
  GO:0042826 & histone deacetylase binding & 25 & 11 & 1.3 & $1.40 \times 10^{-8}$ & $3.46 \times 10^{-5}$ \\ 
  GO:0001067 & regulatory region nucleic acid binding & 92 & 20 & 4.8 & $2.50 \times 10^{-8}$ & $6.18 \times 10^{-5}$ \\ 
  GO:0044212 & transcription regulatory region DNA bind... & 92 & 20 & 4.8 & $2.50 \times 10^{-8}$ & $6.18 \times 10^{-5}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 closeness 0.95 centile  closeness $>=$ 0.000116011972685393 MF background PSP. Alpha = 2.02265372168285e-05} 
\label{tab:Number of nodes 173 closeness 0.95 centile  closeness $>=$ 0.000116011972685393 MF background PSP. Alpha = 2.02265372168285e-05}
\end{table}





% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:16:10 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0031224 & intrinsic component of membrane & 788 & 85 & 38.1 & $3.00 \times 10^{-16}$ & TRUE \\ 
  GO:0016021 & integral component of membrane & 747 & 79 & 36.1 & $2.90 \times 10^{-14}$ & TRUE \\ 
  GO:0031226 & intrinsic component of plasma membrane & 308 & 40 & 14.9 & $1.80 \times 10^{-9}$ & TRUE \\ 
  GO:0005887 & integral component of plasma membrane & 288 & 37 & 13.9 & $1.20 \times 10^{-8}$ & TRUE \\ 
  GO:0044425 & membrane part & 1395 & 98 & 67.5 & $6.50 \times 10^{-7}$ & TRUE \\ 
  GO:0005796 & Golgi lumen & 12 & 5 & 0.6 & $1.50 \times 10^{-4}$ & FALSE \\ 
  GO:0043202 & lysosomal lumen & 26 & 7 & 1.3 & $1.60 \times 10^{-4}$ & FALSE \\ 
  GO:0034703 & cation channel complex & 96 & 13 & 4.6 & $5.90 \times 10^{-4}$ & FALSE \\ 
  GO:1902495 & transmembrane transporter complex & 124 & 15 & 6.0 & $7.70 \times 10^{-4}$ & FALSE \\ 
  GO:0034702 & ion channel complex & 115 & 14 & 5.6 & $1.09 \times 10^{-3}$ & FALSE \\ 
  GO:1990351 & transporter complex & 129 & 15 & 6.2 & $1.17 \times 10^{-3}$ & FALSE \\ 
  GO:0030315 & T-tubule & 26 & 6 & 1.3 & $1.20 \times 10^{-3}$ & FALSE \\ 
  GO:0099055 & integral component of postsynaptic membr... & 47 & 8 & 2.3 & $1.58 \times 10^{-3}$ & FALSE \\ 
  GO:0033268 & node of Ranvier & 12 & 4 & 0.6 & $1.92 \times 10^{-3}$ & FALSE \\ 
  GO:0034706 & sodium channel complex & 12 & 4 & 0.6 & $1.92 \times 10^{-3}$ & FALSE \\ 
  GO:0001518 & voltage-gated sodium channel complex & 6 & 3 & 0.3 & $1.99 \times 10^{-3}$ & FALSE \\ 
  GO:0099699 & integral component of synaptic membrane & 61 & 9 & 3.0 & $2.30 \times 10^{-3}$ & FALSE \\ 
  GO:0098936 & intrinsic component of postsynaptic memb... & 50 & 8 & 2.4 & $2.38 \times 10^{-3}$ & FALSE \\ 
  GO:0099240 & intrinsic component of synaptic membrane & 67 & 9 & 3.2 & $4.42 \times 10^{-3}$ & FALSE \\ 
  GO:0044459 & plasma membrane part & 742 & 50 & 35.9 & $5.31 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 closeness 0.05 centile  closeness $<=$ 7.85311545481246e-05 CC background PSP. Alpha=3.33555703802535e-05} 
\label{tab:Number of nodes 173 closeness 0.05 centile  closeness $<=$ 7.85311545481246e-05 CC background PSP. Alpha=3.33555703802535e-05}
\end{table}





% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:16:13 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0044428 & nuclear part & 952 & 113 & 48.6 & $1.20 \times 10^{-25}$ & $1.80 \times 10^{-22}$ \\ 
  GO:0005634 & nucleus & 1439 & 138 & 73.4 & $7.60 \times 10^{-25}$ & $1.14 \times 10^{-21}$ \\ 
  GO:0031974 & membrane-enclosed lumen & 1171 & 122 & 59.7 & $4.10 \times 10^{-23}$ & $6.15 \times 10^{-20}$ \\ 
  GO:0043233 & organelle lumen & 1171 & 122 & 59.7 & $4.10 \times 10^{-23}$ & $6.15 \times 10^{-20}$ \\ 
  GO:0070013 & intracellular organelle lumen & 1171 & 122 & 59.7 & $4.10 \times 10^{-23}$ & $6.15 \times 10^{-20}$ \\ 
  GO:0031981 & nuclear lumen & 856 & 103 & 43.7 & $8.20 \times 10^{-23}$ & $1.23 \times 10^{-19}$ \\ 
  GO:0005654 & nucleoplasm & 716 & 91 & 36.5 & $5.70 \times 10^{-21}$ & $8.54 \times 10^{-18}$ \\ 
  GO:0032991 & protein-containing complex & 1540 & 136 & 78.6 & $5.40 \times 10^{-20}$ & $8.09 \times 10^{-17}$ \\ 
  GO:1990904 & ribonucleoprotein complex & 293 & 53 & 14.9 & $7.40 \times 10^{-18}$ & $1.11 \times 10^{-14}$ \\ 
  GO:0070062 & extracellular exosome & 886 & 97 & 45.2 & $9.30 \times 10^{-18}$ & $1.39 \times 10^{-14}$ \\ 
  GO:0005829 & cytosol & 1643 & 137 & 83.8 & $1.50 \times 10^{-17}$ & $2.25 \times 10^{-14}$ \\ 
  GO:0043230 & extracellular organelle & 896 & 97 & 45.7 & $2.20 \times 10^{-17}$ & $3.30 \times 10^{-14}$ \\ 
  GO:1903561 & extracellular vesicle & 896 & 97 & 45.7 & $2.20 \times 10^{-17}$ & $3.30 \times 10^{-14}$ \\ 
  GO:0043231 & intracellular membrane-bounded organelle & 2301 & 161 & 117.4 & $2.90 \times 10^{-16}$ & $4.35 \times 10^{-13}$ \\ 
  GO:0005615 & extracellular space & 969 & 99 & 49.4 & $5.50 \times 10^{-16}$ & $8.24 \times 10^{-13}$ \\ 
  GO:0043232 & intracellular non-membrane-bounded organ... & 1252 & 114 & 63.9 & $2.00 \times 10^{-15}$ & $3.00 \times 10^{-12}$ \\ 
  GO:0043228 & non-membrane-bounded organelle & 1260 & 114 & 64.3 & $3.50 \times 10^{-15}$ & $5.25 \times 10^{-12}$ \\ 
  GO:0044421 & extracellular region part & 1007 & 99 & 51.4 & $9.50 \times 10^{-15}$ & $1.42 \times 10^{-11}$ \\ 
  GO:0044446 & intracellular organelle part & 2308 & 159 & 117.8 & $1.60 \times 10^{-14}$ & $2.40 \times 10^{-11}$ \\ 
  GO:0005576 & extracellular region & 1089 & 102 & 55.6 & $7.80 \times 10^{-14}$ & $1.17 \times 10^{-10}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 173 closeness 0.95 centile  closeness $>=$ 0.000116011972685393 CC background PSP. Alpha = 3.33555703802535e-05} 
\label{tab:Number of nodes 173 closeness 0.95 centile  closeness $>=$ 0.000116011972685393 CC background PSP. Alpha = 3.33555703802535e-05}
\end{table}

\subsection{Local transitivity topGO}

Local transitivity isolates set to 0 (therefore 0 includes isolates and nodes with transitivity 0

Low transitivity 935 nodes Biological process 0.05 centile table~\ref{tab:Number of nodes 935 local transitivity 0.05 centile  local transitivity $<=$ 0 BP background PSP. Alpha=4.5583006655119e-06isolates 0}

Biological process high local transitivity 0.95 (184 nodes) table~\ref{tab:Number of nodes 184 local transitivity 0.95 centile  local transitivity $>=$ 0.5 BP background PSP. Alpha = 4.5583006655119e-06isolates 0}

Molecular function low transitivity 0.05 centile 935 nodestable~\ref{tab:Number of nodes 935 local transitivity 0.05 centile  local transitivity $<=$ 0 MF background PSP. Alpha=2.02265372168285e-05isolates 0}

Molecular function high transitivity table~\ref{tab:Number of nodes 184 local transitivity 0.95 centile  local transitivity $>=$ 0.5 MF background PSP. Alpha = 2.02265372168285e-05isolates 0}

Cellular component low transitivity 0.05 centile,isolates to 0, number of nodes 935, table~\ref{tab:Number of nodes 935 local transitivity 0.05 centile  local transitivity $<=$ 0 CC background PSP.Alpha=3.33555703802535e-05isolates 0}

Cellular component high transitivity 0.95 centile table~\ref{tab:Number of nodes 184 local transitivity 0.95 centile  local transitivity $>=$ 0.5 CC background PSP. Alpha = 3.33555703802535e-05isolates 0}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:22:44 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0016054 & organic acid catabolic process & 89 & 41 & 23.4 & $3.90 \times 10^{-5}$ & FALSE \\ 
  GO:0046395 & carboxylic acid catabolic process & 89 & 41 & 23.4 & $3.90 \times 10^{-5}$ & FALSE \\ 
  GO:0007156 & homophilic cell adhesion via plasma memb... & 34 & 20 & 9.0 & $5.90 \times 10^{-5}$ & FALSE \\ 
  GO:0006029 & proteoglycan metabolic process & 12 & 10 & 3.2 & $5.90 \times 10^{-5}$ & FALSE \\ 
  GO:0044255 & cellular lipid metabolic process & 256 & 94 & 67.4 & $9.30 \times 10^{-5}$ & FALSE \\ 
  GO:0098742 & cell-cell adhesion via plasma-membrane a... & 62 & 30 & 16.3 & $1.40 \times 10^{-4}$ & FALSE \\ 
  GO:0086010 & membrane depolarization during action po... & 19 & 13 & 5.0 & $1.40 \times 10^{-4}$ & FALSE \\ 
  GO:0030258 & lipid modification & 80 & 36 & 21.1 & $2.10 \times 10^{-4}$ & FALSE \\ 
  GO:0072329 & monocarboxylic acid catabolic process & 50 & 25 & 13.2 & $2.70 \times 10^{-4}$ & FALSE \\ 
  GO:0006639 & acylglycerol metabolic process & 27 & 16 & 7.1 & $2.90 \times 10^{-4}$ & FALSE \\ 
  GO:0006629 & lipid metabolic process & 313 & 109 & 82.5 & $3.00 \times 10^{-4}$ & FALSE \\ 
  GO:0046486 & glycerolipid metabolic process & 108 & 45 & 28.4 & $3.10 \times 10^{-4}$ & FALSE \\ 
  GO:0008654 & phospholipid biosynthetic process & 76 & 34 & 20.0 & $3.50 \times 10^{-4}$ & FALSE \\ 
  GO:0044282 & small molecule catabolic process & 133 & 53 & 35.0 & $3.60 \times 10^{-4}$ & FALSE \\ 
  GO:0006638 & neutral lipid metabolic process & 28 & 16 & 7.4 & $5.20 \times 10^{-4}$ & FALSE \\ 
  GO:0044242 & cellular lipid catabolic process & 69 & 31 & 18.2 & $5.80 \times 10^{-4}$ & FALSE \\ 
  GO:1901136 & carbohydrate derivative catabolic proces... & 42 & 21 & 11.1 & $8.20 \times 10^{-4}$ & FALSE \\ 
  GO:0035637 & multicellular organismal signaling & 89 & 37 & 23.4 & $1.11 \times 10^{-3}$ & FALSE \\ 
  GO:0006658 & phosphatidylserine metabolic process & 5 & 5 & 1.3 & $1.26 \times 10^{-3}$ & FALSE \\ 
  GO:0052646 & alditol phosphate metabolic process & 5 & 5 & 1.3 & $1.26 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 935 local transitivity 0.05 centile  local transitivity $<=$ 0 BP background PSP. Alpha=4.5583006655119e-06isolates 0} 
\label{tab:Number of nodes 935 local transitivity 0.05 centile  local transitivity $<=$ 0 BP background PSP. Alpha=4.5583006655119e-06isolates 0}
\end{table}

\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0034776 & response to histamine & 5 & 3 & 0.3 & $1.40 \times 10^{-3}$ & $1.54 \times 10^{1}$ \\ 
  GO:0071420 & cellular response to histamine & 5 & 3 & 0.3 & $1.40 \times 10^{-3}$ & $1.54 \times 10^{1}$ \\ 
  GO:0007210 & serotonin receptor signaling pathway & 2 & 2 & 0.1 & $2.80 \times 10^{-3}$ & $3.07 \times 10^{1}$ \\ 
  GO:0040038 & polar body extrusion after meiotic divis... & 2 & 2 & 0.1 & $2.80 \times 10^{-3}$ & $3.07 \times 10^{1}$ \\ 
  GO:0098664 & G protein-coupled serotonin receptor sig... & 2 & 2 & 0.1 & $2.80 \times 10^{-3}$ & $3.07 \times 10^{1}$ \\ 
  GO:0008033 & tRNA processing & 14 & 4 & 0.7 & $5.00 \times 10^{-3}$ & $5.48 \times 10^{1}$ \\ 
  GO:0022412 & cellular process involved in reproductio... & 52 & 8 & 2.8 & $5.40 \times 10^{-3}$ & $5.92 \times 10^{1}$ \\ 
  GO:0008354 & germ cell migration & 3 & 2 & 0.2 & $8.10 \times 10^{-3}$ & $8.88 \times 10^{1}$ \\ 
  GO:0007214 & gamma-aminobutyric acid signaling pathwa... & 9 & 3 & 0.5 & $9.70 \times 10^{-3}$ & $1.06 \times 10^{2}$ \\ 
  GO:0000147 & actin cortical patch assembly & 4 & 2 & 0.2 & $1.56 \times 10^{-2}$ & $1.71 \times 10^{2}$ \\ 
  GO:0000578 & embryonic axis specification & 4 & 2 & 0.2 & $1.56 \times 10^{-2}$ & $1.71 \times 10^{2}$ \\ 
  GO:0033206 & meiotic cytokinesis & 4 & 2 & 0.2 & $1.56 \times 10^{-2}$ & $1.71 \times 10^{2}$ \\ 
  GO:0044396 & actin cortical patch organization & 4 & 2 & 0.2 & $1.56 \times 10^{-2}$ & $1.71 \times 10^{2}$ \\ 
  GO:0051666 & actin cortical patch localization & 4 & 2 & 0.2 & $1.56 \times 10^{-2}$ & $1.71 \times 10^{2}$ \\ 
  GO:0006520 & cellular amino acid metabolic process & 127 & 13 & 6.7 & $1.58 \times 10^{-2}$ & $1.73 \times 10^{2}$ \\ 
  GO:0048599 & oocyte development & 11 & 3 & 0.6 & $1.77 \times 10^{-2}$ & $1.94 \times 10^{2}$ \\ 
  GO:0007026 & negative regulation of microtubule depol... & 12 & 3 & 0.6 & $2.26 \times 10^{-2}$ & $2.48 \times 10^{2}$ \\ 
  GO:0009994 & oocyte differentiation & 12 & 3 & 0.6 & $2.26 \times 10^{-2}$ & $2.48 \times 10^{2}$ \\ 
  GO:0034316 & negative regulation of Arp2/3 complex-me... & 5 & 2 & 0.3 & $2.52 \times 10^{-2}$ & $2.76 \times 10^{2}$ \\ 
  GO:0042537 & benzene-containing compound metabolic pr... & 5 & 2 & 0.3 & $2.52 \times 10^{-2}$ & $2.76 \times 10^{2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 184 local transitivity 0.95 centile  local transitivity $>=$ 0.5 BP background PSP. Alpha = 4.5583006655119e-06isolates 0} 
\label{tab:Number of nodes 184 local transitivity 0.95 centile  local transitivity $>=$ 0.5 BP background PSP. Alpha = 4.5583006655119e-06isolates 0}
\end{table}

\todo{fix up this table so adjusted is not greater than 1}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:24:44 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0046873 & metal ion transmembrane transporter acti... & 123 & 60 & 31.5 & $1.50 \times 10^{-8}$ & TRUE \\ 
  GO:0005215 & transporter activity & 270 & 105 & 69.1 & $3.70 \times 10^{-7}$ & TRUE \\ 
  GO:0046872 & metal ion binding & 706 & 233 & 180.8 & $4.00 \times 10^{-7}$ & TRUE \\ 
  GO:0008324 & cation transmembrane transporter activit... & 177 & 75 & 45.3 & $4.20 \times 10^{-7}$ & TRUE \\ 
  GO:0043169 & cation binding & 718 & 236 & 183.8 & $4.70 \times 10^{-7}$ & TRUE \\ 
  GO:0022857 & transmembrane transporter activity & 250 & 98 & 64.0 & $6.10 \times 10^{-7}$ & TRUE \\ 
  GO:0015081 & sodium ion transmembrane transporter act... & 33 & 22 & 8.4 & $7.40 \times 10^{-7}$ & TRUE \\ 
  GO:0022890 & inorganic cation transmembrane transport... & 169 & 71 & 43.3 & $1.30 \times 10^{-6}$ & TRUE \\ 
  GO:0015077 & monovalent inorganic cation transmembran... & 103 & 48 & 26.4 & $2.40 \times 10^{-6}$ & TRUE \\ 
  GO:0015075 & ion transmembrane transporter activity & 223 & 87 & 57.1 & $3.50 \times 10^{-6}$ & TRUE \\ 
  GO:0015318 & inorganic molecular entity transmembrane... & 205 & 80 & 52.5 & $8.90 \times 10^{-6}$ & TRUE \\ 
  GO:0015079 & potassium ion transmembrane transporter ... & 44 & 24 & 11.3 & $3.60 \times 10^{-5}$ & FALSE \\ 
  GO:0005509 & calcium ion binding & 180 & 70 & 46.1 & $3.80 \times 10^{-5}$ & FALSE \\ 
  GO:0005244 & voltage-gated ion channel activity & 65 & 31 & 16.6 & $8.70 \times 10^{-5}$ & FALSE \\ 
  GO:0022832 & voltage-gated channel activity & 65 & 31 & 16.6 & $8.70 \times 10^{-5}$ & FALSE \\ 
  GO:0005261 & cation channel activity & 96 & 41 & 24.6 & $1.60 \times 10^{-4}$ & FALSE \\ 
  GO:0015267 & channel activity & 119 & 48 & 30.5 & $2.40 \times 10^{-4}$ & FALSE \\ 
  GO:0022803 & passive transmembrane transporter activi... & 119 & 48 & 30.5 & $2.40 \times 10^{-4}$ & FALSE \\ 
  GO:0022836 & gated channel activity & 98 & 41 & 25.1 & $2.70 \times 10^{-4}$ & FALSE \\ 
  GO:0005272 & sodium channel activity & 16 & 11 & 4.1 & $3.50 \times 10^{-4}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 935 local transitivity 0.05 centile  local transitivity $<=$ 0 MF background PSP. Alpha=2.02265372168285e-05isolates 0} 
\label{tab:Number of nodes 935 local transitivity 0.05 centile  local transitivity $<=$ 0 MF background PSP. Alpha=2.02265372168285e-05isolates 0}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:24:46 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0004890 & GABA-A receptor activity & 4 & 3 & 0.2 & $5.00 \times 10^{-4}$ & $1.24 \times 10^{0}$ \\ 
  GO:0022851 & GABA-gated chloride ion channel activity & 4 & 3 & 0.2 & $5.00 \times 10^{-4}$ & $1.24 \times 10^{0}$ \\ 
  GO:0016917 & GABA receptor activity & 6 & 3 & 0.3 & $2.30 \times 10^{-3}$ & $5.69 \times 10^{0}$ \\ 
  GO:0099095 & ligand-gated anion channel activity & 6 & 3 & 0.3 & $2.30 \times 10^{-3}$ & $5.69 \times 10^{0}$ \\ 
  GO:0004993 & G protein-coupled serotonin receptor act... & 2 & 2 & 0.1 & $2.60 \times 10^{-3}$ & $6.43 \times 10^{0}$ \\ 
  GO:0008227 & G protein-coupled amine receptor activit... & 2 & 2 & 0.1 & $2.60 \times 10^{-3}$ & $6.43 \times 10^{0}$ \\ 
  GO:0099589 & serotonin receptor activity & 2 & 2 & 0.1 & $2.60 \times 10^{-3}$ & $6.43 \times 10^{0}$ \\ 
  GO:0005253 & anion channel activity & 16 & 4 & 0.8 & $7.20 \times 10^{-3}$ & $1.78 \times 10^{1}$ \\ 
  GO:0099529 & neurotransmitter receptor activity invol... & 17 & 4 & 0.9 & $9.10 \times 10^{-3}$ & $2.25 \times 10^{1}$ \\ 
  GO:0030594 & neurotransmitter receptor activity & 27 & 5 & 1.4 & $1.03 \times 10^{-2}$ & $2.55 \times 10^{1}$ \\ 
  GO:0051537 & 2 iron, 2 sulfur cluster binding & 10 & 3 & 0.5 & $1.19 \times 10^{-2}$ & $2.94 \times 10^{1}$ \\ 
  GO:0051536 & iron-sulfur cluster binding & 28 & 5 & 1.4 & $1.21 \times 10^{-2}$ & $2.99 \times 10^{1}$ \\ 
  GO:0051540 & metal cluster binding & 28 & 5 & 1.4 & $1.21 \times 10^{-2}$ & $2.99 \times 10^{1}$ \\ 
  GO:0098960 & postsynaptic neurotransmitter receptor a... & 19 & 4 & 1.0 & $1.37 \times 10^{-2}$ & $3.39 \times 10^{1}$ \\ 
  GO:0005112 & Notch binding & 4 & 2 & 0.2 & $1.44 \times 10^{-2}$ & $3.56 \times 10^{1}$ \\ 
  GO:0016874 & ligase activity & 52 & 7 & 2.6 & $1.49 \times 10^{-2}$ & $3.68 \times 10^{1}$ \\ 
  GO:0005254 & chloride channel activity & 11 & 3 & 0.6 & $1.57 \times 10^{-2}$ & $3.88 \times 10^{1}$ \\ 
  GO:0048037 & cofactor binding & 153 & 14 & 7.8 & $2.16 \times 10^{-2}$ & $5.34 \times 10^{1}$ \\ 
  GO:0099528 & G protein-coupled neurotransmitter recep... & 5 & 2 & 0.2 & $2.32 \times 10^{-2}$ & $5.74 \times 10^{1}$ \\ 
  GO:0004298 & threonine-type endopeptidase activity & 13 & 3 & 0.7 & $2.53 \times 10^{-2}$ & $6.25 \times 10^{1}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 184 local transitivity 0.95 centile  local transitivity $>=$ 0.5 MF background PSP. Alpha = 2.02265372168285e-05isolates 0} 
\label{tab:Number of nodes 184 local transitivity 0.95 centile  local transitivity $>=$ 0.5 MF background PSP. Alpha = 2.02265372168285e-05isolates 0}
\end{table}

\todo{fix up this table so adjusted is not greater than 1 too }

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:26:59 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0016021 & integral component of membrane & 747 & 319 & 198.0 & $3.60 \times 10^{-28}$ & TRUE \\ 
  GO:0031224 & intrinsic component of membrane & 788 & 331 & 208.9 & $8.70 \times 10^{-28}$ & TRUE \\ 
  GO:0005887 & integral component of plasma membrane & 288 & 119 & 76.3 & $7.90 \times 10^{-9}$ & TRUE \\ 
  GO:0031226 & intrinsic component of plasma membrane & 308 & 125 & 81.7 & $1.20 \times 10^{-8}$ & TRUE \\ 
  GO:0044425 & membrane part & 1395 & 429 & 369.8 & $1.90 \times 10^{-6}$ & TRUE \\ 
  GO:0033268 & node of Ranvier & 12 & 9 & 3.2 & $6.20 \times 10^{-4}$ & FALSE \\ 
  GO:0034706 & sodium channel complex & 12 & 9 & 3.2 & $6.20 \times 10^{-4}$ & FALSE \\ 
  GO:0034703 & cation channel complex & 96 & 38 & 25.4 & $3.15 \times 10^{-3}$ & FALSE \\ 
  GO:0099699 & integral component of synaptic membrane & 61 & 26 & 16.2 & $4.34 \times 10^{-3}$ & FALSE \\ 
  GO:0099240 & intrinsic component of synaptic membrane & 67 & 28 & 17.8 & $4.39 \times 10^{-3}$ & FALSE \\ 
  GO:1902495 & transmembrane transporter complex & 124 & 46 & 32.9 & $5.46 \times 10^{-3}$ & FALSE \\ 
  GO:0001518 & voltage-gated sodium channel complex & 6 & 5 & 1.6 & $6.08 \times 10^{-3}$ & FALSE \\ 
  GO:0032809 & neuronal cell body membrane & 13 & 8 & 3.5 & $8.18 \times 10^{-3}$ & FALSE \\ 
  GO:0043202 & lysosomal lumen & 26 & 13 & 6.9 & $8.63 \times 10^{-3}$ & FALSE \\ 
  GO:0043194 & axon initial segment & 16 & 9 & 4.2 & $1.11 \times 10^{-2}$ & FALSE \\ 
  GO:0034705 & potassium channel complex & 38 & 17 & 10.1 & $1.13 \times 10^{-2}$ & FALSE \\ 
  GO:1990351 & transporter complex & 129 & 46 & 34.2 & $1.24 \times 10^{-2}$ & FALSE \\ 
  GO:0031045 & dense core granule & 9 & 6 & 2.4 & $1.34 \times 10^{-2}$ & FALSE \\ 
  GO:0098533 & ATPase dependent transmembrane transport... & 9 & 6 & 2.4 & $1.34 \times 10^{-2}$ & FALSE \\ 
  GO:0044298 & cell body membrane & 14 & 8 & 3.7 & $1.47 \times 10^{-2}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 935 local transitivity 0.05 centile  local transitivity $<=$ 0 CC background PSP. Alpha=3.33555703802535e-05isolates 0} 
\label{tab:Number of nodes 935 local transitivity 0.05 centile  local transitivity $<=$ 0 CC background PSP.Alpha=3.33555703802535e-05isolates 0}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:27:01 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{max width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:1902711 & GABA-A receptor complex & 4 & 3 & 0.2 & $5.30 \times 10^{-4}$ & $7.94 \times 10^{-1}$ \\ 
  GO:1902710 & GABA receptor complex & 5 & 3 & 0.3 & $1.27 \times 10^{-3}$ & $1.90 \times 10^{0}$ \\ 
  GO:0098888 & extrinsic component of presynaptic membr... & 6 & 3 & 0.3 & $2.45 \times 10^{-3}$ & $3.67 \times 10^{0}$ \\ 
  GO:0000315 & organellar large ribosomal subunit & 15 & 4 & 0.8 & $6.10 \times 10^{-3}$ & $9.14 \times 10^{0}$ \\ 
  GO:0005762 & mitochondrial large ribosomal subunit & 15 & 4 & 0.8 & $6.10 \times 10^{-3}$ & $9.14 \times 10^{0}$ \\ 
  GO:0098798 & mitochondrial protein complex & 131 & 14 & 6.8 & $7.15 \times 10^{-3}$ & $1.07 \times 10^{1}$ \\ 
  GO:0042719 & mitochondrial intermembrane space protei... & 3 & 2 & 0.2 & $7.76 \times 10^{-3}$ & $1.16 \times 10^{1}$ \\ 
  GO:0034707 & chloride channel complex & 9 & 3 & 0.5 & $9.16 \times 10^{-3}$ & $1.37 \times 10^{1}$ \\ 
  GO:0072669 & tRNA-splicing ligase complex & 4 & 2 & 0.2 & $1.50 \times 10^{-2}$ & $2.25 \times 10^{1}$ \\ 
  GO:0098871 & postsynaptic actin cytoskeleton & 4 & 2 & 0.2 & $1.50 \times 10^{-2}$ & $2.25 \times 10^{1}$ \\ 
  GO:0098793 & presynapse & 267 & 22 & 13.9 & $1.84 \times 10^{-2}$ & $2.76 \times 10^{1}$ \\ 
  GO:0099243 & extrinsic component of synaptic membrane & 12 & 3 & 0.6 & $2.14 \times 10^{-2}$ & $3.20 \times 10^{1}$ \\ 
  GO:0097025 & MPP7-DLG1-LIN7 complex & 5 & 2 & 0.3 & $2.42 \times 10^{-2}$ & $3.62 \times 10^{1}$ \\ 
  GO:0005839 & proteasome core complex & 13 & 3 & 0.7 & $2.67 \times 10^{-2}$ & $4.01 \times 10^{1}$ \\ 
  GO:0098831 & presynaptic active zone cytoplasmic comp... & 13 & 3 & 0.7 & $2.67 \times 10^{-2}$ & $4.01 \times 10^{1}$ \\ 
  GO:0042734 & presynaptic membrane & 84 & 9 & 4.4 & $2.84 \times 10^{-2}$ & $4.26 \times 10^{1}$ \\ 
  GO:0019774 & proteasome core complex, beta-subunit co... & 6 & 2 & 0.3 & $3.50 \times 10^{-2}$ & $5.25 \times 10^{1}$ \\ 
  GO:0031082 & BLOC complex & 6 & 2 & 0.3 & $3.50 \times 10^{-2}$ & $5.25 \times 10^{1}$ \\ 
  GO:0031083 & BLOC-1 complex & 6 & 2 & 0.3 & $3.50 \times 10^{-2}$ & $5.25 \times 10^{1}$ \\ 
  GO:0005884 & actin filament & 61 & 7 & 3.2 & $3.67 \times 10^{-2}$ & $5.51 \times 10^{1}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 184 local transitivity 0.95 centile  local transitivity $>=$ 0.5 CC background PSP. Alpha = 3.33555703802535e-05isolates 0} 
\label{tab:Number of nodes 184 local transitivity 0.95 centile  local transitivity $>=$ 0.5 CC background PSP. Alpha = 3.33555703802535e-05isolates 0}
\end{table}

\subsection{kcoreness Gene ontology enrichment topGO}

Biological process 0.05 centile low kcore kcore less than or equal to 1 308 nodes 0.05 centile table~\ref{tab:Number of nodes 308 k core 0.05 centile  k core $<=$ 1 BP background PSP. Alpha=4.5583006655119e-06}

Biological process 0.95 high centile kcore greater than or equal to 23 table~\ref{tab:Number of nodes 201 k core 0.95 centile  k core $>=$ 23 BP background PSP. Alpha = 4.5583006655119e-06}

Molecular function 0.05 centile low kcore less than or equal to 1 308 nodes \ref{tab:Number of nodes 308 k core 0.05 centile  k core $<=$ 1 MF background PSP. Alpha=2.02265372168285e-05}

Seem to have two molecular function with different tables but with valid MF ids need to check
\ref{tab:Number of nodes 201 k core 0.95 centile  k core $>=$ 23 MF background PSP. Alpha = 2.02265372168285e-05}

\todo{check molecular function topgo kcore and missing cellular component}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:42:08 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0009190 & cyclic nucleotide biosynthetic process & 9 & 5 & 0.8 & $4.20 \times 10^{-4}$ & FALSE \\ 
  GO:0052652 & cyclic purine nucleotide metabolic proce... & 9 & 5 & 0.8 & $4.20 \times 10^{-4}$ & FALSE \\ 
  GO:0097035 & regulation of membrane lipid distributio... & 9 & 5 & 0.8 & $4.20 \times 10^{-4}$ & FALSE \\ 
  GO:0038171 & cannabinoid signaling pathway & 3 & 3 & 0.3 & $6.20 \times 10^{-4}$ & FALSE \\ 
  GO:0071926 & endocannabinoid signaling pathway & 3 & 3 & 0.3 & $6.20 \times 10^{-4}$ & FALSE \\ 
  GO:0009187 & cyclic nucleotide metabolic process & 16 & 6 & 1.4 & $1.41 \times 10^{-3}$ & FALSE \\ 
  GO:0006171 & cAMP biosynthetic process & 4 & 3 & 0.3 & $2.31 \times 10^{-3}$ & FALSE \\ 
  GO:0021681 & cerebellar granular layer development & 8 & 4 & 0.7 & $2.77 \times 10^{-3}$ & FALSE \\ 
  GO:0046058 & cAMP metabolic process & 8 & 4 & 0.7 & $2.77 \times 10^{-3}$ & FALSE \\ 
  GO:0046339 & diacylglycerol metabolic process & 8 & 4 & 0.7 & $2.77 \times 10^{-3}$ & FALSE \\ 
  GO:0044273 & sulfur compound catabolic process & 13 & 5 & 1.1 & $3.18 \times 10^{-3}$ & FALSE \\ 
  GO:0030166 & proteoglycan biosynthetic process & 9 & 4 & 0.8 & $4.65 \times 10^{-3}$ & FALSE \\ 
  GO:1904321 & response to forskolin & 5 & 3 & 0.4 & $5.41 \times 10^{-3}$ & FALSE \\ 
  GO:1904322 & cellular response to forskolin & 5 & 3 & 0.4 & $5.41 \times 10^{-3}$ & FALSE \\ 
  GO:0006639 & acylglycerol metabolic process & 27 & 7 & 2.3 & $6.13 \times 10^{-3}$ & FALSE \\ 
  GO:0046461 & neutral lipid catabolic process & 10 & 4 & 0.8 & $7.23 \times 10^{-3}$ & FALSE \\ 
  GO:0046464 & acylglycerol catabolic process & 10 & 4 & 0.8 & $7.23 \times 10^{-3}$ & FALSE \\ 
  GO:0001539 & cilium or flagellum-dependent cell motil... & 2 & 2 & 0.2 & $7.27 \times 10^{-3}$ & FALSE \\ 
  GO:0006741 & NADP biosynthetic process & 2 & 2 & 0.2 & $7.27 \times 10^{-3}$ & FALSE \\ 
  GO:0007168 & receptor guanylyl cyclase signaling path... & 2 & 2 & 0.2 & $7.27 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 308 k core 0.05 centile  k core $<=$ 1 BP background PSP. Alpha=4.5583006655119e-06} 
\label{tab:Number of nodes 308 k core 0.05 centile  k core $<=$ 1 BP background PSP. Alpha=4.5583006655119e-06}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:42:21 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0016071 & mRNA metabolic process & 254 & 80 & 15.3 & $1.00 \times 10^{-30}$ & $1.10 \times 10^{-26}$ \\ 
  GO:0090304 & nucleic acid metabolic process & 801 & 129 & 48.2 & $1.00 \times 10^{-30}$ & $1.10 \times 10^{-26}$ \\ 
  GO:0044403 & symbiont process & 359 & 83 & 21.6 & $1.00 \times 10^{-30}$ & $1.10 \times 10^{-26}$ \\ 
  GO:0044419 & interspecies interaction between organis... & 363 & 83 & 21.9 & $1.00 \times 10^{-30}$ & $1.10 \times 10^{-26}$ \\ 
  GO:0051704 & multi-organism process & 597 & 106 & 36.0 & $1.00 \times 10^{-30}$ & $1.10 \times 10^{-26}$ \\ 
  GO:0060255 & regulation of macromolecule metabolic pr... & 1167 & 148 & 70.3 & $1.00 \times 10^{-30}$ & $1.10 \times 10^{-26}$ \\ 
  GO:0016032 & viral process & 344 & 79 & 20.7 & $9.10 \times 10^{-30}$ & $9.98 \times 10^{-26}$ \\ 
  GO:0010467 & gene expression & 922 & 130 & 55.5 & $1.10 \times 10^{-29}$ & $1.21 \times 10^{-25}$ \\ 
  GO:0010468 & regulation of gene expression & 766 & 118 & 46.1 & $1.40 \times 10^{-29}$ & $1.54 \times 10^{-25}$ \\ 
  GO:0008380 & RNA splicing & 89 & 42 & 5.4 & $3.40 \times 10^{-29}$ & $3.73 \times 10^{-25}$ \\ 
  GO:0009057 & macromolecule catabolic process & 431 & 87 & 26.0 & $1.00 \times 10^{-28}$ & $1.10 \times 10^{-24}$ \\ 
  GO:0006396 & RNA processing & 161 & 54 & 9.7 & $1.60 \times 10^{-28}$ & $1.76 \times 10^{-24}$ \\ 
  GO:0016070 & RNA metabolic process & 716 & 111 & 43.1 & $2.10 \times 10^{-27}$ & $2.30 \times 10^{-23}$ \\ 
  GO:0044265 & cellular macromolecule catabolic process & 363 & 78 & 21.9 & $3.30 \times 10^{-27}$ & $3.62 \times 10^{-23}$ \\ 
  GO:0010629 & negative regulation of gene expression & 410 & 82 & 24.7 & $1.80 \times 10^{-26}$ & $1.97 \times 10^{-22}$ \\ 
  GO:0000375 & RNA splicing, via transesterification re... & 73 & 36 & 4.4 & $7.00 \times 10^{-26}$ & $7.68 \times 10^{-22}$ \\ 
  GO:0000377 & RNA splicing, via transesterification re... & 73 & 36 & 4.4 & $7.00 \times 10^{-26}$ & $7.68 \times 10^{-22}$ \\ 
  GO:0000398 & mRNA splicing, via spliceosome & 73 & 36 & 4.4 & $7.00 \times 10^{-26}$ & $7.68 \times 10^{-22}$ \\ 
  GO:0043170 & macromolecule metabolic process & 1679 & 170 & 101.1 & $1.20 \times 10^{-25}$ & $1.32 \times 10^{-21}$ \\ 
  GO:0006402 & mRNA catabolic process & 175 & 53 & 10.5 & $1.70 \times 10^{-25}$ & $1.86 \times 10^{-21}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 201 k core 0.95 centile  k core $>=$ 23 BP background PSP. Alpha = 4.5583006655119e-06} 
\label{tab:Number of nodes 201 k core 0.95 centile  k core $>=$ 23 BP background PSP. Alpha = 4.5583006655119e-06}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:44:22 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0022857 & transmembrane transporter activity & 250 & 40 & 20.3 & $1.30 \times 10^{-5}$ & TRUE \\ 
  GO:0005215 & transporter activity & 270 & 41 & 22.0 & $3.80 \times 10^{-5}$ & FALSE \\ 
  GO:0015075 & ion transmembrane transporter activity & 223 & 35 & 18.1 & $7.50 \times 10^{-5}$ & FALSE \\ 
  GO:0016849 & phosphorus-oxygen lyase activity & 8 & 5 & 0.7 & $1.60 \times 10^{-4}$ & FALSE \\ 
  GO:0009975 & cyclase activity & 9 & 5 & 0.7 & $3.30 \times 10^{-4}$ & FALSE \\ 
  GO:0008514 & organic anion transmembrane transporter ... & 36 & 10 & 2.9 & $4.10 \times 10^{-4}$ & FALSE \\ 
  GO:0046873 & metal ion transmembrane transporter acti... & 123 & 21 & 10.0 & $7.30 \times 10^{-4}$ & FALSE \\ 
  GO:0015318 & inorganic molecular entity transmembrane... & 205 & 30 & 16.7 & $8.70 \times 10^{-4}$ & FALSE \\ 
  GO:0015081 & sodium ion transmembrane transporter act... & 33 & 9 & 2.7 & $9.20 \times 10^{-4}$ & FALSE \\ 
  GO:0008509 & anion transmembrane transporter activity & 55 & 12 & 4.5 & $1.20 \times 10^{-3}$ & FALSE \\ 
  GO:0005540 & hyaluronic acid binding & 7 & 4 & 0.6 & $1.23 \times 10^{-3}$ & FALSE \\ 
  GO:0046872 & metal ion binding & 706 & 78 & 57.4 & $1.23 \times 10^{-3}$ & FALSE \\ 
  GO:0015291 & secondary active transmembrane transport... & 29 & 8 & 2.4 & $1.63 \times 10^{-3}$ & FALSE \\ 
  GO:0005244 & voltage-gated ion channel activity & 65 & 13 & 5.3 & $1.77 \times 10^{-3}$ & FALSE \\ 
  GO:0022832 & voltage-gated channel activity & 65 & 13 & 5.3 & $1.77 \times 10^{-3}$ & FALSE \\ 
  GO:0008324 & cation transmembrane transporter activit... & 177 & 26 & 14.4 & $1.83 \times 10^{-3}$ & FALSE \\ 
  GO:0004016 & adenylate cyclase activity & 4 & 3 & 0.3 & $2.00 \times 10^{-3}$ & FALSE \\ 
  GO:0030021 & extracellular matrix structural constitu... & 4 & 3 & 0.3 & $2.00 \times 10^{-3}$ & FALSE \\ 
  GO:0043169 & cation binding & 718 & 78 & 58.4 & $2.06 \times 10^{-3}$ & FALSE \\ 
  GO:0022890 & inorganic cation transmembrane transport... & 169 & 24 & 13.7 & $4.30 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 308 k core 0.05 centile  k core $<=$ 1 MF background PSP. Alpha=2.02265372168285e-05} 
\label{tab:Number of nodes 308 k core 0.05 centile  k core $<=$ 1 MF background PSP. Alpha=2.02265372168285e-05}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:44:23 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0003723 & RNA binding & 549 & 119 & 33.4 & $1.00 \times 10^{-30}$ & $2.47 \times 10^{-27}$ \\ 
  GO:0003676 & nucleic acid binding & 716 & 129 & 43.5 & $1.00 \times 10^{-30}$ & $2.47 \times 10^{-27}$ \\ 
  GO:0097159 & organic cyclic compound binding & 1366 & 155 & 83.0 & $1.60 \times 10^{-26}$ & $3.96 \times 10^{-23}$ \\ 
  GO:1901363 & heterocyclic compound binding & 1355 & 154 & 82.3 & $2.90 \times 10^{-26}$ & $7.17 \times 10^{-23}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 108 & 35 & 6.6 & $9.50 \times 10^{-18}$ & $2.35 \times 10^{-14}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 114 & 35 & 6.9 & $6.70 \times 10^{-17}$ & $1.66 \times 10^{-13}$ \\ 
  GO:0019899 & enzyme binding & 717 & 94 & 43.6 & $2.50 \times 10^{-16}$ & $6.18 \times 10^{-13}$ \\ 
  GO:0003729 & mRNA binding & 87 & 28 & 5.3 & $3.00 \times 10^{-14}$ & $7.42 \times 10^{-11}$ \\ 
  GO:0005515 & protein binding & 2796 & 199 & 169.9 & $5.40 \times 10^{-13}$ & $1.33 \times 10^{-9}$ \\ 
  GO:0003735 & structural constituent of ribosome & 101 & 27 & 6.1 & $1.30 \times 10^{-11}$ & $3.21 \times 10^{-8}$ \\ 
  GO:0019843 & rRNA binding & 25 & 14 & 1.5 & $1.50 \times 10^{-11}$ & $3.71 \times 10^{-8}$ \\ 
  GO:0051082 & unfolded protein binding & 56 & 20 & 3.4 & $2.10 \times 10^{-11}$ & $5.19 \times 10^{-8}$ \\ 
  GO:0003677 & DNA binding & 251 & 44 & 15.2 & $2.30 \times 10^{-11}$ & $5.69 \times 10^{-8}$ \\ 
  GO:0045296 & cadherin binding & 219 & 40 & 13.3 & $5.90 \times 10^{-11}$ & $1.46 \times 10^{-7}$ \\ 
  GO:0050839 & cell adhesion molecule binding & 272 & 45 & 16.5 & $1.00 \times 10^{-10}$ & $2.47 \times 10^{-7}$ \\ 
  GO:0008134 & transcription factor binding & 127 & 29 & 7.7 & $1.50 \times 10^{-10}$ & $3.71 \times 10^{-7}$ \\ 
  GO:0043565 & sequence-specific DNA binding & 112 & 26 & 6.8 & $1.00 \times 10^{-9}$ & $2.47 \times 10^{-6}$ \\ 
  GO:0019904 & protein domain specific binding & 273 & 43 & 16.6 & $1.50 \times 10^{-9}$ & $3.71 \times 10^{-6}$ \\ 
  GO:0005198 & structural molecule activity & 300 & 45 & 18.2 & $2.90 \times 10^{-9}$ & $7.17 \times 10^{-6}$ \\ 
  GO:0003727 & single-stranded RNA binding & 31 & 13 & 1.9 & $8.20 \times 10^{-9}$ & $2.03 \times 10^{-5}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 201 k core 0.95 centile  k core $>=$ 23 MF background PSP. Alpha = 2.02265372168285e-05} 
\label{tab:Number of nodes 201 k core 0.95 centile  k core $>=$ 23 MF background PSP. Alpha = 2.02265372168285e-05}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:45:56 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0022857 & transmembrane transporter activity & 250 & 40 & 20.3 & $1.30 \times 10^{-5}$ & TRUE \\ 
  GO:0005215 & transporter activity & 270 & 41 & 22.0 & $3.80 \times 10^{-5}$ & FALSE \\ 
  GO:0015075 & ion transmembrane transporter activity & 223 & 35 & 18.1 & $7.50 \times 10^{-5}$ & FALSE \\ 
  GO:0016849 & phosphorus-oxygen lyase activity & 8 & 5 & 0.7 & $1.60 \times 10^{-4}$ & FALSE \\ 
  GO:0009975 & cyclase activity & 9 & 5 & 0.7 & $3.30 \times 10^{-4}$ & FALSE \\ 
  GO:0008514 & organic anion transmembrane transporter ... & 36 & 10 & 2.9 & $4.10 \times 10^{-4}$ & FALSE \\ 
  GO:0046873 & metal ion transmembrane transporter acti... & 123 & 21 & 10.0 & $7.30 \times 10^{-4}$ & FALSE \\ 
  GO:0015318 & inorganic molecular entity transmembrane... & 205 & 30 & 16.7 & $8.70 \times 10^{-4}$ & FALSE \\ 
  GO:0015081 & sodium ion transmembrane transporter act... & 33 & 9 & 2.7 & $9.20 \times 10^{-4}$ & FALSE \\ 
  GO:0008509 & anion transmembrane transporter activity & 55 & 12 & 4.5 & $1.20 \times 10^{-3}$ & FALSE \\ 
  GO:0005540 & hyaluronic acid binding & 7 & 4 & 0.6 & $1.23 \times 10^{-3}$ & FALSE \\ 
  GO:0046872 & metal ion binding & 706 & 78 & 57.4 & $1.23 \times 10^{-3}$ & FALSE \\ 
  GO:0015291 & secondary active transmembrane transport... & 29 & 8 & 2.4 & $1.63 \times 10^{-3}$ & FALSE \\ 
  GO:0005244 & voltage-gated ion channel activity & 65 & 13 & 5.3 & $1.77 \times 10^{-3}$ & FALSE \\ 
  GO:0022832 & voltage-gated channel activity & 65 & 13 & 5.3 & $1.77 \times 10^{-3}$ & FALSE \\ 
  GO:0008324 & cation transmembrane transporter activit... & 177 & 26 & 14.4 & $1.83 \times 10^{-3}$ & FALSE \\ 
  GO:0004016 & adenylate cyclase activity & 4 & 3 & 0.3 & $2.00 \times 10^{-3}$ & FALSE \\ 
  GO:0030021 & extracellular matrix structural constitu... & 4 & 3 & 0.3 & $2.00 \times 10^{-3}$ & FALSE \\ 
  GO:0043169 & cation binding & 718 & 78 & 58.4 & $2.06 \times 10^{-3}$ & FALSE \\ 
  GO:0022890 & inorganic cation transmembrane transport... & 169 & 24 & 13.7 & $4.30 \times 10^{-3}$ & FALSE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 308 k core 0.05 centile  k core $<=$ 1 MF background PSP. Alpha=3.33555703802535e-05} 
\label{tab:Number of nodes 308 k core 0.05 centile  k core $<=$ 1 MF background PSP. Alpha=3.33555703802535e-05}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:45:58 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{llrrrrr}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0003723 & RNA binding & 549 & 119 & 33.4 & $1.00 \times 10^{-30}$ & $1.50 \times 10^{-27}$ \\ 
  GO:0003676 & nucleic acid binding & 716 & 129 & 43.5 & $1.00 \times 10^{-30}$ & $1.50 \times 10^{-27}$ \\ 
  GO:0097159 & organic cyclic compound binding & 1366 & 155 & 83.0 & $1.60 \times 10^{-26}$ & $2.40 \times 10^{-23}$ \\ 
  GO:1901363 & heterocyclic compound binding & 1355 & 154 & 82.3 & $2.90 \times 10^{-26}$ & $4.35 \times 10^{-23}$ \\ 
  GO:0031625 & ubiquitin protein ligase binding & 108 & 35 & 6.6 & $9.50 \times 10^{-18}$ & $1.42 \times 10^{-14}$ \\ 
  GO:0044389 & ubiquitin-like protein ligase binding & 114 & 35 & 6.9 & $6.70 \times 10^{-17}$ & $1.00 \times 10^{-13}$ \\ 
  GO:0019899 & enzyme binding & 717 & 94 & 43.6 & $2.50 \times 10^{-16}$ & $3.75 \times 10^{-13}$ \\ 
  GO:0003729 & mRNA binding & 87 & 28 & 5.3 & $3.00 \times 10^{-14}$ & $4.50 \times 10^{-11}$ \\ 
  GO:0005515 & protein binding & 2796 & 199 & 169.9 & $5.40 \times 10^{-13}$ & $8.09 \times 10^{-10}$ \\ 
  GO:0003735 & structural constituent of ribosome & 101 & 27 & 6.1 & $1.30 \times 10^{-11}$ & $1.95 \times 10^{-8}$ \\ 
  GO:0019843 & rRNA binding & 25 & 14 & 1.5 & $1.50 \times 10^{-11}$ & $2.25 \times 10^{-8}$ \\ 
  GO:0051082 & unfolded protein binding & 56 & 20 & 3.4 & $2.10 \times 10^{-11}$ & $3.15 \times 10^{-8}$ \\ 
  GO:0003677 & DNA binding & 251 & 44 & 15.2 & $2.30 \times 10^{-11}$ & $3.45 \times 10^{-8}$ \\ 
  GO:0045296 & cadherin binding & 219 & 40 & 13.3 & $5.90 \times 10^{-11}$ & $8.84 \times 10^{-8}$ \\ 
  GO:0050839 & cell adhesion molecule binding & 272 & 45 & 16.5 & $1.00 \times 10^{-10}$ & $1.50 \times 10^{-7}$ \\ 
  GO:0008134 & transcription factor binding & 127 & 29 & 7.7 & $1.50 \times 10^{-10}$ & $2.25 \times 10^{-7}$ \\ 
  GO:0043565 & sequence-specific DNA binding & 112 & 26 & 6.8 & $1.00 \times 10^{-9}$ & $1.50 \times 10^{-6}$ \\ 
  GO:0019904 & protein domain specific binding & 273 & 43 & 16.6 & $1.50 \times 10^{-9}$ & $2.25 \times 10^{-6}$ \\ 
  GO:0005198 & structural molecule activity & 300 & 45 & 18.2 & $2.90 \times 10^{-9}$ & $4.35 \times 10^{-6}$ \\ 
  GO:0003727 & single-stranded RNA binding & 31 & 13 & 1.9 & $8.20 \times 10^{-9}$ & $1.23 \times 10^{-5}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 201 k core 0.95 centile  k core $>=$ 23 MF background PSP. Alpha = 3.33555703802535e-05} 
\label{tab:Number of nodes 201 k core 0.95 centile  k core $>=$ 23 MF background PSP. Alpha = 3.33555703802535e-05}
\end{table}

Example of range 1-7

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun  2 16:55:30 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{llrrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & classic & bonf \\ 
  \hline
GO:0005215 & transporter activity & 270 & 184 & 130.1 & $4.00 \times 10^{-12}$ & TRUE \\ 
  GO:0022857 & transmembrane transporter activity & 250 & 172 & 120.5 & $6.20 \times 10^{-12}$ & TRUE \\ 
  GO:0015075 & ion transmembrane transporter activity & 223 & 151 & 107.5 & $8.80 \times 10^{-10}$ & TRUE \\ 
  GO:0015318 & inorganic molecular entity transmembrane... & 205 & 138 & 98.8 & $9.20 \times 10^{-9}$ & TRUE \\ 
  GO:0046873 & metal ion transmembrane transporter acti... & 123 & 89 & 59.3 & $2.60 \times 10^{-8}$ & TRUE \\ 
  GO:0008324 & cation transmembrane transporter activit... & 177 & 119 & 85.3 & $1.20 \times 10^{-7}$ & TRUE \\ 
  GO:0038023 & signaling receptor activity & 128 & 90 & 61.7 & $2.00 \times 10^{-7}$ & TRUE \\ 
  GO:0004930 & G protein-coupled receptor activity & 24 & 23 & 11.6 & $6.10 \times 10^{-7}$ & TRUE \\ 
  GO:0048037 & cofactor binding & 153 & 103 & 73.7 & $7.90 \times 10^{-7}$ & TRUE \\ 
  GO:0022890 & inorganic cation transmembrane transport... & 169 & 112 & 81.4 & $8.80 \times 10^{-7}$ & TRUE \\ 
  GO:0015267 & channel activity & 119 & 83 & 57.3 & $1.00 \times 10^{-6}$ & TRUE \\ 
  GO:0022803 & passive transmembrane transporter activi... & 119 & 83 & 57.3 & $1.00 \times 10^{-6}$ & TRUE \\ 
  GO:0022838 & substrate-specific channel activity & 114 & 80 & 54.9 & $1.10 \times 10^{-6}$ & TRUE \\ 
  GO:0046872 & metal ion binding & 706 & 396 & 340.2 & $1.30 \times 10^{-6}$ & TRUE \\ 
  GO:0043169 & cation binding & 718 & 402 & 346.0 & $1.40 \times 10^{-6}$ & TRUE \\ 
  GO:0005216 & ion channel activity & 113 & 79 & 54.5 & $1.60 \times 10^{-6}$ & TRUE \\ 
  GO:0004888 & transmembrane signaling receptor activit... & 99 & 70 & 47.7 & $3.40 \times 10^{-6}$ & TRUE \\ 
  GO:0015077 & monovalent inorganic cation transmembran... & 103 & 72 & 49.6 & $4.90 \times 10^{-6}$ & TRUE \\ 
  GO:0050662 & coenzyme binding & 107 & 74 & 51.6 & $6.70 \times 10^{-6}$ & TRUE \\ 
  GO:0015081 & sodium ion transmembrane transporter act... & 33 & 28 & 15.9 & $1.30 \times 10^{-5}$ & TRUE \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{Number of nodes 1724 k core 0 centile  k core1-7 MF background PSP. Alpha=3.33555703802535e-05} 
\label{tab:kcore range GO Number of nodes 1724 k core 0 centile  k core1-7 MF background PSP. Alpha=3.33555703802535e-05}
\end{table}

code \url{source('~/RProjects/group_sizes/R/plotting/modified_exampleGoObject_summary_PSP_specifickcore.R')}

\section{Data table for Exac}
\label{sec:supplemental data table for exac}
\begin{itemize}
    \item { transcript}
    \item{gene}
    \item{Chromosome - chr}
    \item{number of exons - n\_exons}
    \item{transcript start - tx\_start}
    \item{transcript end - tx\_end}
    \item{base pair - bp}
    \item{mean synonymous - mu\_syn}
    \item{mean missense - mu\_mis}
    \item{mean loss of function - mu\_lof}
    \item{number of synonymous - n\_syn}
    \item{number of missense mutations - n\_mis}
    \item{number of loss of function mutations - n\_lof}
    \item{Expected synonymous - exp\_syn}
    \item{Expected missense - exp\_mis}
    \item{Expected loss of function - exp\_lof}
    \item{Z value synonymous - syn\_z}
    \item{Z value missense - mis\_z}
    \item{Z value loss of function - lof\_z}
    \item{Probability of loss of fucntion intolerance - pLI}
    \item{Number of copy number variations (CNV) - n\_cnv}
    \item{Expected number of copy number variations - exp\_cnv}
    \item{Z value for copy number variations - cnv\_z}
    
\end{itemize}

\subsection{Degree distribution}

Table of $\gamma$ as a function of minimal degree table~\ref{table:gamma}
\begin{table}[ht]
\centering
\begin{tabular}{lr}
  \hline
  from degree $k$ & coefficient $\gamma$ \\ 
  \hline
1 & 1.38 \\ 
  5 & 1.81 \\ 
  10 & 2.10 \\ 
  15 & 2.30 \\ 
  20 & 2.43 \\ 
  25 & 2.51 \\ 
   30 & 2.52 \\ 
  35 & 2.54 \\ 
   40 & 2.54 \\ 
   45 & 2.60 \\ 
  50 & 2.63 \\ 
   60 & 2.60 \\ 
   70 & 2.72 \\ 
   80 & 2.80 \\ 
   90 & 2.90 \\ 
   100 & 2.92 \\ 
  150 & 2.88 \\ 
   200 & 3.44 \\ 
   250 & 3.90 \\ 
  300 & 5.04 \\ 
   350 & 5.51 \\ 
   400 & 6.33 \\ 
   \hline
\end{tabular}
\caption{The $\gamma$ as a function of the degree sequence start (e.g. all degree $>5$). from RProjects/PhDGraphs}
  \label{table:gamma}
\end{table}


\section{GO enrichment panther}

\subsection{Degree}


\textbf{Biological process} background PSP Panther top 0.9 centile degree centrality top 20 terms. Ordered by fold change table~\ref{tab:GO biological process complete BP 3457 90cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 385}

Biological process background PSP. top 0.9 centile. Over representation top 20 terms. 385 significant terms with FDR 0.05 table~\ref{tab:GO biological process complete BP 3457 90cent deg.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 385}

The high degree nodes with the highest fold change that had enriched association with the biological process clade using the PSP as background was the ERBB2 signalling pathway GO:0038128  ( synonyms HER2 signaling pathway , NEU signaling , receptor tyrosine-protein kinase erbB-2 signaling pathway folder change 6.52 FDR p = 0.00381 (10 if 12 in the PSP). In addition to the well described role in cellular processes and as an oncogene \cite{baselga2009novel} ERBB2 compents are involved in synaptic plasticity with components interacting with PSD \cite{funk2020region} and PICK1 interacts with erbb2 receptor \cite{kim2004pdz}


Others were enriched for essential cell cycle components such as  mRNA stabilization (GO:0048255) 6.15 times fold change p=0.00254. There were also a number of references to telomere regulation such as  regulation of telomere maintenance (GO:0032204) 5.15 times 10 of 16 terms in PSP p= 0.00069. 

A number of essential cell processess had the lowest FDR for high degree nodes including cellular response to organic substance (GO:0071310) p= $3.16 \times 10^{-11}$ and    protein metabolism  regulation of protein metabolic process (GO:0051246)

A total of 30 terms were enriched for \textbf{molecular function} in high degree nodes table~\ref{tab:GO molecular function complete MF 3457 90cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 30} ordered by fold change and table~\ref{tab:GO molecular function complete MF 3457 90cent deg.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 30} ordered by fold change. 

ubiquitin protein ligase binding (GO:0031625) total in PSP  96  high degree 40  fold change 3.3 p $3.94 \times 10^{-9}$ FDR $3.60 \times 10^{-6}$ 

 p53 binding (GO:0002039) 11 of 19  fold change 4.5 p $2.54 \times 10^{-4}$ FDR $2.89 \times 10^{-2}$ as expected from the literature 

 protein domain specific binding (GO:0019904) 67 of 242  fold change 2.2 p =  $5.20 \times 10^{-8}$ FDR $3.56 \times 10^{-5}$ 


46 terms were enriched for \textbf{cellular component} sorted by fold change in table~\ref{tab:GO cellular component complete CC 3457 90cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 46} or by FDR in table~\ref{tab:GO cellular component complete CC 3457 90cent deg.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 46}

High degree nodes were located in the  nucleus (GO:0005634) 246 of 1359 fold change 1.4 p $3.67 \times 10^{-11}$ FDR $5.53 \times 10^{-8}$  
and involved  microtubule cytoskeleton (GO:0015630) 75 of 300 fold change 2.0 p= $2.77 \times 10^{-7}$ FDR $5.21 \times 10^{-5}$ and
and  extracellular vesicle (GO:1903561) 141 of 771 fold change 1.4 p= $1.08 \times 10^{-5}$ FDR $8.10 \times 10^{-4}$ 

Pathway enrichment analysis in Panther table~\ref{tab:GO analysis Panther pathways 3457 90cent deg.txtOver representation only} revealed four enriched pathways. The most significant was Parkinson disease (P00049) 27 of 45 terms fold change 4.7 FDR $6.35 \times 10^{-7}$. Parkinsons disease would initially appear to be an example of a collection of disease related genes that are \textit{not} in the periphery of the network as has been suggested by the disease module concept of Barabassi. \cite{barabasi2011network}. Hu found that a Parkinson's gene set for the human interactome had a moderate degree distribution and lower than those associated with cancers \cite{hu2017network}

\subsubsection{Low degree centrality}

Low degree nodes have few typical features on over representation analysis
No over representation enrichment for biological process, molecular function or any GO SLIM. Cell membrane ontology term intrinsic component of membrane is over represented and contains 131 of the 174 genes in the PSP with this label FDR $1.18 \times 10^{-2}$ (table~\ref{tab:GO cellular component complete CC 3457 10cent deg.txt Over representation only. Top 20 Terms. 2 significant terms shown . Total significant terms 2})

\footnote{ enriched for single unclassified reactome term table~\ref{tab:Reactome pathways reactome 3457 10cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 2 significant terms shown. Total significant terms 1}}
% \ref{tab:Reactome pathways reactome 3457 10cent deg.txt Over representation only. Top 20 Terms. 1 significant terms shown . Total significant terms 1}

\subsubsection{No enrichment}

\ref{tab:GO analysis noenrich}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 16:22:28 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrlrr}
  \hline
GO biological process complete &  n in PSP & n found  & Expected & Fold enrichment & p value & FDR \\ 
  \hline
ERBB2 signalling pathway (GO:0038128) & 12 & 10 & 2 & 6.52 & $5.74 \times 10^{-5}$ & $3.81 \times 10^{-3}$ \\ 
  mRNA stabilization (GO:0048255) & 14 & 11 & 2 & 6.15 & $3.51 \times 10^{-5}$ & $2.54 \times 10^{-3}$ \\ 
  regulation of telomere maintenance via telomere lengthening (GO:1904356) & 17 & 12 & 2 & 5.53 & $3.26 \times 10^{-5}$ & $2.40 \times 10^{-3}$ \\ 
  positive regulation of telomerase RNA localization to Cajal body (GO:1904874) & 10 & 7 & 1 & 5.48 & $1.60 \times 10^{-3}$ & $4.80 \times 10^{-2}$ \\ 
  regulation of telomerase RNA localization to Cajal body (GO:1904872) & 10 & 7 & 1 & 5.48 & $1.60 \times 10^{-3}$ & $4.79 \times 10^{-2}$ \\ 
  positive regulation of telomerase activity (GO:0051973) & 13 & 9 & 2 & 5.42 & $3.62 \times 10^{-4}$ & $1.48 \times 10^{-2}$ \\ 
  negative regulation of epidermal growth factor receptor signaling pathway (GO:0042059) & 13 & 9 & 2 & 5.42 & $3.62 \times 10^{-4}$ & $1.48 \times 10^{-2}$ \\ 
  negative regulation of mRNA catabolic process (GO:1902373) & 16 & 11 & 2 & 5.38 & $8.31 \times 10^{-5}$ & $5.06 \times 10^{-3}$ \\ 
  regulation of telomere maintenance via telomerase (GO:0032210) & 16 & 11 & 2 & 5.38 & $8.31 \times 10^{-5}$ & $5.03 \times 10^{-3}$ \\ 
  regulation of telomere maintenance (GO:0032204) & 23 & 15 & 3 & 5.11 & $6.65 \times 10^{-6}$ & $6.79 \times 10^{-4}$ \\ 
  RNA stabilization (GO:0043489) & 17 & 11 & 2 & 5.07 & $1.23 \times 10^{-4}$ & $6.52 \times 10^{-3}$ \\ 
  negative regulation of ERBB signaling pathway (GO:1901185) & 14 & 9 & 2 & 5.03 & $5.36 \times 10^{-4}$ & $2.00 \times 10^{-2}$ \\ 
  positive regulation of telomere maintenance via telomere lengthening (GO:1904358) & 14 & 9 & 2 & 5.03 & $5.36 \times 10^{-4}$ & $1.99 \times 10^{-2}$ \\ 
  negative regulation of mRNA metabolic process (GO:1903312) & 22 & 14 & 3 & 4.98 & $1.67 \times 10^{-5}$ & $1.31 \times 10^{-3}$ \\ 
  regulation of telomerase activity (GO:0051972) & 16 & 10 & 2 & 4.89 & $3.09 \times 10^{-4}$ & $1.31 \times 10^{-2}$ \\ 
  positive regulation of telomere maintenance (GO:0032206) & 16 & 10 & 2 & 4.89 & $3.09 \times 10^{-4}$ & $1.31 \times 10^{-2}$ \\ 
  positive regulation of telomere maintenance via telomerase (GO:0032212) & 13 & 8 & 2 & 4.82 & $1.35 \times 10^{-3}$ & $4.22 \times 10^{-2}$ \\ 
  positive regulation of protein insertion into mitochondrial membrane involved in apoptotic signalling pathway (GO:1900740) & 13 & 8 & 2 & 4.82 & $1.35 \times 10^{-3}$ & $4.21 \times 10^{-2}$ \\ 
  regulation of protein insertion into mitochondrial membrane involved in apoptotic signalling pathway (GO:1900739) & 13 & 8 & 2 & 4.82 & $1.35 \times 10^{-3}$ & $4.19 \times 10^{-2}$ \\ 
  positive regulation of response to cytokine stimulus (GO:0060760) & 13 & 8 & 2 & 4.82 & $1.35 \times 10^{-3}$ & $4.18 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{\textbf{Panther GO enrichment analysis of biological process in high degree nodes} (top 0.9 centile) with the post synaptic proteome as the gene background. The terms are ordered by fold change and only over represented terms and those with FDR $<0.05$ are shown. Top 20 terms. 20 significant terms shown. Total significant terms 385. \textit{n in PSP}, records the total number of occurrences of the term in the n=3457 post synaptic proteome. \textit{n found} refers to the number of occurrences of the term in the top 0.9 centile of high degree nodes.} 
\label{tab:GO biological process complete BP 3457 90cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 385}
\end{table}






% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 16:22:28 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrlrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
cellular response to organic substance (GO:0071310) & 649 & 148 & 83 & 1.79 & $5.70 \times 10^{-12}$ & $3.26 \times 10^{-8}$ \\ 
  regulation of protein metabolic process (GO:0051246) & 684 & 151 & 87 & 1.73 & $3.16 \times 10^{-11}$ & $4.52 \times 10^{-8}$ \\ 
  negative regulation of cellular process (GO:0048523) & 1070 & 208 & 137 & 1.52 & $1.62 \times 10^{-11}$ & $4.62 \times 10^{-8}$ \\ 
  positive regulation of cellular metabolic process (GO:0031325) & 754 & 161 & 96 & 1.67 & $4.07 \times 10^{-11}$ & $4.66 \times 10^{-8}$ \\ 
  regulation of macromolecule biosynthetic process (GO:0010556) & 720 & 156 & 92 & 1.70 & $4.53 \times 10^{-11}$ & $4.71 \times 10^{-8}$ \\ 
  regulation of cellular biosynthetic process (GO:0031326) & 752 & 161 & 96 & 1.68 & $3.80 \times 10^{-11}$ & $4.82 \times 10^{-8}$ \\ 
  positive regulation of metabolic process (GO:0009893) & 829 & 173 & 106 & 1.63 & $3.02 \times 10^{-11}$ & $4.94 \times 10^{-8}$ \\ 
  positive regulation of macromolecule metabolic process (GO:0010604) & 746 & 163 & 95 & 1.71 & $4.48 \times 10^{-12}$ & $5.12 \times 10^{-8}$ \\ 
  positive regulation of nitrogen compound metabolic process (GO:0051173) & 697 & 153 & 89 & 1.72 & $2.81 \times 10^{-11}$ & $5.36 \times 10^{-8}$ \\ 
  regulation of macromolecule metabolic process (GO:0060255) & 1258 & 233 & 161 & 1.45 & $2.35 \times 10^{-11}$ & $5.37 \times 10^{-8}$ \\ 
  regulation of cellular protein metabolic process (GO:0032268) & 638 & 143 & 81 & 1.75 & $5.86 \times 10^{-11}$ & $5.59 \times 10^{-8}$ \\ 
  regulation of nitrogen compound metabolic process (GO:0051171) & 1155 & 220 & 148 & 1.49 & $1.57 \times 10^{-11}$ & $5.99 \times 10^{-8}$ \\ 
  regulation of programmed cell death (GO:0043067) & 402 & 104 & 51 & 2.03 & $7.14 \times 10^{-11}$ & $6.28 \times 10^{-8}$ \\ 
  regulation of apoptotic process (GO:0042981) & 392 & 102 & 50 & 2.04 & $7.86 \times 10^{-11}$ & $6.42 \times 10^{-8}$ \\ 
  positive regulation of cellular proce ss (GO:0048522) & 1336 & 241 & 171 & 1.41 & $1.00 \times 10^{-10}$ & $6.75 \times 10^{-8}$ \\ 
  regulation of cellular macromolecule biosynthetic process (GO:2000112) & 694 & 151 & 89 & 1.70 & $9.03 \times 10^{-11}$ & $6.89 \times 10^{-8}$ \\ 
  positive regulation of biological process (GO:0048518) & 1450 & 256 & 185 & 1.38 & $9.95 \times 10^{-11}$ & $7.11 \times 10^{-8}$ \\ 
  regulation of cellular metabolic process (GO:0031323) & 1254 & 230 & 160 & 1.44 & $1.31 \times 10^{-10}$ & $8.30 \times 10^{-8}$ \\ 
  regulation of biosynthetic process (GO:0009889) & 775 & 162 & 99 & 1.64 & $1.90 \times 10^{-10}$ & $1.04 \times 10^{-7}$ \\ 
  symbiotic process (GO:0044403) & 313 & 87 & 40 & 2.18 & $1.75 \times 10^{-10}$ & $1.05 \times 10^{-7}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO biological process complete BP 3457 90cent deg.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 385} 
\label{tab:GO biological process complete BP 3457 90cent deg.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 385}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 16:27:32 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}


\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
protein phosphorylated amino acid binding (GO:0045309) & 16 & 11 & 2 & 5.4 & $8.31 \times 10^{-5}$ & $1.20 \times 10^{-2}$ \\ 
  protein folding chaperone (GO:0044183) & 14 & 9 & 2 & 5.0 & $5.36 \times 10^{-4}$ & $4.90 \times 10^{-2}$ \\ 
  disordered domain specific binding (GO:0097718) & 18 & 11 & 2 & 4.8 & $1.79 \times 10^{-4}$ & $2.33 \times 10^{-2}$ \\ 
  phosphoprotein binding (GO:0051219) & 30 & 18 & 4 & 4.7 & $1.90 \times 10^{-6}$ & $5.20 \times 10^{-4}$ \\ 
  p53 binding (GO:0002039) & 19 & 11 & 2 & 4.5 & $2.54 \times 10^{-4}$ & $2.89 \times 10^{-2}$ \\ 
  protein kinase regulator activity (GO:0019887) & 40 & 19 & 5 & 3.7 & $1.33 \times 10^{-5}$ & $2.80 \times 10^{-3}$ \\ 
  heat shock protein binding (GO:0031072) & 49 & 23 & 6 & 3.7 & $1.90 \times 10^{-6}$ & $5.77 \times 10^{-4}$ \\ 
  protein N-terminus binding (GO:0047485) & 37 & 17 & 5 & 3.6 & $5.28 \times 10^{-5}$ & $8.04 \times 10^{-3}$ \\ 
  tau protein binding (GO:0048156) & 32 & 14 & 4 & 3.4 & $3.61 \times 10^{-4}$ & $3.53 \times 10^{-2}$ \\ 
  ubiquitin protein ligase binding (GO:0031625) & 96 & 40 & 12 & 3.3 & $3.94 \times 10^{-9}$ & $3.60 \times 10^{-6}$ \\ 
  ubiquitin-like protein ligase binding (GO:0044389) & 99 & 41 & 13 & 3.2 & $2.86 \times 10^{-9}$ & $3.92 \times 10^{-6}$ \\ 
  kinase regulator activity (GO:0019207) & 49 & 20 & 6 & 3.2 & $4.48 \times 10^{-5}$ & $7.21 \times 10^{-3}$ \\ 
  protein phosphatase binding (GO:0019903) & 44 & 17 & 6 & 3.0 & $2.83 \times 10^{-4}$ & $2.98 \times 10^{-2}$ \\ 
  unfolded protein binding (GO:0051082) & 45 & 17 & 6 & 3.0 & $3.50 \times 10^{-4}$ & $3.55 \times 10^{-2}$ \\ 
  mRNA binding (GO:0003729) & 87 & 29 & 11 & 2.6 & $3.29 \times 10^{-5}$ & $5.64 \times 10^{-3}$ \\ 
  protein C-terminus binding (GO:0008022) & 79 & 25 & 10 & 2.5 & $2.04 \times 10^{-4}$ & $2.55 \times 10^{-2}$ \\ 
  transcription factor binding (GO:0008134) & 149 & 42 & 19 & 2.2 & $1.68 \times 10^{-5}$ & $3.06 \times 10^{-3}$ \\ 
  protein domain specific binding (GO:0019904) & 242 & 67 & 31 & 2.2 & $5.20 \times 10^{-8}$ & $3.56 \times 10^{-5}$ \\ 
  protein kinase binding (GO:0019901) & 210 & 56 & 27 & 2.1 & $1.72 \times 10^{-6}$ & $5.90 \times 10^{-4}$ \\ 
  kinase binding (GO:0019900) & 235 & 62 & 30 & 2.1 & $7.55 \times 10^{-7}$ & $2.95 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO molecular function complete MF 3457 90cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 30} 
\label{tab:GO molecular function complete MF 3457 90cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 30}
\end{table}


\clearpage

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 16:27:32 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
enzyme binding (GO:0019899) & 641 & 142 & 82 & 1.7 & $1.46 \times 10^{-10}$ & $4.01 \times 10^{-7}$ \\ 
  ubiquitin protein ligase binding (GO:0031625) & 96 & 40 & 12 & 3.3 & $3.94 \times 10^{-9}$ & $3.60 \times 10^{-6}$ \\ 
  ubiquitin-like protein ligase binding (GO:0044389) & 99 & 41 & 13 & 3.2 & $2.86 \times 10^{-9}$ & $3.92 \times 10^{-6}$ \\ 
  protein domain specific binding (GO:0019904) & 242 & 67 & 31 & 2.2 & $5.20 \times 10^{-8}$ & $3.56 \times 10^{-5}$ \\ 
  RNA binding (GO:0003723) & 452 & 99 & 58 & 1.7 & $4.29 \times 10^{-7}$ & $2.35 \times 10^{-4}$ \\ 
  protein-containing complex binding (GO:0044877) & 422 & 94 & 54 & 1.7 & $6.21 \times 10^{-7}$ & $2.83 \times 10^{-4}$ \\ 
  kinase binding (GO:0019900) & 235 & 62 & 30 & 2.1 & $7.55 \times 10^{-7}$ & $2.95 \times 10^{-4}$ \\ 
  phosphoprotein binding (GO:0051219) & 30 & 18 & 4 & 4.7 & $1.90 \times 10^{-6}$ & $5.20 \times 10^{-4}$ \\ 
  nucleic acid binding (GO:0003676) & 724 & 138 & 92 & 1.5 & $2.11 \times 10^{-6}$ & $5.25 \times 10^{-4}$ \\ 
  heat shock protein binding (GO:0031072) & 49 & 23 & 6 & 3.7 & $1.90 \times 10^{-6}$ & $5.77 \times 10^{-4}$ \\ 
  protein kinase binding (GO:0019901) & 210 & 56 & 27 & 2.1 & $1.72 \times 10^{-6}$ & $5.90 \times 10^{-4}$ \\ 
  organic cyclic compound binding (GO:0097159) & 1288 & 212 & 164 & 1.3 & $1.18 \times 10^{-5}$ & $2.68 \times 10^{-3}$ \\ 
  protein kinase regulator activity (GO:0019887) & 40 & 19 & 5 & 3.7 & $1.33 \times 10^{-5}$ & $2.80 \times 10^{-3}$ \\ 
  heterocyclic compound binding (GO:1901363) & 1270 & 209 & 162 & 1.3 & $1.44 \times 10^{-5}$ & $2.83 \times 10^{-3}$ \\ 
  transcription factor binding (GO:0008134) & 149 & 42 & 19 & 2.2 & $1.68 \times 10^{-5}$ & $3.06 \times 10^{-3}$ \\ 
  mRNA binding (GO:0003729) & 87 & 29 & 11 & 2.6 & $3.29 \times 10^{-5}$ & $5.64 \times 10^{-3}$ \\ 
  kinase regulator activity (GO:0019207) & 49 & 20 & 6 & 3.2 & $4.48 \times 10^{-5}$ & $7.21 \times 10^{-3}$ \\ 
  protein N-terminus binding (GO:0047485) & 37 & 17 & 5 & 3.6 & $5.28 \times 10^{-5}$ & $8.04 \times 10^{-3}$ \\ 
  protein phosphorylated amino acid binding (GO:0045309) & 16 & 11 & 2 & 5.4 & $8.31 \times 10^{-5}$ & $1.20 \times 10^{-2}$ \\ 
  cadherin binding (GO:0045296) & 165 & 42 & 21 & 2.0 & $1.04 \times 10^{-4}$ & $1.43 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO molecular function complete MF 3457 90cent deg.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 30} 
\label{tab:GO molecular function complete MF 3457 90cent deg.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 30}
\end{table}


\clearpage

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 16:29:31 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
chaperone complex (GO:0101031) & 19 & 12 & 2 & 5.0 & $7.16 \times 10^{-5}$ & $3.71 \times 10^{-3}$ \\ 
  catalytic step 2 spliceosome (GO:0071013) & 32 & 17 & 4 & 4.2 & $1.25 \times 10^{-5}$ & $8.56 \times 10^{-4}$ \\ 
  centriole (GO:0005814) & 28 & 12 & 4 & 3.4 & $1.09 \times 10^{-3}$ & $3.91 \times 10^{-2}$ \\ 
  spliceosomal complex (GO:0005681) & 50 & 20 & 6 & 3.1 & $5.62 \times 10^{-5}$ & $3.02 \times 10^{-3}$ \\ 
  inclusion body (GO:0016234) & 33 & 13 & 4 & 3.1 & $1.27 \times 10^{-3}$ & $4.34 \times 10^{-2}$ \\ 
  cytoplasmic ribonucleoprotein granule (GO:0036464) & 59 & 20 & 8 & 2.6 & $4.15 \times 10^{-4}$ & $1.95 \times 10^{-2}$ \\ 
  ficolin-1-rich granule (GO:0101002) & 57 & 19 & 7 & 2.6 & $6.75 \times 10^{-4}$ & $2.67 \times 10^{-2}$ \\ 
  ficolin-1-rich granule lumen (GO:1904813) & 57 & 19 & 7 & 2.6 & $6.75 \times 10^{-4}$ & $2.60 \times 10^{-2}$ \\ 
  ribonucleoprotein granule (GO:0035770) & 62 & 20 & 8 & 2.5 & $6.34 \times 10^{-4}$ & $2.58 \times 10^{-2}$ \\ 
  centrosome (GO:0005813) & 132 & 36 & 17 & 2.1 & $1.42 \times 10^{-4}$ & $6.92 \times 10^{-3}$ \\ 
  nuclear envelope (GO:0005635) & 110 & 30 & 14 & 2.1 & $5.38 \times 10^{-4}$ & $2.31 \times 10^{-2}$ \\ 
  focal adhesion (GO:0005925) & 212 & 57 & 27 & 2.1 & $1.13 \times 10^{-6}$ & $1.89 \times 10^{-4}$ \\ 
  cell-substrate junction (GO:0030055) & 217 & 58 & 28 & 2.1 & $1.40 \times 10^{-6}$ & $2.10 \times 10^{-4}$ \\ 
  ribonucleoprotein complex (GO:1990904) & 208 & 54 & 27 & 2.0 & $6.32 \times 10^{-6}$ & $6.80 \times 10^{-4}$ \\ 
  microtubule organizing center (GO:0005815) & 173 & 44 & 22 & 2.0 & $8.90 \times 10^{-5}$ & $4.46 \times 10^{-3}$ \\ 
  vesicle lumen (GO:0031983) & 118 & 30 & 15 & 2.0 & $1.27 \times 10^{-3}$ & $4.26 \times 10^{-2}$ \\ 
  microtubule cytoskeleton (GO:0015630) & 300 & 75 & 38 & 2.0 & $2.77 \times 10^{-7}$ & $5.21 \times 10^{-5}$ \\ 
  transferase complex (GO:1990234) & 129 & 32 & 16 & 1.9 & $1.28 \times 10^{-3}$ & $4.18 \times 10^{-2}$ \\ 
  catalytic complex (GO:1902494) & 311 & 72 & 40 & 1.8 & $5.24 \times 10^{-6}$ & $6.07 \times 10^{-4}$ \\ 
  nuclear body (GO:0016604) & 173 & 40 & 22 & 1.8 & $1.10 \times 10^{-3}$ & $3.86 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO cellular component complete CC 3457 90cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 46} 
\label{tab:GO cellular component complete CC 3457 90cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 46}
\end{table}




% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 16:29:31 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
nucleus (GO:0005634) & 1359 & 246 & 174 & 1.4 & $3.67 \times 10^{-11}$ & $5.53 \times 10^{-8}$ \\ 
  non-membrane-bounded organelle (GO:0043228) & 1105 & 205 & 141 & 1.4 & $2.13 \times 10^{-9}$ & $1.61 \times 10^{-6}$ \\ 
  intracellular non-membrane-bounded organelle (GO:0043232) & 1098 & 203 & 140 & 1.4 & $3.93 \times 10^{-9}$ & $1.97 \times 10^{-6}$ \\ 
  protein-containing complex (GO:0032991) & 1403 & 239 & 179 & 1.3 & $4.20 \times 10^{-8}$ & $1.58 \times 10^{-5}$ \\ 
  nucleoplasm (GO:0005654) & 761 & 150 & 97 & 1.5 & $5.38 \times 10^{-8}$ & $1.62 \times 10^{-5}$ \\ 
  nuclear lumen (GO:0031981) & 894 & 169 & 114 & 1.5 & $6.50 \times 10^{-8}$ & $1.63 \times 10^{-5}$ \\ 
  cytosol (GO:0005829) & 1430 & 240 & 183 & 1.3 & $1.44 \times 10^{-7}$ & $3.09 \times 10^{-5}$ \\ 
  microtubule cytoskeleton (GO:0015630) & 300 & 75 & 38 & 2.0 & $2.77 \times 10^{-7}$ & $5.21 \times 10^{-5}$ \\ 
  focal adhesion (GO:0005925) & 212 & 57 & 27 & 2.1 & $1.13 \times 10^{-6}$ & $1.89 \times 10^{-4}$ \\ 
  cell-substrate junction (GO:0030055) & 217 & 58 & 28 & 2.1 & $1.40 \times 10^{-6}$ & $2.10 \times 10^{-4}$ \\ 
  cytoskeleton (GO:0005856) & 603 & 119 & 77 & 1.6 & $3.06 \times 10^{-6}$ & $4.19 \times 10^{-4}$ \\ 
  intracellular membrane-bounded organelle (GO:0043231) & 2085 & 314 & 266 & 1.2 & $4.34 \times 10^{-6}$ & $5.45 \times 10^{-4}$ \\ 
  membrane-enclosed lumen (GO:0031974) & 1179 & 199 & 151 & 1.3 & $6.52 \times 10^{-6}$ & $5.77 \times 10^{-4}$ \\ 
  catalytic complex (GO:1902494) & 311 & 72 & 40 & 1.8 & $5.24 \times 10^{-6}$ & $6.07 \times 10^{-4}$ \\ 
  intracellular organelle lumen (GO:0070013) & 1179 & 199 & 151 & 1.3 & $6.52 \times 10^{-6}$ & $6.13 \times 10^{-4}$ \\ 
  extracellular exosome (GO:0070062) & 759 & 140 & 97 & 1.4 & $7.40 \times 10^{-6}$ & $6.19 \times 10^{-4}$ \\ 
  organelle lumen (GO:0043233) & 1179 & 199 & 151 & 1.3 & $6.52 \times 10^{-6}$ & $6.54 \times 10^{-4}$ \\ 
  ribonucleoprotein complex (GO:1990904) & 208 & 54 & 27 & 2.0 & $6.32 \times 10^{-6}$ & $6.80 \times 10^{-4}$ \\ 
  extracellular vesicle (GO:1903561) & 771 & 141 & 98 & 1.4 & $1.08 \times 10^{-5}$ & $8.10 \times 10^{-4}$ \\ 
  extracellular region (GO:0005576) & 1132 & 191 & 145 & 1.3 & $1.18 \times 10^{-5}$ & $8.46 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{High degree GO cellular component complete CC 3457 90cent deg.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 46} 
\label{tab:GO cellular component complete CC 3457 90cent deg.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 46}
\end{table}




\todo{rotational tables commented out here}
% % latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Sat Jun 27 13:33:40 2020
% \begin{sidewaystable}[ht]
% \centering
% {\small %
% \begin{adjustbox}{totalheight=3cm}

% \begin{tabular}{lrrrllrr}
%   \hline
% GO biological process complete & Homo sapiens PSP ref & Test genes & Expected & Over/under & Fold enrichment & p value & FDR \\ 
%   \hline
% ERBB2 signaling pathway (GO:0038128) & 12 & 10 & 2 & + & 6.52 & $5.74 \times 10^{-5}$ & $3.81 \times 10^{-3}$ \\ 
%   mRNA stabilization (GO:0048255) & 14 & 11 & 2 & + & 6.15 & $3.51 \times 10^{-5}$ & $2.54 \times 10^{-3}$ \\ 
%   regulation of telomere maintenance via telomere lengthening (GO:1904356) & 17 & 12 & 2 & + & 5.53 & $3.26 \times 10^{-5}$ & $2.40 \times 10^{-3}$ \\ 
%   positive regulation of telomerase RNA localization to Cajal body (GO:1904874) & 10 & 7 & 1 & + & 5.48 & $1.60 \times 10^{-3}$ & $4.80 \times 10^{-2}$ \\ 
%   regulation of telomerase RNA localization to Cajal body (GO:1904872) & 10 & 7 & 1 & + & 5.48 & $1.60 \times 10^{-3}$ & $4.79 \times 10^{-2}$ \\ 
%   positive regulation of telomerase activity (GO:0051973) & 13 & 9 & 2 & + & 5.42 & $3.62 \times 10^{-4}$ & $1.48 \times 10^{-2}$ \\ 
%   negative regulation of epidermal growth factor receptor signaling pathway (GO:0042059) & 13 & 9 & 2 & + & 5.42 & $3.62 \times 10^{-4}$ & $1.48 \times 10^{-2}$ \\ 
%   negative regulation of mRNA catabolic process (GO:1902373) & 16 & 11 & 2 & + & 5.38 & $8.31 \times 10^{-5}$ & $5.06 \times 10^{-3}$ \\ 
%   regulation of telomere maintenance via telomerase (GO:0032210) & 16 & 11 & 2 & + & 5.38 & $8.31 \times 10^{-5}$ & $5.03 \times 10^{-3}$ \\ 
%   regulation of telomere maintenance (GO:0032204) & 23 & 15 & 3 & + & 5.11 & $6.65 \times 10^{-6}$ & $6.79 \times 10^{-4}$ \\ 
%   \hline
% \end{tabular}
% \end{adjustbox}
% } %

% \caption{GO analysis BP 3457 90cent deg.txt ordered by fold change} 
% \label{tab:GO analysis BP 3457 90cent deg.txt ordered by fold change}
% \end{sidewaystable}




% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sat Jun 27 15:11:11 2020

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sat Jun 27 15:27:00 2020
% \begin{table}[ht]
% \centering
% \begin{adjustbox}{width=\textwidth}

% \begin{tabular}{lrrrrrr}
%   \hline
% GO molecular function complete & Homo sapiens PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
%   \hline
% protein phosphorylated amino acid binding (GO:0045309) & 16 & 11 & 2 & 5.4 & $8.31 \times 10^{-5}$ & $1.20 \times 10^{-2}$ \\ 
%   protein folding chaperone (GO:0044183) & 14 & 9 & 2 & 5.0 & $5.36 \times 10^{-4}$ & $4.90 \times 10^{-2}$ \\ 
%   disordered domain specific binding (GO:0097718) & 18 & 11 & 2 & 4.8 & $1.79 \times 10^{-4}$ & $2.33 \times 10^{-2}$ \\ 
%   phosphoprotein binding (GO:0051219) & 30 & 18 & 4 & 4.7 & $1.90 \times 10^{-6}$ & $5.20 \times 10^{-4}$ \\ 
%   p53 binding (GO:0002039) & 19 & 11 & 2 & 4.5 & $2.54 \times 10^{-4}$ & $2.89 \times 10^{-2}$ \\ 
%   protein kinase regulator activity (GO:0019887) & 40 & 19 & 5 & 3.7 & $1.33 \times 10^{-5}$ & $2.80 \times 10^{-3}$ \\ 
%   heat shock protein binding (GO:0031072) & 49 & 23 & 6 & 3.7 & $1.90 \times 10^{-6}$ & $5.77 \times 10^{-4}$ \\ 
%   protein N-terminus binding (GO:0047485) & 37 & 17 & 5 & 3.6 & $5.28 \times 10^{-5}$ & $8.04 \times 10^{-3}$ \\ 
%   tau protein binding (GO:0048156) & 32 & 14 & 4 & 3.4 & $3.61 \times 10^{-4}$ & $3.53 \times 10^{-2}$ \\ 
%   ubiquitin protein ligase binding (GO:0031625) & 96 & 40 & 12 & 3.3 & $3.94 \times 10^{-9}$ & $3.60 \times 10^{-6}$ \\ 
%   \hline
% \end{tabular}
% \end{adjustbox}
% \caption{GO analysis MF 3457 90cent deg.txt ordered by fold change. Over representation only} 
% \label{tab:GO analysis MF 3457 90cent deg.txt ordered by fold change. Over representation only}
% \end{table}










% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sat Jun 27 15:45:02 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER Pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
Parkinson disease (P00049) & 45 & 27 & 6 & 4.7 & $4.74 \times 10^{-9}$ & $6.35 \times 10^{-7}$ \\ 
  CCKR signaling map (P06959) & 70 & 22 & 9 & 2.5 & $6.03 \times 10^{-4}$ & $2.02 \times 10^{-2}$ \\ 
  EGF receptor signaling pathway (P00018) & 59 & 19 & 8 & 2.5 & $8.92 \times 10^{-4}$ & $2.39 \times 10^{-2}$ \\ 
  Integrin signalling pathway (P00034) & 85 & 25 & 11 & 2.3 & $5.81 \times 10^{-4}$ & $2.60 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis Panther pathways 3457 90cent deg.txtOver representation only} 
\label{tab:GO analysis Panther pathways 3457 90cent deg.txtOver representation only}
\end{table}




\begin{table}[]
    \centering
    \begin{adjustbox}{width=\textwidth}
    
    \begin{tabular}{lllll}
    \toprule
    Centrality measure & GO ontology & centile & comment \\
    \midrule
     Degree   &  BP &  0.1 & under representation (organelle) \\
     Degree    & MF & 0.1 &  under representation of protein binding GO:0005515\\
     Degree & Pathways & 0.1 & none\\
     Degree & BP SLIM & 0.1 & none \\
     Degree & MF SLIM & 0.1 & none \\
     Degree & CC SLIM & 0.1 & none \\
     Degree & Protein class & none \\
     Betweenness & BP SLIM & 0.9 & none \\
      Betweenness & MF SLIM & 0.9 & none \\
      Betweenness & CC SLIM & 0.9 & none \\
      Betweenness & Protein class & 0.9 & none \\
     Betweenness & BP & 0.1 & under representation of organelle organization (GO:0033043)\\
     Betweenness & MF & 0.1 & \\
     Betweeness & all slims, protein class, pathways & 0.1 &\\
     Transitivity NA & BP & 0.9 & none \\
     Transitivity NA & MF & 0.9 & none \\
     Transitivity NA & CC & 0.9 & none \\
     Transitivity NA & BP & 0.1 & N=631 viral process underrepresented only\\
     Transitivity NA & MF & 0.1 & N=631 none\\
     Transitivity NA & CC & 0.1 & N=631 under representation cytosol\\
     \bottomrule
    \end{tabular}
    \end{adjustbox}
    \caption{No enrichment}
    \label{tab:GO analysis noenrich}
\end{table}


\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
intrinsic component of membrane (GO:0031224) & 926 & 174 & 131 & 1.3 & $7.03 \times 10^{-5}$ & $1.18 \times 10^{-2}$ \\ 
  integral component of membrane (GO:0016021) & 889 & 167 & 126 & 1.3 & $1.14 \times 10^{-4}$ & $1.72 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO cellular component complete CC 3457 10cent deg.txt Over representation only. Top 20 Terms. 2 significant terms shown . Total significant terms 2. Ordered by FDR identical to ordered by fold enrichment} 
\label{tab:GO cellular component complete CC 3457 10cent deg.txt Over representation only. Top 20 Terms. 2 significant terms shown . Total significant terms 2}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 08:55:08 2020
\begin{table}[ht]
\centering
\begin{tabular}{lrrrrrr}
  \hline
Reactome pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
Unclassified (UNCLASSIFIED) & 932 & 175 & 132 & 1.3 & $7.25 \times 10^{-5}$ & $4.62 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\caption{Reactome pathways reactome 3457 10cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 2 significant terms shown. Total significant terms 1} 
\label{tab:Reactome pathways reactome 3457 10cent deg.txt ordered by fold change. Over representation only. Top 20 terms. 2 significant terms shown. Total significant terms 1}
\end{table}
% % latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Tue Jun 30 08:55:08 2020
% \begin{table}[ht]
% \centering
% \begin{tabular}{lrrrrrr}
%   \hline
% Reactome pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
%   \hline
% Unclassified (UNCLASSIFIED) & 932 & 175 & 132 & 1.3 & $7.25 \times 10^{-5}$ & $4.62 \times 10^{-2}$ \\ 
%   \hline
% \end{tabular}
% \caption{Reactome pathways reactome 3457 10cent deg.txt Over representation only. Top 20 Terms. 1 significant terms shown . Total significant terms 1} 
% \label{tab:Reactome pathways reactome 3457 10cent deg.txt Over representation only. Top 20 Terms. 1 significant terms shown . Total significant terms 1}
% \end{table}
%%%% BETWEENNESS %%%%%%%%%%%%%%%%%%%%%%%
%
%
%

\subsection{Betweenness centrality}

50 \% of the betweenness mass is in the highest 157 nodes. 

95\% includes response to cocaine

Bottom 10\% are betweenness 0 (betweeness 0.03 10.045\%)

Low betweenness
BP and MF nil significant


\ref{tab:GO biological process complete BP 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms. 3457 significant terms shown. Total significant terms 355}

\ref{tab:GO biological process complete BP 3457 90cent bet.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 355}

\ref{tab:GO molecular function complete MF 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 21}

\ref{tab:GO molecular function complete MF 3457 90cent bet.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 21}

\ref{tab:GO cellular component complete CC 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 36}

\ref{tab:GO cellular component complete CC 3457 90cent bet.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 36}

\ref{tab:GO analysis pathway 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis pathway 3457 90cent bet.txt Over representation only. Top 20 Terms}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 13:10:28 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}


\begin{tabular}{lrrrrrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
ERBB2 signaling pathway (GO:0038128) & 12 & 10 & 1 & 6.7 & $4.67 \times 10^{-5}$ & $3.76 \times 10^{-3}$ \\ 
  negative regulation of epidermal growth factor receptor signaling pathway (GO:0042059) & 13 & 9 & 2 & 5.6 & $3.03 \times 10^{-4}$ & $1.51 \times 10^{-2}$ \\ 
  mRNA stabilization (GO:0048255) & 14 & 9 & 2 & 5.2 & $4.49 \times 10^{-4}$ & $2.06 \times 10^{-2}$ \\ 
  negative regulation of ERBB signaling pathway (GO:1901185) & 14 & 9 & 2 & 5.2 & $4.49 \times 10^{-4}$ & $2.06 \times 10^{-2}$ \\ 
  regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway (GO:1900739) & 13 & 8 & 2 & 5.0 & $1.16 \times 10^{-3}$ & $4.09 \times 10^{-2}$ \\ 
  positive regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway (GO:1900740) & 13 & 8 & 2 & 5.0 & $1.16 \times 10^{-3}$ & $4.10 \times 10^{-2}$ \\ 
  ERBB signaling pathway (GO:0038127) & 25 & 15 & 3 & 4.8 & $1.04 \times 10^{-5}$ & $1.24 \times 10^{-3}$ \\ 
  positive regulation of mitochondrial outer membrane permeabilization involved in apoptotic signaling pathway (GO:1901030) & 16 & 9 & 2 & 4.5 & $9.18 \times 10^{-4}$ & $3.45 \times 10^{-2}$ \\ 
  negative regulation of mRNA catabolic process (GO:1902373) & 16 & 9 & 2 & 4.5 & $9.18 \times 10^{-4}$ & $3.46 \times 10^{-2}$ \\ 
  RNA stabilization (GO:0043489) & 17 & 9 & 2 & 4.3 & $1.27 \times 10^{-3}$ & $4.33 \times 10^{-2}$ \\ 
  protein insertion into membrane (GO:0051205) & 17 & 9 & 2 & 4.3 & $1.27 \times 10^{-3}$ & $4.34 \times 10^{-2}$ \\ 
  regulation of cyclin-dependent protein kinase activity (GO:1904029) & 19 & 10 & 2 & 4.2 & $7.07 \times 10^{-4}$ & $2.80 \times 10^{-2}$ \\ 
  regulation of mitochondrial outer membrane permeabilization involved in apoptotic signaling pathway (GO:1901028) & 19 & 10 & 2 & 4.2 & $7.07 \times 10^{-4}$ & $2.81 \times 10^{-2}$ \\ 
  regulation of establishment of protein localization to mitochondrion (GO:1903747) & 27 & 14 & 3 & 4.2 & $6.97 \times 10^{-5}$ & $5.24 \times 10^{-3}$ \\ 
  cellular response to heat (GO:0034605) & 24 & 12 & 3 & 4.0 & $2.99 \times 10^{-4}$ & $1.51 \times 10^{-2}$ \\ 
  signal transduction by p53 class mediator (GO:0072331) & 22 & 11 & 3 & 4.0 & $5.35 \times 10^{-4}$ & $2.36 \times 10^{-2}$ \\ 
  positive regulation of nucleocytoplasmic transport (GO:0046824) & 20 & 10 & 2 & 4.0 & $9.58 \times 10^{-4}$ & $3.57 \times 10^{-2}$ \\ 
  ciliary basal body-plasma membrane docking (GO:0097711) & 29 & 14 & 4 & 3.9 & $1.26 \times 10^{-4}$ & $8.41 \times 10^{-3}$ \\ 
  substantia nigra development (GO:0021762) & 25 & 12 & 3 & 3.9 & $3.99 \times 10^{-4}$ & $1.89 \times 10^{-2}$ \\ 
  response to dopamine (GO:1903350) & 23 & 11 & 3 & 3.8 & $7.13 \times 10^{-4}$ & $2.79 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO biological process complete BP 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms. 3457 significant terms shown. Total significant terms 355} 
\label{tab:GO biological process complete BP 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms. 3457 significant terms shown. Total significant terms 355}
\end{table}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 13:10:28 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
regulation of cellular protein metabolic process (GO:0032268) & 638 & 142 & 79 & 1.8 & $1.41 \times 10^{-11}$ & $8.07 \times 10^{-8}$ \\ 
  regulation of protein metabolic process (GO:0051246) & 684 & 149 & 85 & 1.8 & $1.21 \times 10^{-11}$ & $1.38 \times 10^{-7}$ \\ 
  negative regulation of cellular process (GO:0048523) & 1070 & 202 & 133 & 1.5 & $4.13 \times 10^{-11}$ & $1.58 \times 10^{-7}$ \\ 
  regulation of protein modification process (GO:0031399) & 448 & 108 & 56 & 1.9 & $2.60 \times 10^{-10}$ & $7.44 \times 10^{-7}$ \\ 
  cellular response to organic substance (GO:0071310) & 649 & 139 & 81 & 1.7 & $3.68 \times 10^{-10}$ & $8.41 \times 10^{-7}$ \\ 
  positive regulation of protein metabolic process (GO:0051247) & 426 & 103 & 53 & 1.9 & $5.21 \times 10^{-10}$ & $9.93 \times 10^{-7}$ \\ 
  positive regulation of metabolic process (GO:0009893) & 829 & 163 & 103 & 1.6 & $1.28 \times 10^{-9}$ & $2.09 \times 10^{-6}$ \\ 
  regulation of catabolic process (GO:0009894) & 282 & 77 & 35 & 2.2 & $1.54 \times 10^{-9}$ & $2.20 \times 10^{-6}$ \\ 
  negative regulation of biological process (GO:0048519) & 1245 & 219 & 155 & 1.4 & $1.74 \times 10^{-9}$ & $2.21 \times 10^{-6}$ \\ 
  regulation of transferase activity (GO:0051338) & 262 & 73 & 33 & 2.2 & $2.40 \times 10^{-9}$ & $2.50 \times 10^{-6}$ \\ 
  positive regulation of macromolecule metabolic process (GO:0010604) & 746 & 150 & 93 & 1.6 & $2.36 \times 10^{-9}$ & $2.70 \times 10^{-6}$ \\ 
  positive regulation of cellular protein metabolic process (GO:0032270) & 406 & 97 & 50 & 1.9 & $3.98 \times 10^{-9}$ & $3.79 \times 10^{-6}$ \\ 
  regulation of programmed cell death (GO:0043067) & 402 & 96 & 50 & 1.9 & $5.41 \times 10^{-9}$ & $4.42 \times 10^{-6}$ \\ 
  response to organic substance (GO:0010033) & 824 & 160 & 102 & 1.6 & $5.07 \times 10^{-9}$ & $4.46 \times 10^{-6}$ \\ 
  regulation of apoptotic process (GO:0042981) & 392 & 94 & 49 & 1.9 & $6.17 \times 10^{-9}$ & $4.71 \times 10^{-6}$ \\ 
  positive regulation of cellular metabolic process (GO:0031325) & 754 & 149 & 94 & 1.6 & $8.21 \times 10^{-9}$ & $4.94 \times 10^{-6}$ \\ 
  regulation of multicellular organismal process (GO:0051239) & 821 & 159 & 102 & 1.6 & $6.98 \times 10^{-9}$ & $4.99 \times 10^{-6}$ \\ 
  regulation of response to stimulus (GO:0048583) & 994 & 183 & 124 & 1.5 & $8.14 \times 10^{-9}$ & $5.17 \times 10^{-6}$ \\ 
  regulation of phosphorylation (GO:0042325) & 412 & 97 & 51 & 1.9 & $7.73 \times 10^{-9}$ & $5.20 \times 10^{-6}$ \\ 
  positive regulation of response to stimulus (GO:0048584) & 594 & 125 & 74 & 1.7 & $9.87 \times 10^{-9}$ & $5.65 \times 10^{-6}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO biological process complete BP 3457 90cent bet.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 355} 
\label{tab:GO biological process complete BP 3457 90cent bet.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 355}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 13:16:38 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
disordered domain specific binding (GO:0097718) & 18 & 12 & 2 & 5.4 & $3.85 \times 10^{-5}$ & $8.11 \times 10^{-3}$ \\ 
  protein phosphorylated amino acid binding (GO:0045309) & 16 & 10 & 2 & 5.0 & $2.55 \times 10^{-4}$ & $3.49 \times 10^{-2}$ \\ 
  phosphoprotein binding (GO:0051219) & 30 & 17 & 4 & 4.6 & $4.74 \times 10^{-6}$ & $1.44 \times 10^{-3}$ \\ 
  protein kinase regulator activity (GO:0019887) & 40 & 20 & 5 & 4.0 & $2.98 \times 10^{-6}$ & $1.17 \times 10^{-3}$ \\ 
  protein N-terminus binding (GO:0047485) & 37 & 18 & 5 & 3.9 & $1.26 \times 10^{-5}$ & $3.14 \times 10^{-3}$ \\ 
  histone deacetylase binding (GO:0042826) & 28 & 13 & 3 & 3.7 & $2.96 \times 10^{-4}$ & $3.86 \times 10^{-2}$ \\ 
  kinase regulator activity (GO:0019207) & 49 & 21 & 6 & 3.5 & $1.11 \times 10^{-5}$ & $3.05 \times 10^{-3}$ \\ 
  protein phosphatase binding (GO:0019903) & 44 & 17 & 5 & 3.1 & $2.12 \times 10^{-4}$ & $3.23 \times 10^{-2}$ \\ 
  ubiquitin protein ligase binding (GO:0031625) & 96 & 36 & 12 & 3.0 & $1.15 \times 10^{-7}$ & $7.86 \times 10^{-5}$ \\ 
  ubiquitin-like protein ligase binding (GO:0044389) & 99 & 37 & 12 & 3.0 & $8.12 \times 10^{-8}$ & $7.41 \times 10^{-5}$ \\ 
  protein C-terminus binding (GO:0008022) & 79 & 25 & 10 & 2.5 & $1.69 \times 10^{-4}$ & $2.89 \times 10^{-2}$ \\ 
  protein domain specific binding (GO:0019904) & 242 & 67 & 30 & 2.2 & $1.34 \times 10^{-8}$ & $1.83 \times 10^{-5}$ \\ 
  kinase binding (GO:0019900) & 235 & 61 & 29 & 2.1 & $5.58 \times 10^{-7}$ & $3.06 \times 10^{-4}$ \\ 
  protein kinase binding (GO:0019901) & 210 & 54 & 26 & 2.1 & $3.40 \times 10^{-6}$ & $1.16 \times 10^{-3}$ \\ 
  cadherin binding (GO:0045296) & 165 & 42 & 21 & 2.0 & $5.56 \times 10^{-5}$ & $1.02 \times 10^{-2}$ \\ 
  cell adhesion molecule binding (GO:0050839) & 235 & 54 & 29 & 1.9 & $5.53 \times 10^{-5}$ & $1.08 \times 10^{-2}$ \\ 
  enzyme binding (GO:0019899) & 641 & 144 & 80 & 1.8 & $4.09 \times 10^{-12}$ & $1.12 \times 10^{-8}$ \\ 
  protein-containing complex binding (GO:0044877) & 422 & 91 & 52 & 1.7 & $9.98 \times 10^{-7}$ & $4.55 \times 10^{-4}$ \\ 
  signaling receptor binding (GO:0005102) & 421 & 85 & 52 & 1.6 & $2.50 \times 10^{-5}$ & $5.72 \times 10^{-3}$ \\ 
  cytoskeletal protein binding (GO:0008092) & 365 & 73 & 45 & 1.6 & $1.74 \times 10^{-4}$ & $2.80 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO molecular function complete MF 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 21} 
\label{tab:GO molecular function complete MF 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 21}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 13:16:38 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
enzyme binding (GO:0019899) & 641 & 144 & 80 & 1.8 & $4.09 \times 10^{-12}$ & $1.12 \times 10^{-8}$ \\ 
  protein domain specific binding (GO:0019904) & 242 & 67 & 30 & 2.2 & $1.34 \times 10^{-8}$ & $1.83 \times 10^{-5}$ \\ 
  ubiquitin-like protein ligase binding (GO:0044389) & 99 & 37 & 12 & 3.0 & $8.12 \times 10^{-8}$ & $7.41 \times 10^{-5}$ \\ 
  ubiquitin protein ligase binding (GO:0031625) & 96 & 36 & 12 & 3.0 & $1.15 \times 10^{-7}$ & $7.86 \times 10^{-5}$ \\ 
  kinase binding (GO:0019900) & 235 & 61 & 29 & 2.1 & $5.58 \times 10^{-7}$ & $3.06 \times 10^{-4}$ \\ 
  protein-containing complex binding (GO:0044877) & 422 & 91 & 52 & 1.7 & $9.98 \times 10^{-7}$ & $4.55 \times 10^{-4}$ \\ 
  protein kinase binding (GO:0019901) & 210 & 54 & 26 & 2.1 & $3.40 \times 10^{-6}$ & $1.16 \times 10^{-3}$ \\ 
  protein kinase regulator activity (GO:0019887) & 40 & 20 & 5 & 4.0 & $2.98 \times 10^{-6}$ & $1.17 \times 10^{-3}$ \\ 
  phosphoprotein binding (GO:0051219) & 30 & 17 & 4 & 4.6 & $4.74 \times 10^{-6}$ & $1.44 \times 10^{-3}$ \\ 
  kinase regulator activity (GO:0019207) & 49 & 21 & 6 & 3.5 & $1.11 \times 10^{-5}$ & $3.05 \times 10^{-3}$ \\ 
  protein N-terminus binding (GO:0047485) & 37 & 18 & 5 & 3.9 & $1.26 \times 10^{-5}$ & $3.14 \times 10^{-3}$ \\ 
  signaling receptor binding (GO:0005102) & 421 & 85 & 52 & 1.6 & $2.50 \times 10^{-5}$ & $5.72 \times 10^{-3}$ \\ 
  disordered domain specific binding (GO:0097718) & 18 & 12 & 2 & 5.4 & $3.85 \times 10^{-5}$ & $8.11 \times 10^{-3}$ \\ 
  cadherin binding (GO:0045296) & 165 & 42 & 21 & 2.0 & $5.56 \times 10^{-5}$ & $1.02 \times 10^{-2}$ \\ 
  cell adhesion molecule binding (GO:0050839) & 235 & 54 & 29 & 1.9 & $5.53 \times 10^{-5}$ & $1.08 \times 10^{-2}$ \\ 
  cytoskeletal protein binding (GO:0008092) & 365 & 73 & 45 & 1.6 & $1.74 \times 10^{-4}$ & $2.80 \times 10^{-2}$ \\ 
  protein C-terminus binding (GO:0008022) & 79 & 25 & 10 & 2.5 & $1.69 \times 10^{-4}$ & $2.89 \times 10^{-2}$ \\ 
  protein phosphatase binding (GO:0019903) & 44 & 17 & 5 & 3.1 & $2.12 \times 10^{-4}$ & $3.23 \times 10^{-2}$ \\ 
  identical protein binding (GO:0042802) & 539 & 98 & 67 & 1.5 & $2.27 \times 10^{-4}$ & $3.27 \times 10^{-2}$ \\ 
  protein phosphorylated amino acid binding (GO:0045309) & 16 & 10 & 2 & 5.0 & $2.55 \times 10^{-4}$ & $3.49 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO molecular function complete MF 3457 90cent bet.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 21} 
\label{tab:GO molecular function complete MF 3457 90cent bet.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 21}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 13:18:47 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}


\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
inclusion body (GO:0016234) & 33 & 14 & 4 & 3.4 & $3.61 \times 10^{-4}$ & $2.18 \times 10^{-2}$ \\ 
  cytoplasmic ribonucleoprotein granule (GO:0036464) & 59 & 19 & 7 & 2.6 & $7.05 \times 10^{-4}$ & $3.31 \times 10^{-2}$ \\ 
  nuclear membrane (GO:0031965) & 64 & 20 & 8 & 2.5 & $6.51 \times 10^{-4}$ & $3.26 \times 10^{-2}$ \\ 
  ribonucleoprotein granule (GO:0035770) & 62 & 19 & 8 & 2.5 & $1.07 \times 10^{-3}$ & $4.47 \times 10^{-2}$ \\ 
  centrosome (GO:0005813) & 132 & 36 & 16 & 2.2 & $6.95 \times 10^{-5}$ & $5.51 \times 10^{-3}$ \\ 
  nuclear envelope (GO:0005635) & 110 & 30 & 14 & 2.2 & $2.78 \times 10^{-4}$ & $1.82 \times 10^{-2}$ \\ 
  microtubule organizing center (GO:0005815) & 173 & 45 & 22 & 2.1 & $1.86 \times 10^{-5}$ & $2.54 \times 10^{-3}$ \\ 
  focal adhesion (GO:0005925) & 212 & 52 & 26 & 2.0 & $1.61 \times 10^{-5}$ & $2.69 \times 10^{-3}$ \\ 
  cell-substrate junction (GO:0030055) & 217 & 53 & 27 & 2.0 & $1.80 \times 10^{-5}$ & $2.70 \times 10^{-3}$ \\ 
  glutamatergic synapse (GO:0098978) & 189 & 45 & 24 & 1.9 & $1.38 \times 10^{-4}$ & $1.04 \times 10^{-2}$ \\ 
  microtubule cytoskeleton (GO:0015630) & 300 & 68 & 37 & 1.8 & $9.71 \times 10^{-6}$ & $2.44 \times 10^{-3}$ \\ 
  vacuole (GO:0005773) & 222 & 48 & 28 & 1.7 & $5.68 \times 10^{-4}$ & $3.05 \times 10^{-2}$ \\ 
  secretory granule (GO:0030141) & 273 & 56 & 34 & 1.6 & $6.18 \times 10^{-4}$ & $3.21 \times 10^{-2}$ \\ 
  anchoring junction (GO:0070161) & 356 & 71 & 44 & 1.6 & $2.09 \times 10^{-4}$ & $1.43 \times 10^{-2}$ \\ 
  cytoskeleton (GO:0005856) & 603 & 115 & 75 & 1.5 & $6.08 \times 10^{-6}$ & $2.29 \times 10^{-3}$ \\ 
  supramolecular complex (GO:0099080) & 373 & 71 & 46 & 1.5 & $6.90 \times 10^{-4}$ & $3.35 \times 10^{-2}$ \\ 
  extracellular exosome (GO:0070062) & 759 & 134 & 94 & 1.4 & $2.99 \times 10^{-5}$ & $3.46 \times 10^{-3}$ \\ 
  extracellular organelle (GO:0043230) & 771 & 135 & 96 & 1.4 & $4.25 \times 10^{-5}$ & $4.56 \times 10^{-3}$ \\ 
  extracellular vesicle (GO:1903561) & 771 & 135 & 96 & 1.4 & $4.25 \times 10^{-5}$ & $4.26 \times 10^{-3}$ \\ 
  nucleoplasm (GO:0005654) & 761 & 130 & 95 & 1.4 & $1.81 \times 10^{-4}$ & $1.30 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO cellular component complete CC 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 36} 
\label{tab:GO cellular component complete CC 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown. Total significant terms 36}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 13:18:47 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
cytosol (GO:0005829) & 1430 & 231 & 178 & 1.3 & $9.32 \times 10^{-7}$ & $7.02 \times 10^{-4}$ \\ 
  nucleus (GO:0005634) & 1359 & 223 & 169 & 1.3 & $4.80 \times 10^{-7}$ & $7.22 \times 10^{-4}$ \\ 
  cytoskeleton (GO:0005856) & 603 & 115 & 75 & 1.5 & $6.08 \times 10^{-6}$ & $2.29 \times 10^{-3}$ \\ 
  intracellular non-membrane-bounded organelle (GO:0043232) & 1098 & 183 & 137 & 1.3 & $8.01 \times 10^{-6}$ & $2.41 \times 10^{-3}$ \\ 
  microtubule cytoskeleton (GO:0015630) & 300 & 68 & 37 & 1.8 & $9.71 \times 10^{-6}$ & $2.44 \times 10^{-3}$ \\ 
  non-membrane-bounded organelle (GO:0043228) & 1105 & 185 & 137 & 1.4 & $4.93 \times 10^{-6}$ & $2.47 \times 10^{-3}$ \\ 
  microtubule organizing center (GO:0005815) & 173 & 45 & 22 & 2.1 & $1.86 \times 10^{-5}$ & $2.54 \times 10^{-3}$ \\ 
  focal adhesion (GO:0005925) & 212 & 52 & 26 & 2.0 & $1.61 \times 10^{-5}$ & $2.69 \times 10^{-3}$ \\ 
  cell-substrate junction (GO:0030055) & 217 & 53 & 27 & 2.0 & $1.80 \times 10^{-5}$ & $2.70 \times 10^{-3}$ \\ 
  extracellular region (GO:0005576) & 1132 & 186 & 141 & 1.3 & $1.53 \times 10^{-5}$ & $2.87 \times 10^{-3}$ \\ 
  extracellular space (GO:0005615) & 966 & 163 & 120 & 1.4 & $2.45 \times 10^{-5}$ & $3.08 \times 10^{-3}$ \\ 
  protein-containing complex (GO:0032991) & 1403 & 221 & 175 & 1.3 & $1.51 \times 10^{-5}$ & $3.24 \times 10^{-3}$ \\ 
  extracellular exosome (GO:0070062) & 759 & 134 & 94 & 1.4 & $2.99 \times 10^{-5}$ & $3.46 \times 10^{-3}$ \\ 
  intracellular membrane-bounded organelle (GO:0043231) & 2085 & 301 & 259 & 1.2 & $4.32 \times 10^{-5}$ & $4.07 \times 10^{-3}$ \\ 
  extracellular vesicle (GO:1903561) & 771 & 135 & 96 & 1.4 & $4.25 \times 10^{-5}$ & $4.26 \times 10^{-3}$ \\ 
  extracellular organelle (GO:0043230) & 771 & 135 & 96 & 1.4 & $4.25 \times 10^{-5}$ & $4.56 \times 10^{-3}$ \\ 
  vesicle (GO:0031982) & 1187 & 190 & 148 & 1.3 & $6.00 \times 10^{-5}$ & $5.02 \times 10^{-3}$ \\ 
  intracellular organelle (GO:0043229) & 2465 & 343 & 307 & 1.1 & $5.94 \times 10^{-5}$ & $5.26 \times 10^{-3}$ \\ 
  centrosome (GO:0005813) & 132 & 36 & 16 & 2.2 & $6.95 \times 10^{-5}$ & $5.51 \times 10^{-3}$ \\ 
  glutamatergic synapse (GO:0098978) & 189 & 45 & 24 & 1.9 & $1.38 \times 10^{-4}$ & $1.04 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO cellular component complete CC 3457 90cent bet.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 36} 
\label{tab:GO cellular component complete CC 3457 90cent bet.txt Over representation only. Top 20 Terms. 20 significant terms shown . Total significant terms 36}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 13:31:19 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER Pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
Parkinson disease (P00049) & 45 & 26 & 6 & 4.6 & $1.00 \times 10^{-8}$ & $1.34 \times 10^{-6}$ \\ 
  Heterotrimeric G-protein signaling pathway-rod outer segment phototransduction (P00028) & 13 & 7 & 2 & 4.3 & $4.14 \times 10^{-3}$ & $3.96 \times 10^{-2}$ \\ 
  Opioid prodynorphin pathway (P05916) & 14 & 7 & 2 & 4.0 & $5.63 \times 10^{-3}$ & $4.44 \times 10^{-2}$ \\ 
  Angiotensin II-stimulated signaling through G proteins and beta-arrestin (P05911) & 19 & 9 & 2 & 3.8 & $2.29 \times 10^{-3}$ & $2.79 \times 10^{-2}$ \\ 
  Opioid proenkephalin pathway (P05915) & 15 & 7 & 2 & 3.8 & $7.48 \times 10^{-3}$ & $4.78 \times 10^{-2}$ \\ 
  Enkephalin release (P05913) & 15 & 7 & 2 & 3.8 & $7.48 \times 10^{-3}$ & $4.56 \times 10^{-2}$ \\ 
  Nicotine pharmacodynamics pathway (P06587) & 18 & 8 & 2 & 3.6 & $5.40 \times 10^{-3}$ & $4.53 \times 10^{-2}$ \\ 
  PI3 kinase pathway (P00048) & 23 & 10 & 3 & 3.5 & $2.17 \times 10^{-3}$ & $2.91 \times 10^{-2}$ \\ 
  Metabotropic glutamate receptor group II pathway (P00040) & 25 & 10 & 3 & 3.2 & $3.52 \times 10^{-3}$ & $3.93 \times 10^{-2}$ \\ 
  Dopamine receptor mediated signaling pathway (P05912) & 30 & 12 & 4 & 3.2 & $1.42 \times 10^{-3}$ & $2.38 \times 10^{-2}$ \\ 
  B cell activation (P00010) & 31 & 12 & 4 & 3.1 & $1.78 \times 10^{-3}$ & $2.65 \times 10^{-2}$ \\ 
  Muscarinic acetylcholine receptor 2 and 4 signaling pathway (P00043) & 26 & 10 & 3 & 3.1 & $4.41 \times 10^{-3}$ & $3.94 \times 10^{-2}$ \\ 
  CCKR signaling map (P06959) & 70 & 24 & 9 & 2.8 & $6.71 \times 10^{-5}$ & $3.00 \times 10^{-3}$ \\ 
  T cell activation (P00053) & 35 & 12 & 4 & 2.8 & $4.03 \times 10^{-3}$ & $4.15 \times 10^{-2}$ \\ 
  EGF receptor signaling pathway (P00018) & 59 & 20 & 7 & 2.7 & $3.38 \times 10^{-4}$ & $7.55 \times 10^{-3}$ \\ 
  FGF signaling pathway (P00021) & 59 & 19 & 7 & 2.6 & $7.05 \times 10^{-4}$ & $1.35 \times 10^{-2}$ \\ 
  Integrin signalling pathway (P00034) & 85 & 27 & 11 & 2.5 & $7.92 \times 10^{-5}$ & $2.65 \times 10^{-3}$ \\ 
  Gonadotropin-releasing hormone receptor pathway (P06664) & 92 & 28 & 11 & 2.5 & $8.42 \times 10^{-5}$ & $2.26 \times 10^{-3}$ \\ 
  Heterotrimeric G-protein signaling pathway-Gi alpha and Gs alpha mediated pathway (P00026) & 57 & 16 & 7 & 2.3 & $6.89 \times 10^{-3}$ & $4.86 \times 10^{-2}$ \\ 
  Angiogenesis (P00005) & 62 & 17 & 8 & 2.2 & $5.75 \times 10^{-3}$ & $4.28 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis pathway 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis pathway 3457 90cent bet.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 13:31:19 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER Pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
Parkinson disease (P00049) & 45 & 26 & 6 & 4.6 & $1.00 \times 10^{-8}$ & $1.34 \times 10^{-6}$ \\ 
  Gonadotropin-releasing hormone receptor pathway (P06664) & 92 & 28 & 11 & 2.5 & $8.42 \times 10^{-5}$ & $2.26 \times 10^{-3}$ \\ 
  Integrin signalling pathway (P00034) & 85 & 27 & 11 & 2.5 & $7.92 \times 10^{-5}$ & $2.65 \times 10^{-3}$ \\ 
  CCKR signaling map (P06959) & 70 & 24 & 9 & 2.8 & $6.71 \times 10^{-5}$ & $3.00 \times 10^{-3}$ \\ 
  EGF receptor signaling pathway (P00018) & 59 & 20 & 7 & 2.7 & $3.38 \times 10^{-4}$ & $7.55 \times 10^{-3}$ \\ 
  FGF signaling pathway (P00021) & 59 & 19 & 7 & 2.6 & $7.05 \times 10^{-4}$ & $1.35 \times 10^{-2}$ \\ 
  Dopamine receptor mediated signaling pathway (P05912) & 30 & 12 & 4 & 3.2 & $1.42 \times 10^{-3}$ & $2.38 \times 10^{-2}$ \\ 
  B cell activation (P00010) & 31 & 12 & 4 & 3.1 & $1.78 \times 10^{-3}$ & $2.65 \times 10^{-2}$ \\ 
  Angiotensin II-stimulated signaling through G proteins and beta-arrestin (P05911) & 19 & 9 & 2 & 3.8 & $2.29 \times 10^{-3}$ & $2.79 \times 10^{-2}$ \\ 
  PI3 kinase pathway (P00048) & 23 & 10 & 3 & 3.5 & $2.17 \times 10^{-3}$ & $2.91 \times 10^{-2}$ \\ 
  Metabotropic glutamate receptor group II pathway (P00040) & 25 & 10 & 3 & 3.2 & $3.52 \times 10^{-3}$ & $3.93 \times 10^{-2}$ \\ 
  Muscarinic acetylcholine receptor 2 and 4 signaling pathway (P00043) & 26 & 10 & 3 & 3.1 & $4.41 \times 10^{-3}$ & $3.94 \times 10^{-2}$ \\ 
  Heterotrimeric G-protein signaling pathway-rod outer segment phototransduction (P00028) & 13 & 7 & 2 & 4.3 & $4.14 \times 10^{-3}$ & $3.96 \times 10^{-2}$ \\ 
  T cell activation (P00053) & 35 & 12 & 4 & 2.8 & $4.03 \times 10^{-3}$ & $4.15 \times 10^{-2}$ \\ 
  Angiogenesis (P00005) & 62 & 17 & 8 & 2.2 & $5.75 \times 10^{-3}$ & $4.28 \times 10^{-2}$ \\ 
  Opioid prodynorphin pathway (P05916) & 14 & 7 & 2 & 4.0 & $5.63 \times 10^{-3}$ & $4.44 \times 10^{-2}$ \\ 
  Nicotine pharmacodynamics pathway (P06587) & 18 & 8 & 2 & 3.6 & $5.40 \times 10^{-3}$ & $4.53 \times 10^{-2}$ \\ 
  Enkephalin release (P05913) & 15 & 7 & 2 & 3.8 & $7.48 \times 10^{-3}$ & $4.56 \times 10^{-2}$ \\ 
  Opioid proenkephalin pathway (P05915) & 15 & 7 & 2 & 3.8 & $7.48 \times 10^{-3}$ & $4.78 \times 10^{-2}$ \\ 
  Heterotrimeric G-protein signaling pathway-Gi alpha and Gs alpha mediated pathway (P00026) & 57 & 16 & 7 & 2.3 & $6.89 \times 10^{-3}$ & $4.86 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis pathway 3457 90cent bet.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis pathway 3457 90cent bet.txt Over representation only. Top 20 Terms}
\end{table}

\todo{reactome}




%%%%%%%%%%%%%%%%%%% BETWEENNESS LOW %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\subsubsection{Low betweenness}


\ref{tab:GO analysis CC 3457 10cent bet.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis CC 3457 10cent bet.txt Over representation only. Top 20 Terms}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 13:36:29 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}


\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
integral component of synaptic membrane (GO:0099699) & 55 & 14 & 4 & 3.1 & $4.81 \times 10^{-4}$ & $4.53 \times 10^{-2}$ \\ 
  intrinsic component of synaptic membrane (GO:0099240) & 61 & 15 & 5 & 3.0 & $4.16 \times 10^{-4}$ & $4.17 \times 10^{-2}$ \\ 
  integral component of membrane (GO:0016021) & 889 & 106 & 72 & 1.5 & $2.09 \times 10^{-5}$ & $6.29 \times 10^{-3}$ \\ 
  intrinsic component of membrane (GO:0031224) & 926 & 110 & 75 & 1.5 & $1.85 \times 10^{-5}$ & $6.95 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis CC 3457 10cent bet.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis CC 3457 10cent bet.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Tue Jun 30 13:36:29 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
integral component of membrane (GO:0016021) & 889 & 106 & 72 & 1.5 & $2.09 \times 10^{-5}$ & $6.29 \times 10^{-3}$ \\ 
  intrinsic component of membrane (GO:0031224) & 926 & 110 & 75 & 1.5 & $1.85 \times 10^{-5}$ & $6.95 \times 10^{-3}$ \\ 
  intrinsic component of synaptic membrane (GO:0099240) & 61 & 15 & 5 & 3.0 & $4.16 \times 10^{-4}$ & $4.17 \times 10^{-2}$ \\ 
  integral component of synaptic membrane (GO:0099699) & 55 & 14 & 4 & 3.1 & $4.81 \times 10^{-4}$ & $4.53 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis CC 3457 10cent bet.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis CC 3457 10cent bet.txt Over representation only. Top 20 Terms}
\end{table}

% % latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Sat Jun 27 16:20:02 2020
% \begin{table}[ht]
% \centering
% \begin{adjustbox}{width=\textwidth}

% \begin{tabular}{lrrrrrr}
%   \hline
% GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
%   \hline
% integral component of membrane (GO:0016021) & 889 & 106 & 72 & 1.5 & $2.09 \times 10^{-5}$ & $6.29 \times 10^{-3}$ \\ 
%   intrinsic component of membrane (GO:0031224) & 926 & 110 & 75 & 1.5 & $1.85 \times 10^{-5}$ & $6.95 \times 10^{-3}$ \\ 
%   intrinsic component of synaptic membrane (GO:0099240) & 61 & 15 & 5 & 3.0 & $4.16 \times 10^{-4}$ & $4.17 \times 10^{-2}$ \\ 
%   integral component of synaptic membrane (GO:0099699) & 55 & 14 & 4 & 3.1 & $4.81 \times 10^{-4}$ & $4.53 \times 10^{-2}$ \\ 
%   \hline
% \end{tabular}
% \end{adjustbox}
% \caption{GO analysis CC 3457 10cent bet.txt Over representation only} 
% \label{tab:GO analysis CC 3457 10cent bet.txt Over representation only}
% \end{table}


% % latex table generated in R 3.6.3 by xtable 1.8-4 package
% % Sat Jun 27 16:20:02 2020
% \begin{table}[ht]
% \centering
% \begin{adjustbox}{width=\textwidth}

% \begin{tabular}{lrrrrrr}
%   \hline
% GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
%   \hline
% integral component of membrane (GO:0016021) & 889 & 106 & 72 & 1.5 & $2.09 \times 10^{-5}$ & $6.29 \times 10^{-3}$ \\ 
%   intrinsic component of membrane (GO:0031224) & 926 & 110 & 75 & 1.5 & $1.85 \times 10^{-5}$ & $6.95 \times 10^{-3}$ \\ 
%   intrinsic component of synaptic membrane (GO:0099240) & 61 & 15 & 5 & 3.0 & $4.16 \times 10^{-4}$ & $4.17 \times 10^{-2}$ \\ 
%   integral component of synaptic membrane (GO:0099699) & 55 & 14 & 4 & 3.1 & $4.81 \times 10^{-4}$ & $4.53 \times 10^{-2}$ \\ 
%   \hline
% \end{tabular}
% \end{adjustbox}
% \caption{GO analysis CC 3457 10cent bet.txt Over representation only} 
% \label{tab:GO analysis CC 3457 10cent bet.txt Over representation only}
% \end{table}



\subsection{Eigenvector}

Top10 may need to redo earlier as used head file to order by fold change ie both same

\ref{tab:GO analysis BP 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis BP 3457 90cent eigvec.txt Over representation only. Top 20 Terms}

\ref{tab:GO analysis MF 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

Says top 10 but is top 20 \ref{tab:GO analysis MF 3457 90cent eigvec.txt Over representation only. Top 10 Terms}

\ref{tab:GO analysis CC 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis CC 3457 90cent eigvec.txt Over representation only. Top 20 Terms}

Pathway table~\ref{tab:GO analysis pantherpathway 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis pantherpathway 3457 90cent eigvec.txt Over representation only. Top 20 Terms}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:14:24 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrlrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
ribosomal large subunit assembly (GO:0000027) & 11 & 11 & 1 & 7.41 & $1.24 \times 10^{-5}$ & $9.54 \times 10^{-4}$ \\ 
  ribosome assembly (GO:0042255) & 20 & 18 & 3 & 6.67 & $5.62 \times 10^{-8}$ & $7.22 \times 10^{-6}$ \\ 
  ribosomal small subunit assembly (GO:0000028) & 8 & 7 & 1 & 6.48 & $8.78 \times 10^{-4}$ & $4.08 \times 10^{-2}$ \\ 
  ribosomal large subunit biogenesis (GO:0042273) & 16 & 14 & 2 & 6.48 & $2.22 \times 10^{-6}$ & $2.04 \times 10^{-4}$ \\ 
  mRNA stabilization (GO:0048255) & 14 & 12 & 2 & 6.35 & $1.39 \times 10^{-5}$ & $1.05 \times 10^{-3}$ \\ 
  ribosomal small subunit biogenesis (GO:0042274) & 13 & 11 & 2 & 6.27 & $3.49 \times 10^{-5}$ & $2.51 \times 10^{-3}$ \\ 
  negative regulation of mRNA catabolic process (GO:1902373) & 16 & 12 & 2 & 5.56 & $3.50 \times 10^{-5}$ & $2.50 \times 10^{-3}$ \\ 
  ERBB2 signaling pathway (GO:0038128) & 12 & 9 & 2 & 5.56 & $3.46 \times 10^{-4}$ & $1.81 \times 10^{-2}$ \\ 
  negative regulation of mRNA metabolic process (GO:1903312) & 22 & 16 & 3 & 5.39 & $2.26 \times 10^{-6}$ & $2.06 \times 10^{-4}$ \\ 
  RNA stabilization (GO:0043489) & 17 & 12 & 2 & 5.23 & $5.34 \times 10^{-5}$ & $3.59 \times 10^{-3}$ \\ 
  nuclear-transcribed mRNA catabolic process, nonsense-mediated decay (GO:0000184) & 77 & 51 & 10 & 4.91 & $1.58 \times 10^{-16}$ & $1.06 \times 10^{-13}$ \\ 
  regulation of telomere maintenance via telomere lengthening (GO:1904356) & 17 & 11 & 2 & 4.79 & $1.92 \times 10^{-4}$ & $1.08 \times 10^{-2}$ \\ 
  rRNA processing (GO:0006364) & 61 & 39 & 8 & 4.74 & $1.66 \times 10^{-12}$ & $4.13 \times 10^{-10}$ \\ 
  viral gene expression (GO:0019080) & 83 & 53 & 11 & 4.73 & $1.20 \times 10^{-16}$ & $8.57 \times 10^{-14}$ \\ 
  viral transcription (GO:0019083) & 77 & 49 & 10 & 4.72 & $2.09 \times 10^{-15}$ & $9.18 \times 10^{-13}$ \\ 
  rRNA metabolic process (GO:0016072) & 63 & 40 & 8 & 4.70 & $9.95 \times 10^{-13}$ & $2.78 \times 10^{-10}$ \\ 
  ribosome biogenesis (GO:0042254) & 71 & 45 & 10 & 4.70 & $3.56 \times 10^{-14}$ & $1.20 \times 10^{-11}$ \\ 
  negative regulation of RNA catabolic process (GO:1902369) & 19 & 12 & 3 & 4.68 & $1.16 \times 10^{-4}$ & $6.99 \times 10^{-3}$ \\ 
  regulation of telomere maintenance via telomerase (GO:0032210) & 16 & 10 & 2 & 4.63 & $4.63 \times 10^{-4}$ & $2.35 \times 10^{-2}$ \\ 
  SRP-dependent cotranslational protein targeting to membrane (GO:0006614) & 76 & 47 & 10 & 4.58 & $1.81 \times 10^{-14}$ & $6.48 \times 10^{-12}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis BP 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis BP 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:14:24 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}


\begin{tabular}{lrrrlrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
mRNA metabolic process (GO:0016071) & 198 & 105 & 27 & 3.93 & $5.71 \times 10^{-28}$ & $6.53 \times 10^{-24}$ \\ 
  symbiotic process (GO:0044403) & 313 & 124 & 42 & 2.94 & $6.83 \times 10^{-24}$ & $3.90 \times 10^{-20}$ \\ 
  RNA processing (GO:0006396) & 179 & 91 & 24 & 3.77 & $4.48 \times 10^{-23}$ & $1.71 \times 10^{-19}$ \\ 
  interspecies interaction between organisms (GO:0044419) & 553 & 167 & 75 & 2.24 & $4.65 \times 10^{-22}$ & $1.33 \times 10^{-18}$ \\ 
  nucleic acid metabolic process (GO:0090304) & 429 & 142 & 58 & 2.45 & $2.37 \times 10^{-21}$ & $4.52 \times 10^{-18}$ \\ 
  RNA metabolic process (GO:0016070) & 330 & 122 & 45 & 2.74 & $2.02 \times 10^{-21}$ & $4.62 \times 10^{-18}$ \\ 
  gene expression (GO:0010467) & 414 & 138 & 56 & 2.47 & $5.65 \times 10^{-21}$ & $9.23 \times 10^{-18}$ \\ 
  viral process (GO:0016032) & 285 & 110 & 38 & 2.86 & $2.36 \times 10^{-20}$ & $3.37 \times 10^{-17}$ \\ 
  ribonucleoprotein complex biogenesis (GO:0022613) & 115 & 64 & 16 & 4.12 & $1.04 \times 10^{-17}$ & $1.32 \times 10^{-14}$ \\ 
  macromolecule metabolic process (GO:0043170) & 1257 & 265 & 170 & 1.56 & $1.74 \times 10^{-17}$ & $1.99 \times 10^{-14}$ \\ 
  RNA catabolic process (GO:0006401) & 100 & 59 & 14 & 4.37 & $2.96 \times 10^{-17}$ & $3.08 \times 10^{-14}$ \\ 
  nucleobase-containing compound metabolic process (GO:0006139) & 572 & 157 & 77 & 2.03 & $5.57 \times 10^{-17}$ & $5.31 \times 10^{-14}$ \\ 
  cellular aromatic compound metabolic process (GO:0006725) & 628 & 166 & 85 & 1.96 & $7.72 \times 10^{-17}$ & $6.79 \times 10^{-14}$ \\ 
  negative regulation of macromolecule metabolic process (GO:0010605) & 600 & 161 & 81 & 1.99 & $1.10 \times 10^{-16}$ & $8.37 \times 10^{-14}$ \\ 
  viral gene expression (GO:0019080) & 83 & 53 & 11 & 4.73 & $1.20 \times 10^{-16}$ & $8.57 \times 10^{-14}$ \\ 
  mRNA catabolic process (GO:0006402) & 93 & 56 & 13 & 4.46 & $1.05 \times 10^{-16}$ & $8.61 \times 10^{-14}$ \\ 
  nuclear-transcribed mRNA catabolic process, nonsense-mediated decay (GO:0000184) & 77 & 51 & 10 & 4.91 & $1.58 \times 10^{-16}$ & $1.06 \times 10^{-13}$ \\ 
  cellular macromolecule catabolic process (GO:0044265) & 223 & 88 & 30 & 2.92 & $1.83 \times 10^{-16}$ & $1.16 \times 10^{-13}$ \\ 
  translational initiation (GO:0006413) & 92 & 55 & 12 & 4.43 & $2.58 \times 10^{-16}$ & $1.55 \times 10^{-13}$ \\ 
  regulation of gene expression (GO:0010468) & 859 & 202 & 116 & 1.74 & $3.36 \times 10^{-16}$ & $1.83 \times 10^{-13}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis BP 3457 90cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis BP 3457 90cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}


%%%%%%%%%%%% EIG MF %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:11:53 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}


\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
rRNA binding (GO:0019843) & 20 & 16 & 3 & 5.9 & $9.32 \times 10^{-7}$ & $2.32 \times 10^{-4}$ \\ 
  mRNA 5'-UTR binding (GO:0048027) & 12 & 9 & 2 & 5.6 & $3.46 \times 10^{-4}$ & $3.51 \times 10^{-2}$ \\ 
  protein folding chaperone (GO:0044183) & 14 & 10 & 2 & 5.3 & $2.13 \times 10^{-4}$ & $2.34 \times 10^{-2}$ \\ 
  disordered domain specific binding (GO:0097718) & 18 & 12 & 2 & 4.9 & $7.96 \times 10^{-5}$ & $1.04 \times 10^{-2}$ \\ 
  structural constituent of ribosome (GO:0003735) & 72 & 47 & 10 & 4.8 & $4.03 \times 10^{-15}$ & $3.68 \times 10^{-12}$ \\ 
  single-stranded RNA binding (GO:0003727) & 25 & 16 & 3 & 4.7 & $7.43 \times 10^{-6}$ & $1.45 \times 10^{-3}$ \\ 
  mRNA 3'-UTR binding (GO:0003730) & 30 & 19 & 4 & 4.7 & $1.14 \times 10^{-6}$ & $2.61 \times 10^{-4}$ \\ 
  histone deacetylase binding (GO:0042826) & 28 & 14 & 4 & 3.7 & $2.07 \times 10^{-4}$ & $2.36 \times 10^{-2}$ \\ 
  phosphoprotein binding (GO:0051219) & 30 & 15 & 4 & 3.7 & $1.23 \times 10^{-4}$ & $1.47 \times 10^{-2}$ \\ 
  mRNA binding (GO:0003729) & 87 & 43 & 12 & 3.7 & $7.21 \times 10^{-11}$ & $3.29 \times 10^{-8}$ \\ 
  heat shock protein binding (GO:0031072) & 49 & 23 & 7 & 3.5 & $4.46 \times 10^{-6}$ & $9.40 \times 10^{-4}$ \\ 
  ubiquitin protein ligase binding (GO:0031625) & 96 & 44 & 13 & 3.4 & $2.87 \times 10^{-10}$ & $9.82 \times 10^{-8}$ \\ 
  ubiquitin-like protein ligase binding (GO:0044389) & 99 & 45 & 13 & 3.4 & $2.19 \times 10^{-10}$ & $8.56 \times 10^{-8}$ \\ 
  double-stranded RNA binding (GO:0003725) & 31 & 14 & 4 & 3.4 & $4.66 \times 10^{-4}$ & $4.26 \times 10^{-2}$ \\ 
  unfolded protein binding (GO:0051082) & 45 & 20 & 6 & 3.3 & $3.51 \times 10^{-5}$ & $5.06 \times 10^{-3}$ \\ 
  translation regulator activity (GO:0045182) & 55 & 22 & 7 & 3.0 & $5.16 \times 10^{-5}$ & $7.06 \times 10^{-3}$ \\ 
  translation regulator activity, nucleic acid binding (GO:0090079) & 43 & 17 & 6 & 2.9 & $4.10 \times 10^{-4}$ & $3.88 \times 10^{-2}$ \\ 
  RNA binding (GO:0003723) & 452 & 163 & 61 & 2.7 & $8.70 \times 10^{-29}$ & $2.38 \times 10^{-25}$ \\ 
  structural molecule activity (GO:0005198) & 244 & 76 & 33 & 2.3 & $6.10 \times 10^{-10}$ & $1.86 \times 10^{-7}$ \\ 
  nucleic acid binding (GO:0003676) & 724 & 198 & 98 & 2.0 & $1.19 \times 10^{-22}$ & $1.63 \times 10^{-19}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis MF 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis MF 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:11:53 2020

\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
RNA binding (GO:0003723) & 452 & 163 & 61 & 2.7 & $8.70 \times 10^{-29}$ & $2.38 \times 10^{-25}$ \\ 
  nucleic acid binding (GO:0003676) & 724 & 198 & 98 & 2.0 & $1.19 \times 10^{-22}$ & $1.63 \times 10^{-19}$ \\ 
  structural constituent of ribosome (GO:0003735) & 72 & 47 & 10 & 4.8 & $4.03 \times 10^{-15}$ & $3.68 \times 10^{-12}$ \\ 
  organic cyclic compound binding (GO:0097159) & 1288 & 258 & 174 & 1.5 & $7.02 \times 10^{-14}$ & $4.81 \times 10^{-11}$ \\ 
  heterocyclic compound binding (GO:1901363) & 1270 & 254 & 171 & 1.5 & $1.47 \times 10^{-13}$ & $8.05 \times 10^{-11}$ \\ 
  mRNA binding (GO:0003729) & 87 & 43 & 12 & 3.7 & $7.21 \times 10^{-11}$ & $3.29 \times 10^{-8}$ \\ 
  ubiquitin-like protein ligase binding (GO:0044389) & 99 & 45 & 13 & 3.4 & $2.19 \times 10^{-10}$ & $8.56 \times 10^{-8}$ \\ 
  ubiquitin protein ligase binding (GO:0031625) & 96 & 44 & 13 & 3.4 & $2.87 \times 10^{-10}$ & $9.82 \times 10^{-8}$ \\ 
  enzyme binding (GO:0019899) & 641 & 146 & 87 & 1.7 & $6.22 \times 10^{-10}$ & $1.70 \times 10^{-7}$ \\ 
  structural molecule activity (GO:0005198) & 244 & 76 & 33 & 2.3 & $6.10 \times 10^{-10}$ & $1.86 \times 10^{-7}$ \\ 
  rRNA binding (GO:0019843) & 20 & 16 & 3 & 5.9 & $9.32 \times 10^{-7}$ & $2.32 \times 10^{-4}$ \\ 
  mRNA 3'-UTR binding (GO:0003730) & 30 & 19 & 4 & 4.7 & $1.14 \times 10^{-6}$ & $2.61 \times 10^{-4}$ \\ 
  heat shock protein binding (GO:0031072) & 49 & 23 & 7 & 3.5 & $4.46 \times 10^{-6}$ & $9.40 \times 10^{-4}$ \\ 
  single-stranded RNA binding (GO:0003727) & 25 & 16 & 3 & 4.7 & $7.43 \times 10^{-6}$ & $1.45 \times 10^{-3}$ \\ 
  protein kinase binding (GO:0019901) & 210 & 56 & 28 & 2.0 & $1.17 \times 10^{-5}$ & $2.14 \times 10^{-3}$ \\ 
  kinase binding (GO:0019900) & 235 & 60 & 32 & 1.9 & $1.34 \times 10^{-5}$ & $2.29 \times 10^{-3}$ \\ 
  binding (GO:0005488) & 2944 & 425 & 397 & 1.1 & $2.14 \times 10^{-5}$ & $3.44 \times 10^{-3}$ \\ 
  protein-containing complex binding (GO:0044877) & 422 & 91 & 57 & 1.6 & $2.63 \times 10^{-5}$ & $4.00 \times 10^{-3}$ \\ 
  unfolded protein binding (GO:0051082) & 45 & 20 & 6 & 3.3 & $3.51 \times 10^{-5}$ & $5.06 \times 10^{-3}$ \\ 
  translation regulator activity (GO:0045182) & 55 & 22 & 7 & 3.0 & $5.16 \times 10^{-5}$ & $7.06 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis MF 3457 90cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis MF 3457 90cent eigvec.txt Over representation only. Top 10 Terms}
\end{table}




%%%% EIG TOP CC %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%




% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:20:22 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}


\begin{tabular}{lrrrlrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
U2-type catalytic step 2 spliceosome (GO:0071007) & 10 & 8 & 1 & 5.93 & $5.55 \times 10^{-4}$ & $1.71 \times 10^{-2}$ \\ 
  cytosolic small ribosomal subunit (GO:0022627) & 28 & 21 & 4 & 5.56 & $3.88 \times 10^{-8}$ & $2.54 \times 10^{-6}$ \\ 
  catalytic step 2 spliceosome (GO:0071013) & 32 & 23 & 4 & 5.33 & $1.51 \times 10^{-8}$ & $1.04 \times 10^{-6}$ \\ 
  small ribosomal subunit (GO:0015935) & 30 & 21 & 4 & 5.19 & $9.03 \times 10^{-8}$ & $5.44 \times 10^{-6}$ \\ 
  cytosolic ribosome (GO:0022626) & 68 & 47 & 9 & 5.12 & $8.21 \times 10^{-16}$ & $6.18 \times 10^{-13}$ \\ 
  cytosolic large ribosomal subunit (GO:0022625) & 39 & 26 & 5 & 4.94 & $5.28 \times 10^{-9}$ & $3.98 \times 10^{-7}$ \\ 
  chaperone complex (GO:0101031) & 19 & 12 & 3 & 4.68 & $1.16 \times 10^{-4}$ & $4.26 \times 10^{-3}$ \\ 
  ribosomal subunit (GO:0044391) & 75 & 47 & 10 & 4.64 & $1.26 \times 10^{-14}$ & $3.78 \times 10^{-12}$ \\ 
  U2-type spliceosomal complex (GO:0005684) & 21 & 13 & 3 & 4.59 & $7.00 \times 10^{-5}$ & $2.77 \times 10^{-3}$ \\ 
  polysomal ribosome (GO:0042788) & 26 & 16 & 4 & 4.56 & $1.07 \times 10^{-5}$ & $5.20 \times 10^{-4}$ \\ 
  protein complex involved in cell adhesion (GO:0098636) & 23 & 14 & 3 & 4.51 & $4.21 \times 10^{-5}$ & $1.81 \times 10^{-3}$ \\ 
  integrin complex (GO:0008305) & 22 & 13 & 3 & 4.38 & $9.94 \times 10^{-5}$ & $3.74 \times 10^{-3}$ \\ 
  spliceosomal complex (GO:0005681) & 50 & 29 & 7 & 4.30 & $7.10 \times 10^{-9}$ & $5.09 \times 10^{-7}$ \\ 
  large ribosomal subunit (GO:0015934) & 45 & 26 & 6 & 4.28 & $4.65 \times 10^{-8}$ & $2.92 \times 10^{-6}$ \\ 
  ribosome (GO:0005840) & 90 & 52 & 12 & 4.28 & $5.21 \times 10^{-15}$ & $2.61 \times 10^{-12}$ \\ 
  ribonucleoprotein complex (GO:1990904) & 208 & 110 & 28 & 3.92 & $2.59 \times 10^{-29}$ & $3.90 \times 10^{-26}$ \\ 
  polysome (GO:0005844) & 42 & 19 & 6 & 3.35 & $4.56 \times 10^{-5}$ & $1.91 \times 10^{-3}$ \\ 
  ribonucleoprotein granule (GO:0035770) & 62 & 25 & 8 & 2.99 & $1.42 \times 10^{-5}$ & $6.69 \times 10^{-4}$ \\ 
  cytoplasmic ribonucleoprotein granule (GO:0036464) & 59 & 23 & 8 & 2.89 & $7.56 \times 10^{-5}$ & $2.92 \times 10^{-3}$ \\ 
  focal adhesion (GO:0005925) & 212 & 76 & 29 & 2.66 & $1.68 \times 10^{-12}$ & $2.53 \times 10^{-10}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis CC 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis CC 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}




% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:20:22 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrlrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
ribonucleoprotein complex (GO:1990904) & 208 & 110 & 28 & 3.92 & $2.59 \times 10^{-29}$ & $3.90 \times 10^{-26}$ \\ 
  cytosolic ribosome (GO:0022626) & 68 & 47 & 9 & 5.12 & $8.21 \times 10^{-16}$ & $6.18 \times 10^{-13}$ \\ 
  ribosome (GO:0005840) & 90 & 52 & 12 & 4.28 & $5.21 \times 10^{-15}$ & $2.61 \times 10^{-12}$ \\ 
  protein-containing complex (GO:0032991) & 1403 & 277 & 189 & 1.46 & $7.75 \times 10^{-15}$ & $2.91 \times 10^{-12}$ \\ 
  ribosomal subunit (GO:0044391) & 75 & 47 & 10 & 4.64 & $1.26 \times 10^{-14}$ & $3.78 \times 10^{-12}$ \\ 
  nucleus (GO:0005634) & 1359 & 268 & 183 & 1.46 & $5.01 \times 10^{-14}$ & $1.26 \times 10^{-11}$ \\ 
  non-membrane-bounded organelle (GO:0043228) & 1105 & 232 & 149 & 1.56 & $6.96 \times 10^{-14}$ & $1.31 \times 10^{-11}$ \\ 
  intracellular non-membrane-bounded organelle (GO:0043232) & 1098 & 231 & 148 & 1.56 & $6.42 \times 10^{-14}$ & $1.38 \times 10^{-11}$ \\ 
  focal adhesion (GO:0005925) & 212 & 76 & 29 & 2.66 & $1.68 \times 10^{-12}$ & $2.53 \times 10^{-10}$ \\ 
  cell-substrate junction (GO:0030055) & 217 & 77 & 29 & 2.63 & $1.54 \times 10^{-12}$ & $2.57 \times 10^{-10}$ \\ 
  cytosol (GO:0005829) & 1430 & 272 & 193 & 1.41 & $2.10 \times 10^{-12}$ & $2.87 \times 10^{-10}$ \\ 
  nuclear lumen (GO:0031981) & 894 & 195 & 121 & 1.62 & $2.45 \times 10^{-12}$ & $3.07 \times 10^{-10}$ \\ 
  membrane-enclosed lumen (GO:0031974) & 1179 & 232 & 159 & 1.46 & $5.85 \times 10^{-11}$ & $5.87 \times 10^{-9}$ \\ 
  nucleoplasm (GO:0005654) & 761 & 169 & 103 & 1.65 & $6.25 \times 10^{-11}$ & $5.88 \times 10^{-9}$ \\ 
  intracellular organelle lumen (GO:0070013) & 1179 & 232 & 159 & 1.46 & $5.85 \times 10^{-11}$ & $6.29 \times 10^{-9}$ \\ 
  organelle lumen (GO:0043233) & 1179 & 232 & 159 & 1.46 & $5.85 \times 10^{-11}$ & $6.77 \times 10^{-9}$ \\ 
  extracellular exosome (GO:0070062) & 759 & 163 & 102 & 1.59 & $1.81 \times 10^{-9}$ & $1.60 \times 10^{-7}$ \\ 
  extracellular vesicle (GO:1903561) & 771 & 164 & 104 & 1.58 & $3.09 \times 10^{-9}$ & $2.45 \times 10^{-7}$ \\ 
  extracellular organelle (GO:0043230) & 771 & 164 & 104 & 1.58 & $3.09 \times 10^{-9}$ & $2.59 \times 10^{-7}$ \\ 
  cytosolic large ribosomal subunit (GO:0022625) & 39 & 26 & 5 & 4.94 & $5.28 \times 10^{-9}$ & $3.98 \times 10^{-7}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis CC 3457 90cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis CC 3457 90cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:24:02 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER Pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
Parkinson disease (P00049) & 45 & 24 & 6 & 4.0 & $4.75 \times 10^{-7}$ & $6.36 \times 10^{-5}$ \\ 
  Integrin signalling pathway (P00034) & 85 & 29 & 11 & 2.5 & $4.58 \times 10^{-5}$ & $3.07 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis pantherpathway 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis pantherpathway 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:24:02 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER Pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
Parkinson disease (P00049) & 45 & 24 & 6 & 4.0 & $4.75 \times 10^{-7}$ & $6.36 \times 10^{-5}$ \\ 
  Integrin signalling pathway (P00034) & 85 & 29 & 11 & 2.5 & $4.58 \times 10^{-5}$ & $3.07 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis pantherpathway 3457 90cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis pantherpathway 3457 90cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}

\subsubsection{Eigenvector centrality low}

\ref{tab:GO analysis BP 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

Trans synaptic signalling \ref{tab:GO analysis BP 3457 10cent eigvec.txt Over representation only. Top 20 Terms}

\ref{tab:GO analysis MF 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis CC 3457 10cent eigvec.txt Over representation only. Top 20 Terms}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:39:50 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
anterograde trans-synaptic signaling (GO:0098916) & 161 & 35 & 15 & 2.4 & $1.35 \times 10^{-5}$ & $3.85 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis BP 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis BP 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:39:50 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
anterograde trans-synaptic signaling (GO:0098916) & 161 & 35 & 15 & 2.4 & $1.35 \times 10^{-5}$ & $3.85 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis BP 3457 10cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis BP 3457 10cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}




% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:42:29 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
sodium ion transmembrane transporter activity (GO:0015081) & 39 & 14 & 4 & 3.9 & $7.69 \times 10^{-5}$ & $3.01 \times 10^{-2}$ \\ 
  potassium channel activity (GO:0005267) & 44 & 15 & 4 & 3.7 & $6.91 \times 10^{-5}$ & $3.16 \times 10^{-2}$ \\ 
  voltage-gated cation channel activity (GO:0022843) & 55 & 16 & 5 & 3.2 & $1.88 \times 10^{-4}$ & $3.68 \times 10^{-2}$ \\ 
  potassium ion transmembrane transporter activity (GO:0015079) & 52 & 15 & 5 & 3.1 & $3.22 \times 10^{-4}$ & $4.90 \times 10^{-2}$ \\ 
  metal ion transmembrane transporter activity (GO:0046873) & 125 & 33 & 11 & 2.9 & $4.57 \times 10^{-7}$ & $1.25 \times 10^{-3}$ \\ 
  voltage-gated channel activity (GO:0022832) & 74 & 19 & 7 & 2.8 & $1.91 \times 10^{-4}$ & $3.50 \times 10^{-2}$ \\ 
  voltage-gated ion channel activity (GO:0005244) & 74 & 19 & 7 & 2.8 & $1.91 \times 10^{-4}$ & $3.28 \times 10^{-2}$ \\ 
  cation channel activity (GO:0005261) & 116 & 26 & 11 & 2.4 & $1.35 \times 10^{-4}$ & $3.70 \times 10^{-2}$ \\ 
  gated channel activity (GO:0022836) & 117 & 26 & 11 & 2.4 & $1.44 \times 10^{-4}$ & $3.57 \times 10^{-2}$ \\ 
  monovalent inorganic cation transmembrane transporter activity (GO:0015077) & 117 & 26 & 11 & 2.4 & $1.44 \times 10^{-4}$ & $3.28 \times 10^{-2}$ \\ 
  ion channel activity (GO:0005216) & 139 & 29 & 13 & 2.3 & $1.49 \times 10^{-4}$ & $3.15 \times 10^{-2}$ \\ 
  passive transmembrane transporter activity (GO:0022803) & 145 & 30 & 13 & 2.2 & $1.17 \times 10^{-4}$ & $4.02 \times 10^{-2}$ \\ 
  channel activity (GO:0015267) & 145 & 30 & 13 & 2.2 & $1.17 \times 10^{-4}$ & $3.57 \times 10^{-2}$ \\ 
  inorganic cation transmembrane transporter activity (GO:0022890) & 176 & 36 & 16 & 2.2 & $3.01 \times 10^{-5}$ & $2.06 \times 10^{-2}$ \\ 
  cation transmembrane transporter activity (GO:0008324) & 180 & 36 & 17 & 2.2 & $3.98 \times 10^{-5}$ & $2.18 \times 10^{-2}$ \\ 
  inorganic molecular entity transmembrane transporter activity (GO:0015318) & 226 & 40 & 21 & 1.9 & $1.97 \times 10^{-4}$ & $3.17 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis MF 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis MF 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:42:29 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
metal ion transmembrane transporter activity (GO:0046873) & 125 & 33 & 11 & 2.9 & $4.57 \times 10^{-7}$ & $1.25 \times 10^{-3}$ \\ 
  inorganic cation transmembrane transporter activity (GO:0022890) & 176 & 36 & 16 & 2.2 & $3.01 \times 10^{-5}$ & $2.06 \times 10^{-2}$ \\ 
  cation transmembrane transporter activity (GO:0008324) & 180 & 36 & 17 & 2.2 & $3.98 \times 10^{-5}$ & $2.18 \times 10^{-2}$ \\ 
  sodium ion transmembrane transporter activity (GO:0015081) & 39 & 14 & 4 & 3.9 & $7.69 \times 10^{-5}$ & $3.01 \times 10^{-2}$ \\ 
  ion channel activity (GO:0005216) & 139 & 29 & 13 & 2.3 & $1.49 \times 10^{-4}$ & $3.15 \times 10^{-2}$ \\ 
  potassium channel activity (GO:0005267) & 44 & 15 & 4 & 3.7 & $6.91 \times 10^{-5}$ & $3.16 \times 10^{-2}$ \\ 
  inorganic molecular entity transmembrane transporter activity (GO:0015318) & 226 & 40 & 21 & 1.9 & $1.97 \times 10^{-4}$ & $3.17 \times 10^{-2}$ \\ 
  voltage-gated ion channel activity (GO:0005244) & 74 & 19 & 7 & 2.8 & $1.91 \times 10^{-4}$ & $3.28 \times 10^{-2}$ \\ 
  monovalent inorganic cation transmembrane transporter activity (GO:0015077) & 117 & 26 & 11 & 2.4 & $1.44 \times 10^{-4}$ & $3.28 \times 10^{-2}$ \\ 
  voltage-gated channel activity (GO:0022832) & 74 & 19 & 7 & 2.8 & $1.91 \times 10^{-4}$ & $3.50 \times 10^{-2}$ \\ 
  gated channel activity (GO:0022836) & 117 & 26 & 11 & 2.4 & $1.44 \times 10^{-4}$ & $3.57 \times 10^{-2}$ \\ 
  channel activity (GO:0015267) & 145 & 30 & 13 & 2.2 & $1.17 \times 10^{-4}$ & $3.57 \times 10^{-2}$ \\ 
  voltage-gated cation channel activity (GO:0022843) & 55 & 16 & 5 & 3.2 & $1.88 \times 10^{-4}$ & $3.68 \times 10^{-2}$ \\ 
  cation channel activity (GO:0005261) & 116 & 26 & 11 & 2.4 & $1.35 \times 10^{-4}$ & $3.70 \times 10^{-2}$ \\ 
  passive transmembrane transporter activity (GO:0022803) & 145 & 30 & 13 & 2.2 & $1.17 \times 10^{-4}$ & $4.02 \times 10^{-2}$ \\ 
  potassium ion transmembrane transporter activity (GO:0015079) & 52 & 15 & 5 & 3.1 & $3.22 \times 10^{-4}$ & $4.90 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis MF 3457 10cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis MF 3457 10cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:44:34 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
sodium channel complex (GO:0034706) & 14 & 8 & 1 & 6.2 & $2.55 \times 10^{-4}$ & $1.67 \times 10^{-2}$ \\ 
  integral component of postsynaptic membrane (GO:0099055) & 44 & 15 & 4 & 3.7 & $6.91 \times 10^{-5}$ & $5.78 \times 10^{-3}$ \\ 
  potassium channel complex (GO:0034705) & 45 & 15 & 4 & 3.6 & $8.53 \times 10^{-5}$ & $6.76 \times 10^{-3}$ \\ 
  intrinsic component of postsynaptic membrane (GO:0098936) & 48 & 16 & 4 & 3.6 & $4.94 \times 10^{-5}$ & $5.31 \times 10^{-3}$ \\ 
  postsynaptic density membrane (GO:0098839) & 41 & 13 & 4 & 3.5 & $3.80 \times 10^{-4}$ & $2.12 \times 10^{-2}$ \\ 
  integral component of synaptic membrane (GO:0099699) & 55 & 17 & 5 & 3.4 & $6.33 \times 10^{-5}$ & $6.35 \times 10^{-3}$ \\ 
  intrinsic component of synaptic membrane (GO:0099240) & 61 & 18 & 6 & 3.2 & $6.39 \times 10^{-5}$ & $6.01 \times 10^{-3}$ \\ 
  cation channel complex (GO:0034703) & 96 & 27 & 9 & 3.1 & $1.97 \times 10^{-6}$ & $4.24 \times 10^{-4}$ \\ 
  ion channel complex (GO:0034702) & 118 & 32 & 11 & 3.0 & $4.16 \times 10^{-7}$ & $1.57 \times 10^{-4}$ \\ 
  transmembrane transporter complex (GO:1902495) & 127 & 32 & 12 & 2.8 & $2.27 \times 10^{-6}$ & $4.28 \times 10^{-4}$ \\ 
  transporter complex (GO:1990351) & 131 & 33 & 12 & 2.7 & $1.52 \times 10^{-6}$ & $4.57 \times 10^{-4}$ \\ 
  postsynaptic membrane (GO:0045211) & 120 & 27 & 11 & 2.5 & $9.16 \times 10^{-5}$ & $6.57 \times 10^{-3}$ \\ 
  synaptic membrane (GO:0097060) & 164 & 33 & 15 & 2.2 & $9.20 \times 10^{-5}$ & $6.29 \times 10^{-3}$ \\ 
  integral component of plasma membrane (GO:0005887) & 389 & 64 & 36 & 1.8 & $1.15 \times 10^{-5}$ & $1.57 \times 10^{-3}$ \\ 
  intrinsic component of plasma membrane (GO:0031226) & 406 & 66 & 37 & 1.8 & $1.11 \times 10^{-5}$ & $1.67 \times 10^{-3}$ \\ 
  integral component of membrane (GO:0016021) & 889 & 128 & 82 & 1.6 & $5.21 \times 10^{-8}$ & $2.62 \times 10^{-5}$ \\ 
  intrinsic component of membrane (GO:0031224) & 926 & 133 & 85 & 1.6 & $2.43 \times 10^{-8}$ & $1.83 \times 10^{-5}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis CC 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis CC 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:44:34 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
intrinsic component of membrane (GO:0031224) & 926 & 133 & 85 & 1.6 & $2.43 \times 10^{-8}$ & $1.83 \times 10^{-5}$ \\ 
  integral component of membrane (GO:0016021) & 889 & 128 & 82 & 1.6 & $5.21 \times 10^{-8}$ & $2.62 \times 10^{-5}$ \\ 
  ion channel complex (GO:0034702) & 118 & 32 & 11 & 3.0 & $4.16 \times 10^{-7}$ & $1.57 \times 10^{-4}$ \\ 
  cation channel complex (GO:0034703) & 96 & 27 & 9 & 3.1 & $1.97 \times 10^{-6}$ & $4.24 \times 10^{-4}$ \\ 
  transmembrane transporter complex (GO:1902495) & 127 & 32 & 12 & 2.8 & $2.27 \times 10^{-6}$ & $4.28 \times 10^{-4}$ \\ 
  transporter complex (GO:1990351) & 131 & 33 & 12 & 2.7 & $1.52 \times 10^{-6}$ & $4.57 \times 10^{-4}$ \\ 
  integral component of plasma membrane (GO:0005887) & 389 & 64 & 36 & 1.8 & $1.15 \times 10^{-5}$ & $1.57 \times 10^{-3}$ \\ 
  intrinsic component of plasma membrane (GO:0031226) & 406 & 66 & 37 & 1.8 & $1.11 \times 10^{-5}$ & $1.67 \times 10^{-3}$ \\ 
  intrinsic component of postsynaptic membrane (GO:0098936) & 48 & 16 & 4 & 3.6 & $4.94 \times 10^{-5}$ & $5.31 \times 10^{-3}$ \\ 
  integral component of postsynaptic membrane (GO:0099055) & 44 & 15 & 4 & 3.7 & $6.91 \times 10^{-5}$ & $5.78 \times 10^{-3}$ \\ 
  intrinsic component of synaptic membrane (GO:0099240) & 61 & 18 & 6 & 3.2 & $6.39 \times 10^{-5}$ & $6.01 \times 10^{-3}$ \\ 
  synaptic membrane (GO:0097060) & 164 & 33 & 15 & 2.2 & $9.20 \times 10^{-5}$ & $6.29 \times 10^{-3}$ \\ 
  integral component of synaptic membrane (GO:0099699) & 55 & 17 & 5 & 3.4 & $6.33 \times 10^{-5}$ & $6.35 \times 10^{-3}$ \\ 
  postsynaptic membrane (GO:0045211) & 120 & 27 & 11 & 2.5 & $9.16 \times 10^{-5}$ & $6.57 \times 10^{-3}$ \\ 
  potassium channel complex (GO:0034705) & 45 & 15 & 4 & 3.6 & $8.53 \times 10^{-5}$ & $6.76 \times 10^{-3}$ \\ 
  sodium channel complex (GO:0034706) & 14 & 8 & 1 & 6.2 & $2.55 \times 10^{-4}$ & $1.67 \times 10^{-2}$ \\ 
  postsynaptic density membrane (GO:0098839) & 41 & 13 & 4 & 3.5 & $3.80 \times 10^{-4}$ & $2.12 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis CC 3457 10cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis CC 3457 10cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}

\clearpage
\subsection{Closeness centrality}

\subsubsection{High closeness centrality}

\ref{tab:GO analysis BP 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis BP 3457 90cent clo.txt Over representation only. Top 20 Terms}

\ref{tab:GO analysis MF 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis MF 3457 90cent clo.txt Over representation only. Top 20 Terms}

\ref{tab:GO analysis CC 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis CC 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

Pathway

\ref{tab:GO analysis pathways 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis pathways 3457 90cent clo.txt Over representation only. Top 20 Terms}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:00:15 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrlrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
mRNA stabilization (GO:0048255) & 14 & 11 & 2 & 5.95 & $4.62 \times 10^{-5}$ & $3.05 \times 10^{-3}$ \\ 
  ERBB2 signalling pathway (GO:0038128) & 12 & 9 & 2 & 5.68 & $2.97 \times 10^{-4}$ & $1.29 \times 10^{-2}$ \\ 
  negative regulation of epidermal growth factor receptor signalling pathway (GO:0042059) & 13 & 9 & 2 & 5.25 & $4.51 \times 10^{-4}$ & $1.81 \times 10^{-2}$ \\ 
  negative regulation of mRNA catabolic process (GO:1902373) & 16 & 11 & 2 & 5.21 & $1.09 \times 10^{-4}$ & $6.24 \times 10^{-3}$ \\ 
  RNA stabilization (GO:0043489) & 17 & 11 & 2 & 4.90 & $1.60 \times 10^{-4}$ & $8.30 \times 10^{-3}$ \\ 
  regulation of telomere maintenance via telomere lengthening (GO:1904356) & 17 & 11 & 2 & 4.90 & $1.60 \times 10^{-4}$ & $8.26 \times 10^{-3}$ \\ 
  negative regulation of ERBB signalling pathway (GO:1901185) & 14 & 9 & 2 & 4.87 & $6.66 \times 10^{-4}$ & $2.49 \times 10^{-2}$ \\ 
  negative regulation of mRNA metabolic process (GO:1903312) & 22 & 14 & 3 & 4.82 & $2.33 \times 10^{-5}$ & $1.79 \times 10^{-3}$ \\ 
  regulation of telomere maintenance via telomerase (GO:0032210) & 16 & 10 & 2 & 4.74 & $3.93 \times 10^{-4}$ & $1.60 \times 10^{-2}$ \\ 
  positive regulation of telomerase activity (GO:0051973) & 13 & 8 & 2 & 4.66 & $1.64 \times 10^{-3}$ & $4.99 \times 10^{-2}$ \\ 
  positive regulation of protein insertion into mitochondrial membrane involved in apoptotic signalling pathway (GO:1900740) & 13 & 8 & 2 & 4.66 & $1.64 \times 10^{-3}$ & $4.98 \times 10^{-2}$ \\ 
  regulation of protein insertion into mitochondrial membrane involved in apoptotic signalling pathway (GO:1900739) & 13 & 8 & 2 & 4.66 & $1.64 \times 10^{-3}$ & $4.97 \times 10^{-2}$ \\ 
  ribosome assembly (GO:0042255) & 20 & 12 & 3 & 4.55 & $1.37 \times 10^{-4}$ & $7.44 \times 10^{-3}$ \\ 
  regulation of defence response to virus by virus (GO:0050690) & 17 & 10 & 2 & 4.46 & $5.60 \times 10^{-4}$ & $2.14 \times 10^{-2}$ \\ 
  negative regulation of RNA catabolic process (GO:1902369) & 19 & 11 & 3 & 4.39 & $3.28 \times 10^{-4}$ & $1.40 \times 10^{-2}$ \\ 
  regulation of telomere maintenance (GO:0032204) & 23 & 13 & 3 & 4.28 & $1.13 \times 10^{-4}$ & $6.40 \times 10^{-3}$ \\ 
  regulation of telomerase activity (GO:0051972) & 16 & 9 & 2 & 4.26 & $1.34 \times 10^{-3}$ & $4.30 \times 10^{-2}$ \\ 
  positive regulation of telomere maintenance (GO:0032206) & 16 & 9 & 2 & 4.26 & $1.34 \times 10^{-3}$ & $4.29 \times 10^{-2}$ \\ 
  positive regulation of mitochondrial outer membrane permeabilization involved in apoptotic signalling pathway (GO:1901030) & 16 & 9 & 2 & 4.26 & $1.34 \times 10^{-3}$ & $4.28 \times 10^{-2}$ \\ 
  protein refolding (GO:0042026) & 16 & 9 & 2 & 4.26 & $1.34 \times 10^{-3}$ & $4.27 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis BP 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis BP 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:00:15 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}


\begin{tabular}{lrrrlrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
interspecies interaction between organisms (GO:0044419) & 553 & 150 & 73 & 2.06 & $1.08 \times 10^{-16}$ & $6.18 \times 10^{-13}$ \\ 
  symbiotic process (GO:0044403) & 313 & 106 & 41 & 2.57 & $7.76 \times 10^{-17}$ & $8.88 \times 10^{-13}$ \\ 
  regulation of macromolecule metabolic process (GO:0060255) & 1258 & 250 & 166 & 1.51 & $3.14 \times 10^{-14}$ & $1.20 \times 10^{-10}$ \\ 
  viral process (GO:0016032) & 285 & 92 & 38 & 2.45 & $1.66 \times 10^{-13}$ & $4.76 \times 10^{-10}$ \\ 
  mRNA metabolic process (GO:0016071) & 198 & 73 & 26 & 2.79 & $4.60 \times 10^{-13}$ & $1.05 \times 10^{-9}$ \\ 
  regulation of protein metabolic process (GO:0051246) & 684 & 158 & 90 & 1.75 & $3.41 \times 10^{-12}$ & $6.50 \times 10^{-9}$ \\ 
  regulation of gene expression (GO:0010468) & 859 & 185 & 113 & 1.63 & $4.20 \times 10^{-12}$ & $6.86 \times 10^{-9}$ \\ 
  negative regulation of macromolecule metabolic process (GO:0010605) & 600 & 143 & 79 & 1.81 & $1.17 \times 10^{-11}$ & $1.67 \times 10^{-8}$ \\ 
  regulation of nitrogen compound metabolic process (GO:0051171) & 1155 & 226 & 152 & 1.48 & $1.55 \times 10^{-11}$ & $1.78 \times 10^{-8}$ \\ 
  regulation of primary metabolic process (GO:0080090) & 1198 & 232 & 158 & 1.47 & $1.53 \times 10^{-11}$ & $1.94 \times 10^{-8}$ \\ 
  regulation of cellular protein metabolic process (GO:0032268) & 638 & 148 & 84 & 1.76 & $2.14 \times 10^{-11}$ & $2.23 \times 10^{-8}$ \\ 
  positive regulation of macromolecule metabolic process (GO:0010604) & 746 & 164 & 98 & 1.67 & $4.48 \times 10^{-11}$ & $3.66 \times 10^{-8}$ \\ 
  regulation of metabolic process (GO:0019222) & 1393 & 257 & 184 & 1.40 & $4.18 \times 10^{-11}$ & $3.68 \times 10^{-8}$ \\ 
  negative regulation of biological process (GO:0048519) & 1245 & 237 & 164 & 1.44 & $4.15 \times 10^{-11}$ & $3.96 \times 10^{-8}$ \\ 
  response to organic substance (GO:0010033) & 824 & 176 & 109 & 1.62 & $5.42 \times 10^{-11}$ & $4.14 \times 10^{-8}$ \\ 
  cellular response to organic substance (GO:0071310) & 649 & 148 & 86 & 1.73 & $6.63 \times 10^{-11}$ & $4.74 \times 10^{-8}$ \\ 
  regulation of cellular macromolecule biosynthetic process (GO:2000112) & 694 & 155 & 92 & 1.69 & $7.79 \times 10^{-11}$ & $5.24 \times 10^{-8}$ \\ 
  regulation of macromolecule biosynthetic process (GO:0010556) & 720 & 159 & 95 & 1.67 & $8.77 \times 10^{-11}$ & $5.28 \times 10^{-8}$ \\ 
  positive regulation of metabolic process (GO:0009893) & 829 & 176 & 109 & 1.61 & $8.64 \times 10^{-11}$ & $5.49 \times 10^{-8}$ \\ 
  macromolecule metabolic process (GO:0043170) & 1257 & 237 & 166 & 1.43 & $1.25 \times 10^{-10}$ & $7.12 \times 10^{-8}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis BP 3457 90cent clo.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis BP 3457 90cent clo.txt Over representation only. Top 20 Terms}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:02:39 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
protein folding chaperone (GO:0044183) & 14 & 10 & 2 & 5.4 & $1.80 \times 10^{-4}$ & $2.06 \times 10^{-2}$ \\ 
  disordered domain specific binding (GO:0097718) & 18 & 12 & 2 & 5.0 & $6.52 \times 10^{-5}$ & $8.50 \times 10^{-3}$ \\ 
  Hsp70 protein binding (GO:0030544) & 16 & 10 & 2 & 4.7 & $3.93 \times 10^{-4}$ & $3.99 \times 10^{-2}$ \\ 
  protein phosphorylated amino acid binding (GO:0045309) & 16 & 10 & 2 & 4.7 & $3.93 \times 10^{-4}$ & $3.84 \times 10^{-2}$ \\ 
  phosphoprotein binding (GO:0051219) & 30 & 17 & 4 & 4.3 & $9.73 \times 10^{-6}$ & $1.67 \times 10^{-3}$ \\ 
  mRNA 3'-UTR binding (GO:0003730) & 30 & 17 & 4 & 4.3 & $9.73 \times 10^{-6}$ & $1.57 \times 10^{-3}$ \\ 
  single-stranded RNA binding (GO:0003727) & 25 & 14 & 3 & 4.2 & $6.64 \times 10^{-5}$ & $8.26 \times 10^{-3}$ \\ 
  unfolded protein binding (GO:0051082) & 45 & 23 & 6 & 3.9 & $1.02 \times 10^{-6}$ & $2.53 \times 10^{-4}$ \\ 
  heat shock protein binding (GO:0031072) & 49 & 25 & 6 & 3.9 & $3.49 \times 10^{-7}$ & $1.06 \times 10^{-4}$ \\ 
  ubiquitin protein ligase binding (GO:0031625) & 96 & 45 & 13 & 3.5 & $4.93 \times 10^{-11}$ & $2.70 \times 10^{-8}$ \\ 
  tau protein binding (GO:0048156) & 32 & 15 & 4 & 3.5 & $1.71 \times 10^{-4}$ & $2.03 \times 10^{-2}$ \\ 
  chaperone binding (GO:0051087) & 45 & 21 & 6 & 3.5 & $9.13 \times 10^{-6}$ & $1.67 \times 10^{-3}$ \\ 
  ubiquitin-like protein ligase binding (GO:0044389) & 99 & 46 & 13 & 3.5 & $3.73 \times 10^{-11}$ & $2.55 \times 10^{-8}$ \\ 
  histone deacetylase binding (GO:0042826) & 28 & 13 & 4 & 3.5 & $4.97 \times 10^{-4}$ & $4.39 \times 10^{-2}$ \\ 
  double-stranded RNA binding (GO:0003725) & 31 & 14 & 4 & 3.4 & $3.78 \times 10^{-4}$ & $3.99 \times 10^{-2}$ \\ 
  mRNA binding (GO:0003729) & 87 & 37 & 11 & 3.2 & $2.27 \times 10^{-8}$ & $1.03 \times 10^{-5}$ \\ 
  structural constituent of ribosome (GO:0003735) & 72 & 23 & 10 & 2.4 & $4.89 \times 10^{-4}$ & $4.47 \times 10^{-2}$ \\ 
  RNA binding (GO:0003723) & 452 & 135 & 60 & 2.3 & $8.27 \times 10^{-18}$ & $2.27 \times 10^{-14}$ \\ 
  transcription factor binding (GO:0008134) & 149 & 41 & 20 & 2.1 & $6.13 \times 10^{-5}$ & $8.39 \times 10^{-3}$ \\ 
  protein kinase binding (GO:0019901) & 210 & 56 & 28 & 2.0 & $5.97 \times 10^{-6}$ & $1.17 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis MF 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis MF 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:02:39 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
RNA binding (GO:0003723) & 452 & 135 & 60 & 2.3 & $8.27 \times 10^{-18}$ & $2.27 \times 10^{-14}$ \\ 
  enzyme binding (GO:0019899) & 641 & 156 & 85 & 1.8 & $1.06 \times 10^{-13}$ & $1.46 \times 10^{-10}$ \\ 
  nucleic acid binding (GO:0003676) & 724 & 168 & 96 & 1.8 & $2.72 \times 10^{-13}$ & $2.48 \times 10^{-10}$ \\ 
  ubiquitin-like protein ligase binding (GO:0044389) & 99 & 46 & 13 & 3.5 & $3.73 \times 10^{-11}$ & $2.55 \times 10^{-8}$ \\ 
  ubiquitin protein ligase binding (GO:0031625) & 96 & 45 & 13 & 3.5 & $4.93 \times 10^{-11}$ & $2.70 \times 10^{-8}$ \\ 
  heterocyclic compound binding (GO:1901363) & 1270 & 229 & 168 & 1.4 & $2.42 \times 10^{-8}$ & $9.48 \times 10^{-6}$ \\ 
  mRNA binding (GO:0003729) & 87 & 37 & 11 & 3.2 & $2.27 \times 10^{-8}$ & $1.03 \times 10^{-5}$ \\ 
  organic cyclic compound binding (GO:0097159) & 1288 & 231 & 170 & 1.4 & $3.42 \times 10^{-8}$ & $1.17 \times 10^{-5}$ \\ 
  heat shock protein binding (GO:0031072) & 49 & 25 & 6 & 3.9 & $3.49 \times 10^{-7}$ & $1.06 \times 10^{-4}$ \\ 
  protein-containing complex binding (GO:0044877) & 422 & 97 & 56 & 1.7 & $4.27 \times 10^{-7}$ & $1.17 \times 10^{-4}$ \\ 
  unfolded protein binding (GO:0051082) & 45 & 23 & 6 & 3.9 & $1.02 \times 10^{-6}$ & $2.53 \times 10^{-4}$ \\ 
  protein domain specific binding (GO:0019904) & 242 & 62 & 32 & 1.9 & $3.97 \times 10^{-6}$ & $9.05 \times 10^{-4}$ \\ 
  kinase binding (GO:0019900) & 235 & 61 & 31 & 2.0 & $4.39 \times 10^{-6}$ & $9.24 \times 10^{-4}$ \\ 
  protein kinase binding (GO:0019901) & 210 & 56 & 28 & 2.0 & $5.97 \times 10^{-6}$ & $1.17 \times 10^{-3}$ \\ 
  mRNA 3'-UTR binding (GO:0003730) & 30 & 17 & 4 & 4.3 & $9.73 \times 10^{-6}$ & $1.57 \times 10^{-3}$ \\ 
  phosphoprotein binding (GO:0051219) & 30 & 17 & 4 & 4.3 & $9.73 \times 10^{-6}$ & $1.67 \times 10^{-3}$ \\ 
  chaperone binding (GO:0051087) & 45 & 21 & 6 & 3.5 & $9.13 \times 10^{-6}$ & $1.67 \times 10^{-3}$ \\ 
  signaling receptor binding (GO:0005102) & 421 & 89 & 56 & 1.6 & $2.92 \times 10^{-5}$ & $4.45 \times 10^{-3}$ \\ 
  protein binding (GO:0005515) & 2580 & 377 & 340 & 1.1 & $3.61 \times 10^{-5}$ & $5.21 \times 10^{-3}$ \\ 
  single-stranded RNA binding (GO:0003727) & 25 & 14 & 3 & 4.2 & $6.64 \times 10^{-5}$ & $8.26 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis MF 3457 90cent clo.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis MF 3457 90cent clo.txt Over representation only. Top 20 Terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:04:29 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
chaperone complex (GO:0101031) & 19 & 13 & 3 & 5.2 & $2.64 \times 10^{-5}$ & $1.59 \times 10^{-3}$ \\ 
  postsynaptic specialization, intracellular component (GO:0099091) & 17 & 10 & 2 & 4.5 & $5.60 \times 10^{-4}$ & $2.34 \times 10^{-2}$ \\ 
  COP9 signalosome (GO:0008180) & 20 & 10 & 3 & 3.8 & $1.44 \times 10^{-3}$ & $4.94 \times 10^{-2}$ \\ 
  catalytic step 2 spliceosome (GO:0071013) & 32 & 16 & 4 & 3.8 & $5.72 \times 10^{-5}$ & $3.19 \times 10^{-3}$ \\ 
  cytosolic small ribosomal subunit (GO:0022627) & 28 & 13 & 4 & 3.5 & $4.97 \times 10^{-4}$ & $2.14 \times 10^{-2}$ \\ 
  cytoplasmic ribonucleoprotein granule (GO:0036464) & 59 & 26 & 8 & 3.3 & $1.80 \times 10^{-6}$ & $1.29 \times 10^{-4}$ \\ 
  ribonucleoprotein granule (GO:0035770) & 62 & 27 & 8 & 3.3 & $1.36 \times 10^{-6}$ & $1.02 \times 10^{-4}$ \\ 
  small ribosomal subunit (GO:0015935) & 30 & 13 & 4 & 3.3 & $8.29 \times 10^{-4}$ & $3.28 \times 10^{-2}$ \\ 
  spliceosomal complex (GO:0005681) & 50 & 21 & 7 & 3.2 & $3.20 \times 10^{-5}$ & $1.85 \times 10^{-3}$ \\ 
  melanosome (GO:0042470) & 46 & 17 & 6 & 2.8 & $9.84 \times 10^{-4}$ & $3.80 \times 10^{-2}$ \\ 
  pigment granule (GO:0048770) & 46 & 17 & 6 & 2.8 & $9.84 \times 10^{-4}$ & $3.70 \times 10^{-2}$ \\ 
  ribonucleoprotein complex (GO:1990904) & 208 & 74 & 27 & 2.7 & $1.38 \times 10^{-12}$ & $1.04 \times 10^{-9}$ \\ 
  cytosolic ribosome (GO:0022626) & 68 & 23 & 9 & 2.6 & $2.12 \times 10^{-4}$ & $1.07 \times 10^{-2}$ \\ 
  focal adhesion (GO:0005925) & 212 & 71 & 28 & 2.5 & $5.62 \times 10^{-11}$ & $1.69 \times 10^{-8}$ \\ 
  cell-substrate junction (GO:0030055) & 217 & 72 & 29 & 2.5 & $4.85 \times 10^{-11}$ & $1.82 \times 10^{-8}$ \\ 
  ribosomal subunit (GO:0044391) & 75 & 23 & 10 & 2.3 & $1.10 \times 10^{-3}$ & $4.03 \times 10^{-2}$ \\ 
  ribosome (GO:0005840) & 90 & 27 & 12 & 2.3 & $3.52 \times 10^{-4}$ & $1.61 \times 10^{-2}$ \\ 
  nuclear speck (GO:0016607) & 93 & 26 & 12 & 2.1 & $1.19 \times 10^{-3}$ & $4.26 \times 10^{-2}$ \\ 
  anchoring junction (GO:0070161) & 356 & 88 & 47 & 1.9 & $9.55 \times 10^{-8}$ & $9.58 \times 10^{-6}$ \\ 
  nucleolus (GO:0005730) & 189 & 45 & 25 & 1.8 & $4.67 \times 10^{-4}$ & $2.07 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis CC 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis CC 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}




% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:04:29 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
protein-containing complex (GO:0032991) & 1403 & 267 & 185 & 1.4 & $1.63 \times 10^{-13}$ & $2.45 \times 10^{-10}$ \\ 
  ribonucleoprotein complex (GO:1990904) & 208 & 74 & 27 & 2.7 & $1.38 \times 10^{-12}$ & $1.04 \times 10^{-9}$ \\ 
  nucleus (GO:0005634) & 1359 & 255 & 179 & 1.4 & $8.75 \times 10^{-12}$ & $4.39 \times 10^{-9}$ \\ 
  focal adhesion (GO:0005925) & 212 & 71 & 28 & 2.5 & $5.62 \times 10^{-11}$ & $1.69 \times 10^{-8}$ \\ 
  cell-substrate junction (GO:0030055) & 217 & 72 & 29 & 2.5 & $4.85 \times 10^{-11}$ & $1.82 \times 10^{-8}$ \\ 
  cytosol (GO:0005829) & 1430 & 260 & 189 & 1.4 & $1.32 \times 10^{-10}$ & $3.32 \times 10^{-8}$ \\ 
  non-membrane-bounded organelle (GO:0043228) & 1105 & 215 & 146 & 1.5 & $1.78 \times 10^{-10}$ & $3.82 \times 10^{-8}$ \\ 
  extracellular exosome (GO:0070062) & 759 & 162 & 100 & 1.6 & $5.75 \times 10^{-10}$ & $1.08 \times 10^{-7}$ \\ 
  intracellular non-membrane-bounded organelle (GO:0043232) & 1098 & 212 & 145 & 1.5 & $6.54 \times 10^{-10}$ & $1.09 \times 10^{-7}$ \\ 
  extracellular vesicle (GO:1903561) & 771 & 163 & 102 & 1.6 & $1.01 \times 10^{-9}$ & $1.38 \times 10^{-7}$ \\ 
  extracellular organelle (GO:0043230) & 771 & 163 & 102 & 1.6 & $1.01 \times 10^{-9}$ & $1.52 \times 10^{-7}$ \\ 
  extracellular space (GO:0005615) & 966 & 190 & 127 & 1.5 & $3.05 \times 10^{-9}$ & $3.82 \times 10^{-7}$ \\ 
  nucleoplasm (GO:0005654) & 761 & 156 & 100 & 1.6 & $2.15 \times 10^{-8}$ & $2.49 \times 10^{-6}$ \\ 
  extracellular region (GO:0005576) & 1132 & 208 & 149 & 1.4 & $7.14 \times 10^{-8}$ & $7.67 \times 10^{-6}$ \\ 
  nuclear lumen (GO:0031981) & 894 & 173 & 118 & 1.5 & $9.99 \times 10^{-8}$ & $9.39 \times 10^{-6}$ \\ 
  anchoring junction (GO:0070161) & 356 & 88 & 47 & 1.9 & $9.55 \times 10^{-8}$ & $9.58 \times 10^{-6}$ \\ 
  membrane-enclosed lumen (GO:0031974) & 1179 & 211 & 156 & 1.4 & $3.85 \times 10^{-7}$ & $3.05 \times 10^{-5}$ \\ 
  intracellular organelle lumen (GO:0070013) & 1179 & 211 & 156 & 1.4 & $3.85 \times 10^{-7}$ & $3.22 \times 10^{-5}$ \\ 
  organelle lumen (GO:0043233) & 1179 & 211 & 156 & 1.4 & $3.85 \times 10^{-7}$ & $3.41 \times 10^{-5}$ \\ 
  ribonucleoprotein granule (GO:0035770) & 62 & 27 & 8 & 3.3 & $1.36 \times 10^{-6}$ & $1.02 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis CC 3457 90cent clo.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis CC 3457 90cent clo.txt Over representation only. Top 20 Terms}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:06:16 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER Pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
Parkinson disease (P00049) & 45 & 26 & 6 & 4.4 & $3.06 \times 10^{-8}$ & $4.09 \times 10^{-6}$ \\ 
  Cytoskeletal regulation by Rho GTPase (P00016) & 36 & 14 & 5 & 3.0 & $1.23 \times 10^{-3}$ & $3.29 \times 10^{-2}$ \\ 
  CCKR signaling map (P06959) & 70 & 24 & 9 & 2.6 & $1.31 \times 10^{-4}$ & $5.86 \times 10^{-3}$ \\ 
  FGF signaling pathway (P00021) & 59 & 19 & 8 & 2.4 & $1.82 \times 10^{-3}$ & $4.07 \times 10^{-2}$ \\ 
  Integrin signalling pathway (P00034) & 85 & 27 & 11 & 2.4 & $1.48 \times 10^{-4}$ & $4.97 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis pathways 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis pathways 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:06:16 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER Pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
Parkinson disease (P00049) & 45 & 26 & 6 & 4.4 & $3.06 \times 10^{-8}$ & $4.09 \times 10^{-6}$ \\ 
  Integrin signalling pathway (P00034) & 85 & 27 & 11 & 2.4 & $1.48 \times 10^{-4}$ & $4.97 \times 10^{-3}$ \\ 
  CCKR signaling map (P06959) & 70 & 24 & 9 & 2.6 & $1.31 \times 10^{-4}$ & $5.86 \times 10^{-3}$ \\ 
  Cytoskeletal regulation by Rho GTPase (P00016) & 36 & 14 & 5 & 3.0 & $1.23 \times 10^{-3}$ & $3.29 \times 10^{-2}$ \\ 
  FGF signaling pathway (P00021) & 59 & 19 & 8 & 2.4 & $1.82 \times 10^{-3}$ & $4.07 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis pathways 3457 90cent clo.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis pathways 3457 90cent clo.txt Over representation only. Top 20 Terms}
\end{table}

\subsubsection{Low closeness} 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\ref{tab:GO analysis BP 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis BP 3457 10cent clo.txt Over representation only. Top 20 Terms}

Glutamate
\ref{tab:GO analysis MF 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis MF 3457 10cent clo.txt Over representation only. Top 20 Terms}

\ref{tab:GO analysis CC 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO analysis CC 3457 10cent clo.txt Over representation only. Top 20 Terms}

\ref{tab:GO analysis pathway 3457 10cent clo.txt Over representation only. Top 20 Terms}

\ref{tab:GO analysis pathway 3457 10cent clo.txt Over representation only. Top 20 Terms}



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:08:21 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}


\begin{tabular}{lrrrrrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
regulation of presynaptic membrane potential (GO:0099505) & 6 & 3 & 0 & 35.1 & $2.16 \times 10^{-4}$ & $2.47 \times 10^{-2}$ \\ 
  adenylate cyclase-inhibiting G protein-coupled glutamate receptor signaling pathway (GO:0007196) & 7 & 3 & 0 & 30.1 & $3.06 \times 10^{-4}$ & $3.15 \times 10^{-2}$ \\ 
  xenobiotic catabolic process (GO:0042178) & 12 & 4 & 0 & 23.4 & $6.07 \times 10^{-5}$ & $1.01 \times 10^{-2}$ \\ 
  neuronal action potential propagation (GO:0019227) & 9 & 3 & 0 & 23.4 & $5.49 \times 10^{-4}$ & $4.69 \times 10^{-2}$ \\ 
  action potential propagation (GO:0098870) & 9 & 3 & 0 & 23.4 & $5.49 \times 10^{-4}$ & $4.66 \times 10^{-2}$ \\ 
  relaxation of muscle (GO:0090075) & 21 & 4 & 0 & 13.4 & $3.82 \times 10^{-4}$ & $3.67 \times 10^{-2}$ \\ 
  protein import into peroxisome membrane (GO:0045046) & 22 & 4 & 0 & 12.8 & $4.47 \times 10^{-4}$ & $4.14 \times 10^{-2}$ \\ 
  response to lead ion (GO:0010288) & 23 & 4 & 0 & 12.2 & $5.18 \times 10^{-4}$ & $4.45 \times 10^{-2}$ \\ 
  membrane depolarization during action potential (GO:0086010) & 30 & 5 & 0 & 11.7 & $1.21 \times 10^{-4}$ & $1.64 \times 10^{-2}$ \\ 
  acylglycerol catabolic process (GO:0046464) & 32 & 5 & 0 & 11.0 & $1.59 \times 10^{-4}$ & $2.00 \times 10^{-2}$ \\ 
  neutral lipid catabolic process (GO:0046461) & 32 & 5 & 0 & 11.0 & $1.59 \times 10^{-4}$ & $1.99 \times 10^{-2}$ \\ 
  membrane depolarization (GO:0051899) & 52 & 8 & 1 & 10.8 & $1.87 \times 10^{-6}$ & $6.94 \times 10^{-4}$ \\ 
  synaptic transmission, glutamatergic (GO:0035249) & 40 & 6 & 1 & 10.5 & $4.26 \times 10^{-5}$ & $7.47 \times 10^{-3}$ \\ 
  cardiac muscle cell contraction (GO:0086003) & 42 & 6 & 1 & 10.0 & $5.45 \times 10^{-5}$ & $9.36 \times 10^{-3}$ \\ 
  cardiac muscle cell action potential (GO:0086001) & 50 & 7 & 1 & 9.8 & $1.46 \times 10^{-5}$ & $3.52 \times 10^{-3}$ \\ 
  glutamate receptor signaling pathway (GO:0007215) & 48 & 6 & 1 & 8.8 & $1.07 \times 10^{-4}$ & $1.50 \times 10^{-2}$ \\ 
  membrane biogenesis (GO:0044091) & 42 & 5 & 1 & 8.4 & $5.00 \times 10^{-4}$ & $4.41 \times 10^{-2}$ \\ 
  receptor localization to synapse (GO:0097120) & 42 & 5 & 1 & 8.4 & $5.00 \times 10^{-4}$ & $4.38 \times 10^{-2}$ \\ 
  regulation of synaptic transmission, glutamatergic (GO:0051966) & 71 & 8 & 1 & 7.9 & $1.51 \times 10^{-5}$ & $3.61 \times 10^{-3}$ \\ 
  negative regulation of synaptic transmission (GO:0050805) & 71 & 8 & 1 & 7.9 & $1.51 \times 10^{-5}$ & $3.55 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis BP 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis BP 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:08:21 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO biological process complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
cellular process (GO:0009987) & 15458 & 272 & 220 & 1.2 & $3.90 \times 10^{-14}$ & $6.22 \times 10^{-10}$ \\ 
  trans-synaptic signaling (GO:0099537) & 445 & 31 & 6 & 4.9 & $1.39 \times 10^{-12}$ & $5.56 \times 10^{-9}$ \\ 
  regulation of membrane potential (GO:0042391) & 441 & 31 & 6 & 4.9 & $1.12 \times 10^{-12}$ & $5.95 \times 10^{-9}$ \\ 
  anterograde trans-synaptic signaling (GO:0098916) & 426 & 30 & 6 & 4.9 & $2.54 \times 10^{-12}$ & $6.75 \times 10^{-9}$ \\ 
  regulation of biological quality (GO:0065008) & 4134 & 113 & 59 & 1.9 & $8.52 \times 10^{-13}$ & $6.80 \times 10^{-9}$ \\ 
  chemical synaptic transmission (GO:0007268) & 426 & 30 & 6 & 4.9 & $2.54 \times 10^{-12}$ & $8.09 \times 10^{-9}$ \\ 
  synaptic signaling (GO:0099536) & 470 & 31 & 7 & 4.6 & $5.25 \times 10^{-12}$ & $1.20 \times 10^{-8}$ \\ 
  localization (GO:0051179) & 5802 & 138 & 83 & 1.7 & $1.68 \times 10^{-11}$ & $2.67 \times 10^{-8}$ \\ 
  regulation of trans-synaptic signaling (GO:0099177) & 460 & 30 & 7 & 4.6 & $1.54 \times 10^{-11}$ & $2.73 \times 10^{-8}$ \\ 
  modulation of chemical synaptic transmission (GO:0050804) & 459 & 30 & 7 & 4.6 & $1.46 \times 10^{-11}$ & $2.92 \times 10^{-8}$ \\ 
  transport (GO:0006810) & 4550 & 116 & 65 & 1.8 & $2.80 \times 10^{-11}$ & $4.07 \times 10^{-8}$ \\ 
  establishment of localization (GO:0051234) & 4680 & 118 & 67 & 1.8 & $4.23 \times 10^{-11}$ & $5.62 \times 10^{-8}$ \\ 
  multicellular organismal process (GO:0032501) & 7037 & 154 & 100 & 1.5 & $2.05 \times 10^{-10}$ & $2.52 \times 10^{-7}$ \\ 
  system development (GO:0048731) & 4460 & 111 & 64 & 1.8 & $5.76 \times 10^{-10}$ & $6.57 \times 10^{-7}$ \\ 
  nervous system development (GO:0007399) & 2395 & 72 & 34 & 2.1 & $1.36 \times 10^{-9}$ & $1.45 \times 10^{-6}$ \\ 
  multicellular organism development (GO:0007275) & 5080 & 120 & 72 & 1.7 & $1.52 \times 10^{-9}$ & $1.51 \times 10^{-6}$ \\ 
  ion transport (GO:0006811) & 1342 & 48 & 19 & 2.5 & $7.86 \times 10^{-9}$ & $7.38 \times 10^{-6}$ \\ 
  anatomical structure development (GO:0048856) & 5469 & 124 & 78 & 1.6 & $8.71 \times 10^{-9}$ & $7.73 \times 10^{-6}$ \\ 
  regulation of ion transport (GO:0043269) & 705 & 32 & 10 & 3.2 & $1.69 \times 10^{-8}$ & $1.42 \times 10^{-5}$ \\ 
  cell-cell signaling (GO:0007267) & 1175 & 43 & 17 & 2.6 & $2.35 \times 10^{-8}$ & $1.87 \times 10^{-5}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis BP 3457 10cent clo.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis BP 3457 10cent clo.txt Over representation only. Top 20 Terms}
\end{table}


% MF %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LOW CLO %%%%%%%%%%%%%%%%%%%%%

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:22:08 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
kainate selective glutamate receptor activity (GO:0015277) & 5 & 3 & 0 & 42.1 & $1.46 \times 10^{-4}$ & $2.03 \times 10^{-2}$ \\ 
  adenylate cyclase inhibiting G protein-coupled glutamate receptor activity (GO:0001640) & 7 & 3 & 0 & 30.1 & $3.06 \times 10^{-4}$ & $3.53 \times 10^{-2}$ \\ 
  G protein-coupled glutamate receptor activity (GO:0098988) & 7 & 3 & 0 & 30.1 & $3.06 \times 10^{-4}$ & $3.44 \times 10^{-2}$ \\ 
  glutamate receptor activity (GO:0008066) & 28 & 6 & 0 & 15.0 & $7.05 \times 10^{-6}$ & $1.45 \times 10^{-3}$ \\ 
  ionotropic glutamate receptor activity (GO:0004970) & 20 & 4 & 0 & 14.0 & $3.24 \times 10^{-4}$ & $3.49 \times 10^{-2}$ \\ 
  sodium channel activity (GO:0005272) & 45 & 7 & 1 & 10.9 & $7.82 \times 10^{-6}$ & $1.54 \times 10^{-3}$ \\ 
  oxidoreductase activity, acting on the aldehyde or oxo group of donors (GO:0016903) & 44 & 6 & 1 & 9.6 & $6.90 \times 10^{-5}$ & $1.02 \times 10^{-2}$ \\ 
  ATP-dependent microtubule motor activity (GO:1990939) & 38 & 5 & 1 & 9.2 & $3.28 \times 10^{-4}$ & $3.45 \times 10^{-2}$ \\ 
  voltage-gated potassium channel activity (GO:0005249) & 89 & 8 & 1 & 6.3 & $6.75 \times 10^{-5}$ & $1.03 \times 10^{-2}$ \\ 
  potassium channel activity (GO:0005267) & 123 & 11 & 2 & 6.3 & $3.10 \times 10^{-6}$ & $6.66 \times 10^{-4}$ \\ 
  voltage-gated cation channel activity (GO:0022843) & 141 & 12 & 2 & 6.0 & $1.79 \times 10^{-6}$ & $4.02 \times 10^{-4}$ \\ 
  sodium ion transmembrane transporter activity (GO:0015081) & 158 & 13 & 2 & 5.8 & $9.44 \times 10^{-7}$ & $2.48 \times 10^{-4}$ \\ 
  voltage-gated ion channel activity (GO:0005244) & 198 & 15 & 3 & 5.3 & $3.69 \times 10^{-7}$ & $1.09 \times 10^{-4}$ \\ 
  voltage-gated channel activity (GO:0022832) & 199 & 15 & 3 & 5.3 & $3.92 \times 10^{-7}$ & $1.09 \times 10^{-4}$ \\ 
  ion channel regulator activity (GO:0099106) & 120 & 9 & 2 & 5.3 & $8.85 \times 10^{-5}$ & $1.27 \times 10^{-2}$ \\ 
  potassium ion transmembrane transporter activity (GO:0015079) & 159 & 11 & 2 & 4.9 & $2.98 \times 10^{-5}$ & $5.22 \times 10^{-3}$ \\ 
  motor activity (GO:0003774) & 135 & 9 & 2 & 4.7 & $2.05 \times 10^{-4}$ & $2.69 \times 10^{-2}$ \\ 
  metal ion transmembrane transporter activity (GO:0046873) & 439 & 28 & 6 & 4.5 & $1.23 \times 10^{-10}$ & $5.82 \times 10^{-7}$ \\ 
  channel regulator activity (GO:0016247) & 144 & 9 & 2 & 4.4 & $3.21 \times 10^{-4}$ & $3.53 \times 10^{-2}$ \\ 
  carboxylic ester hydrolase activity (GO:0052689) & 145 & 9 & 2 & 4.4 & $3.37 \times 10^{-4}$ & $3.46 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis MF 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis MF 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:22:08 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO molecular function complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
metal ion transmembrane transporter activity (GO:0046873) & 439 & 28 & 6 & 4.5 & $1.23 \times 10^{-10}$ & $5.82 \times 10^{-7}$ \\ 
  inorganic cation transmembrane transporter activity (GO:0022890) & 610 & 31 & 9 & 3.6 & $2.35 \times 10^{-9}$ & $5.54 \times 10^{-6}$ \\ 
  cation transmembrane transporter activity (GO:0008324) & 652 & 31 & 9 & 3.3 & $1.04 \times 10^{-8}$ & $1.64 \times 10^{-5}$ \\ 
  inorganic molecular entity transmembrane transporter activity (GO:0015318) & 833 & 35 & 12 & 3.0 & $2.22 \times 10^{-8}$ & $2.62 \times 10^{-5}$ \\ 
  passive transmembrane transporter activity (GO:0022803) & 485 & 25 & 7 & 3.6 & $6.94 \times 10^{-8}$ & $3.65 \times 10^{-5}$ \\ 
  cation channel activity (GO:0005261) & 344 & 21 & 5 & 4.3 & $5.66 \times 10^{-8}$ & $3.82 \times 10^{-5}$ \\ 
  channel activity (GO:0015267) & 484 & 25 & 7 & 3.6 & $6.69 \times 10^{-8}$ & $3.95 \times 10^{-5}$ \\ 
  gated channel activity (GO:0022836) & 342 & 21 & 5 & 4.3 & $5.15 \times 10^{-8}$ & $4.06 \times 10^{-5}$ \\ 
  ion transmembrane transporter activity (GO:0015075) & 887 & 35 & 13 & 2.8 & $9.72 \times 10^{-8}$ & $4.60 \times 10^{-5}$ \\ 
  ion channel activity (GO:0005216) & 441 & 24 & 6 & 3.8 & $4.92 \times 10^{-8}$ & $4.65 \times 10^{-5}$ \\ 
  small molecule binding (GO:0036094) & 2592 & 70 & 37 & 1.9 & $1.40 \times 10^{-7}$ & $5.51 \times 10^{-5}$ \\ 
  monovalent inorganic cation transmembrane transporter activity (GO:0015077) & 400 & 22 & 6 & 3.9 & $1.52 \times 10^{-7}$ & $5.54 \times 10^{-5}$ \\ 
  anion binding (GO:0043168) & 2865 & 75 & 41 & 1.8 & $1.73 \times 10^{-7}$ & $5.82 \times 10^{-5}$ \\ 
  transmembrane transporter activity (GO:0022857) & 1074 & 39 & 15 & 2.5 & $1.38 \times 10^{-7}$ & $5.94 \times 10^{-5}$ \\ 
  transporter activity (GO:0005215) & 1235 & 42 & 18 & 2.4 & $2.51 \times 10^{-7}$ & $7.92 \times 10^{-5}$ \\ 
  voltage-gated ion channel activity (GO:0005244) & 198 & 15 & 3 & 5.3 & $3.69 \times 10^{-7}$ & $1.09 \times 10^{-4}$ \\ 
  voltage-gated channel activity (GO:0022832) & 199 & 15 & 3 & 5.3 & $3.92 \times 10^{-7}$ & $1.09 \times 10^{-4}$ \\ 
  sodium ion transmembrane transporter activity (GO:0015081) & 158 & 13 & 2 & 5.8 & $9.44 \times 10^{-7}$ & $2.48 \times 10^{-4}$ \\ 
  voltage-gated cation channel activity (GO:0022843) & 141 & 12 & 2 & 6.0 & $1.79 \times 10^{-6}$ & $4.02 \times 10^{-4}$ \\ 
  molecular\_function (GO:0003674) & 17682 & 279 & 252 & 1.1 & $1.76 \times 10^{-6}$ & $4.38 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis MF 3457 10cent clo.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis MF 3457 10cent clo.txt Over representation only. Top 20 Terms}
\end{table}



% CC %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LOW CLO %%%%%%%%%%%%%%%%%%%%%

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:24:18 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
kainate selective glutamate receptor complex (GO:0032983) & 5 & 3 & 0 & 42.1 & $1.46 \times 10^{-4}$ & $3.95 \times 10^{-3}$ \\ 
  node of Ranvier (GO:0033268) & 15 & 6 & 0 & 28.1 & $3.31 \times 10^{-7}$ & $1.62 \times 10^{-5}$ \\ 
  sodium channel complex (GO:0034706) & 26 & 7 & 0 & 18.9 & $3.14 \times 10^{-7}$ & $1.57 \times 10^{-5}$ \\ 
  voltage-gated sodium channel complex (GO:0001518) & 17 & 4 & 0 & 16.5 & $1.89 \times 10^{-4}$ & $5.06 \times 10^{-3}$ \\ 
  axon initial segment (GO:0043194) & 19 & 4 & 0 & 14.8 & $2.73 \times 10^{-4}$ & $7.04 \times 10^{-3}$ \\ 
  synaptic cleft (GO:0043083) & 19 & 4 & 0 & 14.8 & $2.73 \times 10^{-4}$ & $6.95 \times 10^{-3}$ \\ 
  hippocampal mossy fiber to CA3 synapse (GO:0098686) & 35 & 6 & 0 & 12.0 & $2.17 \times 10^{-5}$ & $6.70 \times 10^{-4}$ \\ 
  ionotropic glutamate receptor complex (GO:0008328) & 52 & 7 & 1 & 9.4 & $1.83 \times 10^{-5}$ & $6.03 \times 10^{-4}$ \\ 
  T-tubule (GO:0030315) & 52 & 7 & 1 & 9.4 & $1.83 \times 10^{-5}$ & $5.93 \times 10^{-4}$ \\ 
  excitatory synapse (GO:0060076) & 53 & 7 & 1 & 9.3 & $2.05 \times 10^{-5}$ & $6.43 \times 10^{-4}$ \\ 
  neurotransmitter receptor complex (GO:0098878) & 54 & 7 & 1 & 9.1 & $2.29 \times 10^{-5}$ & $6.96 \times 10^{-4}$ \\ 
  potassium channel complex (GO:0034705) & 98 & 12 & 1 & 8.6 & $4.87 \times 10^{-8}$ & $3.15 \times 10^{-6}$ \\ 
  dendrite cytoplasm (GO:0032839) & 33 & 4 & 0 & 8.5 & $1.75 \times 10^{-3}$ & $3.58 \times 10^{-2}$ \\ 
  main axon (GO:0044304) & 71 & 8 & 1 & 7.9 & $1.51 \times 10^{-5}$ & $5.07 \times 10^{-4}$ \\ 
  intrinsic component of postsynaptic membrane (GO:0098936) & 128 & 14 & 2 & 7.7 & $1.37 \times 10^{-8}$ & $1.10 \times 10^{-6}$ \\ 
  microbody membrane (GO:0031903) & 65 & 7 & 1 & 7.6 & $6.78 \times 10^{-5}$ & $1.92 \times 10^{-3}$ \\ 
  peroxisomal membrane (GO:0005778) & 65 & 7 & 1 & 7.6 & $6.78 \times 10^{-5}$ & $1.89 \times 10^{-3}$ \\ 
  postsynaptic density membrane (GO:0098839) & 103 & 11 & 1 & 7.5 & $6.22 \times 10^{-7}$ & $2.71 \times 10^{-5}$ \\ 
  integral component of postsynaptic membrane (GO:0099055) & 122 & 13 & 2 & 7.5 & $6.03 \times 10^{-8}$ & $3.78 \times 10^{-6}$ \\ 
  terminal bouton (GO:0043195) & 57 & 6 & 1 & 7.4 & $2.53 \times 10^{-4}$ & $6.60 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis CC 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis CC 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:24:18 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
GO cellular component complete & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
synapse (GO:0045202) & 1375 & 70 & 20 & 3.6 & $2.12 \times 10^{-20}$ & $4.26 \times 10^{-17}$ \\ 
  cell junction (GO:0030054) & 2107 & 85 & 30 & 2.8 & $9.96 \times 10^{-19}$ & $1.00 \times 10^{-15}$ \\ 
  neuron projection (GO:0043005) & 1384 & 62 & 20 & 3.1 & $1.68 \times 10^{-15}$ & $1.12 \times 10^{-12}$ \\ 
  ion channel complex (GO:0034702) & 305 & 27 & 4 & 6.2 & $2.35 \times 10^{-13}$ & $1.18 \times 10^{-10}$ \\ 
  transporter complex (GO:1990351) & 336 & 28 & 5 & 5.8 & $3.28 \times 10^{-13}$ & $1.32 \times 10^{-10}$ \\ 
  synaptic membrane (GO:0097060) & 397 & 30 & 6 & 5.3 & $4.74 \times 10^{-13}$ & $1.58 \times 10^{-10}$ \\ 
  cation channel complex (GO:0034703) & 224 & 23 & 3 & 7.2 & $9.09 \times 10^{-13}$ & $2.61 \times 10^{-10}$ \\ 
  transmembrane transporter complex (GO:1902495) & 328 & 27 & 5 & 5.8 & $1.16 \times 10^{-12}$ & $2.91 \times 10^{-10}$ \\ 
  plasma membrane bounded cell projection (GO:0120025) & 2219 & 73 & 32 & 2.3 & $1.26 \times 10^{-11}$ & $2.81 \times 10^{-9}$ \\ 
  postsynaptic membrane (GO:0045211) & 290 & 24 & 4 & 5.8 & $1.97 \times 10^{-11}$ & $3.96 \times 10^{-9}$ \\ 
  membrane (GO:0016020) & 9861 & 198 & 140 & 1.4 & $3.32 \times 10^{-11}$ & $6.06 \times 10^{-9}$ \\ 
  axon (GO:0030424) & 651 & 35 & 9 & 3.8 & $4.69 \times 10^{-11}$ & $7.84 \times 10^{-9}$ \\ 
  cell projection (GO:0042995) & 2307 & 73 & 33 & 2.2 & $9.82 \times 10^{-11}$ & $1.52 \times 10^{-8}$ \\ 
  intrinsic component of plasma membrane (GO:0031226) & 1712 & 60 & 24 & 2.5 & $1.22 \times 10^{-10}$ & $1.75 \times 10^{-8}$ \\ 
  integral component of plasma membrane (GO:0005887) & 1635 & 58 & 23 & 2.5 & $1.79 \times 10^{-10}$ & $2.39 \times 10^{-8}$ \\ 
  cytoplasm (GO:0005737) & 11515 & 217 & 164 & 1.3 & $4.41 \times 10^{-10}$ & $5.53 \times 10^{-8}$ \\ 
  plasma membrane region (GO:0098590) & 1215 & 47 & 17 & 2.7 & $8.56 \times 10^{-10}$ & $1.01 \times 10^{-7}$ \\ 
  cell periphery (GO:0071944) & 5885 & 134 & 84 & 1.6 & $1.09 \times 10^{-9}$ & $1.22 \times 10^{-7}$ \\ 
  plasma membrane (GO:0005886) & 5755 & 131 & 82 & 1.6 & $1.93 \times 10^{-9}$ & $1.94 \times 10^{-7}$ \\ 
  glutamatergic synapse (GO:0098978) & 369 & 24 & 5 & 4.6 & $1.92 \times 10^{-9}$ & $2.03 \times 10^{-7}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis CC 3457 10cent clo.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis CC 3457 10cent clo.txt Over representation only. Top 20 Terms}
\end{table}


% Pathway %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LOW CLO %%%%%%%%%%%%%%%%%%%%%


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:26:34 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER Pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
5-Hydroxytryptamine degredation (P04372) & 21 & 4 & 0 & 13.4 & $3.82 \times 10^{-4}$ & $2.09 \times 10^{-2}$ \\ 
  Ionotropic glutamate receptor pathway (P00037) & 51 & 8 & 1 & 11.0 & $1.64 \times 10^{-6}$ & $1.35 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis pathway 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO analysis pathway 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:26:34 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER Pathways & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
Ionotropic glutamate receptor pathway (P00037) & 51 & 8 & 1 & 11.0 & $1.64 \times 10^{-6}$ & $1.35 \times 10^{-4}$ \\ 
  5-Hydroxytryptamine degredation (P04372) & 21 & 4 & 0 & 13.4 & $3.82 \times 10^{-4}$ & $2.09 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO analysis pathway 3457 10cent clo.txt Over representation only. Top 20 Terms} 
\label{tab:GO analysis pathway 3457 10cent clo.txt Over representation only. Top 20 Terms}
\end{table}
\clearpage

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%                                                                                                                               %                                           
% GO SLIM                                                                                                                       %
%                                                                                                                               %
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\subsection{GOSLIM}

MF Deg over 0.9 - nil

BP Deg over 0.9 - nil




Bet over 0.9 nil all categories
bet no enrichment protein class same with degree

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection{G S Eigenvector centrality high}
%%%%%%%%%%%%%%%%%% GO SLIM EIGENVECTOR CENTRALITY HIGH %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\ref{tab:GO SLIM analysis SLIM BP 3457 90cent eig.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO SLIM analysis SLIM BP 3457 90cent eig.txt Over representation only. Top 20 Terms}

\ref{tab:GO SLIM analysis GS MF 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO SLIM analysis GS MF 3457 90cent eigvec.txt Over representation only. Top 20 Terms}

\ref{tab:GO SLIM analysis GS CC 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO SLIM analysis GS CC 3457 90cent eigvec.txt Over representation only. Top 20 Terms}
%% GO SLIM BP %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% HIGH EIG %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:31:45 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Biological Process & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
ribosomal large subunit assembly (GO:0000027) & 9 & 9 & 1 & 7.4 & $7.95 \times 10^{-5}$ & $8.25 \times 10^{-3}$ \\ 
  ribosomal large subunit biogenesis (GO:0042273) & 14 & 12 & 2 & 6.3 & $1.39 \times 10^{-5}$ & $2.00 \times 10^{-3}$ \\ 
  ribosomal small subunit biogenesis (GO:0042274) & 10 & 8 & 1 & 5.9 & $5.55 \times 10^{-4}$ & $3.34 \times 10^{-2}$ \\ 
  ribosome biogenesis (GO:0042254) & 28 & 21 & 4 & 5.6 & $3.88 \times 10^{-8}$ & $1.81 \times 10^{-5}$ \\ 
  ribonucleoprotein complex assembly (GO:0022618) & 37 & 25 & 5 & 5.0 & $8.64 \times 10^{-9}$ & $8.06 \times 10^{-6}$ \\ 
  ribonucleoprotein complex biogenesis (GO:0022613) & 48 & 31 & 6 & 4.8 & $2.98 \times 10^{-10}$ & $5.56 \times 10^{-7}$ \\ 
  ribonucleoprotein complex subunit organization (GO:0071826) & 39 & 25 & 5 & 4.8 & $1.87 \times 10^{-8}$ & $1.16 \times 10^{-5}$ \\ 
  organelle assembly (GO:0070925) & 47 & 21 & 6 & 3.3 & $2.10 \times 10^{-5}$ & $2.61 \times 10^{-3}$ \\ 
  mRNA processing (GO:0006397) & 41 & 17 & 6 & 3.1 & $2.63 \times 10^{-4}$ & $2.13 \times 10^{-2}$ \\ 
  translational elongation (GO:0006414) & 80 & 33 & 11 & 3.1 & $3.86 \times 10^{-7}$ & $1.44 \times 10^{-4}$ \\ 
  translation (GO:0006412) & 80 & 33 & 11 & 3.1 & $3.86 \times 10^{-7}$ & $1.20 \times 10^{-4}$ \\ 
  peptide biosynthetic process (GO:0043043) & 80 & 33 & 11 & 3.1 & $3.86 \times 10^{-7}$ & $1.03 \times 10^{-4}$ \\ 
  RNA splicing (GO:0008380) & 39 & 16 & 5 & 3.0 & $4.37 \times 10^{-4}$ & $2.81 \times 10^{-2}$ \\ 
  RNA processing (GO:0006396) & 62 & 25 & 8 & 3.0 & $1.42 \times 10^{-5}$ & $1.90 \times 10^{-3}$ \\ 
  amide biosynthetic process (GO:0043604) & 83 & 33 & 11 & 3.0 & $1.08 \times 10^{-6}$ & $2.24 \times 10^{-4}$ \\ 
  mRNA metabolic process (GO:0016071) & 50 & 19 & 7 & 2.8 & $4.32 \times 10^{-4}$ & $2.88 \times 10^{-2}$ \\ 
  peptide metabolic process (GO:0006518) & 97 & 35 & 13 & 2.7 & $2.93 \times 10^{-6}$ & $5.47 \times 10^{-4}$ \\ 
  cellular amide metabolic process (GO:0043603) & 111 & 36 & 15 & 2.4 & $1.25 \times 10^{-5}$ & $1.94 \times 10^{-3}$ \\ 
  cellular protein-containing complex assembly (GO:0034622) & 106 & 33 & 14 & 2.3 & $7.46 \times 10^{-5}$ & $8.19 \times 10^{-3}$ \\ 
  protein-containing complex assembly (GO:0065003) & 114 & 33 & 15 & 2.1 & $2.17 \times 10^{-4}$ & $1.84 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis SLIM BP 3457 90cent eig.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO SLIM analysis SLIM BP 3457 90cent eig.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:31:45 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Biological Process & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
ribonucleoprotein complex biogenesis (GO:0022613) & 48 & 31 & 6 & 4.8 & $2.98 \times 10^{-10}$ & $5.56 \times 10^{-7}$ \\ 
  ribonucleoprotein complex assembly (GO:0022618) & 37 & 25 & 5 & 5.0 & $8.64 \times 10^{-9}$ & $8.06 \times 10^{-6}$ \\ 
  ribonucleoprotein complex subunit organization (GO:0071826) & 39 & 25 & 5 & 4.8 & $1.87 \times 10^{-8}$ & $1.16 \times 10^{-5}$ \\ 
  ribosome biogenesis (GO:0042254) & 28 & 21 & 4 & 5.6 & $3.88 \times 10^{-8}$ & $1.81 \times 10^{-5}$ \\ 
  peptide biosynthetic process (GO:0043043) & 80 & 33 & 11 & 3.1 & $3.86 \times 10^{-7}$ & $1.03 \times 10^{-4}$ \\ 
  translation (GO:0006412) & 80 & 33 & 11 & 3.1 & $3.86 \times 10^{-7}$ & $1.20 \times 10^{-4}$ \\ 
  gene expression (GO:0010467) & 313 & 79 & 42 & 1.9 & $5.87 \times 10^{-7}$ & $1.37 \times 10^{-4}$ \\ 
  translational elongation (GO:0006414) & 80 & 33 & 11 & 3.1 & $3.86 \times 10^{-7}$ & $1.44 \times 10^{-4}$ \\ 
  amide biosynthetic process (GO:0043604) & 83 & 33 & 11 & 3.0 & $1.08 \times 10^{-6}$ & $2.24 \times 10^{-4}$ \\ 
  peptide metabolic process (GO:0006518) & 97 & 35 & 13 & 2.7 & $2.93 \times 10^{-6}$ & $5.47 \times 10^{-4}$ \\ 
  macromolecule metabolic process (GO:0043170) & 662 & 133 & 89 & 1.5 & $4.57 \times 10^{-6}$ & $7.76 \times 10^{-4}$ \\ 
  RNA processing (GO:0006396) & 62 & 25 & 8 & 3.0 & $1.42 \times 10^{-5}$ & $1.90 \times 10^{-3}$ \\ 
  cellular amide metabolic process (GO:0043603) & 111 & 36 & 15 & 2.4 & $1.25 \times 10^{-5}$ & $1.94 \times 10^{-3}$ \\ 
  ribosomal large subunit biogenesis (GO:0042273) & 14 & 12 & 2 & 6.3 & $1.39 \times 10^{-5}$ & $2.00 \times 10^{-3}$ \\ 
  organelle assembly (GO:0070925) & 47 & 21 & 6 & 3.3 & $2.10 \times 10^{-5}$ & $2.61 \times 10^{-3}$ \\ 
  cellular protein metabolic process (GO:0044267) & 340 & 76 & 46 & 1.7 & $5.34 \times 10^{-5}$ & $6.23 \times 10^{-3}$ \\ 
  cellular protein-containing complex assembly (GO:0034622) & 106 & 33 & 14 & 2.3 & $7.46 \times 10^{-5}$ & $8.19 \times 10^{-3}$ \\ 
  ribosomal large subunit assembly (GO:0000027) & 9 & 9 & 1 & 7.4 & $7.95 \times 10^{-5}$ & $8.25 \times 10^{-3}$ \\ 
  cellular nitrogen compound metabolic process (GO:0034641) & 444 & 92 & 60 & 1.5 & $9.37 \times 10^{-5}$ & $9.20 \times 10^{-3}$ \\ 
  organonitrogen compound biosynthetic process (GO:1901566) & 156 & 42 & 21 & 2.0 & $1.08 \times 10^{-4}$ & $1.01 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis SLIM BP 3457 90cent eig.txt Over representation only. Top 20 Terms} 
\label{tab:GO SLIM analysis SLIM BP 3457 90cent eig.txt Over representation only. Top 20 Terms}
\end{table}

%% GO SLIM MF %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% HIGH EIG %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:35:11 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Molecular Function & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
rRNA binding (GO:0019843) & 7 & 7 & 1 & 7.4 & $5.21 \times 10^{-4}$ & $3.18 \times 10^{-2}$ \\ 
  structural constituent of ribosome (GO:0003735) & 56 & 40 & 8 & 5.3 & $6.18 \times 10^{-14}$ & $1.01 \times 10^{-11}$ \\ 
  structural molecule activity (GO:0005198) & 71 & 46 & 10 & 4.8 & $9.93 \times 10^{-15}$ & $2.43 \times 10^{-12}$ \\ 
  unfolded protein binding (GO:0051082) & 17 & 10 & 2 & 4.4 & $6.58 \times 10^{-4}$ & $3.58 \times 10^{-2}$ \\ 
  RNA binding (GO:0003723) & 129 & 62 & 17 & 3.6 & $7.49 \times 10^{-15}$ & $3.66 \times 10^{-12}$ \\ 
  nucleic acid binding (GO:0003676) & 248 & 79 & 33 & 2.4 & $7.89 \times 10^{-11}$ & $9.65 \times 10^{-9}$ \\ 
  heterocyclic compound binding (GO:1901363) & 348 & 97 & 47 & 2.1 & $2.24 \times 10^{-10}$ & $1.82 \times 10^{-8}$ \\ 
  organic cyclic compound binding (GO:0097159) & 354 & 98 & 48 & 2.0 & $2.00 \times 10^{-10}$ & $1.95 \times 10^{-8}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis GS MF 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO SLIM analysis GS MF 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:35:11 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Molecular Function & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
structural molecule activity (GO:0005198) & 71 & 46 & 10 & 4.8 & $9.93 \times 10^{-15}$ & $2.43 \times 10^{-12}$ \\ 
  RNA binding (GO:0003723) & 129 & 62 & 17 & 3.6 & $7.49 \times 10^{-15}$ & $3.66 \times 10^{-12}$ \\ 
  structural constituent of ribosome (GO:0003735) & 56 & 40 & 8 & 5.3 & $6.18 \times 10^{-14}$ & $1.01 \times 10^{-11}$ \\ 
  nucleic acid binding (GO:0003676) & 248 & 79 & 33 & 2.4 & $7.89 \times 10^{-11}$ & $9.65 \times 10^{-9}$ \\ 
  heterocyclic compound binding (GO:1901363) & 348 & 97 & 47 & 2.1 & $2.24 \times 10^{-10}$ & $1.82 \times 10^{-8}$ \\ 
  organic cyclic compound binding (GO:0097159) & 354 & 98 & 48 & 2.0 & $2.00 \times 10^{-10}$ & $1.95 \times 10^{-8}$ \\ 
  rRNA binding (GO:0019843) & 7 & 7 & 1 & 7.4 & $5.21 \times 10^{-4}$ & $3.18 \times 10^{-2}$ \\ 
  unfolded protein binding (GO:0051082) & 17 & 10 & 2 & 4.4 & $6.58 \times 10^{-4}$ & $3.58 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis GS MF 3457 90cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO SLIM analysis GS MF 3457 90cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}



%% GO SLIM MF %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% HIGH EIG %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:36:39 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Cellular Component & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
cytosolic small ribosomal subunit (GO:0022627) & 24 & 18 & 3 & 5.6 & $3.74 \times 10^{-7}$ & $1.26 \times 10^{-5}$ \\ 
  small ribosomal subunit (GO:0015935) & 26 & 19 & 4 & 5.4 & $2.34 \times 10^{-7}$ & $9.18 \times 10^{-6}$ \\ 
  cytosolic ribosome (GO:0022626) & 59 & 42 & 8 & 5.3 & $1.52 \times 10^{-14}$ & $2.37 \times 10^{-12}$ \\ 
  cytosolic large ribosomal subunit (GO:0022625) & 35 & 24 & 5 & 5.1 & $1.41 \times 10^{-8}$ & $9.46 \times 10^{-7}$ \\ 
  ribosomal subunit (GO:0044391) & 64 & 43 & 9 & 5.0 & $3.13 \times 10^{-14}$ & $2.94 \times 10^{-12}$ \\ 
  Prp19 complex (GO:0000974) & 15 & 10 & 2 & 4.9 & $3.18 \times 10^{-4}$ & $6.50 \times 10^{-3}$ \\ 
  catalytic step 2 spliceosome (GO:0071013) & 15 & 10 & 2 & 4.9 & $3.18 \times 10^{-4}$ & $6.23 \times 10^{-3}$ \\ 
  ribosome (GO:0005840) & 65 & 43 & 9 & 4.9 & $4.64 \times 10^{-14}$ & $3.63 \times 10^{-12}$ \\ 
  large ribosomal subunit (GO:0015934) & 38 & 24 & 5 & 4.7 & $4.47 \times 10^{-8}$ & $2.10 \times 10^{-6}$ \\ 
  U5 snRNP (GO:0005682) & 16 & 10 & 2 & 4.6 & $4.63 \times 10^{-4}$ & $8.70 \times 10^{-3}$ \\ 
  spliceosomal complex (GO:0005681) & 24 & 15 & 3 & 4.6 & $1.77 \times 10^{-5}$ & $4.63 \times 10^{-4}$ \\ 
  cytosolic part (GO:0044445) & 81 & 50 & 11 & 4.6 & $2.59 \times 10^{-15}$ & $6.09 \times 10^{-13}$ \\ 
  spliceosomal snRNP complex (GO:0097525) & 17 & 10 & 2 & 4.4 & $6.58 \times 10^{-4}$ & $1.19 \times 10^{-2}$ \\ 
  small nuclear ribonucleoprotein complex (GO:0030532) & 17 & 10 & 2 & 4.4 & $6.58 \times 10^{-4}$ & $1.15 \times 10^{-2}$ \\ 
  Sm-like protein family complex (GO:0120114) & 17 & 10 & 2 & 4.4 & $6.58 \times 10^{-4}$ & $1.11 \times 10^{-2}$ \\ 
  ribonucleoprotein complex (GO:1990904) & 129 & 70 & 17 & 4.0 & $6.93 \times 10^{-19}$ & $3.26 \times 10^{-16}$ \\ 
  cytosol (GO:0005829) & 232 & 85 & 31 & 2.7 & $2.58 \times 10^{-14}$ & $3.04 \times 10^{-12}$ \\ 
  intracellular non-membrane-bounded organelle (GO:0043232) & 299 & 82 & 40 & 2.0 & $1.41 \times 10^{-8}$ & $8.29 \times 10^{-7}$ \\ 
  non-membrane-bounded organelle (GO:0043228) & 299 & 82 & 40 & 2.0 & $1.41 \times 10^{-8}$ & $7.37 \times 10^{-7}$ \\ 
  nuclear part (GO:0044428) & 179 & 42 & 24 & 1.7 & $1.73 \times 10^{-3}$ & $2.46 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis GS CC 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO SLIM analysis GS CC 3457 90cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:36:39 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Cellular Component & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
ribonucleoprotein complex (GO:1990904) & 129 & 70 & 17 & 4.0 & $6.93 \times 10^{-19}$ & $3.26 \times 10^{-16}$ \\ 
  cytosolic part (GO:0044445) & 81 & 50 & 11 & 4.6 & $2.59 \times 10^{-15}$ & $6.09 \times 10^{-13}$ \\ 
  cytosolic ribosome (GO:0022626) & 59 & 42 & 8 & 5.3 & $1.52 \times 10^{-14}$ & $2.37 \times 10^{-12}$ \\ 
  ribosomal subunit (GO:0044391) & 64 & 43 & 9 & 5.0 & $3.13 \times 10^{-14}$ & $2.94 \times 10^{-12}$ \\ 
  cytosol (GO:0005829) & 232 & 85 & 31 & 2.7 & $2.58 \times 10^{-14}$ & $3.04 \times 10^{-12}$ \\ 
  ribosome (GO:0005840) & 65 & 43 & 9 & 4.9 & $4.64 \times 10^{-14}$ & $3.63 \times 10^{-12}$ \\ 
  non-membrane-bounded organelle (GO:0043228) & 299 & 82 & 40 & 2.0 & $1.41 \times 10^{-8}$ & $7.37 \times 10^{-7}$ \\ 
  intracellular non-membrane-bounded organelle (GO:0043232) & 299 & 82 & 40 & 2.0 & $1.41 \times 10^{-8}$ & $8.29 \times 10^{-7}$ \\ 
  cytosolic large ribosomal subunit (GO:0022625) & 35 & 24 & 5 & 5.1 & $1.41 \times 10^{-8}$ & $9.46 \times 10^{-7}$ \\ 
  large ribosomal subunit (GO:0015934) & 38 & 24 & 5 & 4.7 & $4.47 \times 10^{-8}$ & $2.10 \times 10^{-6}$ \\ 
  protein-containing complex (GO:0032991) & 643 & 137 & 87 & 1.6 & $1.49 \times 10^{-7}$ & $6.38 \times 10^{-6}$ \\ 
  nucleus (GO:0005634) & 466 & 107 & 63 & 1.7 & $2.50 \times 10^{-7}$ & $9.04 \times 10^{-6}$ \\ 
  small ribosomal subunit (GO:0015935) & 26 & 19 & 4 & 5.4 & $2.34 \times 10^{-7}$ & $9.18 \times 10^{-6}$ \\ 
  cytosolic small ribosomal subunit (GO:0022627) & 24 & 18 & 3 & 5.6 & $3.74 \times 10^{-7}$ & $1.26 \times 10^{-5}$ \\ 
  intracellular organelle (GO:0043229) & 1038 & 188 & 140 & 1.3 & $9.82 \times 10^{-6}$ & $3.08 \times 10^{-4}$ \\ 
  intracellular organelle part (GO:0044446) & 552 & 114 & 74 & 1.5 & $1.20 \times 10^{-5}$ & $3.51 \times 10^{-4}$ \\ 
  intracellular part (GO:0044424) & 1357 & 232 & 183 & 1.3 & $1.39 \times 10^{-5}$ & $3.85 \times 10^{-4}$ \\ 
  spliceosomal complex (GO:0005681) & 24 & 15 & 3 & 4.6 & $1.77 \times 10^{-5}$ & $4.63 \times 10^{-4}$ \\ 
  intracellular (GO:0005622) & 1365 & 232 & 184 & 1.3 & $2.23 \times 10^{-5}$ & $5.24 \times 10^{-4}$ \\ 
  organelle (GO:0043226) & 1068 & 190 & 144 & 1.3 & $2.35 \times 10^{-5}$ & $5.25 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis GS CC 3457 90cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO SLIM analysis GS CC 3457 90cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}

\clearpage

%%%%%%%%%%%%%%%%%% GO SLIM EIGENVECTOR CENTRALITY HIGH %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsubsection{G S Eigenvector centrality low}

\ref{tab:GO SLIM analysis BP SLIM 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO SLIM analysis BP SLIM 3457 10cent eigvec.txt Over representation only. Top 20 Terms}

\ref{tab:GO SLIM analysis MF SLIM 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO SLIM analysis MF SLIM 3457 10cent eigvec.txt Over representation only. Top 20 Terms}

\ref{tab:GO SLIM analysis CC SLIM 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO SLIM analysis SLIM MF 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

%%%%%%%%%%%%%%%%%% GO SLIM EIGENVECTOR CENTRALITY HIGH %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%% GO SLIM BP %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LOW EIG %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:46:44 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Biological Process & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
regulation of trans-synaptic signaling (GO:0099177) & 32 & 12 & 3 & 4.1 & $1.81 \times 10^{-4}$ & $4.21 \times 10^{-2}$ \\ 
  modulation of chemical synaptic transmission (GO:0050804) & 32 & 12 & 3 & 4.1 & $1.81 \times 10^{-4}$ & $3.75 \times 10^{-2}$ \\ 
  chemical synaptic transmission (GO:0007268) & 84 & 23 & 8 & 3.0 & $1.69 \times 10^{-5}$ & $3.16 \times 10^{-2}$ \\ 
  anterograde trans-synaptic signaling (GO:0098916) & 84 & 23 & 8 & 3.0 & $1.69 \times 10^{-5}$ & $1.58 \times 10^{-2}$ \\ 
  trans-synaptic signaling (GO:0099537) & 85 & 23 & 8 & 3.0 & $1.99 \times 10^{-5}$ & $1.24 \times 10^{-2}$ \\ 
  synaptic signaling (GO:0099536) & 85 & 23 & 8 & 3.0 & $1.99 \times 10^{-5}$ & $9.28 \times 10^{-3}$ \\ 
  metal ion transport (GO:0030001) & 77 & 20 & 7 & 2.8 & $1.13 \times 10^{-4}$ & $3.02 \times 10^{-2}$ \\ 
  cell-cell signaling (GO:0007267) & 101 & 24 & 9 & 2.6 & $9.30 \times 10^{-5}$ & $2.89 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis BP SLIM 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO SLIM analysis BP SLIM 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:46:44 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Biological Process & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
synaptic signaling (GO:0099536) & 85 & 23 & 8 & 3.0 & $1.99 \times 10^{-5}$ & $9.28 \times 10^{-3}$ \\ 
  trans-synaptic signaling (GO:0099537) & 85 & 23 & 8 & 3.0 & $1.99 \times 10^{-5}$ & $1.24 \times 10^{-2}$ \\ 
  anterograde trans-synaptic signaling (GO:0098916) & 84 & 23 & 8 & 3.0 & $1.69 \times 10^{-5}$ & $1.58 \times 10^{-2}$ \\ 
  cell-cell signaling (GO:0007267) & 101 & 24 & 9 & 2.6 & $9.30 \times 10^{-5}$ & $2.89 \times 10^{-2}$ \\ 
  metal ion transport (GO:0030001) & 77 & 20 & 7 & 2.8 & $1.13 \times 10^{-4}$ & $3.02 \times 10^{-2}$ \\ 
  chemical synaptic transmission (GO:0007268) & 84 & 23 & 8 & 3.0 & $1.69 \times 10^{-5}$ & $3.16 \times 10^{-2}$ \\ 
  modulation of chemical synaptic transmission (GO:0050804) & 32 & 12 & 3 & 4.1 & $1.81 \times 10^{-4}$ & $3.75 \times 10^{-2}$ \\ 
  regulation of trans-synaptic signaling (GO:0099177) & 32 & 12 & 3 & 4.1 & $1.81 \times 10^{-4}$ & $4.21 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis BP SLIM 3457 10cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO SLIM analysis BP SLIM 3457 10cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}

%% GO SLIM MF %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LOW EIG %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:50:54 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Molecular Function & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
cyclase activity (GO:0009975) & 8 & 5 & 1 & 6.8 & $2.97 \times 10^{-3}$ & $4.84 \times 10^{-2}$ \\ 
  glutamate receptor activity (GO:0008066) & 19 & 8 & 2 & 4.6 & $1.22 \times 10^{-3}$ & $2.71 \times 10^{-2}$ \\ 
  sodium ion transmembrane transporter activity (GO:0015081) & 27 & 11 & 2 & 4.4 & $1.88 \times 10^{-4}$ & $6.14 \times 10^{-3}$ \\ 
  amino acid binding (GO:0016597) & 23 & 9 & 2 & 4.3 & $9.20 \times 10^{-4}$ & $2.14 \times 10^{-2}$ \\ 
  voltage-gated potassium channel activity (GO:0005249) & 22 & 8 & 2 & 4.0 & $2.57 \times 10^{-3}$ & $4.66 \times 10^{-2}$ \\ 
  potassium channel activity (GO:0005267) & 34 & 12 & 3 & 3.9 & $2.86 \times 10^{-4}$ & $8.22 \times 10^{-3}$ \\ 
  carboxylic acid binding (GO:0031406) & 32 & 11 & 3 & 3.8 & $6.15 \times 10^{-4}$ & $1.50 \times 10^{-2}$ \\ 
  voltage-gated cation channel activity (GO:0022843) & 39 & 13 & 4 & 3.6 & $2.54 \times 10^{-4}$ & $7.76 \times 10^{-3}$ \\ 
  neurotransmitter binding (GO:0042165) & 33 & 10 & 3 & 3.3 & $2.37 \times 10^{-3}$ & $4.46 \times 10^{-2}$ \\ 
  voltage-gated channel activity (GO:0022832) & 43 & 13 & 4 & 3.3 & $5.55 \times 10^{-4}$ & $1.51 \times 10^{-2}$ \\ 
  voltage-gated ion channel activity (GO:0005244) & 43 & 13 & 4 & 3.3 & $5.55 \times 10^{-4}$ & $1.43 \times 10^{-2}$ \\ 
  cation channel activity (GO:0005261) & 77 & 23 & 7 & 3.2 & $5.09 \times 10^{-6}$ & $4.98 \times 10^{-4}$ \\ 
  organic acid binding (GO:0043177) & 38 & 11 & 3 & 3.1 & $1.97 \times 10^{-3}$ & $3.86 \times 10^{-2}$ \\ 
  metal ion transmembrane transporter activity (GO:0046873) & 90 & 26 & 8 & 3.1 & $2.03 \times 10^{-6}$ & $2.48 \times 10^{-4}$ \\ 
  ion gated channel activity (GO:0022839) & 73 & 21 & 7 & 3.1 & $2.14 \times 10^{-5}$ & $1.75 \times 10^{-3}$ \\ 
  potassium ion transmembrane transporter activity (GO:0015079) & 42 & 12 & 4 & 3.1 & $1.36 \times 10^{-3}$ & $2.90 \times 10^{-2}$ \\ 
  ion channel activity (GO:0005216) & 95 & 27 & 9 & 3.1 & $1.66 \times 10^{-6}$ & $8.14 \times 10^{-4}$ \\ 
  passive transmembrane transporter activity (GO:0022803) & 96 & 27 & 9 & 3.1 & $1.97 \times 10^{-6}$ & $4.82 \times 10^{-4}$ \\ 
  channel activity (GO:0015267) & 96 & 27 & 9 & 3.1 & $1.97 \times 10^{-6}$ & $3.21 \times 10^{-4}$ \\ 
  gated channel activity (GO:0022836) & 78 & 21 & 7 & 2.9 & $4.91 \times 10^{-5}$ & $2.67 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis MF SLIM 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO SLIM analysis MF SLIM 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:50:54 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Molecular Function & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
metal ion transmembrane transporter activity (GO:0046873) & 90 & 26 & 8 & 3.1 & $2.03 \times 10^{-6}$ & $2.48 \times 10^{-4}$ \\ 
  channel activity (GO:0015267) & 96 & 27 & 9 & 3.1 & $1.97 \times 10^{-6}$ & $3.21 \times 10^{-4}$ \\ 
  passive transmembrane transporter activity (GO:0022803) & 96 & 27 & 9 & 3.1 & $1.97 \times 10^{-6}$ & $4.82 \times 10^{-4}$ \\ 
  cation channel activity (GO:0005261) & 77 & 23 & 7 & 3.2 & $5.09 \times 10^{-6}$ & $4.98 \times 10^{-4}$ \\ 
  ion channel activity (GO:0005216) & 95 & 27 & 9 & 3.1 & $1.66 \times 10^{-6}$ & $8.14 \times 10^{-4}$ \\ 
  inorganic cation transmembrane transporter activity (GO:0022890) & 118 & 28 & 11 & 2.6 & $2.22 \times 10^{-5}$ & $1.55 \times 10^{-3}$ \\ 
  ion gated channel activity (GO:0022839) & 73 & 21 & 7 & 3.1 & $2.14 \times 10^{-5}$ & $1.75 \times 10^{-3}$ \\ 
  ion transmembrane transporter activity (GO:0015075) & 156 & 33 & 14 & 2.3 & $3.53 \times 10^{-5}$ & $2.16 \times 10^{-3}$ \\ 
  inorganic molecular entity transmembrane transporter activity (GO:0015318) & 152 & 32 & 14 & 2.3 & $5.16 \times 10^{-5}$ & $2.52 \times 10^{-3}$ \\ 
  gated channel activity (GO:0022836) & 78 & 21 & 7 & 2.9 & $4.91 \times 10^{-5}$ & $2.67 \times 10^{-3}$ \\ 
  cation transmembrane transporter activity (GO:0008324) & 124 & 28 & 11 & 2.5 & $6.21 \times 10^{-5}$ & $2.76 \times 10^{-3}$ \\ 
  transmembrane transporter activity (GO:0022857) & 185 & 36 & 17 & 2.1 & $7.78 \times 10^{-5}$ & $3.17 \times 10^{-3}$ \\ 
  monovalent inorganic cation transmembrane transporter activity (GO:0015077) & 76 & 20 & 7 & 2.9 & $9.74 \times 10^{-5}$ & $3.66 \times 10^{-3}$ \\ 
  transporter activity (GO:0005215) & 207 & 38 & 19 & 2.0 & $1.73 \times 10^{-4}$ & $6.05 \times 10^{-3}$ \\ 
  sodium ion transmembrane transporter activity (GO:0015081) & 27 & 11 & 2 & 4.4 & $1.88 \times 10^{-4}$ & $6.14 \times 10^{-3}$ \\ 
  voltage-gated cation channel activity (GO:0022843) & 39 & 13 & 4 & 3.6 & $2.54 \times 10^{-4}$ & $7.76 \times 10^{-3}$ \\ 
  potassium channel activity (GO:0005267) & 34 & 12 & 3 & 3.9 & $2.86 \times 10^{-4}$ & $8.22 \times 10^{-3}$ \\ 
  voltage-gated ion channel activity (GO:0005244) & 43 & 13 & 4 & 3.3 & $5.55 \times 10^{-4}$ & $1.43 \times 10^{-2}$ \\ 
  carboxylic acid binding (GO:0031406) & 32 & 11 & 3 & 3.8 & $6.15 \times 10^{-4}$ & $1.50 \times 10^{-2}$ \\ 
  voltage-gated channel activity (GO:0022832) & 43 & 13 & 4 & 3.3 & $5.55 \times 10^{-4}$ & $1.51 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis MF SLIM 3457 10cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO SLIM analysis MF SLIM 3457 10cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}

%% GO SLIM CC %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LOW EIG %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:52:59 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Cellular Component & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
cation channel complex (GO:0034703) & 46 & 17 & 4 & 4.0 & $9.54 \times 10^{-6}$ & $8.97 \times 10^{-4}$ \\ 
  ion channel complex (GO:0034702) & 56 & 19 & 5 & 3.7 & $7.81 \times 10^{-6}$ & $9.17 \times 10^{-4}$ \\ 
  transporter complex (GO:1990351) & 60 & 20 & 6 & 3.6 & $5.57 \times 10^{-6}$ & $8.73 \times 10^{-4}$ \\ 
  transmembrane transporter complex (GO:1902495) & 57 & 19 & 5 & 3.6 & $9.63 \times 10^{-6}$ & $7.54 \times 10^{-4}$ \\ 
  plasma membrane protein complex (GO:0098797) & 90 & 21 & 8 & 2.5 & $3.26 \times 10^{-4}$ & $1.02 \times 10^{-2}$ \\ 
  integral component of plasma membrane (GO:0005887) & 153 & 33 & 14 & 2.4 & $2.92 \times 10^{-5}$ & $1.96 \times 10^{-3}$ \\ 
  integral component of membrane (GO:0016021) & 204 & 44 & 19 & 2.4 & $1.13 \times 10^{-6}$ & $5.29 \times 10^{-4}$ \\ 
  intrinsic component of plasma membrane (GO:0031226) & 154 & 33 & 14 & 2.3 & $3.09 \times 10^{-5}$ & $1.81 \times 10^{-3}$ \\ 
  intrinsic component of membrane (GO:0031224) & 206 & 44 & 19 & 2.3 & $1.25 \times 10^{-6}$ & $2.93 \times 10^{-4}$ \\ 
  plasma membrane part (GO:0044459) & 356 & 53 & 33 & 1.6 & $8.63 \times 10^{-4}$ & $2.54 \times 10^{-2}$ \\ 
  plasma membrane (GO:0005886) & 695 & 95 & 64 & 1.5 & $6.27 \times 10^{-5}$ & $3.27 \times 10^{-3}$ \\ 
  membrane (GO:0016020) & 761 & 102 & 70 & 1.5 & $7.60 \times 10^{-5}$ & $3.25 \times 10^{-3}$ \\ 
  cell periphery (GO:0071944) & 724 & 95 & 66 & 1.4 & $3.25 \times 10^{-4}$ & $1.09 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis CC SLIM 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO SLIM analysis CC SLIM 3457 10cent eigvec.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 14:52:59 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Cellular Component & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
intrinsic component of membrane (GO:0031224) & 206 & 44 & 19 & 2.3 & $1.25 \times 10^{-6}$ & $2.93 \times 10^{-4}$ \\ 
  integral component of membrane (GO:0016021) & 204 & 44 & 19 & 2.4 & $1.13 \times 10^{-6}$ & $5.29 \times 10^{-4}$ \\ 
  transmembrane transporter complex (GO:1902495) & 57 & 19 & 5 & 3.6 & $9.63 \times 10^{-6}$ & $7.54 \times 10^{-4}$ \\ 
  transporter complex (GO:1990351) & 60 & 20 & 6 & 3.6 & $5.57 \times 10^{-6}$ & $8.73 \times 10^{-4}$ \\ 
  cation channel complex (GO:0034703) & 46 & 17 & 4 & 4.0 & $9.54 \times 10^{-6}$ & $8.97 \times 10^{-4}$ \\ 
  ion channel complex (GO:0034702) & 56 & 19 & 5 & 3.7 & $7.81 \times 10^{-6}$ & $9.17 \times 10^{-4}$ \\ 
  intrinsic component of plasma membrane (GO:0031226) & 154 & 33 & 14 & 2.3 & $3.09 \times 10^{-5}$ & $1.81 \times 10^{-3}$ \\ 
  integral component of plasma membrane (GO:0005887) & 153 & 33 & 14 & 2.4 & $2.92 \times 10^{-5}$ & $1.96 \times 10^{-3}$ \\ 
  membrane (GO:0016020) & 761 & 102 & 70 & 1.5 & $7.60 \times 10^{-5}$ & $3.25 \times 10^{-3}$ \\ 
  plasma membrane (GO:0005886) & 695 & 95 & 64 & 1.5 & $6.27 \times 10^{-5}$ & $3.27 \times 10^{-3}$ \\ 
  plasma membrane protein complex (GO:0098797) & 90 & 21 & 8 & 2.5 & $3.26 \times 10^{-4}$ & $1.02 \times 10^{-2}$ \\ 
  cell periphery (GO:0071944) & 724 & 95 & 66 & 1.4 & $3.25 \times 10^{-4}$ & $1.09 \times 10^{-2}$ \\ 
  plasma membrane part (GO:0044459) & 356 & 53 & 33 & 1.6 & $8.63 \times 10^{-4}$ & $2.54 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis CC SLIM 3457 10cent eigvec.txt Over representation only. Top 20 Terms} 
\label{tab:GO SLIM analysis CC SLIM 3457 10cent eigvec.txt Over representation only. Top 20 Terms}
\end{table}
\clearpage

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%% GO SLIM HIGH  CLOSENESS CENTRALITY %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection{ SLIM high centrality}

BP no enrichment

MF 
\ref{tab:GO SLIM analysis SLIM MF 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

\ref{tab:GO SLIM analysis SLIM MF 3457 90cent clo.txt Over representation only. Top 20 Terms}

\ref{tab:PANTHER GO-Slim Cellular Component SLIM CC 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown}

\ref{tab:GO SLIM analysis GO SLIM CC 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:31:34 2020

%% GO SLIM MF %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% HIGH CLO %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Molecular Function & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
unfolded protein binding (GO:0051082) & 17 & 10 & 2 & 4.5 & $5.60 \times 10^{-4}$ & $3.91 \times 10^{-2}$ \\ 
  structural molecule activity (GO:0005198) & 71 & 28 & 9 & 3.0 & $4.07 \times 10^{-6}$ & $9.96 \times 10^{-4}$ \\ 
  structural constituent of ribosome (GO:0003735) & 56 & 21 & 7 & 2.8 & $1.84 \times 10^{-4}$ & $1.50 \times 10^{-2}$ \\ 
  RNA binding (GO:0003723) & 129 & 43 & 17 & 2.5 & $5.64 \times 10^{-7}$ & $2.76 \times 10^{-4}$ \\ 
  nucleic acid binding (GO:0003676) & 248 & 59 & 33 & 1.8 & $5.62 \times 10^{-5}$ & $5.50 \times 10^{-3}$ \\ 
  heterocyclic compound binding (GO:1901363) & 348 & 77 & 46 & 1.7 & $3.71 \times 10^{-5}$ & $4.54 \times 10^{-3}$ \\ 
  organic cyclic compound binding (GO:0097159) & 354 & 78 & 47 & 1.7 & $3.03 \times 10^{-5}$ & $4.93 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis SLIM MF 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO SLIM analysis SLIM MF 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:31:34 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}


\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Molecular Function & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
RNA binding (GO:0003723) & 129 & 43 & 17 & 2.5 & $5.64 \times 10^{-7}$ & $2.76 \times 10^{-4}$ \\ 
  structural molecule activity (GO:0005198) & 71 & 28 & 9 & 3.0 & $4.07 \times 10^{-6}$ & $9.96 \times 10^{-4}$ \\ 
  heterocyclic compound binding (GO:1901363) & 348 & 77 & 46 & 1.7 & $3.71 \times 10^{-5}$ & $4.54 \times 10^{-3}$ \\ 
  organic cyclic compound binding (GO:0097159) & 354 & 78 & 47 & 1.7 & $3.03 \times 10^{-5}$ & $4.93 \times 10^{-3}$ \\ 
  nucleic acid binding (GO:0003676) & 248 & 59 & 33 & 1.8 & $5.62 \times 10^{-5}$ & $5.50 \times 10^{-3}$ \\ 
  structural constituent of ribosome (GO:0003735) & 56 & 21 & 7 & 2.8 & $1.84 \times 10^{-4}$ & $1.50 \times 10^{-2}$ \\ 
  unfolded protein binding (GO:0051082) & 17 & 10 & 2 & 4.5 & $5.60 \times 10^{-4}$ & $3.91 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis SLIM MF 3457 90cent clo.txt Over representation only. Top 20 Terms} 
\label{tab:GO SLIM analysis SLIM MF 3457 90cent clo.txt Over representation only. Top 20 Terms}
\end{table}

%% GO SLIM CC %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% HIGH CLO %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 16:06:19 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Cellular Component & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
cytosolic small ribosomal subunit (GO:0022627) & 24 & 12 & 3 & 3.8 & $4.89 \times 10^{-4}$ & $2.55 \times 10^{-2}$ \\ 
  small ribosomal subunit (GO:0015935) & 26 & 13 & 3 & 3.8 & $2.86 \times 10^{-4}$ & $1.92 \times 10^{-2}$ \\ 
  cytosolic part (GO:0044445) & 81 & 30 & 11 & 2.8 & $5.97 \times 10^{-6}$ & $1.40 \times 10^{-3}$ \\ 
  cytosolic ribosome (GO:0022626) & 59 & 21 & 8 & 2.7 & $2.60 \times 10^{-4}$ & $2.03 \times 10^{-2}$ \\ 
  ribosomal subunit (GO:0044391) & 64 & 22 & 8 & 2.6 & $2.50 \times 10^{-4}$ & $2.35 \times 10^{-2}$ \\ 
  ribosome (GO:0005840) & 65 & 22 & 9 & 2.6 & $2.93 \times 10^{-4}$ & $1.72 \times 10^{-2}$ \\ 
  ribonucleoprotein complex (GO:1990904) & 129 & 39 & 17 & 2.3 & $1.92 \times 10^{-5}$ & $3.00 \times 10^{-3}$ \\ 
  cytosol (GO:0005829) & 232 & 63 & 31 & 2.1 & $6.17 \times 10^{-7}$ & $2.90 \times 10^{-4}$ \\ 
  intracellular non-membrane-bounded organelle (GO:0043232) & 299 & 63 & 39 & 1.6 & $7.23 \times 10^{-4}$ & $3.40 \times 10^{-2}$ \\ 
  non-membrane-bounded organelle (GO:0043228) & 299 & 63 & 39 & 1.6 & $7.23 \times 10^{-4}$ & $3.09 \times 10^{-2}$ \\ 
  nucleus (GO:0005634) & 466 & 97 & 61 & 1.6 & $2.36 \times 10^{-5}$ & $2.77 \times 10^{-3}$ \\ 
  intracellular part (GO:0044424) & 1357 & 216 & 179 & 1.2 & $9.49 \times 10^{-4}$ & $3.72 \times 10^{-2}$ \\ 
  intracellular (GO:0005622) & 1365 & 216 & 180 & 1.2 & $1.16 \times 10^{-3}$ & $4.20 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{PANTHER GO-Slim Cellular Component SLIM CC 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown} 
\label{tab:PANTHER GO-Slim Cellular Component SLIM CC 3457 90cent clo.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 16:06:19 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Cellular Component & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
cytosol (GO:0005829) & 232 & 63 & 31 & 2.1 & $6.17 \times 10^{-7}$ & $2.90 \times 10^{-4}$ \\ 
  cytosolic part (GO:0044445) & 81 & 30 & 11 & 2.8 & $5.97 \times 10^{-6}$ & $1.40 \times 10^{-3}$ \\ 
  nucleus (GO:0005634) & 466 & 97 & 61 & 1.6 & $2.36 \times 10^{-5}$ & $2.77 \times 10^{-3}$ \\ 
  ribonucleoprotein complex (GO:1990904) & 129 & 39 & 17 & 2.3 & $1.92 \times 10^{-5}$ & $3.00 \times 10^{-3}$ \\ 
  ribosome (GO:0005840) & 65 & 22 & 9 & 2.6 & $2.93 \times 10^{-4}$ & $1.72 \times 10^{-2}$ \\ 
  small ribosomal subunit (GO:0015935) & 26 & 13 & 3 & 3.8 & $2.86 \times 10^{-4}$ & $1.92 \times 10^{-2}$ \\ 
  cytosolic ribosome (GO:0022626) & 59 & 21 & 8 & 2.7 & $2.60 \times 10^{-4}$ & $2.03 \times 10^{-2}$ \\ 
  ribosomal subunit (GO:0044391) & 64 & 22 & 8 & 2.6 & $2.50 \times 10^{-4}$ & $2.35 \times 10^{-2}$ \\ 
  cytosolic small ribosomal subunit (GO:0022627) & 24 & 12 & 3 & 3.8 & $4.89 \times 10^{-4}$ & $2.55 \times 10^{-2}$ \\ 
  non-membrane-bounded organelle (GO:0043228) & 299 & 63 & 39 & 1.6 & $7.23 \times 10^{-4}$ & $3.09 \times 10^{-2}$ \\ 
  intracellular non-membrane-bounded organelle (GO:0043232) & 299 & 63 & 39 & 1.6 & $7.23 \times 10^{-4}$ & $3.40 \times 10^{-2}$ \\ 
  intracellular part (GO:0044424) & 1357 & 216 & 179 & 1.2 & $9.49 \times 10^{-4}$ & $3.72 \times 10^{-2}$ \\ 
  intracellular (GO:0005622) & 1365 & 216 & 180 & 1.2 & $1.16 \times 10^{-3}$ & $4.20 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{PANTHER GO-Slim Cellular Component SLIM CC 3457 90cent clo.txt Over representation only. Top 20 Terms. 20 significant terms shown} 
\label{tab:PANTHER GO-Slim Cellular Component SLIM CC 3457 90cent clo.txt Over representation only. Top 20 Terms. 20 significant terms shown}
\end{table}

\ref{tab:PANTHER GO-Slim Cellular Component SLIM CC 3457 90cent clo.txt Over representation only. Top 20 Terms. 20 significant terms shown}







\clearpage
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%% GO SLIM LOW  CLOSENESS CENTRALITY %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection{SLIM low centrality}

\ref{tab:PANTHER GO-Slim Biological Process GO SLIM BP 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown}

\ref{tab:PANTHER GO-Slim Biological Process GO SLIM BP 3457 10cent clo.txt Over representation only. Top 20 Terms. 20 significant terms shown}

Glutamate

\ref{tab:PANTHER GO-Slim Molecular Function GO SLIM MF 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown}

\ref{tab:PANTHER GO-Slim Molecular Function GO SLIM MF 3457 10cent clo.txt Over representation only. Top 20 Terms. 20 significant terms shown}

\ref{tab:GO SLIM analysis GO SLIM CC 3457 10cent clo.txt Over representation only. Top 20 Terms}

\ref{tab:GO SLIM analysis GO SLIM CC 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms}

%% GO SLIM BP %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LOW CLO %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:57:47 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Biological Process & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
synaptic transmission, glutamatergic (GO:0035249) & 31 & 8 & 0 & 18.1 & $5.81 \times 10^{-8}$ & $2.40 \times 10^{-5}$ \\ 
  regulation of trans-synaptic signaling (GO:0099177) & 71 & 10 & 1 & 9.9 & $2.00 \times 10^{-7}$ & $5.90 \times 10^{-5}$ \\ 
  modulation of chemical synaptic transmission (GO:0050804) & 71 & 10 & 1 & 9.9 & $2.00 \times 10^{-7}$ & $5.16 \times 10^{-5}$ \\ 
  regulation of exocytosis (GO:0017157) & 59 & 6 & 1 & 7.1 & $3.01 \times 10^{-4}$ & $2.82 \times 10^{-2}$ \\ 
  chemical synaptic transmission (GO:0007268) & 245 & 21 & 3 & 6.0 & $2.05 \times 10^{-10}$ & $4.23 \times 10^{-7}$ \\ 
  anterograde trans-synaptic signaling (GO:0098916) & 245 & 21 & 3 & 6.0 & $2.05 \times 10^{-10}$ & $2.11 \times 10^{-7}$ \\ 
  trans-synaptic signaling (GO:0099537) & 247 & 21 & 4 & 6.0 & $2.35 \times 10^{-10}$ & $1.62 \times 10^{-7}$ \\ 
  synaptic signaling (GO:0099536) & 249 & 21 & 4 & 5.9 & $2.70 \times 10^{-10}$ & $1.39 \times 10^{-7}$ \\ 
  metal ion transport (GO:0030001) & 288 & 17 & 4 & 4.1 & $1.65 \times 10^{-6}$ & $3.78 \times 10^{-4}$ \\ 
  cell-cell signaling (GO:0007267) & 381 & 22 & 5 & 4.0 & $6.85 \times 10^{-8}$ & $2.36 \times 10^{-5}$ \\ 
  regulation of transport (GO:0051049) & 244 & 12 & 3 & 3.5 & $2.87 \times 10^{-4}$ & $2.82 \times 10^{-2}$ \\ 
  cation transport (GO:0006812) & 397 & 19 & 6 & 3.4 & $7.50 \times 10^{-6}$ & $1.11 \times 10^{-3}$ \\ 
  inorganic cation transmembrane transport (GO:0098662) & 274 & 13 & 4 & 3.3 & $2.26 \times 10^{-4}$ & $2.45 \times 10^{-2}$ \\ 
  cation transmembrane transport (GO:0098655) & 302 & 14 & 4 & 3.2 & $1.64 \times 10^{-4}$ & $2.00 \times 10^{-2}$ \\ 
  inorganic ion transmembrane transport (GO:0098660) & 305 & 14 & 4 & 3.2 & $1.81 \times 10^{-4}$ & $2.08 \times 10^{-2}$ \\ 
  ion transport (GO:0006811) & 608 & 25 & 9 & 2.9 & $3.68 \times 10^{-6}$ & $6.90 \times 10^{-4}$ \\ 
  transmembrane transport (GO:0055085) & 479 & 18 & 7 & 2.6 & $2.53 \times 10^{-4}$ & $2.61 \times 10^{-2}$ \\ 
  establishment of localization (GO:0051234) & 1785 & 51 & 25 & 2.0 & $2.18 \times 10^{-6}$ & $4.51 \times 10^{-4}$ \\ 
  transport (GO:0006810) & 1760 & 50 & 25 & 2.0 & $4.82 \times 10^{-6}$ & $7.65 \times 10^{-4}$ \\ 
  localization (GO:0051179) & 2181 & 58 & 31 & 1.9 & $3.89 \times 10^{-6}$ & $6.68 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{PANTHER GO-Slim Biological Process GO SLIM BP 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown} 
\label{tab:PANTHER GO-Slim Biological Process GO SLIM BP 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown}
\end{table}



% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:57:47 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Biological Process & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
synaptic signaling (GO:0099536) & 249 & 21 & 4 & 5.9 & $2.70 \times 10^{-10}$ & $1.39 \times 10^{-7}$ \\ 
  trans-synaptic signaling (GO:0099537) & 247 & 21 & 4 & 6.0 & $2.35 \times 10^{-10}$ & $1.62 \times 10^{-7}$ \\ 
  anterograde trans-synaptic signaling (GO:0098916) & 245 & 21 & 3 & 6.0 & $2.05 \times 10^{-10}$ & $2.11 \times 10^{-7}$ \\ 
  chemical synaptic transmission (GO:0007268) & 245 & 21 & 3 & 6.0 & $2.05 \times 10^{-10}$ & $4.23 \times 10^{-7}$ \\ 
  cell-cell signaling (GO:0007267) & 381 & 22 & 5 & 4.0 & $6.85 \times 10^{-8}$ & $2.36 \times 10^{-5}$ \\ 
  synaptic transmission, glutamatergic (GO:0035249) & 31 & 8 & 0 & 18.1 & $5.81 \times 10^{-8}$ & $2.40 \times 10^{-5}$ \\ 
  modulation of chemical synaptic transmission (GO:0050804) & 71 & 10 & 1 & 9.9 & $2.00 \times 10^{-7}$ & $5.16 \times 10^{-5}$ \\ 
  regulation of trans-synaptic signaling (GO:0099177) & 71 & 10 & 1 & 9.9 & $2.00 \times 10^{-7}$ & $5.90 \times 10^{-5}$ \\ 
  metal ion transport (GO:0030001) & 288 & 17 & 4 & 4.1 & $1.65 \times 10^{-6}$ & $3.78 \times 10^{-4}$ \\ 
  establishment of localization (GO:0051234) & 1785 & 51 & 25 & 2.0 & $2.18 \times 10^{-6}$ & $4.51 \times 10^{-4}$ \\ 
  localization (GO:0051179) & 2181 & 58 & 31 & 1.9 & $3.89 \times 10^{-6}$ & $6.68 \times 10^{-4}$ \\ 
  ion transport (GO:0006811) & 608 & 25 & 9 & 2.9 & $3.68 \times 10^{-6}$ & $6.90 \times 10^{-4}$ \\ 
  transport (GO:0006810) & 1760 & 50 & 25 & 2.0 & $4.82 \times 10^{-6}$ & $7.65 \times 10^{-4}$ \\ 
  cation transport (GO:0006812) & 397 & 19 & 6 & 3.4 & $7.50 \times 10^{-6}$ & $1.11 \times 10^{-3}$ \\ 
  cation transmembrane transport (GO:0098655) & 302 & 14 & 4 & 3.2 & $1.64 \times 10^{-4}$ & $2.00 \times 10^{-2}$ \\ 
  biological\_process (GO:0008150) & 8857 & 159 & 126 & 1.3 & $1.50 \times 10^{-4}$ & $2.06 \times 10^{-2}$ \\ 
  inorganic ion transmembrane transport (GO:0098660) & 305 & 14 & 4 & 3.2 & $1.81 \times 10^{-4}$ & $2.08 \times 10^{-2}$ \\ 
  inorganic cation transmembrane transport (GO:0098662) & 274 & 13 & 4 & 3.3 & $2.26 \times 10^{-4}$ & $2.45 \times 10^{-2}$ \\ 
  transmembrane transport (GO:0055085) & 479 & 18 & 7 & 2.6 & $2.53 \times 10^{-4}$ & $2.61 \times 10^{-2}$ \\ 
  regulation of exocytosis (GO:0017157) & 59 & 6 & 1 & 7.1 & $3.01 \times 10^{-4}$ & $2.82 \times 10^{-2}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{PANTHER GO-Slim Biological Process GO SLIM BP 3457 10cent clo.txt Over representation only. Top 20 Terms. 20 significant terms shown} 
\label{tab:PANTHER GO-Slim Biological Process GO SLIM BP 3457 10cent clo.txt Over representation only. Top 20 Terms. 20 significant terms shown}
\end{table}


%% GO SLIM MF %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LOW CLO %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 16:01:28 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}
\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Molecular Function & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
glutamate receptor activity (GO:0008066) & 25 & 6 & 0 & 16.9 & $4.00 \times 10^{-6}$ & $1.25 \times 10^{-4}$ \\ 
  adenylate cyclase activity (GO:0004016) & 21 & 5 & 0 & 16.7 & $2.73 \times 10^{-5}$ & $6.31 \times 10^{-4}$ \\ 
  phosphorus-oxygen lyase activity (GO:0016849) & 25 & 5 & 0 & 14.0 & $5.64 \times 10^{-5}$ & $1.07 \times 10^{-3}$ \\ 
  cyclase activity (GO:0009975) & 25 & 5 & 0 & 14.0 & $5.64 \times 10^{-5}$ & $1.04 \times 10^{-3}$ \\ 
  amino acid binding (GO:0016597) & 42 & 7 & 1 & 11.7 & $5.20 \times 10^{-6}$ & $1.54 \times 10^{-4}$ \\ 
  voltage-gated calcium channel activity (GO:0005245) & 28 & 4 & 0 & 10.0 & $1.01 \times 10^{-3}$ & $1.27 \times 10^{-2}$ \\ 
  extracellular ligand-gated ion channel activity (GO:0005230) & 39 & 5 & 1 & 9.0 & $3.66 \times 10^{-4}$ & $5.27 \times 10^{-3}$ \\ 
  sodium ion transmembrane transporter activity (GO:0015081) & 99 & 11 & 1 & 7.8 & $4.33 \times 10^{-7}$ & $1.65 \times 10^{-5}$ \\ 
  calmodulin binding (GO:0005516) & 46 & 5 & 1 & 7.6 & $7.31 \times 10^{-4}$ & $9.98 \times 10^{-3}$ \\ 
  carboxylic acid binding (GO:0031406) & 85 & 9 & 1 & 7.4 & $7.05 \times 10^{-6}$ & $1.97 \times 10^{-4}$ \\ 
  organic acid binding (GO:0043177) & 94 & 9 & 1 & 6.7 & $1.49 \times 10^{-5}$ & $3.77 \times 10^{-4}$ \\ 
  postsynaptic neurotransmitter receptor activity (GO:0098960) & 42 & 4 & 1 & 6.7 & $3.91 \times 10^{-3}$ & $4.43 \times 10^{-2}$ \\ 
  solute:sodium symporter activity (GO:0015370) & 42 & 4 & 1 & 6.7 & $3.91 \times 10^{-3}$ & $4.34 \times 10^{-2}$ \\ 
  ligand-gated channel activity (GO:0022834) & 77 & 7 & 1 & 6.4 & $1.81 \times 10^{-4}$ & $2.91 \times 10^{-3}$ \\ 
  ligand-gated ion channel activity (GO:0015276) & 77 & 7 & 1 & 6.4 & $1.81 \times 10^{-4}$ & $2.83 \times 10^{-3}$ \\ 
  voltage-gated cation channel activity (GO:0022843) & 103 & 9 & 1 & 6.1 & $2.92 \times 10^{-5}$ & $6.48 \times 10^{-4}$ \\ 
  ion gated channel activity (GO:0022839) & 187 & 16 & 3 & 6.0 & $3.14 \times 10^{-8}$ & $2.09 \times 10^{-6}$ \\ 
  potassium channel activity (GO:0005267) & 107 & 9 & 2 & 5.9 & $3.86 \times 10^{-5}$ & $8.21 \times 10^{-4}$ \\ 
  calcium channel activity (GO:0005262) & 73 & 6 & 1 & 5.8 & $8.59 \times 10^{-4}$ & $1.14 \times 10^{-2}$ \\ 
  voltage-gated channel activity (GO:0022832) & 110 & 9 & 2 & 5.7 & $4.72 \times 10^{-5}$ & $9.66 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{PANTHER GO-Slim Molecular Function GO SLIM MF 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown} 
\label{tab:PANTHER GO-Slim Molecular Function GO SLIM MF 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms. 20 significant terms shown}
\end{table}


% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 16:01:28 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Molecular Function & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
ion channel activity (GO:0005216) & 290 & 22 & 4 & 5.3 & $6.49 \times 10^{-10}$ & $1.73 \times 10^{-7}$ \\ 
  metal ion transmembrane transporter activity (GO:0046873) & 309 & 23 & 4 & 5.2 & $3.70 \times 10^{-10}$ & $1.97 \times 10^{-7}$ \\ 
  channel activity (GO:0015267) & 307 & 22 & 4 & 5.0 & $1.75 \times 10^{-9}$ & $2.33 \times 10^{-7}$ \\ 
  passive transmembrane transporter activity (GO:0022803) & 307 & 22 & 4 & 5.0 & $1.75 \times 10^{-9}$ & $3.11 \times 10^{-7}$ \\ 
  cation channel activity (GO:0005261) & 234 & 19 & 3 & 5.7 & $3.50 \times 10^{-9}$ & $3.72 \times 10^{-7}$ \\ 
  inorganic cation transmembrane transporter activity (GO:0022890) & 393 & 24 & 6 & 4.3 & $6.14 \times 10^{-9}$ & $5.44 \times 10^{-7}$ \\ 
  cation transmembrane transporter activity (GO:0008324) & 422 & 24 & 6 & 4.0 & $2.24 \times 10^{-8}$ & $1.70 \times 10^{-6}$ \\ 
  ion gated channel activity (GO:0022839) & 187 & 16 & 3 & 6.0 & $3.14 \times 10^{-8}$ & $2.09 \times 10^{-6}$ \\ 
  inorganic molecular entity transmembrane transporter activity (GO:0015318) & 547 & 27 & 8 & 3.5 & $4.74 \times 10^{-8}$ & $2.10 \times 10^{-6}$ \\ 
  transporter activity (GO:0005215) & 776 & 33 & 11 & 3.0 & $4.41 \times 10^{-8}$ & $2.13 \times 10^{-6}$ \\ 
  ion transmembrane transporter activity (GO:0015075) & 578 & 28 & 8 & 3.4 & $3.85 \times 10^{-8}$ & $2.27 \times 10^{-6}$ \\ 
  transmembrane transporter activity (GO:0022857) & 696 & 31 & 10 & 3.1 & $4.32 \times 10^{-8}$ & $2.30 \times 10^{-6}$ \\ 
  gated channel activity (GO:0022836) & 197 & 16 & 3 & 5.7 & $6.15 \times 10^{-8}$ & $2.52 \times 10^{-6}$ \\ 
  monovalent inorganic cation transmembrane transporter activity (GO:0015077) & 261 & 17 & 4 & 4.6 & $4.58 \times 10^{-7}$ & $1.62 \times 10^{-5}$ \\ 
  sodium ion transmembrane transporter activity (GO:0015081) & 99 & 11 & 1 & 7.8 & $4.33 \times 10^{-7}$ & $1.65 \times 10^{-5}$ \\ 
  small molecule binding (GO:0036094) & 377 & 20 & 5 & 3.7 & $9.69 \times 10^{-7}$ & $3.22 \times 10^{-5}$ \\ 
  glutamate receptor activity (GO:0008066) & 25 & 6 & 0 & 16.9 & $4.00 \times 10^{-6}$ & $1.25 \times 10^{-4}$ \\ 
  amino acid binding (GO:0016597) & 42 & 7 & 1 & 11.7 & $5.20 \times 10^{-6}$ & $1.54 \times 10^{-4}$ \\ 
  carboxylic acid binding (GO:0031406) & 85 & 9 & 1 & 7.4 & $7.05 \times 10^{-6}$ & $1.97 \times 10^{-4}$ \\ 
  anion binding (GO:0043168) & 484 & 21 & 7 & 3.0 & $1.04 \times 10^{-5}$ & $2.78 \times 10^{-4}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{PANTHER GO-Slim Molecular Function GO SLIM MF 3457 10cent clo.txt Over representation only. Top 20 Terms. 20 significant terms shown} 
\label{tab:PANTHER GO-Slim Molecular Function GO SLIM MF 3457 10cent clo.txt Over representation only. Top 20 Terms. 20 significant terms shown}
\end{table}

%% GO SLIM CC %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LOW CLO %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:45:48 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Cellular Component & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
integral component of peroxisomal membrane (GO:0005779) & 4 & 2 & 0 & 35.1 & $2.84 \times 10^{-3}$ & $3.52 \times 10^{-2}$ \\ 
  postsynaptic specialization membrane (GO:0099634) & 22 & 5 & 0 & 16.0 & $3.31 \times 10^{-5}$ & $9.56 \times 10^{-4}$ \\ 
  postsynaptic density membrane (GO:0098839) & 22 & 5 & 0 & 16.0 & $3.31 \times 10^{-5}$ & $9.06 \times 10^{-4}$ \\ 
  presynaptic membrane (GO:0042734) & 22 & 4 & 0 & 12.8 & $4.47 \times 10^{-4}$ & $8.01 \times 10^{-3}$ \\ 
  postsynaptic membrane (GO:0045211) & 43 & 6 & 1 & 9.8 & $6.14 \times 10^{-5}$ & $1.45 \times 10^{-3}$ \\ 
  cation channel complex (GO:0034703) & 111 & 14 & 2 & 8.8 & $2.59 \times 10^{-9}$ & $2.25 \times 10^{-7}$ \\ 
  ionotropic glutamate receptor complex (GO:0008328) & 33 & 4 & 0 & 8.5 & $1.75 \times 10^{-3}$ & $2.46 \times 10^{-2}$ \\ 
  ion channel complex (GO:0034702) & 128 & 15 & 2 & 8.2 & $1.75 \times 10^{-9}$ & $1.82 \times 10^{-7}$ \\ 
  transporter complex (GO:1990351) & 138 & 16 & 2 & 8.1 & $5.73 \times 10^{-10}$ & $7.45 \times 10^{-8}$ \\ 
  synaptic membrane (GO:0097060) & 62 & 7 & 1 & 7.9 & $5.14 \times 10^{-5}$ & $1.34 \times 10^{-3}$ \\ 
  transmembrane transporter complex (GO:1902495) & 133 & 15 & 2 & 7.9 & $2.82 \times 10^{-9}$ & $2.10 \times 10^{-7}$ \\ 
  neuron to neuron synapse (GO:0098984) & 56 & 5 & 1 & 6.3 & $1.65 \times 10^{-3}$ & $2.60 \times 10^{-2}$ \\ 
  postsynaptic specialization (GO:0099572) & 56 & 5 & 1 & 6.3 & $1.65 \times 10^{-3}$ & $2.53 \times 10^{-2}$ \\ 
  asymmetric synapse (GO:0032279) & 56 & 5 & 1 & 6.3 & $1.65 \times 10^{-3}$ & $2.46 \times 10^{-2}$ \\ 
  postsynaptic density (GO:0014069) & 56 & 5 & 1 & 6.3 & $1.65 \times 10^{-3}$ & $2.39 \times 10^{-2}$ \\ 
  plasma membrane protein complex (GO:0098797) & 237 & 18 & 3 & 5.3 & $2.39 \times 10^{-8}$ & $1.38 \times 10^{-6}$ \\ 
  presynapse (GO:0098793) & 113 & 8 & 2 & 5.0 & $3.14 \times 10^{-4}$ & $5.84 \times 10^{-3}$ \\ 
  axon (GO:0030424) & 100 & 7 & 1 & 4.9 & $7.91 \times 10^{-4}$ & $1.37 \times 10^{-2}$ \\ 
  postsynapse (GO:0098794) & 88 & 6 & 1 & 4.8 & $2.11 \times 10^{-3}$ & $2.75 \times 10^{-2}$ \\ 
  synapse part (GO:0044456) & 192 & 11 & 3 & 4.0 & $1.47 \times 10^{-4}$ & $3.07 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis GO SLIM CC 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms} 
\label{tab:GO SLIM analysis GO SLIM CC 3457 10cent clo.txt ordered by fold change. Over representation only. Top 20 terms}
\end{table}
% latex table generated in R 3.6.3 by xtable 1.8-4 package
% Sun Jun 28 15:45:48 2020
\begin{table}[ht]
\centering
\begin{adjustbox}{width=\textwidth}

\begin{tabular}{lrrrrrr}
  \hline
PANTHER GO-Slim Cellular Component & PSP ref & Test genes & Expected & Fold enrichment & p value & FDR \\ 
  \hline
plasma membrane (GO:0005886) & 3105 & 88 & 44 & 2.0 & $1.45 \times 10^{-10}$ & $3.76 \times 10^{-8}$ \\ 
  membrane (GO:0016020) & 3445 & 95 & 49 & 1.9 & $1.00 \times 10^{-10}$ & $5.20 \times 10^{-8}$ \\ 
  cell periphery (GO:0071944) & 3175 & 88 & 45 & 1.9 & $4.14 \times 10^{-10}$ & $7.17 \times 10^{-8}$ \\ 
  transporter complex (GO:1990351) & 138 & 16 & 2 & 8.1 & $5.73 \times 10^{-10}$ & $7.45 \times 10^{-8}$ \\ 
  ion channel complex (GO:0034702) & 128 & 15 & 2 & 8.2 & $1.75 \times 10^{-9}$ & $1.82 \times 10^{-7}$ \\ 
  transmembrane transporter complex (GO:1902495) & 133 & 15 & 2 & 7.9 & $2.82 \times 10^{-9}$ & $2.10 \times 10^{-7}$ \\ 
  cation channel complex (GO:0034703) & 111 & 14 & 2 & 8.8 & $2.59 \times 10^{-9}$ & $2.25 \times 10^{-7}$ \\ 
  integral component of membrane (GO:0016021) & 986 & 39 & 14 & 2.8 & $1.57 \times 10^{-8}$ & $1.02 \times 10^{-6}$ \\ 
  plasma membrane protein complex (GO:0098797) & 237 & 18 & 3 & 5.3 & $2.39 \times 10^{-8}$ & $1.38 \times 10^{-6}$ \\ 
  intrinsic component of membrane (GO:0031224) & 1012 & 39 & 14 & 2.7 & $3.07 \times 10^{-8}$ & $1.60 \times 10^{-6}$ \\ 
  plasma membrane part (GO:0044459) & 1404 & 47 & 20 & 2.4 & $7.98 \times 10^{-8}$ & $3.77 \times 10^{-6}$ \\ 
  membrane part (GO:0044425) & 2076 & 61 & 30 & 2.1 & $8.96 \times 10^{-8}$ & $3.88 \times 10^{-6}$ \\ 
  integral component of plasma membrane (GO:0005887) & 652 & 28 & 9 & 3.0 & $4.10 \times 10^{-7}$ & $1.64 \times 10^{-5}$ \\ 
  intrinsic component of plasma membrane (GO:0031226) & 660 & 28 & 9 & 3.0 & $5.18 \times 10^{-7}$ & $1.92 \times 10^{-5}$ \\ 
  neuron part (GO:0097458) & 451 & 21 & 6 & 3.3 & $3.72 \times 10^{-6}$ & $1.29 \times 10^{-4}$ \\ 
  membrane protein complex (GO:0098796) & 548 & 23 & 8 & 3.0 & $6.62 \times 10^{-6}$ & $2.15 \times 10^{-4}$ \\ 
  postsynaptic density membrane (GO:0098839) & 22 & 5 & 0 & 16.0 & $3.31 \times 10^{-5}$ & $9.06 \times 10^{-4}$ \\ 
  postsynaptic specialization membrane (GO:0099634) & 22 & 5 & 0 & 16.0 & $3.31 \times 10^{-5}$ & $9.56 \times 10^{-4}$ \\ 
  synaptic membrane (GO:0097060) & 62 & 7 & 1 & 7.9 & $5.14 \times 10^{-5}$ & $1.34 \times 10^{-3}$ \\ 
  plasma membrane region (GO:0098590) & 587 & 22 & 8 & 2.6 & $5.48 \times 10^{-5}$ & $1.36 \times 10^{-3}$ \\ 
   \hline
\end{tabular}
\end{adjustbox}
\caption{GO SLIM analysis GO SLIM CC 3457 10cent clo.txt Over representation only. Top 20 Terms} 
\label{tab:GO SLIM analysis GO SLIM CC 3457 10cent clo.txt Over representation only. Top 20 Terms}
\end{table}

